
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <title>Descriptions of Tinker Routines &#8212; Tinker User&#39;s Guide  documentation</title>
    <link rel="stylesheet" href="../_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    <script id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/language_data.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Description of Modules &amp; Global Variables" href="modules.html" />
    <link rel="prev" title="Use of the Keyword Control File" href="keywords.html" />
   
  <link rel="stylesheet" href="../_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <div class="section" id="descriptions-of-tinker-routines">
<h1>Descriptions of Tinker Routines<a class="headerlink" href="#descriptions-of-tinker-routines" title="Permalink to this headline">¶</a></h1>
<p>The distribution version of the Tinker package contains over 700 separate programs, subroutines and functions. This section contains a brief description of the purpose of most of these code units. Further information can be found in the comments located at the top of each source code file.</p>
<p><strong>ACTIVE Subroutine</strong></p>
<p>“active” sets the list of atoms that are used during each potential energy Function** calculation</p>
<p><strong>ADDBASE Subroutine</strong></p>
<p>“addbase” builds the Cartesian coordinates for a single nucleic acid base; coordinates are read from the Protein Data Bank file or found from internal coordinates, then atom types are assigned and connectivity data generated</p>
<p><strong>ADDBOND Subroutine</strong></p>
<p>“addbond” adds entries to the attached atoms list in order to generate a direct connection between two atoms</p>
<p><strong>ADDSIDE Subroutine</strong></p>
<p>“addside” builds the Cartesian coordinates for a single amino acid side chain; coordinates are read from the Protein Data Bank file or found from internal coordinates, then atom types are assigned and connectivity data generated</p>
<p><strong>ADJACENT Function</strong></p>
<p>“adjacent” finds an atom connected to atom “i1” other than atom “i2”; if no such atom exists, then the closest atom in space is returned</p>
<p><strong>ALCHEMY Program</strong></p>
<p>“alchemy” computes the free energy difference corresponding to a small perturbation by Boltzmann weighting the potential energy difference over a number of sample states; current version (incorrectly) considers the charge energy to be intermolecular in finding the perturbation energies</p>
<p><strong>ANALYSIS Subroutine</strong></p>
<p>“analysis” calls the series of routines needed to calculate the potential energy and perform energy partitioning analysis in terms of type of interaction or atom number</p>
<p><strong>ANALYZ4 Subroutine</strong></p>
<p>“analyz4” prints the energy to 4 decimal places and number of interactions for each component of the potential energy</p>
<p><strong>ANALYZ6 Subroutine</strong></p>
<p>“analyz6” prints the energy to 6 decimal places and number of interactions for each component of the potential energy</p>
<p><strong>ANALYZ8 Subroutine</strong></p>
<p>“analyz8” prints the energy to 8 decimal places and number of interactions for each component of the potential energy</p>
<p><strong>ANALYZE Program</strong></p>
<p>“analyze” computes and displays the total potential; options are provided to partition the energy by atom or by potential Function** type; parameters used in computing interactions can also be displayed by atom; output of large energy interactions and of electrostatic and inertial properties is available</p>
<p><strong>ANGLES Subroutine</strong></p>
<p>“angles” finds the total number of bond angles and stores the atom numbers of the atoms defining each angle; for each angle to a trivalent central atom, the third bonded atom is stored for use in out-of-plane bending</p>
<p><strong>ANNEAL Program</strong></p>
<p>“anneal” performs a simulated annealing protocol by means of variable temperature molecular dynamics using either linear, exponential or sigmoidal cooling schedules</p>
<p><strong>ANORM Function</strong></p>
<p>“anorm” finds the norm (length) of a vector; used as a service routine by the Connolly surface area and volume computation</p>
<p><strong>ARCHIVE Program</strong></p>
<p>“archive” is a utility Program** for coordinate files which concatenates multiple coordinate sets into a single archive file, or extracts individual coordinate sets from an archive</p>
<p><strong>ASET Subroutine</strong></p>
<p>“aset” computes by recursion the A Function**s used in the evaluation of Slater-type (STO) overlap integrals</p>
<p><strong>ATOMYZE Subroutine</strong></p>
<p>“atomyze” prints the potential energy components broken down by atom and to a choice of precision</p>
<p><strong>ATTACH Subroutine</strong></p>
<p>“attach” generates lists of 1-3, 1-4 and 1-5 connectivities starting from the previously determined list of attached atoms (ie, 1-2 connectivity)</p>
<p><strong>BASEFILE Subroutine</strong></p>
<p>“basefile” extracts from an input filename the portion consisting of any directory name and the base filename</p>
<p><strong>BCUCOF Subroutine</strong></p>
<p>“bcucof” determines the coefficient matrix needed for bicubic interpolation of a Function**, gradients and cross derivatives</p>
<p><strong>BCUINT Subroutine</strong></p>
<p>“bcuint” performs a bicubic interpolation of the Function** value on a 2D spline grid</p>
<p><strong>BCUINT1 Subroutine</strong></p>
<p>“bcuint1” performs a bicubic interpolation of the Function** value and gradient along the directions of a 2D spline grid</p>
<p><strong>BCUINT2 Subroutine</strong></p>
<p>“bcuint2” performs a bicubic interpolation of the Function** value, gradient and Hessain along the directions of a 2D spline grid</p>
<p><strong>BEEMAN Subroutine</strong></p>
<p>“beeman” performs a single molecular dynamics time step by means of a Beeman multistep recursion formula; the actual coefficients are Brooks’ “Better Beeman” values</p>
<p><strong>BETACF Function</strong></p>
<p>“betacf” computes a rapidly convergent continued fraction needed by routine “betai” to evaluate the cumulative Beta distribution</p>
<p><strong>BETAI Function</strong></p>
<p>“betai” evaluates the cumulative Beta distribution Function** as the probability that a random variable from a distribution with Beta parameters “a” and “b” will be less than “x”</p>
<p><strong>BIGBLOCK Subroutine</strong></p>
<p>“bigblock” replicates the coordinates of a single unit cell to give a larger block of repeated units</p>
<p><strong>BITORS Subroutine</strong></p>
<p>“bitors” finds the total number of bitorsions, pairs of overlapping dihedral angles, and the numbers of the five atoms defining each bitorsion</p>
<p><strong>BMAX Function</strong></p>
<p>“bmax” computes the maximum order of the B Function**s needed for evaluation of Slater-type (STO) overlap integrals</p>
<p><strong>BNDERR Function</strong></p>
<p>“bnderr” is the distance bound error Function** and derivatives; this version implements the original and Havel’s normalized lower bound penalty, the normalized version is preferred when lower bounds are small (as with NMR NOE restraints), the original penalty is needed if large lower bounds are present</p>
<p><strong>BONDS Subroutine</strong></p>
<p>“bonds” finds the total number of covalent bonds and stores the atom numbers of the atoms defining each bond</p>
<p><strong>BORN Subroutine</strong></p>
<p>“born” computes the Born radius of each atom for use with the various GB/SA solvation models</p>
<p><strong>BORN1 Subroutine</strong></p>
<p>“born1” computes derivatives of the Born radii with respect to atomic coordinates and increments total energy derivatives and virial components for potentials involving Born radii</p>
<p><strong>BOUNDS Subroutine</strong></p>
<p>“bounds” finds the center of mass of each molecule and translates any stray molecules back into the periodic box</p>
<p><strong>BSET Subroutine</strong></p>
<p>“bset” computes by downward recursion the B Function**s used in the evaluation of Slater-type (STO) overlap integrals</p>
<p><strong>BSPLINE Subroutine</strong></p>
<p>“bspline” calculates the coefficients for an n-th order B-spline approximation</p>
<p><strong>BSPLINE1 Subroutine</strong></p>
<p>“bspline1” calculates the coefficients and derivative coefficients for an n-th order B-spline approximation</p>
<p><strong>BSSTEP Subroutine</strong></p>
<p>“bsstep” takes a single Bulirsch-Stoer step with monitoring of local truncation error to ensure accuracy</p>
<p><strong>CALENDAR Subroutine</strong></p>
<p>“calendar” returns the current time as a set of integer values representing the year, month, day, hour, minute and second</p>
<p><strong>CELLATOM Subroutine</strong></p>
<p>“cellatom” completes the addition of a symmetry related atom to a unit cell by updating the atom type and attachment arrays</p>
<p><strong>CENTER Subroutine</strong></p>
<p>“center” moves the weighted centroid of each coordinate set to the origin during least squares superposition</p>
<p><strong>CERROR Subroutine</strong></p>
<p>“cerror” is the error handling routine for the Connolly surface area and volume computation</p>
<p><strong>CFFTB Subroutine</strong></p>
<p>“cfftb” computes the backward complex discrete Fourier transform, the Fourier synthesis</p>
<p><strong>CFFTB1 Subroutine</strong></p>
<p><strong>CFFTF Subroutine</strong></p>
<p>“cfftf” computes the forward complex discrete Fourier transform, the Fourier analysis</p>
<p><strong>CFFTF1 Subroutine</strong></p>
<p><strong>CFFTI Subroutine</strong></p>
<p>“cffti” initializes the array “wsave” which is used in both forward and backward transforms; the prime factorization of “n” together with a tabulation of the trigonometric Function**s are computed and stored in “wsave”</p>
<p><strong>CFFTI1 Subroutine</strong></p>
<p><strong>CHIRER Function</strong></p>
<p>“chirer” computes the chirality error and its derivatives with respect to atomic Cartesian coordinates as a sum the squares of deviations of chiral volumes from target values</p>
<p><strong>CHKCLASH Subroutine</strong></p>
<p>“chkclash” determines if there are any atom clashes which might cause trouble on subsequent energy evaluation</p>
<p><strong>CHKPOLE Subroutine</strong></p>
<p>“chkpole” inverts atomic multipole moments as necessary at sites with chiral local reference frame definitions</p>
<p><strong>CHKRING Subroutine</strong></p>
<p>“chkring” tests angles to be constrained for their presence in small rings and removes constraints that are redundant</p>
<p><strong>CHKSIZE Subroutine</strong></p>
<p>“chksize” computes a measure of overall global structural expansion or compaction from the number of excess upper or lower bounds matrix violations</p>
<p><strong>CHKTREE Subroutine</strong></p>
<p>“chktree” tests a minimum energy structure to see if it belongs to the correct progenitor in the existing map</p>
<p><strong>CHKXYZ Subroutine</strong></p>
<p>“chkxyz” finds any pairs of atoms with identical Cartesian coordinates, and prints a warning message</p>
<p><strong>CHOLESKY Subroutine</strong></p>
<p>“cholesky” uses a modified Cholesky method to solve the linear system Ax = b, returning “x” in “b”; “A” is assumed to be a real symmetric positive definite matrix with its diagonal and upper triangle stored by rows</p>
<p><strong>CIRPLN Subroutine</strong></p>
<p><strong>CJKM Function</strong></p>
<p>“cjkm” computes the coefficients of spherical harmonics expressed in prolate spheroidal coordinates</p>
<p><strong>CLIMBER Subroutine</strong></p>
<p><strong>CLIMBRGD Subroutine</strong></p>
<p><strong>CLIMBROT Subroutine</strong></p>
<p><strong>CLIMBTOR Subroutine</strong></p>
<p><strong>CLIMBXYZ Subroutine</strong></p>
<p><strong>CLOCK Subroutine</strong></p>
<p>“clock” determines elapsed CPU time in seconds since the start of the job</p>
<p><strong>CLUSTER Subroutine</strong></p>
<p>“cluster” gets the partitioning of the system into groups and stores a list of the group to which each atom belongs</p>
<p><strong>COLUMN Subroutine</strong></p>
<p>“column” takes the off-diagonal Hessian elements stored as sparse rows and sets up indices to allow column access</p>
<p><strong>COMMAND Subroutine</strong></p>
<p>“command” uses the standard Unix-like iargc/getarg routines to get the number and values of arguments specified on the command line at Program** runtime</p>
<p><strong>COMPRESS Subroutine</strong></p>
<p>“compress” transfers only the non-buried tori from the temporary tori arrays to the final tori arrays</p>
<p><strong>CONNECT Subroutine</strong></p>
<p>“connect” sets up the attached atom arrays starting from a set of internal coordinates</p>
<p><strong>CONNOLLY Subroutine</strong></p>
<p>“connolly” uses the algorithms from the AMS/VAM Program**s of Michael Connolly to compute the analytical molecular surface area and volume of a collection of spherical atoms; thus it implements Fred Richards’ molecular surface definition as a set of analytically defined spherical and toroidal polygons</p>
<p><strong>CONTACT Subroutine</strong></p>
<p>“contact” constructs the contact surface, cycles and convex faces</p>
<p><strong>CONTROL Subroutine</strong></p>
<p>“control” gets initial values for parameters that determine the output style and information level provided by Tinker</p>
<p><strong>COORDS Subroutine</strong></p>
<p>“coords” converts the three principal eigenvalues/vectors from the metric matrix into atomic coordinates, and calls a routine to compute the rms deviation from the bounds</p>
<p><strong>CORRELATE Program</strong></p>
<p>“correlate” computes the time correlation Function** of some user-supplied property from individual snapshot frames taken from a molecular dynamics or other trajectory</p>
<p><strong>CREATEJVM Subroutine</strong></p>
<p><strong>CREATESERVER Subroutine</strong></p>
<p><strong>CREATESYSTEM Subroutine</strong></p>
<p><strong>CREATEUPDATE Subroutine</strong></p>
<p><strong>CRYSTAL Program</strong></p>
<p>“crystal” is a utility Program** which converts between fractional and Cartesian coordinates, and can generate full unit cells from asymmetric units</p>
<p><strong>CUTOFFS Subroutine</strong></p>
<p>“cutoffs” initializes and stores spherical energy cutoff distance windows, Hessian element and Ewald sum cutoffs, and the pairwise neighbor generation method</p>
<p><strong>CYTSY Subroutine</strong></p>
<p>“cytsy” solves a system of linear equations for a cyclically tridiagonal, symmetric, positive definite matrix</p>
<p><strong>CYTSYP Subroutine</strong></p>
<p>“cytsyp” finds the Cholesky factors of a cyclically tridiagonal symmetric, positive definite matrix given by two vectors</p>
<p><strong>CYTSYS Subroutine</strong></p>
<p>“cytsys” solves a cyclically tridiagonal linear system given the Cholesky factors</p>
<p><strong>D1D2 Function</strong></p>
<p>“d1d2” is a utility Function** used in computation of the reaction field recursive summation elements</p>
<p><strong>DELETE Subroutine</strong></p>
<p>“delete” removes a specified atom from the Cartesian coordinates list and shifts the remaining atoms</p>
<p><strong>DEPTH Function</strong></p>
<p><strong>DESTROYJVM Subroutine</strong></p>
<p><strong>DESTROYSERVER Subroutine</strong></p>
<p><strong>DFTMOD Subroutine</strong></p>
<p>“dftmod” computes the modulus of the discrete Fourier transform of “bsarray”, storing it into “bsmod”</p>
<p><strong>DIAGQ Subroutine</strong></p>
<p>“diagq” is a matrix diagonalization routine which is derived from the classical given, housec, and eigen algorithms with several modifications to increase the efficiency and accuracy</p>
<p><strong>DIFFEQ Subroutine</strong></p>
<p>“diffeq” performs the numerical integration of an ordinary differential equation using an adaptive stepsize method to solve the corresponding coupled first-order equations of the general form dyi/dx = f(x,y1,…,yn) for yi = y1,…,yn</p>
<p><strong>DIFFUSE Program</strong></p>
<p>“diffuse” finds the self-diffusion constant for a homogeneous liquid via the Einstein relation from a set of stored molecular dynamics frames; molecular centers of mass are unfolded and mean squared displacements are computed versus time separation</p>
<p><strong>DIST2 Function</strong></p>
<p>“dist2” finds the distance squared between two points; used as a service routine by the Connolly surface area and volume computation</p>
<p><strong>DISTGEOM Program</strong></p>
<p>“distgeom” uses a metric matrix distance geometry procedure to generate structures with interpoint distances that lie within specified bounds, with chiral centers that maintain chirality, and with torsional angles restrained to desired values; the user also has the ability to interactively inspect and alter the triangle smoothed bounds matrix prior to embedding</p>
<p><strong>DMDUMP Subroutine</strong></p>
<p>“dmdump” puts the distance matrix of the final structure into the upper half of a matrix, the distance of each atom to the centroid on the diagonal, and the individual terms of the bounds errors into the lower half of the matrix</p>
<p><strong>DOCUMENT Program</strong></p>
<p>“document” generates a formatted description of all the code modules or common blocks, an index of routines called by each source code module, a listing of all valid keywords, a list of include file dependencies as needed by a Unix-style Makefile, or a formatted force field parameter set summary</p>
<p><strong>DOT Function</strong></p>
<p>“dot” finds the dot product of two vectors</p>
<p><strong>DSTMAT Subroutine</strong></p>
<p>“dstmat” selects a distance matrix containing values between the previously smoothed upper and lower bounds; the distance values are chosen from uniform distributions, in a triangle correlated fashion, or using random partial metrization</p>
<p><strong>DYNAMIC Program</strong></p>
<p>“dynamic” computes a molecular dynamics trajectory in any of several statistical mechanical ensembles with optional periodic boundaries and optional coupling to temperature and pressure baths alternatively a stochastic dynamics trajectory can be generated</p>
<p><strong>EANGANG Subroutine</strong></p>
<p>“eangang” calculates the angle-angle potential energy</p>
<p><strong>EANGANG1 Subroutine</strong></p>
<p>“eangang1” calculates the angle-angle potential energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EANGANG2 Subroutine</strong></p>
<p>“eangang2” calculates the angle-angle potential energy second derivatives with respect to Cartesian coordinates using finite difference methods</p>
<p><strong>EANGANG2A Subroutine</strong></p>
<p>“eangang2a” calculates the angle-angle first derivatives for a single interaction with respect to Cartesian coordinates; used in computation of finite difference second derivatives</p>
<p><strong>EANGANG3 Subroutine</strong></p>
<p>“eangang3” calculates the angle-angle potential energy; also partitions the energy among the atoms</p>
<p><strong>EANGLE Subroutine</strong></p>
<p>“eangle” calculates the angle bending potential energy; projected in-plane angles at trigonal centers or Fourier angle bending terms are optionally used</p>
<p><strong>EANGLE1 Subroutine</strong></p>
<p>“eangle1” calculates the angle bending potential energy and the first derivatives with respect to Cartesian coordinates; projected in-plane angles at trigonal centers or Fourier angle bending terms are optionally used</p>
<p><strong>EANGLE2 Subroutine</strong></p>
<p>“eangle2” calculates second derivatives of the angle bending energy for a single atom using a mixture of analytical and finite difference methods; projected in-plane angles at trigonal centers or Fourier angle bending terms are optionally used</p>
<p><strong>EANGLE2A Subroutine</strong></p>
<p>“eangle2a” calculates bond angle bending potential energy second derivatives with respect to Cartesian coordinates</p>
<p><strong>EANGLE2B Subroutine</strong></p>
<p>“eangle2b” computes projected in-plane bending first derivatives for a single angle with respect to Cartesian coordinates; used in computation of finite difference second derivatives</p>
<p><strong>EANGLE3 Subroutine</strong></p>
<p>“eangle3” calculates the angle bending potential energy, also partitions the energy among the atoms; projected in-plane angles at trigonal centers or Fourier angle bending terms are optionally used</p>
<p><strong>EBOND Subroutine</strong></p>
<p>“ebond” calculates the bond stretching energy</p>
<p><strong>EBOND1 Subroutine</strong></p>
<p>“ebond1” calculates the bond stretching energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EBOND2 Subroutine</strong></p>
<p>“ebond2” calculates second derivatives of the bond stretching energy for a single atom at a time</p>
<p><strong>EBOND3 Subroutine</strong></p>
<p>“ebond3” calculates the bond stretching energy; also partitions the energy among the atoms</p>
<p><strong>EBUCK Subroutine</strong></p>
<p>“ebuck” calculates the Buckingham exp-6 van der Waals energy</p>
<p><strong>EBUCK0A Subroutine</strong></p>
<p>“ebuck0a” calculates the Buckingham exp-6 van der Waals energy using a pairwise double loop</p>
<p><strong>EBUCK0B Subroutine</strong></p>
<p>“ebuck0b” calculates the Buckingham exp-6 van der Waals energy using the method of lights to locate neighboring atoms</p>
<p><strong>EBUCK0C Subroutine</strong></p>
<p>“ebuck0c” calculates the Buckingham exp-6 van der Waals energy via a Gaussian approximation for potential energy smoothing</p>
<p><strong>EBUCK1 Subroutine</strong></p>
<p>“ebuck1” calculates the Buckingham exp-6 van der Waals energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EBUCK1A Subroutine</strong></p>
<p>“ebuck1a” calculates the Buckingham exp-6 van der Waals energy and its first derivatives using a pairwise double loop</p>
<p><strong>EBUCK1B Subroutine</strong></p>
<p>“ebuck1b” calculates the Buckingham exp-6 van der Waals energy and its first derivatives using the method of lights to locate neighboring atoms</p>
<p><strong>EBUCK1C Subroutine</strong></p>
<p>“ebuck1c” calculates the Buckingham exp-6 van der Waals energy and its first derivatives via a Gaussian approximation for potential energy smoothing</p>
<p><strong>EBUCK2 Subroutine</strong></p>
<p>“ebuck2” calculates the Buckingham exp-6 van der Waals second derivatives for a single atom at a time</p>
<p><strong>EBUCK2A Subroutine</strong></p>
<p>“ebuck2a” calculates the Buckingham exp-6 van der Waals second derivatives using a double loop over relevant atom pairs</p>
<p><strong>EBUCK2B Subroutine</strong></p>
<p>“ebuck2b” calculates the Buckingham exp-6 van der Waals second derivatives via a Gaussian approximation for use with potential energy smoothing</p>
<p><strong>EBUCK3 Subroutine</strong></p>
<p>“ebuck3” calculates the Buckingham exp-6 van der Waals energy and partitions the energy among the atoms</p>
<p><strong>EBUCK3A Subroutine</strong></p>
<p>“ebuck3a” calculates the Buckingham exp-6 van der Waals energy and partitions the energy among the atoms using a pairwise double loop</p>
<p><strong>EBUCK3B Subroutine</strong></p>
<p>“ebuck3b” calculates the Buckingham exp-6 van der Waals energy and also partitions the energy among the atoms using the method of lights to locate neighboring atoms</p>
<p><strong>EBUCK3C Subroutine</strong></p>
<p>“ebuck3c” calculates the Buckingham exp-6 van der Waals energy via a Gaussian approximation for potential energy smoothing</p>
<p><strong>ECHARGE Subroutine</strong></p>
<p>“echarge” calculates the charge-charge interaction energy</p>
<p><strong>ECHARGE0A Subroutine</strong></p>
<p>“echarge0a” calculates the charge-charge interaction energy using a pairwise double loop</p>
<p><strong>ECHARGE0B Subroutine</strong></p>
<p>“echarge0b” calculates the charge-charge interaction energy using the method of lights to locate neighboring atoms</p>
<p><strong>ECHARGE0C Subroutine</strong></p>
<p>“echarge0c” calculates the charge-charge interaction energy for use with potential smoothing methods</p>
<p><strong>ECHARGE0D Subroutine</strong></p>
<p>“echarge0d” calculates the charge-charge interaction energy using a particle mesh Ewald summation</p>
<p><strong>ECHARGE0E Subroutine</strong></p>
<p>“echarge0e” calculates the charge-charge interaction energy using a particle mesh Ewald summation and the method of lights to locate neighboring atoms</p>
<p><strong>ECHARGE1 Subroutine</strong></p>
<p>“echarge1” calculates the charge-charge interaction energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ECHARGE1A Subroutine</strong></p>
<p>“echarge1a” calculates the charge-charge interaction energy and first derivatives with respect to Cartesian coordinates using a pairwise double loop</p>
<p><strong>ECHARGE1B Subroutine</strong></p>
<p>“echarge1b” calculates the charge-charge interaction energy and first derivatives with respect to Cartesian coordinates using the method of lights to locate neighboring atoms</p>
<p><strong>ECHARGE1C Subroutine</strong></p>
<p>“echarge1c” calculates the charge-charge interaction energy and first derivatives with respect to Cartesian coordinates for use with potential smoothing methods</p>
<p><strong>ECHARGE1D Subroutine</strong></p>
<p>“echarge1d” calculates the charge-charge interaction energy and first derivatives with respect to Cartesian coordinates using a particle mesh Ewald summation</p>
<p><strong>ECHARGE2 Subroutine</strong></p>
<p>“echarge2” calculates second derivatives of the charge-charge interaction energy for a single atom</p>
<p><strong>ECHARGE2A Subroutine</strong></p>
<p>“echarge2a” calculates second derivatives of the charge-charge interaction energy for a single atom using a pairwise double loop</p>
<p><strong>ECHARGE2B Subroutine</strong></p>
<p>“echarge2b” calculates second derivatives of the charge-charge interaction energy for a single atom for use with potential smoothing methods</p>
<p><strong>ECHARGE2C Subroutine</strong></p>
<p>“echarge2c” calculates second derivatives of the charge-charge interaction energy for a single atom using a particle mesh Ewald summation</p>
<p><strong>ECHARGE3 Subroutine</strong></p>
<p>“echarge3” calculates the charge-charge interaction energy and partitions the energy among the atoms</p>
<p><strong>ECHARGE3A Subroutine</strong></p>
<p>“echarge3a” calculates the charge-charge interaction energy and partitions the energy among the atoms using a pairwise double loop</p>
<p><strong>ECHARGE3B Subroutine</strong></p>
<p>“echarge3b” calculates the charge-charge interaction energy and partitions the energy among the atoms using the method of lights to locate neighboring atoms</p>
<p><strong>ECHARGE3C Subroutine</strong></p>
<p>“echarge3c” calculates the charge-charge interaction energy and partitions the energy among the atoms for use with potential smoothing methods</p>
<p><strong>ECHARGE3D Subroutine</strong></p>
<p>“echarge3d” calculates the charge-charge interaction energy and partitions the energy among the atoms using a particle mesh Ewald summation</p>
<p><strong>ECHARGE3E Subroutine</strong></p>
<p>“echarge3e” calculates the charge-charge interaction energy and partitions the energy among the atoms using a particle mesh Ewald summation and the method of lights to locate neighboring atoms</p>
<p><strong>ECHGDPL Subroutine</strong></p>
<p>“echgdpl” calculates the charge-dipole interaction energy</p>
<p><strong>ECHGDPL1 Subroutine</strong></p>
<p>“echgdpl1” calculates the charge-dipole interaction energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ECHGDPL2 Subroutine</strong></p>
<p>“echgdpl2” calculates second derivatives of the charge-dipole interaction energy for a single atom</p>
<p><strong>ECHGDPL3 Subroutine</strong></p>
<p>“echgdpl3” calculates the charge-dipole interaction energy; also partitions the energy among the atoms</p>
<p><strong>EDIPOLE Subroutine</strong></p>
<p>“edipole” calculates the dipole-dipole interaction energy</p>
<p><strong>EDIPOLE1 Subroutine</strong></p>
<p>“edipole1” calculates the dipole-dipole interaction energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EDIPOLE2 Subroutine</strong></p>
<p>“edipole2” calculates second derivatives of the dipole-dipole interaction energy for a single atom</p>
<p><strong>EDIPOLE3 Subroutine</strong></p>
<p>“edipole3” calculates the dipole-dipole interaction energy; also partitions the energy among the atoms</p>
<p><strong>EGAUSS Subroutine</strong></p>
<p>“egauss” calculates the Gaussian expansion van der Waals interaction energy</p>
<p><strong>EGAUSS0A Subroutine</strong></p>
<p>“egauss0a” calculates the Gaussian expansion van der Waals interaction energy using a pairwise double loop</p>
<p><strong>EGAUSS0B Subroutine</strong></p>
<p>“egauss0b” calculates the Gaussian expansion van der Waals interaction energy for use with potential energy smoothing</p>
<p><strong>EGAUSS1 Subroutine</strong></p>
<p>“egauss1” calculates the Gaussian expansion van der Waals interaction energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EGAUSS1A Subroutine</strong></p>
<p>“egauss1a” calculates the Gaussian expansion van der Waals interaction energy and its first derivatives using a pairwise double loop</p>
<p><strong>EGAUSS1B Subroutine</strong></p>
<p>“egauss1b” calculates the Gaussian expansion van der Waals interaction energy and its first derivatives for use with stophat potential energy smoothing</p>
<p><strong>EGAUSS2 Subroutine</strong></p>
<p>“egauss2” calculates the Gaussian expansion van der Waals second derivatives for a single atom at a time</p>
<p><strong>EGAUSS2A Subroutine</strong></p>
<p>“egauss2a” calculates the Gaussian expansion van der Waals second derivatives using a pairwise double loop</p>
<p><strong>EGAUSS2B Subroutine</strong></p>
<p>“egauss2b” calculates the Gaussian expansion van der Waals second derivatives for stophat potential energy smoothing</p>
<p><strong>EGAUSS3 Subroutine</strong></p>
<p>“egauss3” calculates the Gaussian expansion van der Waals interaction energy and partitions the energy among the atoms</p>
<p><strong>EGAUSS3A Subroutine</strong></p>
<p>“egauss3a” calculates the Gaussian expansion van der Waals interaction energy and partitions the energy among the atoms using a pairwise double loop</p>
<p><strong>EGAUSS3B Subroutine</strong></p>
<p>“egauss3b” calculates the Gaussian expansion van der Waals interaction energy and partitions the energy among the atoms using a pairwise double loop</p>
<p><strong>EGBSA0A Subroutine</strong></p>
<p>“egbsa0a” calculates the generalized Born polarization energy for the GB/SA solvation models</p>
<p><strong>EGBSA0B Subroutine</strong></p>
<p>“egbsa0b” calculates the generalized Born polarization energy for the GB/SA solvation models for use with potential smoothing methods via analogy to the smoothing of Coulomb’s law</p>
<p><strong>EGBSA1A Subroutine</strong></p>
<p>“egbsa1a” calculates the generalized Born energy and first derivatives of the GB/SA solvation models</p>
<p><strong>EGBSA1B Subroutine</strong></p>
<p>“egbsa1b” calculates the generalized Born energy and first derivatives of the GB/SA solvation models for use with potential smoothing methods</p>
<p><strong>EGBSA2A Subroutine</strong></p>
<p>“egbsa2a” calculates second derivatives of the generalized Born energy term for the GB/SA solvation models</p>
<p><strong>EGBSA2B Subroutine</strong></p>
<p>“egbsa2b” calculates second derivatives of the generalized Born energy term for the GB/SA solvation models for use with potential smoothing methods</p>
<p><strong>EGBSA3A Subroutine</strong></p>
<p>“egbsa3a” calculates the generalized Born energy term for the GB/SA solvation models; also partitions the energy among the atoms</p>
<p><strong>EGBSA3B Subroutine</strong></p>
<p>“egbsa3b” calculates the generalized Born polarization energy for the GB/SA solvation models for use with potential smoothing methods via analogy to the smoothing of Coulomb’s law; also partitions the energy among the atoms</p>
<p><strong>EGEOM Subroutine</strong></p>
<p>“egeom” calculates the energy due to restraints on positions, distances, angles and torsions as well as Gaussian basin and spherical droplet restraints</p>
<p><strong>EGEOM1 Subroutine</strong></p>
<p>“egeom1” calculates the energy and first derivatives with respect to Cartesian coordinates due to restraints on positions, distances, angles and torsions as well as Gaussian basin and spherical droplet restraints</p>
<p><strong>EGEOM2 Subroutine</strong></p>
<p>“egeom2” calculates second derivatives of restraints on positions, distances, angles and torsions as well as Gaussian basin and spherical droplet restraints</p>
<p><strong>EGEOM3 Subroutine</strong></p>
<p>“egeom3” calculates the energy due to restraints on positions, distances, angles and torsions as well as Gaussian basin and droplet restraints; also partitions energy among the atoms</p>
<p><strong>EHAL Subroutine</strong></p>
<p>“ehal” calculates the buffered 14-7 van der Waals energy</p>
<p><strong>EHAL0A Subroutine</strong></p>
<p>“ehal0a” calculates the buffered 14-7 van der Waals energy using a pairwise double loop</p>
<p><strong>EHAL0B Subroutine</strong></p>
<p>“ehal0a” calculates the buffered 14-7 van der Waals energy using the method of lights to locate neighboring atoms</p>
<p><strong>EHAL1 Subroutine</strong></p>
<p>“ehal1” calculates the buffered 14-7 van der Waals energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EHAL1A Subroutine</strong></p>
<p>“ehal1a” calculates the buffered 14-7 van der Waals energy and its first derivatives with respect to Cartesian coordinates using a pairwise double loop</p>
<p><strong>EHAL1B Subroutine</strong></p>
<p>“ehal1b” calculates the buffered 14-7 van der Waals energy and its first derivatives with respect to Cartesian coordinates using the method of lights to locate neighboring atoms</p>
<p><strong>EHAL2 Subroutine</strong></p>
<p>“ehal2” calculates the buffered 14-7 van der Waals second derivatives for a single atom at a time</p>
<p><strong>EHAL3 Subroutine</strong></p>
<p>“ehal3” calculates the buffered 14-7 van der Waals energy and partitions the energy among the atoms</p>
<p><strong>EHAL3A Subroutine</strong></p>
<p>“ehal3a” calculates the buffered 14-7 van der Waals energy and partitions the energy among the atoms using a pairwise double loop</p>
<p><strong>EHAL3B Subroutine</strong></p>
<p>“ehal3b” calculates the buffered 14-7 van der Waals energy and also partitions the energy among the atoms using the method of lights to locate neighboring atoms</p>
<p><strong>EIGEN Subroutine</strong></p>
<p>“eigen” uses the power method to compute the largest eigenvalues and eigenvectors of the metric matrix, “valid” is set true if the first three eigenvalues are positive</p>
<p><strong>EIGENRGD Subroutine</strong></p>
<p><strong>EIGENROT Subroutine</strong></p>
<p><strong>EIGENROT Subroutine</strong></p>
<p><strong>EIGENTOR Subroutine</strong></p>
<p><strong>EIGENXYZ Subroutine</strong></p>
<p><strong>EIMPROP Subroutine</strong></p>
<p>“eimprop” calculates the improper dihedral potential energy</p>
<p><strong>EIMPROP1 Subroutine</strong></p>
<p>“eimprop1” calculates improper dihedral energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EIMPROP2 Subroutine</strong></p>
<p>“eimprop2” calculates second derivatives of the improper dihedral angle energy for a single atom</p>
<p><strong>EIMPROP3 Subroutine</strong></p>
<p>“eimprop3” calculates the improper dihedral potential energy; also partitions the energy terms among the atoms</p>
<p><strong>EIMPTOR Subroutine</strong></p>
<p>“eimptor” calculates the improper torsion potential energy</p>
<p><strong>EIMPTOR1 Subroutine</strong></p>
<p>“eimptor1” calculates improper torsion energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EIMPTOR2 Subroutine</strong></p>
<p>“eimptor2” calculates second derivatives of the improper torsion energy for a single atom</p>
<p><strong>EIMPTOR3 Subroutine</strong></p>
<p>“eimptor3” calculates the improper torsion potential energy; also partitions the energy terms among the atoms</p>
<p><strong>ELJ Subroutine</strong></p>
<p>“elj” calculates the Lennard-Jones 6-12 van der Waals energy</p>
<p><strong>ELJ0A Subroutine</strong></p>
<p>“elj0a” calculates the Lennard-Jones 6-12 van der Waals energy using a pairwise double loop</p>
<p><strong>ELJ0B Subroutine</strong></p>
<p>“elj0b” calculates the Lennard-Jones 6-12 van der Waals energy using the method of lights to locate neighboring atoms</p>
<p><strong>ELJ0C Subroutine</strong></p>
<p>“elj0c” calculates the Lennard-Jones 6-12 van der Waals energy via a Gaussian approximation for potential energy smoothing</p>
<p><strong>ELJ0D Subroutine</strong></p>
<p>“elj0d” calculates the Lennard-Jones 6-12 van der Waals energy for use with stophat potential energy smoothing</p>
<p><strong>ELJ1 Subroutine</strong></p>
<p>“elj1” calculates the Lennard-Jones 6-12 van der Waals energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>ELJ1A Subroutine</strong></p>
<p>“elj1a” calculates the Lennard-Jones 6-12 van der Waals energy and its first derivatives using a pairwise double loop</p>
<p><strong>ELJ1B Subroutine</strong></p>
<p>“elj1b” calculates the Lennard-Jones 6-12 van der Waals energy and its first derivatives using the method of lights to locate neighboring atoms</p>
<p><strong>ELJ1C Subroutine</strong></p>
<p>“elj1c” calculates the Lennard-Jones 6-12 van der Waals energy  and its first derivatives via a Gaussian approximation for  potential energy smoothing</p>
<p><strong>ELJ1D Subroutine</strong></p>
<p>“elj1d” calculates the van der Waals interaction energy and its first derivatives for use with stophat potential energy smoothing</p>
<p><strong>ELJ2 Subroutine</strong></p>
<p>“elj2” calculates the Lennard-Jones 6-12 van der Waals second derivatives for a single atom at a time</p>
<p><strong>ELJ2A Subroutine</strong></p>
<p>“elj2a” calculates the Lennard-Jones 6-12 van der Waals second derivatives using a double loop over relevant atom pairs</p>
<p><strong>ELJ2B Subroutine</strong></p>
<p>“elj2b” calculates the Lennard-Jones 6-12 van der Waals second derivatives via a Gaussian approximation for use with potential energy smoothing</p>
<p><strong>ELJ2C Subroutine</strong></p>
<p>“elj2c” calculates the Lennard-Jones 6-12 van der Waals second derivatives for use with stophat potential energy smoothing</p>
<p><strong>ELJ3 Subroutine</strong></p>
<p>“elj3” calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the energy among the atoms</p>
<p><strong>ELJ3A Subroutine</strong></p>
<p>“elj3a” calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the energy among the atoms using a pairwise double loop</p>
<p><strong>ELJ3B Subroutine</strong></p>
<p>“elj3b” calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the energy among the atoms using the method of lights to locate neighboring atoms</p>
<p><strong>ELJ3C Subroutine</strong></p>
<p>“elj3c” calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the energy among the atoms via a Gaussian approximation for potential energy smoothing</p>
<p><strong>ELJ3D Subroutine</strong></p>
<p>“elj3d” calculates the Lennard-Jones 6-12 van der Waals energy and also partitions the energy among the atoms for use with stophat potential energy smoothing</p>
<p><strong>EMBED Subroutine</strong></p>
<p>“embed” is a distance geometry routine patterned after the ideas of Gordon Crippen, Irwin Kuntz and Tim Havel; it takes as input a set of upper and lower bounds on the interpoint distances, chirality restraints and torsional restraints, and attempts to generate a set of coordinates that satisfy the input bounds and restraints</p>
<p><strong>EMETAL Subroutine</strong></p>
<p>“emetal” calculates the transition metal ligand field energy</p>
<p><strong>EMETAL1 Subroutine</strong></p>
<p>“emetal1” calculates the transition metal ligand field energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EMETAL2 Subroutine</strong></p>
<p>“emetal2” calculates the transition metal ligand field second derivatives for a single atom at a time</p>
<p><strong>EMETAL3 Subroutine</strong></p>
<p>“emetal3” calculates the transition metal ligand field energy and also partitions the energy among the atoms</p>
<p><strong>EMM3HB Subroutine</strong></p>
<p>“emm3hb” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy</p>
<p><strong>EMM3HB0A Subroutine</strong></p>
<p>“emm3hb0a” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy using a pairwise double loop</p>
<p><strong>EMM3HB0B Subroutine</strong></p>
<p>“emm3hb0b” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy using the method of lights to locate neighboring atoms</p>
<p><strong>EMM3HB1 Subroutine</strong></p>
<p>“emm3hb1” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy with respect to Cartesian coordinates</p>
<p><strong>EMM3HB1A Subroutine</strong></p>
<p>“emm3hb1a” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy with respect to Cartesian coordinates using a pairwise double loop</p>
<p><strong>EMM3HB1B Subroutine</strong></p>
<p>“emm3hb1b” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy with respect to Cartesian coordinates using the method of lights to locate neighboring atoms</p>
<p><strong>EMM3HB2 Subroutine</strong></p>
<p>“emm3hb2” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding second derivatives for a single atom at a time</p>
<p><strong>EMM3HB3 Subroutine</strong></p>
<p>“emm3hb3” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy, and partitions the energy among the atoms</p>
<p><strong>EMM3HB3A Subroutine</strong></p>
<p>“emm3hb3” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy, and partitions the energy among the atoms</p>
<p><strong>EMM3HB3B Subroutine</strong></p>
<p>“emm3hb3b” calculates the MM3 exp-6 van der Waals and directional charge transfer hydrogen bonding energy using the method of lights to locate neighboring atoms</p>
<p><strong>EMPOLE Subroutine</strong></p>
<p>“empole” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability</p>
<p><strong>EMPOLE0A Subroutine</strong></p>
<p>“empole0a” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability using a pairwise double loop</p>
<p><strong>EMPOLE0B Subroutine</strong></p>
<p>“empole0b” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability using a regular Ewald summation</p>
<p><strong>EMPOLE1 Subroutine</strong></p>
<p>“empole1” calculates the multipole and dipole polarization energy and derivatives with respect to Cartesian coordinates</p>
<p><strong>EMPOLE1A Subroutine</strong></p>
<p>“empole1a” calculates the multipole and dipole polarization energy and derivatives with respect to Cartesian coordinates using a pairwise double loop</p>
<p><strong>EMPOLE1B Subroutine</strong></p>
<p>“empole1b” calculates the multipole and dipole polarization energy and derivatives with respect to Cartesian coordinates using a regular Ewald summation</p>
<p><strong>EMPOLE2 Subroutine</strong></p>
<p>“empole2” calculates second derivatives of the multipole and dipole polarization energy for a single atom at a time</p>
<p><strong>EMPOLE2A Subroutine</strong></p>
<p>“empole2a” computes multipole and dipole polarization first derivatives for a single atom with respect to Cartesian coordinates; used to get finite difference second derivatives</p>
<p><strong>EMPOLE3 Subroutine</strong></p>
<p>“empole3” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability, and partitions the energy among the atoms</p>
<p><strong>EMPOLE3A Subroutine</strong></p>
<p>“empole3a” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability, and partitions the energy among the atoms using a double loop</p>
<p><strong>EMPOLE3B Subroutine</strong></p>
<p>“empole3b” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability, and partitions the energy among the atoms using a regular Ewald summation</p>
<p><strong>ENERGY Function</strong></p>
<p>“energy” calls the Subroutine**s to calculate the potential energy terms and sums up to form the total energy</p>
<p><strong>ENRGYZE Subroutine</strong></p>
<p>“energyze” is an auxiliary routine for the analyze Program** that performs the energy analysis and prints the total and intermolecular energies</p>
<p><strong>EOPBEND Subroutine</strong></p>
<p>“eopbend” computes the out-of-plane bend potential energy at trigonal centers via a Wilson-Decius-Cross angle bend</p>
<p><strong>EOPBEND1 Subroutine</strong></p>
<p>“eopbend1” computes the out-of-plane bend potential energy and first derivatives at trigonal centers via a Wilson-Decius-Cross angle bend</p>
<p><strong>EOPBEND2 Subroutine</strong></p>
<p>“eopbend2” calculates second derivatives of the out-of-plane bend energy via a Wilson-Decius-Cross angle bend for a single atom using finite difference methods</p>
<p><strong>EOPBEND2A Subroutine</strong></p>
<p>“eopbend2a” calculates out-of-plane bending first derivatives at a trigonal center via a Wilson-Decius-Cross angle bend; used in computation of finite difference second derivatives</p>
<p><strong>EOPBEND3 Subroutine</strong></p>
<p>“eopbend3” computes the out-of-plane bend potential energy at trigonal centers via a Wilson-Decius-Cross angle bend; also partitions the energy among the atoms</p>
<p><strong>EOPDIST Subroutine</strong></p>
<p>“eopdist” computes the out-of-plane distance potential energy at trigonal centers via the central atom height</p>
<p><strong>EOPDIST1 Subroutine</strong></p>
<p>“eopdist1” computes the out-of-plane distance potential energy and first derivatives at trigonal centers via the central atom height</p>
<p><strong>EOPDIST2 Subroutine</strong></p>
<p>“eopdist2” calculates second derivatives of the out-of-plane distance energy for a single atom via the central atom height</p>
<p><strong>EOPDIST3 Subroutine</strong></p>
<p>“eopdist3” computes the out-of-plane distance potential energy at trigonal centers via the central atom height; also partitions the energy among the atoms</p>
<p><strong>EPITORS Subroutine</strong></p>
<p>“epitors” calculates the pi-orbital torsion potential energy</p>
<p><strong>EPITORS1 Subroutine</strong></p>
<p>“epitors1” calculates the pi-orbital torsion potential energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EPITORS2 Subroutine</strong></p>
<p>“epitors2” calculates the second derivatives of the pi-orbital torsion energy for a single atom using finite difference methods</p>
<p><strong>EPITORS2A Subroutine</strong></p>
<p>“epitors2a” calculates the pi-orbital torsion first derivatives; used in computation of finite difference second derivatives</p>
<p><strong>EPITORS3 Subroutine</strong></p>
<p>“epitors3” calculates the pi-orbital torsion potential energy; also partitions the energy terms among the atoms</p>
<p><strong>EPME Subroutine</strong></p>
<p>“epme” computes the reciprocal space energy for a particle mesh Ewald summation over partial charges</p>
<p><strong>EPME1 Subroutine</strong></p>
<p>“epme1” computes the reciprocal space energy and first derivatives for a particle mesh Ewald summation</p>
<p><strong>EPME3 Subroutine</strong></p>
<p>“epme3” computes the reciprocal space energy for a particle mesh Ewald summation over partial charges and prints information about the energy over the charge grid points</p>
<p><strong>EPUCLC Subroutine</strong></p>
<p><strong>EREAL Subroutine</strong></p>
<p>“ereal” evaluates the real space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability</p>
<p><strong>EREAL1 Subroutine</strong></p>
<p>“ereal1” evaluates the real space portion of the regular Ewald summation energy and gradient due to atomic multipole interactions and dipole polarizability</p>
<p><strong>EREAL3 Subroutine</strong></p>
<p>“ereal3” evaluates the real space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability and partitions the energy among the atoms</p>
<p><strong>ERECIP Subroutine</strong></p>
<p>“erecip” evaluates the reciprocal space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability</p>
<p><strong>ERECIP1 Subroutine</strong></p>
<p>“erecip1” evaluates the reciprocal space portion of the regular Ewald summation energy and gradient due to atomic multipole interactions and dipole polarizability</p>
<p><strong>ERECIP3 Subroutine</strong></p>
<p>“erecip3” evaluates the reciprocal space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability, and prints information about the energy over the reciprocal lattice vectors</p>
<p><strong>ERF Function</strong></p>
<p>“erf” computes a numerical approximation to the value of the error Function** via a Chebyshev approximation</p>
<p><strong>ERFC Function</strong></p>
<p>“erfc” computes a numerical approximation to the value of the complementary error Function** via a Chebyshev approximation</p>
<p><strong>ERFCORE Subroutine</strong></p>
<p>“erfcore” evaluates erf(x) or erfc(x) for a real argument x; when called with mode set to 0 it returns erf, a mode of 1 returns erfc; uses rational Function**s that approximate erf(x) and erfc(x) to at least 18 significant decimal digits</p>
<p><strong>ERFIK Subroutine</strong></p>
<p>“erfik” compute the reaction field energy due to a single pair of atomic multipoles</p>
<p><strong>ERFINV Function</strong></p>
<p>“erfinv” evaluates the inverse of the error Function** erf for a real argument in the range (-1,1) using a rational Function** approximation followed by cycles of Newton-Raphson correction</p>
<p><strong>ERXNFLD Subroutine</strong></p>
<p>“erxnfld” calculates the macroscopic reaction field energy arising from a set of atomic multipoles</p>
<p><strong>ERXNFLD1 Subroutine</strong></p>
<p>“erxnfld1” calculates the macroscopic reaction field energy and derivatives with respect to Cartesian coordinates</p>
<p><strong>ERXNFLD2 Subroutine</strong></p>
<p>“erxnfld2” calculates second derivatives of the macroscopic reaction field energy for a single atom at a time</p>
<p><strong>ERXNFLD3 Subroutine</strong></p>
<p>“erxnfld3” calculates the macroscopic reaction field energy, and also partitions the energy among the atoms</p>
<p><strong>ESOLV Subroutine</strong></p>
<p>“esolv” calculates the continuum solvation energy via either the Eisenberg-McLachlan ASP model, Ooi-Scheraga SASA model, various GB/SA methods or the ACE model</p>
<p><strong>ESOLV1 Subroutine</strong></p>
<p>“esolv1” calculates the continuum solvation energy and first derivatives with respect to Cartesian coordinates using either the Eisenberg-McLachlan ASP, Ooi-Scheraga SASA or various GB/SA solvation models</p>
<p><strong>ESOLV2 Subroutine</strong></p>
<p>“esolv2” calculates second derivatives of the continuum solvation energy using either the Eisenberg-McLachlan ASP, Ooi-Scheraga SASA or various GB/SA solvation models</p>
<p><strong>ESOLV3 Subroutine</strong></p>
<p>“esolv3” calculates the continuum solvation energy using either the Eisenberg-McLachlan ASP model, Ooi-Scheraga SASA model, various GB/SA methods or the ACE model; also partitions the energy among the atoms</p>
<p><strong>ESTRBND Subroutine</strong></p>
<p>“estrbnd” calculates the stretch-bend potential energy</p>
<p><strong>ESTRBND1 Subroutine</strong></p>
<p>“estrbnd1” calculates the stretch-bend potential energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRBND2 Subroutine</strong></p>
<p>“estrbnd2” calculates the stretch-bend potential energy second derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRBND3 Subroutine</strong></p>
<p>“estrbnd3” calculates the stretch-bend potential energy; also partitions the energy among the atoms</p>
<p><strong>ESTRTOR Subroutine</strong></p>
<p>“estrtor” calculates the stretch-torsion potential energy</p>
<p><strong>ESTRTOR1 Subroutine</strong></p>
<p>“estrtor1” calculates the stretch-torsion energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRTOR2 Subroutine</strong></p>
<p>“estrtor2” calculates the stretch-torsion potential energy second derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRTOR3 Subroutine</strong></p>
<p>“estrtor3” calculates the stretch-torsion potential energy; also partitions the energy terms among the atoms</p>
<p><strong>ETORS Subroutine</strong></p>
<p>“etors” calculates the torsional potential energy</p>
<p><strong>ETORS0A Subroutine</strong></p>
<p>“etors0a” calculates the torsional potential energy using a standard sum of Fourier terms</p>
<p><strong>ETORS0B Subroutine</strong></p>
<p>“etors0b” calculates the torsional potential energy for use with potential energy smoothing methods</p>
<p><strong>ETORS1 Subroutine</strong></p>
<p>“etors1” calculates the torsional potential energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ETORS1A Subroutine</strong></p>
<p>“etors1a” calculates the torsional potential energy and first derivatives with respect to Cartesian coordinates using a standard sum of Fourier terms</p>
<p><strong>ETORS1B Subroutine</strong></p>
<p>“etors1b” calculates the torsional potential energy and first derivatives with respect to Cartesian coordinates for use with potential energy smoothing methods</p>
<p><strong>ETORS2 Subroutine</strong></p>
<p>“etors2” calculates the second derivatives of the torsional energy for a single atom</p>
<p><strong>ETORS2A Subroutine</strong></p>
<p>“etors2a” calculates the second derivatives of the torsional energy for a single atom using a standard sum of Fourier terms</p>
<p><strong>ETORS2B Subroutine</strong></p>
<p>“etors2b” calculates the second derivatives of the torsional energy for a single atom for use with potential energy smoothing methods</p>
<p><strong>ETORS3 Subroutine</strong></p>
<p>“etors3” calculates the torsional potential energy; also partitions the energy among the atoms</p>
<p><strong>ETORS3A Subroutine</strong></p>
<p>“etors3a” calculates the torsional potential energy using a standard sum of Fourier terms and partitions the energy among the atoms</p>
<p><strong>ETORS3B Subroutine</strong></p>
<p>“etors3b” calculates the torsional potential energy for use with potential energy smoothing methods and partitions the energy among the atoms</p>
<p><strong>ETORTOR Subroutine</strong></p>
<p>“etortor” calculates the torsion-torsion potential energy</p>
<p><strong>ETORTOR1 Subroutine</strong></p>
<p>“etortor1” calculates the torsion-torsion energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ETORTOR2 Subroutine</strong></p>
<p>“etortor2” calculates the torsion-torsion potential energy second derivatives with respect to Cartesian coordinates</p>
<p><strong>ETORTOR3 Subroutine</strong></p>
<p>“etortor3” calculates the torsion-torsion potential energy; also partitions the energy terms among the atoms</p>
<p><strong>EUREY Subroutine</strong></p>
<p>“eurey” calculates the Urey-Bradley 1-3 interaction energy</p>
<p><strong>EUREY1 Subroutine</strong></p>
<p>“eurey1” calculates the Urey-Bradley interaction energy and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EUREY2 Subroutine</strong></p>
<p>“eurey2” calculates second derivatives of the Urey-Bradley interaction energy for a single atom at a time</p>
<p><strong>EUREY3 Subroutine</strong></p>
<p>“eurey3” calculates the Urey-Bradley energy; also partitions the energy among the atoms</p>
<p><strong>EWALDCOF Subroutine</strong></p>
<p>“ewaldcof” finds a value of the Ewald coefficient such that all terms beyond the specified cutoff distance will have an value less than a specified tolerance</p>
<p><strong>EXPLORE Subroutine</strong></p>
<p>“explore” uses simulated annealing on an initial crude embedded distance geoemtry structure to refine versus the bound, chirality, planarity and torsional error Function**s</p>
<p><strong>EXTRA Subroutine</strong></p>
<p>“extra” calculates any additional user defined potential energy contribution</p>
<p><strong>EXTRA1 Subroutine</strong></p>
<p>“extra1” calculates any additional user defined potential energy contribution and its first derivatives</p>
<p><strong>EXTRA2 Subroutine</strong></p>
<p>“extra2” calculates second derivatives of any additional user defined potential energy contribution for a single atom at a time</p>
<p><strong>EXTRA3 Subroutine</strong></p>
<p>“extra3” calculates any additional user defined potential contribution and also partitions the energy among the atoms</p>
<p><strong>FATAL Subroutine</strong></p>
<p>“fatal” terminates execution due to a user request, a severe error or some other nonstandard condition</p>
<p><strong>FFTBACK Subroutine</strong></p>
<p><strong>FFTFRONT Subroutine</strong></p>
<p><strong>FFTSETUP Subroutine</strong></p>
<p><strong>FIELD Subroutine</strong></p>
<p>“field” sets the force field potential energy Function**s from a parameter file and modifications specified in a keyfile</p>
<p><strong>FINAL Subroutine</strong></p>
<p>“final” performs any final Program** actions, prints a status message, and then pauses if necessary to avoid closing the execution window</p>
<p><strong>FINDATM Subroutine</strong></p>
<p>“findatm” locates a specific PDB atom name type within a range of atoms from the PDB file, returns zero if the name type was not found</p>
<p><strong>FIXPDB Subroutine</strong></p>
<p>“fixpdb” corrects problems with PDB files by converting residue and atom names to the forms used by Tinker</p>
<p><strong>FRACDIST Subroutine</strong></p>
<p>“fracdist” computes a normalized distribution of the pairwise fractional distances between the smoothed upper and lower bounds</p>
<p><strong>FREEUNIT Function</strong></p>
<p>“freeunit” finds an unopened Fortran I/O unit and returns its numerical value from 1 to 99; the units already assigned to “input” and “iout” (usually 5 and 6) are skipped since they have special meaning as the default I/O units</p>
<p><strong>GAMMLN Function</strong></p>
<p>“gammln” uses a series expansion due to Lanczos to compute the natural logarithm of the Gamma Function** at “x” in [0,1]</p>
<p><strong>GDA Program</strong></p>
<p>“gda” implements Gaussian Density Annealing (GDA) algorithm for global optimization via simulated annealing</p>
<p><strong>GDA1 Subroutine</strong></p>
<p><strong>GDA2 Function</strong></p>
<p><strong>GDA3 Subroutine</strong></p>
<p><strong>GDASTAT Subroutine</strong></p>
<p><strong>GENDOT Subroutine</strong></p>
<p>“gendot” finds the coordinates of a specified number of surface points for a sphere with the input radius and coordinate center</p>
<p><strong>GEODESIC Subroutine</strong></p>
<p>“geodesic” smooths the upper and lower distance bounds via the triangle inequality using a sparse matrix version of a shortest path algorithm</p>
<p><strong>GEOMETRY Function</strong></p>
<p>“geometry” finds the value of the interatomic distance, angle or dihedral angle defined by two to four input atoms</p>
<p><strong>GETBASE Subroutine</strong></p>
<p>“getbase” finds the base heavy atoms for a single nucleotide residue and copies the names and coordinates to the Protein Data Bank file</p>
<p><strong>GETIME Subroutine</strong></p>
<p>“getime” gets elapsed CPU time in seconds for an interval</p>
<p><strong>GETINT Subroutine</strong></p>
<p>“getint” asks for an internal coordinate file name, then reads the internal coordinates and computes Cartesian coordinates</p>
<p><strong>GETKEY Subroutine</strong></p>
<p>“getkey” finds a valid keyfile and stores its contents as line images for subsequent keyword parameter searching</p>
<p><strong>GETMOL2 Subroutine</strong></p>
<p>“getmol2” asks for a Sybyl MOL2 molecule file name, then reads the coordinates from the file</p>
<p><strong>GETMONITOR Subroutine</strong></p>
<p><strong>GETNUCH Subroutine</strong></p>
<p>“getnuch” finds the nucleotide hydrogen atoms for a single residue and copies the names and coordinates to the Protein Data Bank file</p>
<p><strong>GETNUMB Subroutine</strong></p>
<p>“getnumb” searchs an input string from left to right for an integer and puts the numeric value in “number”; returns zero with “next” unchanged if no integer value is found</p>
<p><strong>GETPDB Subroutine</strong></p>
<p>“getpdb” asks for a Protein Data Bank file name, then reads in the coordinates file</p>
<p><strong>GETPRB Subroutine</strong></p>
<p>“getprb” tests for a possible probe position at the interface between three neighboring atoms</p>
<p><strong>GETPRM Subroutine</strong></p>
<p>“getprm” finds the potential energy parameter file and then opens and reads the parameters</p>
<p><strong>GETPROH Subroutine</strong></p>
<p>“getproh” finds the hydrogen atoms for a single amino acid residue and copies the names and coordinates to the Protein Data Bank file</p>
<p><strong>GETREF Subroutine</strong></p>
<p>“getref” copies structure information from the reference area into the standard variables for the current system structure</p>
<p><strong>GETSEQ Subroutine</strong></p>
<p>“getseq” asks the user for the amino acid sequence and torsional angle values needed to define a peptide</p>
<p><strong>GETSEQN Subroutine</strong></p>
<p>“getseqn” asks the user for the nucleotide sequence and torsional angle values needed to define a nucleic acid</p>
<p><strong>GETSIDE Subroutine</strong></p>
<p>“getside” finds the side chain heavy atoms for a single amino acid residue and copies the names and coordinates to the Protein Data Bank file</p>
<p><strong>GETSTRING Subroutine</strong></p>
<p>“getstring” searchs for a quoted text string within an input character string; the region between the first and second quotes is returned as the “text”; if the actual text is too long, only the first part is returned</p>
<p><strong>GETTEXT Subroutine</strong></p>
<p>“gettext” searchs an input string for the first string of non-blank characters; the region from a non-blank character to the first blank space is returned as “text”; if the actual text is too long, only the first part is returned</p>
<p><strong>GETTOR Subroutine</strong></p>
<p>“gettor” tests for a possible torus position at the interface between two atoms, and finds the torus radius, center and axis</p>
<p><strong>GETWORD Subroutine</strong></p>
<p>“getword” searchs an input string for the first alphabetic character (A-Z or a-z); the region from this first character to the first blank space or comma is returned as a “word”; if the actual word is too long, only the first part is returned</p>
<p><strong>GETXYZ Subroutine</strong></p>
<p>“getxyz” asks for a Cartesian coordinate file name, then reads in the coordinates file</p>
<p><strong>GRADIENT Subroutine</strong></p>
<p>“gradient” calls Subroutine**s to calculate the potential energy and first derivatives with respect to Cartesian coordinates</p>
<p><strong>GRADRGD Subroutine</strong></p>
<p>“gradrgd” calls Subroutine**s to calculate the potential energy and first derivatives with respect to rigid body coordinates</p>
<p><strong>GRADROT Subroutine</strong></p>
<p>“gradrot” calls Subroutine**s to calculate the potential energy and its torsional first derivatives</p>
<p><strong>GRAFIC Subroutine</strong></p>
<p>“grafic” outputs the upper &amp; lower triangles and diagonal of a square matrix in a schematic form for visual inspection</p>
<p><strong>GROUPS Subroutine</strong></p>
<p>“groups” tests a set of atoms to see if all are members of a single atom group or a pair of atom groups; if so, then the correct intra- or intergroup weight is assigned</p>
<p><strong>GRPLINE Subroutine</strong></p>
<p>“grpline” tests each atom group for linearity of the sites contained in the group</p>
<p><strong>GYRATE Subroutine</strong></p>
<p>“gyrate” computes the radius of gyration of a molecular system from its atomic coordinates</p>
<p><strong>HANGLE Subroutine</strong></p>
<p>“hangle” constructs hybrid angle bending parameters given an initial state, final state and “lambda” value</p>
<p><strong>HATOM Subroutine</strong></p>
<p>“hatom” assigns a new atom type to each hybrid site</p>
<p><strong>HBOND Subroutine</strong></p>
<p>“hbond” constructs hybrid bond stretch parameters given an initial state, final state and “lambda” value</p>
<p><strong>HCHARGE Subroutine</strong></p>
<p>“hcharge” constructs hybrid charge interaction parameters given an initial state, final state and “lambda” value</p>
<p><strong>HDIPOLE Subroutine</strong></p>
<p>“hdipole” constructs hybrid dipole interaction parameters given an initial state, final state and “lambda” value</p>
<p><strong>HESSIAN Subroutine</strong></p>
<p>“hessian” calls Subroutine**s to calculate the Hessian elements for each atom in turn with respect to Cartesian coordinates</p>
<p><strong>HESSRGD Subroutine</strong></p>
<p>“hessrgd” computes the numerical Hessian elements with respect to rigid body coordinates via 6*ngroup+1 gradient evaluations</p>
<p><strong>HESSROT Subroutine</strong></p>
<p>“hessrot” computes the numerical Hessian elements with respect to torsional angles; either the full matrix or just the diagonal can be calculated; the full matrix needs nomega+1 gradient evaluations while the diagonal requires just two gradient calls</p>
<p><strong>HIMPTOR Subroutine</strong></p>
<p>“himptor” constructs hybrid improper torsional parameters given an initial state, final state and “lambda” value</p>
<p><strong>HSTRBND Subroutine</strong></p>
<p>“hstrbnd” constructs hybrid stretch-bend parameters given an initial state, final state and “lambda” value</p>
<p><strong>HSTRTOR Subroutine</strong></p>
<p>“hstrtor” constructs hybrid stretch-torsion parameters given an initial state, final state and “lambda” value</p>
<p><strong>HTORS Subroutine</strong></p>
<p>“htors” constructs hybrid torsional parameters for a given initial state, final state and “lambda” value</p>
<p><strong>HVDW Subroutine</strong></p>
<p>“hvdw” constructs hybrid van der Waals  parameters given an initial state, final state and “lambda” value</p>
<p><strong>HYBRID Subroutine</strong></p>
<p>“hybrid” constructs the hybrid hamiltonian for a specified initial state, final state and mutation parameter “lambda”</p>
<p><strong>IJKPTS Subroutine</strong></p>
<p>“ijkpts” stores a set of indices used during calculation of macroscopic reaction field energetics</p>
<p><strong>IMAGE Subroutine</strong></p>
<p>“image” takes the components of pairwise distance between two points in the same or neighboring periodic boxes and converts to the components of the minimum image distance</p>
<p><strong>IMPOSE Subroutine</strong></p>
<p>“impose” performs the least squares best superposition of two atomic coordinate sets via a quaternion method; upon return, the first coordinate set is unchanged while the second set is translated and rotated to give best fit; the final root mean square fit is returned in “rmsvalue”</p>
<p><strong>INDUCE Subroutine</strong></p>
<p>“induce” computes the induced dipole moment at each polarizable site due to direct or mutual polarization; assumes that multipole components have already been rotated into the global coordinate frame</p>
<p><strong>INDUCE0A Subroutine</strong></p>
<p>“induce0a” computes the induced dipole moment at each polarizable site using a pairwise double loop</p>
<p><strong>INDUCE0B Subroutine</strong></p>
<p>“induce0b” computes the induced dipole moment at each polarizable site using a regular Ewald summation</p>
<p><strong>INEDGE Subroutine</strong></p>
<p>“inedge” inserts a concave edge into the linked list for its temporary torus</p>
<p><strong>INERTIA Subroutine</strong></p>
<p>“inertia” computes the principal moments of inertia for the system, and optionally translates the center of mass to the origin and rotates the principal axes onto the global axes</p>
<p><strong>INITERR Function</strong></p>
<p>“initerr” is the initial error Function** and derivatives for a distance geometry embedding; it includes components from the local geometry and torsional restraint errors</p>
<p><strong>INITIAL Subroutine</strong></p>
<p>“initial” sets up original values for some parameters and variables that might not otherwise get initialized</p>
<p><strong>INITPRM Subroutine</strong></p>
<p>“initprm” completely initializes a force field by setting all parameters to zero and using defaults for control values</p>
<p><strong>INITRES Subroutine</strong></p>
<p>“initres” sets names for biopolymer residue types used in PDB file conversion and automated generation of structures</p>
<p><strong>INITROT Subroutine</strong></p>
<p>“initrot” sets the torsional angles which are to be rotated in subsequent computation, by default automatically selects all rotatable single bonds; assumes internal coordinates have already been setup</p>
<p><strong>INSERT Subroutine</strong></p>
<p>“insert” adds the specified atom to the Cartesian coordinates list and shifts the remaining atoms</p>
<p><strong>INTEDIT Program</strong></p>
<p>“intedit” allows the user to extract information from or alter the values within an internal coordinates file</p>
<p><strong>INTXYZ Program</strong></p>
<p>“intxyz” takes as input an internal coordinates file, converts to and then writes out Cartesian coordinates</p>
<p><strong>INVBETA Function</strong></p>
<p>“invbeta” computes the inverse Beta distribution Function** via a combination of Newton iteration and bisection search</p>
<p><strong>INVERT Subroutine</strong></p>
<p>“invert” inverts a matrix using the Gauss-Jordan method</p>
<p><strong>IPEDGE Subroutine</strong></p>
<p>“ipedge” inserts convex edge into linked list for atom</p>
<p><strong>ISPLPE Subroutine</strong></p>
<p>“isplpe” computes the coefficients for a cubic periodic interpolating spline</p>
<p><strong>JACOBI Subroutine</strong></p>
<p>“jacobi” performs a matrix diagonalization of a real symmetric matrix by the method of Jacobi rotations</p>
<p><strong>KANGANG Subroutine</strong></p>
<p>“kangang” assigns the parameters for angle-angle cross term interactions and processes new or changed parameter values</p>
<p><strong>KANGLE Subroutine</strong></p>
<p>“kangle” assigns the force constants and ideal angles for the bond angles; also processes new or changed parameters</p>
<p><strong>KATOM Subroutine</strong></p>
<p>“katom” assigns an atom type definitions to each atom in the structure and processes any new or changed values</p>
<p><strong>KBOND Subroutine</strong></p>
<p>“kbond” assigns a force constant and ideal bond length to each bond in the structure and processes any new or changed parameter values</p>
<p><strong>KCHARGE Subroutine</strong></p>
<p>“kcharge” assigns partial charges to the atoms within the structure and processes any new or changed values</p>
<p><strong>KCHIRAL Subroutine</strong></p>
<p>“kchiral” determines the target value for each chirality and planarity restraint as the signed volume of the parallelpiped spanned by vectors from a common atom to each of three other atoms</p>
<p><strong>KDIPOLE Subroutine</strong></p>
<p>“kdipole” assigns bond dipoles to the bonds within the structure and processes any new or changed values</p>
<p><strong>KENEG Subroutine</strong></p>
<p>“keneg” applies primary and secondary electronegativity bond length corrections to applicable bond parameters</p>
<p><strong>KEWALD Subroutine</strong></p>
<p>“kewald” assigns both regular Ewald summation and particle mesh Ewald parameters for a periodic box</p>
<p><strong>KGEOM Subroutine</strong></p>
<p>“kgeom” asisgns parameters for geometric restraint terms to be included in the potential energy calculation</p>
<p><strong>KIMPROP Subroutine</strong></p>
<p>“kimprop” assigns potential parameters to each improper dihedral in the structure and processes any changed values</p>
<p><strong>KIMPTOR Subroutine</strong></p>
<p>“kimptor” assigns torsional parameters to each improper torsion in the structure and processes any changed values</p>
<p><strong>KINETIC Subroutine</strong></p>
<p>“kinetic” computes the total kinetic energy and kinetic energy contributions to the pressure tensor by summing over velocities</p>
<p><strong>KMETAL Subroutine</strong></p>
<p>“kmetal” assigns ligand field parameters to transition metal atoms and processes any new or changed parameter values</p>
<p><strong>KMPOLE Subroutine</strong></p>
<p>“kmpole” assigns atomic multipole moments to the atoms of the structure and processes any new or changed values</p>
<p><strong>KOPBEND Subroutine</strong></p>
<p>“kopbend” assigns the force constants for out-of-plane bending at trigonal centers via Wilson-Decius-Cross angle bends; also processes any new or changed parameter values</p>
<p><strong>KOPDIST Subroutine</strong></p>
<p>“kopdist” assigns the force constants for out-of-plane distance at trigonal centers via the central atom height; also processes any new or changed parameter values</p>
<p><strong>KORBIT Subroutine</strong></p>
<p>“korbit” assigns pi-orbital parameters to conjugated systems and processes any new or changed parameters</p>
<p><strong>KPITORS Subroutine</strong></p>
<p>“kpitors” assigns pi-orbital torsion parameters to torsions needing them, and processes any new or changed values</p>
<p><strong>KPOLAR Subroutine</strong></p>
<p>“kpolar” assigns atomic dipole polarizabilities to the atoms within the structure and processes any new or changed values</p>
<p><strong>KSOLV Subroutine</strong></p>
<p>“ksolv” assigns continuum solvation energy parameters for the Eisenberg-McLachlan ASP, Ooi-Scheraga SASA or various GB/SA solvation models</p>
<p><strong>KSTRBND Subroutine</strong></p>
<p>“kstrbnd” assigns the parameters for the stretch-bend interactions and processes new or changed parameter values</p>
<p><strong>KSTRTOR Subroutine</strong></p>
<p>“kstrtor” assigns stretch-torsion parameters to torsions needing them, and processes any new or changed values</p>
<p><strong>KTORS Subroutine</strong></p>
<p>“ktors” assigns torsional parameters to each torsion in the structure and processes any new or changed values</p>
<p><strong>KTORTOR Subroutine</strong></p>
<p>“ktortor” assigns torsion-torsion parameters to adjacent torsion pairs and processes any new or changed values</p>
<p><strong>KUREY Subroutine</strong></p>
<p>“kurey” assigns the force constants and ideal distances for the Urey-Bradley 1-3 interactions; also processes any new or changed parameter values</p>
<p><strong>KVDW Subroutine</strong></p>
<p>“kvdw” assigns the parameters to be used in computing the van der Waals interactions and processes any new or changed values for these parameters</p>
<p><strong>LATTICE Subroutine</strong></p>
<p>“lattice” stores the periodic box dimensions and sets angle values to be used in computing fractional coordinates</p>
<p><strong>LBFGS Subroutine</strong></p>
<p>“lbfgs” is a limited memory BFGS quasi-newton nonlinear optimization routine</p>
<p><strong>LIGASE Subroutine</strong></p>
<p>“ligase” translates a nucleic acid structure in Protein Data Bank format to a Cartesian coordinate file and sequence file</p>
<p><strong>LIGHTS Subroutine</strong></p>
<p>“lights” computes the set of nearest neighbor interactions using the method of lights algorithm</p>
<p><strong>LINBODY Subroutine</strong></p>
<p>“linbody” finds the angular velocity of a linear rigid body given the inertia tensor and angular momentum</p>
<p><strong>LMSTEP Subroutine</strong></p>
<p>“lmstep” computes the Levenberg-Marquardt step during a nonlinear least squares calculation; this version is based upon ideas from the Minpack routine LMPAR together with with the internal doubling strategy of Dennis and Schnabel</p>
<p><strong>LOCALMIN Subroutine</strong></p>
<p>“localmin” is used during normal mode local search to perform a Cartesian coordinate energy minimization</p>
<p><strong>LOCALRGD Subroutine</strong></p>
<p>“localrgd” is used during the PSS local search procedure to perform a rigid body energy minimization</p>
<p><strong>LOCALROT Subroutine</strong></p>
<p>“localrot” is used during the PSS local search procedure to perform a torsional space energy minimization</p>
<p><strong>LOCALXYZ Subroutine</strong></p>
<p>“localxyz” is used during the potential smoothing and search procedure to perform a local optimization at the current smoothing level</p>
<p><strong>LOCERR Function</strong></p>
<p>“locerr” is the local geometry error Function** and derivatives including the 1-2, 1-3 and 1-4 distance bound restraints</p>
<p><strong>LOWCASE Subroutine</strong></p>
<p>“lowcase” converts a text string to all lower case letters</p>
<p><strong>MAJORIZE Subroutine</strong></p>
<p>“majorize” refines the projected coordinates by attempting to minimize the least square residual between the trial distance matrix and the distances computed from the coordinates</p>
<p><strong>MAKEINT Subroutine</strong></p>
<p>“makeint” converts Cartesian to internal coordinates where selection of internal coordinates is controlled by “mode”</p>
<p><strong>MAKEPDB Subroutine</strong></p>
<p>“makexyz” converts a set of Cartesian coordinates to Protein Data Bank format with special handling for systems consisting of polypeptide chains, ligands and water molecules</p>
<p><strong>MAKEREF Subroutine</strong></p>
<p>“makeref” copies the information contained in the “xyz” file of the current structure into corresponding reference areas</p>
<p><strong>MAKEXYZ Subroutine</strong></p>
<p>“makexyz” generates a complete set of Cartesian coordinates for a full structure from the internal coordinate values</p>
<p><strong>MAPCHECK Subroutine</strong></p>
<p>“mapcheck” checks the current minimum energy structure for possible addition to the master list of local minima</p>
<p><strong>MAXWELL Function</strong></p>
<p>“maxwell” returns a speed in Angstroms/picosecond randomly selected from a 3-D Maxwell-Boltzmann distribution for the specified particle mass and system temperature</p>
<p><strong>MCM1 Function</strong></p>
<p>“mcm1” is a service routine that computes the energy and gradient for truncated Newton optimization in Cartesian coordinate space</p>
<p><strong>MCM2 Subroutine</strong></p>
<p>“mcm2” is a service routine that computes the sparse matrix Hessian elements for truncated Newton optimization in Cartesian coordinate space</p>
<p><strong>MCMSTEP Function</strong></p>
<p>“mcmstep” implements the minimization phase of an MCM step via Cartesian minimization following a Monte Carlo step</p>
<p><strong>MDINIT Subroutine</strong></p>
<p>“mdinit” initializes the velocities and accelerations for a molecular dynamics trajectory, including restarts</p>
<p><strong>MDREST Subroutine</strong></p>
<p>“mdrest” finds and removes any translational or rotational kinetic energy of the overall system center of mass</p>
<p><strong>MDSAVE Subroutine</strong></p>
<p>“mdsave” writes molecular dynamics trajectory snapshots and auxiliary files with velocity and induced dipole information; also checks for user requested termination of a simulation</p>
<p><strong>MDSTAT Subroutine</strong></p>
<p>“mdstat” is called at each molecular dynamics time step to form statistics on various average values and fluctuations, and to periodically save the state of the trajectory</p>
<p><strong>MEASFN Subroutine</strong></p>
<p><strong>MEASFP Subroutine</strong></p>
<p><strong>MEASFS Subroutine</strong></p>
<p><strong>MEASPM Subroutine</strong></p>
<p>“measpm” computes the volume of a single prism section of the full interior polyhedron</p>
<p><strong>MECHANIC Subroutine</strong></p>
<p>“mechanic” sets up needed parameters for the potential energy calculation and reads in many of the user selectable options</p>
<p><strong>MERGE Subroutine</strong></p>
<p>“merge” combines the reference and current structures into a single new “current” structure containing the reference atoms followed by the atoms of the current structure</p>
<p><strong>METRIC Subroutine</strong></p>
<p>“metric” takes as input the trial distance matrix and computes the metric matrix of all possible dot products between the atomic vectors and the center of mass using the law of cosines and the following formula for the distances to the center of mass:</p>
<p><strong>MIDERR Function</strong></p>
<p>“miderr” is the secondary error Function** and derivatives for a distance geometry embedding; it includes components from the distance bounds, local geometry, chirality and torsional restraint errors</p>
<p><strong>MINIMIZ1 Function</strong></p>
<p>“minimiz1” is a service routine that computes the energy and gradient for a low storage BFGS optimization in Cartesian coordinate space</p>
<p><strong>MINIMIZE Program</strong></p>
<p>“minimize” performs energy minimization in Cartesian coordinate space using a low storage BFGS nonlinear optimization</p>
<p><strong>MINIROT Program</strong></p>
<p>“minirot” performs an energy minimization in torsional angle space using a low storage BFGS nonlinear optimization</p>
<p><strong>MINIROT1 Function</strong></p>
<p>“minirot1” is a service routine that computes the energy and gradient for a low storage BFGS nonlinear optimization in torsional angle space</p>
<p><strong>MINPATH Subroutine</strong></p>
<p>“minpath” is a routine for finding the triangle smoothed upper and lower bounds of each atom to a specified root atom using a sparse variant of the Bellman-Ford shortest path algorithm</p>
<p><strong>MINRIGID Program</strong></p>
<p>“minrigid” performs an energy minimization of rigid body atom groups using a low storage BFGS nonlinear optimization</p>
<p><strong>MINRIGID1 Function</strong></p>
<p>“minrigid1” is a service routine that computes the energy and gradient for a low storage BFGS nonlinear optimization of rigid bodies</p>
<p><strong>MMID Subroutine</strong></p>
<p>“mmid” implements a modified midpoint method to advance the integration of a set of first order differential equations</p>
<p><strong>MODECART Subroutine</strong></p>
<p><strong>MODEROT Subroutine</strong></p>
<p><strong>MODESRCH Subroutine</strong></p>
<p><strong>MODETORS Subroutine</strong></p>
<p><strong>MODULI Subroutine</strong></p>
<p>“moduli” sets the moduli of the inverse discrete Fourier transform of the B-splines; bsmod[1-3] hold these values, nfft[1-3] are the grid dimensions, bsorder is the order of B-spline approximation</p>
<p><strong>MOLECULE Subroutine</strong></p>
<p>“molecule” counts the molecules, assigns each atom to its molecule and computes the mass of each molecule</p>
<p><strong>MOLUIND Subroutine</strong></p>
<p>“moluind” computes the molecular induced dipole components in the presence of an external electric field</p>
<p><strong>MOMENTS Subroutine</strong></p>
<p>“moments” computes the total electric charge, dipole and quadrupole moments for the entire system as a sum over the partial charges, bond dipoles and atomic multipole moments</p>
<p><strong>MONTE Program</strong></p>
<p>“monte” performs a Monte Carlo/MCM conformational search using either Cartesian single atom or torsional move sets</p>
<p><strong>MUTATE Subroutine</strong></p>
<p>“mutate” constructs the hybrid hamiltonian for a specified initial state, final state and mutation parameter “lambda”</p>
<p><strong>NEEDUPDATE Subroutine</strong></p>
<p><strong>NEIGHBOR Subroutine</strong></p>
<p>“neighbor” finds all of the neighbors of each atom</p>
<p><strong>NEWATM Subroutine</strong></p>
<p>“newatm” creates and defines an atom needed for the Cartesian coordinates file, but which may not present in the original Protein Data Bank file</p>
<p><strong>NEWTON Program</strong></p>
<p>“newton” performs an energy minimization in Cartesian coordinate space using a truncated Newton method</p>
<p><strong>NEWTON1 Function</strong></p>
<p>“newton1” is a service routine that computes the energy and gradient for truncated Newton optimization in Cartesian coordinate space</p>
<p><strong>NEWTON2 Subroutine</strong></p>
<p>“newton2” is a service routine that computes the sparse matrix Hessian elements for truncated Newton optimization in Cartesian coordinate space</p>
<p><strong>NEWTROT Program</strong></p>
<p>“newtrot” performs an energy minimization in torsional angle space using a truncated Newton conjugate gradient method</p>
<p><strong>NEWTROT1 Function</strong></p>
<p>“newtrot1” is a service routine that computes the energy and gradient for truncated Newton conjugate gradient optimization in torsional angle space</p>
<p><strong>NEWTROT2 Subroutine</strong></p>
<p>“newtrot2” is a service routine that computes the sparse matrix Hessian elements for truncated Newton optimization in torsional angle space</p>
<p><strong>NEXTARG Subroutine</strong></p>
<p>“nextarg” finds the next unused command line argument and returns it in the input character string</p>
<p><strong>NEXTTEXT Function</strong></p>
<p>“nexttext” finds and returns the location of the first non-blank character within an input text string; zero is returned if no such character is found</p>
<p><strong>NORMAL Function</strong></p>
<p>“normal” generates a random number from a normal Gaussian distribution with a mean of zero and a variance of one</p>
<p><strong>NUCBASE Subroutine</strong></p>
<p>“nucbase” builds the side chain for a single nucleotide base in terms of internal coordinates</p>
<p><strong>NUCCHAIN Subroutine</strong></p>
<p>“nucchain” builds up the internal coordinates for a nucleic acid sequence from the sugar type, backbone and glycosidic torsional values</p>
<p><strong>NUCLEIC Program</strong></p>
<p>“nucleic” builds the internal and Cartesian coordinates of a polynucleotide from nucleic acid sequence and torsional angle values for the nucleic acid backbone and side chains</p>
<p><strong>NUMBER Function</strong></p>
<p>“number” converts a text numeral into an integer value; the input string must contain only numeric characters</p>
<p><strong>NUMERAL Subroutine</strong></p>
<p>“numeral” converts an input integer number into the corresponding right- or left-justified text numeral</p>
<p><strong>NUMGRAD Subroutine</strong></p>
<p>“numgrad” computes the gradient of the objective Function** “fvalue” with respect to Cartesian coordinates of the atoms via a two-sided numerical differentiation</p>
<p><strong>OCVM Subroutine</strong></p>
<p>“ocvm” is an optimally conditioned variable metric nonlinear optimization routine without line searches</p>
<p><strong>OLDATM Subroutine</strong></p>
<p>“oldatm” get the Cartesian coordinates for an atom from the Protein Data Bank file, then assigns the atom type and atomic connectivities</p>
<p><strong>OPENEND Subroutine</strong></p>
<p>“openend” opens a file on a Fortran unit such that the position is set to the bottom for appending to the end of the file</p>
<p><strong>OPTIMIZ1 Function</strong></p>
<p>“optimiz1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization in Cartesian coordinate space</p>
<p><strong>OPTIMIZE Program</strong></p>
<p>“optimize” performs energy minimization in Cartesian coordinate space using an optimally conditioned variable metric method</p>
<p><strong>OPTIROT Program</strong></p>
<p>“optirot” performs an energy minimization in torsional angle space using an optimally conditioned variable metric method</p>
<p><strong>OPTIROT1 Function</strong></p>
<p>“optirot1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization in torsional angle space</p>
<p><strong>OPTRIGID Program</strong></p>
<p>“optrigid” performs an energy minimization of rigid body atom groups using an optimally conditioned variable metric method</p>
<p><strong>OPTRIGID1 Function</strong></p>
<p>“optrigid1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization of rigid bodies</p>
<p><strong>OPTSAVE Subroutine</strong></p>
<p>“optsave” is used by the optimizers to write imtermediate coordinates and other relevant information; also checks for user requested termination of an optimization</p>
<p><strong>ORBITAL Subroutine</strong></p>
<p>“orbital” finds and organizes lists of atoms in a pisystem, bonds connecting pisystem atoms and torsions whose two central atoms are both pisystem atoms</p>
<p><strong>ORIENT Subroutine</strong></p>
<p>“orient” computes a set of reference Cartesian coordinates in standard orientation for each rigid body atom group</p>
<p><strong>ORTHOG Subroutine</strong></p>
<p>“orthog” performs an orthogonalization of an input matrix via the modified Gram-Schmidt algorithm</p>
<p><strong>OVERLAP Subroutine</strong></p>
<p>“overlap” computes the overlap for two parallel p-orbitals given the atomic numbers and distance of separation</p>
<p><strong>PARAMYZE Subroutine</strong></p>
<p>“paramyze” prints the force field parameters used in the computation of each of the potential energy terms</p>
<p><strong>PASSB Subroutine</strong></p>
<p><strong>PASSB2 Subroutine</strong></p>
<p><strong>PASSB3 Subroutine</strong></p>
<p><strong>PASSB4 Subroutine</strong></p>
<p><strong>PASSB5 Subroutine</strong></p>
<p><strong>PASSF Subroutine</strong></p>
<p><strong>PASSF2 Subroutine</strong></p>
<p><strong>PASSF3 Subroutine</strong></p>
<p><strong>PASSF4 Subroutine</strong></p>
<p><strong>PASSF5 Subroutine</strong></p>
<p><strong>PATH Program</strong></p>
<p>“path” locates a series of structures equally spaced along a conformational pathway connecting the input reactant and product structures; a series of constrained optimizations orthogonal to the path is done via Lagrangian multipliers</p>
<p><strong>PATH1 Function</strong></p>
<p><strong>PATHPNT Subroutine</strong></p>
<p>“pathpnt” finds a structure on the synchronous transit path with the specified path value “t”</p>
<p><strong>PATHSCAN Subroutine</strong></p>
<p>“pathscan” makes a scan of a synchronous transit pathway by computing structures and energies for specific path values</p>
<p><strong>PATHVAL Subroutine</strong></p>
<p>“pathval” computes the synchronous transit path value for the specified structure</p>
<p><strong>PDBATM Subroutine</strong></p>
<p>“pdbatm” adds an atom to the Protein Data Bank file</p>
<p><strong>PDBXYZ Program</strong></p>
<p>“pdbxyz” takes as input a Protein Data Bank file and then converts to and writes out a Cartesian coordinates file and, for biopolymers, a sequence file</p>
<p><strong>PIALTER Subroutine</strong></p>
<p>“pialter” first modifies bond lengths and force constants according to the standard bond slope parameters and the bond order values stored in “pnpl”; also alters some 2-fold torsional parameters based on the bond-order * beta matrix</p>
<p><strong>PIMOVE Subroutine</strong></p>
<p>“pimove” rotates the vector between atoms “list(1)” and “list(2)” so that atom 1 is at the origin and atom 2 along the x-axis; the atoms defining the respective planes are also moved and their bond lengths normalized</p>
<p><strong>PIPLANE Subroutine</strong></p>
<p>“piplane” selects the three atoms which specify the plane perpendicular to each p-orbital; the current version will fail in certain situations, including ketenes, allenes, and isolated or adjacent triple bonds</p>
<p><strong>PISCF Subroutine</strong></p>
<p>“piscf” performs an scf molecular orbital calculation for the pisystem using a modified Pariser-Parr-Pople method</p>
<p><strong>PITILT Subroutine</strong></p>
<p>“pitilt” calculates for each pibond the ratio of the actual p-orbital overlap integral to the ideal overlap if the same orbitals were perfectly parallel</p>
<p><strong>PLACE Subroutine</strong></p>
<p>“place” finds the probe sites by putting the probe sphere tangent to each triple of neighboring atoms</p>
<p><strong>POLARGRP Subroutine</strong></p>
<p>“polargrp” generates members of the polarization group of each atom and separate lists of the 1-2, 1-3 and 1-4 group connectivities</p>
<p><strong>POLARIZE Program</strong></p>
<p>“polarize” computes the molecular polarizability by applying an external field along each axis followed by diagonalization of the resulting polarizability tensor</p>
<p><strong>POLYMER Subroutine</strong></p>
<p>“polymer” tests for the presence of an infinite polymer extending across periodic boundaries</p>
<p><strong>POLYP Subroutine</strong></p>
<p>“polyp” is a polynomial product routine that multiplies two algebraic forms</p>
<p><strong>POTNRG Function</strong></p>
<p><strong>POTOFF Subroutine</strong></p>
<p>“potoff” clears the forcefield definition by turning off the use of each of the potential energy Function**s</p>
<p><strong>POWER Subroutine</strong></p>
<p>“power” uses the power method with deflation to compute the few largest eigenvalues and eigenvectors of a symmetric matrix</p>
<p><strong>PRECISE Function</strong></p>
<p>“precise” finds a machine precision value as selected by the input argument: (1) the smallest positive floating point value, (2) the smallest relative floating point spacing, (3) the largest relative floating point spacing</p>
<p><strong>PRECOND Subroutine</strong></p>
<p>“precond” solves a simplified version of the Newton equations Ms = r, and uses the result to precondition linear conjugate gradient iterations on the full Newton equations in “tnsolve”</p>
<p><strong>PRESSURE Subroutine</strong></p>
<p>“pressure” uses the internal virial to find the pressure in a periodic box and maintains a constant desired pressure by scaling the coordinates via coupling to an external constant pressure bath</p>
<p><strong>PRMKEY Subroutine</strong></p>
<p>“field” parses a text string to extract keywords related to force field potential energy Function**al forms and constants</p>
<p><strong>PROCHAIN Subroutine</strong></p>
<p>“prochain” builds up the internal coordinates for an amino acid sequence from the phi, psi, omega and chi values</p>
<p><strong>PROJCT Subroutine</strong></p>
<p><strong>PROMO Subroutine</strong></p>
<p>“promo” writes a short message containing information about the Tinker version number and the copyright notice</p>
<p><strong>PROPERTY Function</strong></p>
<p>“property” takes two input snapshot frames and computes the value of the property for which the correlation Function** is being accumulated</p>
<p><strong>PROPYZE Subroutine</strong></p>
<p>“propyze” finds and prints the total charge, dipole moment components, radius of gyration and moments of inertia</p>
<p><strong>PROSIDE Subroutine</strong></p>
<p>“proside” builds the side chain for a single amino acid residue in terms of internal coordinates</p>
<p><strong>PROTEIN Program</strong></p>
<p>“protein” builds the internal and Cartesian coordinates of a polypeptide from amino acid sequence and torsional angle values for the peptide backbone and side chains</p>
<p><strong>PRTARC Subroutine</strong></p>
<p>“prtarc” writes out a set of Cartesian coordinates for all active atoms in the Tinker XYZ archive format</p>
<p><strong>PRTCAR Subroutine</strong></p>
<p>“prtcar” writes out a set of Cartesian coordinates for all active atoms in the Accelerys InsightII .car format</p>
<p><strong>PRTDYN Subroutine</strong></p>
<p>“prtdyn” writes out the information needed to restart a molecular dynamics trajectory to an external disk file</p>
<p><strong>PRTERR Subroutine</strong></p>
<p>“prterr” writes out a set of coordinates to a disk file prior to aborting on a serious error</p>
<p><strong>PRTINT Subroutine</strong></p>
<p>“prtint” writes out a set of Z-matrix internal coordinates to an external disk file</p>
<p><strong>PRTMOL2 Program</strong></p>
<p>“prtmol2” writes out a set of coordinates in Sybyl MOL2 format to an external disk file</p>
<p><strong>PRTPDB Subroutine</strong></p>
<p>“prtpdb” writes out a set of Protein Data Bank coordinates to an external disk file</p>
<p><strong>PRTPRM Subroutine</strong></p>
<p>“prtprm” writes out a formatted listing of the default set of potential energy parameters for a force field</p>
<p><strong>PRTSEQ Subroutine</strong></p>
<p>“prtseq” writes out a biopolymer sequence to an external disk file with 15 residues per line and distinct chains separated by blank lines</p>
<p><strong>PRTXMOL Subroutine</strong></p>
<p>“prtxmol” writes out a set of Cartesian coordinates for all active atoms in a simple, generic XYZ format originally used by the XMOL Program**</p>
<p><strong>PRTXYZ Subroutine</strong></p>
<p>“prtxyz” writes out a set of Cartesian coordinates to an external disk file</p>
<p><strong>PSS Program</strong></p>
<p>“pss” implements the potential smoothing plus search method for global optimization in Cartesian coordinate space with local searches performed in Cartesian or torsional space</p>
<p><strong>PSS1 Function</strong></p>
<p>“pss1” is a service routine that computes the energy and gradient during PSS global optimization in Cartesian coordinate space</p>
<p><strong>PSS2 Subroutine</strong></p>
<p>“pss2” is a service routine that computes the sparse matrix Hessian elements during PSS global optimization in Cartesian coordinate space</p>
<p><strong>PSSRGD1 Function</strong></p>
<p>“pssrgd1” is a service routine that computes the energy and gradient during PSS global optimization over rigid bodies</p>
<p><strong>PSSRIGID Program</strong></p>
<p>“pssrigid” implements the potential smoothing plus search method for global optimization for a set of rigid bodies</p>
<p><strong>PSSROT Program</strong></p>
<p>“pssrot” implements the potential smoothing plus search method for global optimization in torsional space</p>
<p><strong>PSSROT1 Function</strong></p>
<p>“pssrot1” is a service routine that computes the energy and gradient during PSS global optimization in torsional space</p>
<p><strong>PSSWRITE Subroutine</strong></p>
<p><strong>PTINCY Function</strong></p>
<p><strong>PZEXTR Subroutine</strong></p>
<p>“pzextr” is a polynomial extrapolation routine used during Bulirsch-Stoer integration of ordinary differential equations</p>
<p><strong>QRFACT Subroutine</strong></p>
<p>“qrfact” performs Householder transformations with column pivoting (optional) to compute a QR factorization of the m by n matrix a; the routine determines an orthogonal matrix q, a permutation matrix p, and an upper trapezoidal matrix r with diagonal elements of nonincreasing magnitude, such that a*p = q*r; the Householder transformation for column k, k = 1,2,…,min(m,n), is of the form</p>
<p><strong>QRSOLVE Subroutine</strong></p>
<p>“qrsolve” solves a*x=b and d*x=0 in the least squares sense; normally used in combination with routine “qrfact” to solve least squares problems</p>
<p><strong>QUATFIT Subroutine</strong></p>
<p>“quatfit” uses a quaternion-based method to achieve the best fit superposition of two sets of coordinates</p>
<p><strong>RADIAL Program</strong></p>
<p>“radial” finds the radial distribution Function** for a specified pair of atom types via analysis of a set of coordinate frames</p>
<p><strong>RANDOM Function</strong></p>
<p>“random” generates a random number on [0,1] via a long period generator due to L’Ecuyer with Bays-Durham shuffle</p>
<p><strong>RANVEC Subroutine</strong></p>
<p>“ranvec” generates a unit vector in 3-dimensional space with uniformly distributed random orientation</p>
<p><strong>RATTLE Subroutine</strong></p>
<p>“rattle” implements the first portion of the rattle algorithm by correcting atomic positions and half-step velocities to maintain interatomic distance and absolute spatial constraints</p>
<p><strong>RATTLE2 Subroutine</strong></p>
<p>“rattle2” implements the second portion of the rattle algorithm by correcting the full-step velocities in order to maintain interatomic distance constraints</p>
<p><strong>READBLK Subroutine</strong></p>
<p>“readblk” reads in a set of snapshot frames and transfers the values to internal arrays for use in the computation of time correlation Function**s</p>
<p><strong>READDYN Subroutine</strong></p>
<p>“readdyn” get the positions, velocities and accelerations for a molecular dynamics restart from an external disk file</p>
<p><strong>READINT Subroutine</strong></p>
<p>“readint” gets a set of Z-matrix internal coordinates from an external file</p>
<p><strong>READMOL2 Subroutine</strong></p>
<p>“readmol2” gets a set of Sybyl MOL2 coordinates from an external disk file</p>
<p><strong>READPDB Subroutine</strong></p>
<p>“readpdb” gets a set of Protein Data Bank coordinates from an external disk file</p>
<p><strong>READPRM Subroutine</strong></p>
<p>“readprm” processes the potential energy parameter file in order to define the default force field parameters</p>
<p><strong>READSEQ Subroutine</strong></p>
<p>“readseq” gets a biopolymer sequence containing one or more separate chains from an external file; all lines containing sequence must begin with the starting sequence number, the actual sequence is read from subsequent nonblank characters</p>
<p><strong>READXYZ Subroutine</strong></p>
<p>“readxyz” gets a set of Cartesian coordinates from an external disk file</p>
<p><strong>REFINE Subroutine</strong></p>
<p>“refine” performs minimization of the atomic coordinates of an initial crude embedded distance geometry structure versus the bound, chirality, planarity and torsional error Function**s</p>
<p><strong>RELEASEMONITOR Subroutine</strong></p>
<p><strong>REPLICA Subroutine</strong></p>
<p>“replica” decides between images and replicates for generation of periodic boundary conditions, and sets the cell replicate list if the replicates method is to be used</p>
<p><strong>RFINDEX Subroutine</strong></p>
<p>“rfindex” finds indices for each multipole site for use in computing reaction field energetics</p>
<p><strong>RGDSRCH Subroutine</strong></p>
<p><strong>RGDSTEP Subroutine</strong></p>
<p>“rgdstep” performs a single molecular dynamics time step for a rigid body calculation</p>
<p><strong>RIBOSOME Subroutine</strong></p>
<p>“ribosome” translates a polypeptide structure in Protein Data Bank format to a Cartesian coordinate file and sequence file</p>
<p><strong>RIGIDXYZ Subroutine</strong></p>
<p>“rigidxyz” computes Cartesian coordinates for a rigid body group via rotation and translation of reference coordinates</p>
<p><strong>RINGS Subroutine</strong></p>
<p>“rings” searches the structure for small rings and stores their constituent atoms</p>
<p><strong>RMSERROR Subroutine</strong></p>
<p>“rmserror” computes the maximum absolute deviation and the rms deviation from the distance bounds, and the number and rms value of the distance restraint violations</p>
<p><strong>RMSFIT Function</strong></p>
<p>“rmsfit” computes the rms fit of two coordinate sets</p>
<p><strong>ROTANG Function</strong></p>
<p><strong>ROTCHECK Function</strong></p>
<p>“rotcheck” tests a specified candidate rotatable bond for the disallowed case where inactive atoms are found on both sides of the candidate bond</p>
<p><strong>ROTEULER Subroutine</strong></p>
<p>“roteuler” computes a set of Euler angle values consistent with an input rotation matrix</p>
<p><strong>ROTLIST Subroutine</strong></p>
<p>“rotlist” generates the minimum list of all the atoms lying to one side of a pair of directly bonded atoms; optionally finds the minimal list by choosing the side with fewer atoms</p>
<p><strong>ROTMAT Subroutine</strong></p>
<p>“rotmat” finds the rotation matrix that converts from the local coordinate system to the global frame at a multipole site</p>
<p><strong>ROTPOLE Subroutine</strong></p>
<p>“rotpole” constructs the set of atomic multipoles in the global frame by applying the correct rotation matrix for each site</p>
<p><strong>ROTRGD Subroutine</strong></p>
<p>“rotrgd” finds the rotation matrix for a rigid body due to a single step of dynamics</p>
<p><strong>ROTSITE Subroutine</strong></p>
<p>“rotsite” computes the atomic multipoles at a specified site in the global coordinate frame by applying a rotation matrix</p>
<p><strong>SADDLE Program</strong></p>
<p>“saddle” finds a transition state between two conformational minima using a combination of ideas from the synchronous transit (Halgren-Lipscomb) and quadratic path (Bell-Crighton) methods</p>
<p><strong>SADDLE1 Function</strong></p>
<p>“saddle1” is a service routine that computes the energy and gradient for transition state optimization</p>
<p><strong>SADDLES Subroutine</strong></p>
<p>“saddles” constructs circles, convex edges and saddle faces</p>
<p><strong>SCAN Program</strong></p>
<p>“scan” attempts to find all the local minima on a potential energy surface via an iterative series of local searches</p>
<p><strong>SCAN1 Function</strong></p>
<p>“scan1” is a service routine that computes the energy and gradient during exploration of a potential energy surface via iterative local search</p>
<p><strong>SCAN2 Subroutine</strong></p>
<p>“scan2” is a service routine that computes the sparse matrix Hessian elements during exploration of a potential energy surface via iterative local search</p>
<p><strong>SDAREA Subroutine</strong></p>
<p>“sdarea” optionally scales the atomic friction coefficient of each atom based on its accessible surface area</p>
<p><strong>SDSTEP Subroutine</strong></p>
<p>“sdstep” performs a single stochastic dynamics time step via a velocity Verlet integration algorithm</p>
<p><strong>SDTERM Subroutine</strong></p>
<p>“sdterm” gets frictional and random force terms needed to update positions and velocities via stochastic dynamics</p>
<p><strong>SEARCH Subroutine</strong></p>
<p>“search” is a unidimensional line search based upon parabolic extrapolation and cubic interpolation using both Function** and gradient values; if forced to search in an uphill direction, return is after the initial step</p>
<p><strong>SETACCELERATION Subroutine</strong></p>
<p><strong>SETATOMIC Subroutine</strong></p>
<p><strong>SETATOMTYPES Subroutine</strong></p>
<p><strong>SETCHARGE Subroutine</strong></p>
<p><strong>SETCONNECTIVITY Subroutine</strong></p>
<p><strong>SETCOORDINATES Subroutine</strong></p>
<p><strong>SETENERGY Subroutine</strong></p>
<p><strong>SETFILE Subroutine</strong></p>
<p><strong>SETFORCEFIELD Subroutine</strong></p>
<p><strong>SETGRADIENTS Subroutine</strong></p>
<p><strong>SETIME Subroutine</strong></p>
<p>“setime” initializes the elapsed interval CPU timer</p>
<p><strong>SETINDUCED Subroutine</strong></p>
<p><strong>SETKEYWORD Subroutine</strong></p>
<p><strong>SETMASS Subroutine</strong></p>
<p><strong>SETNAME Subroutine</strong></p>
<p><strong>SETSTEP Subroutine</strong></p>
<p><strong>SETSTORY Subroutine</strong></p>
<p><strong>SETTIME Subroutine</strong></p>
<p><strong>SETUPDATED Subroutine</strong></p>
<p><strong>SETVELOCITY Subroutine</strong></p>
<p><strong>SHAKEUP Subroutine</strong></p>
<p>“shakeup” initializes any holonomic constraints for use with the rattle algorithm during molecular dynamics</p>
<p><strong>SIGMOID Function</strong></p>
<p>“sigmoid” implements a normalized sigmoidal Function** on the interval [0,1]; the curves connect (0,0) to (1,1) and have a cooperativity controlled by beta, they approach a straight line as beta -&gt; 0 and get more nonlinear as beta increases</p>
<p><strong>SKTDYN Subroutine</strong></p>
<p>“sktdyn” sends the current dynamics info via a socket</p>
<p><strong>SKTINIT Subroutine</strong></p>
<p>“sktinit” sets up socket communication with the graphical user interface by starting a Java virtual machine, initiating a server, and loading an object with system information</p>
<p><strong>SKTKILL Subroutine</strong></p>
<p>“sktkill” closes the server and Java virtual machine</p>
<p><strong>SKTOPT Subroutine</strong></p>
<p>“sktopt” sends the current optimization info via a socket</p>
<p><strong>SLATER Subroutine</strong></p>
<p>“slater” is a general routine for computing the overlap integrals between two Slater-type orbitals</p>
<p><strong>SMOOTH Subroutine</strong></p>
<p>“smooth” sets the type of smoothing method and the extent of surface deformation for use with potential energy smoothing</p>
<p><strong>SNIFFER Program</strong></p>
<p>“sniffer” performs a global energy minimization using a discrete version of Griewank’s global search trajectory</p>
<p><strong>SNIFFER1 Function</strong></p>
<p>“sniffer1” is a service routine that computes the energy and gradient for the Sniffer global optimization method</p>
<p><strong>SOAK Subroutine</strong></p>
<p>“soak” takes a currently defined solute system and places it into a solvent box, with removal of any solvent molecules that overlap the solute</p>
<p><strong>SORT Subroutine</strong></p>
<p>“sort” takes an input list of integers and sorts it into ascending order using the Heapsort algorithm</p>
<p><strong>SORT10 Subroutine</strong></p>
<p>“sort10” takes an input list of character strings and sorts it into alphabetical order using the Heapsort algorithm, duplicate values are removed from the final sorted list</p>
<p><strong>SORT2 Subroutine</strong></p>
<p>“sort2” takes an input list of reals and sorts it into ascending order using the Heapsort algorithm; it also returns a key into the original ordering</p>
<p><strong>SORT3 Subroutine</strong></p>
<p>“sort3” takes an input list of integers and sorts it into ascending order using the Heapsort algorithm; it also returns a key into the original ordering</p>
<p><strong>SORT4 Subroutine</strong></p>
<p>“sort4” takes an input list of integers and sorts it into ascending absolute value using the Heapsort algorithm</p>
<p><strong>SORT5 Subroutine</strong></p>
<p>“sort5” takes an input list of integers and sorts it into ascending order based on each value modulo “m”</p>
<p><strong>SORT6 Subroutine</strong></p>
<p>“sort6” takes an input list of character strings and sorts it into alphabetical order using the Heapsort algorithm</p>
<p><strong>SORT7 Subroutine</strong></p>
<p>“sort7” takes an input list of character strings and sorts it into alphabetical order using the Heapsort algorithm; it also returns a key into the original ordering</p>
<p><strong>SORT8 Subroutine</strong></p>
<p>“sort8” takes an input list of integers and sorts it into ascending order using the Heapsort algorithm, duplicate values are removed from the final sorted list</p>
<p><strong>SORT9 Subroutine</strong></p>
<p>“sort9” takes an input list of reals and sorts it into ascending order using the Heapsort algorithm, duplicate values are removed from the final sorted list</p>
<p><strong>SPACEFILL Program</strong></p>
<p>“spacefill” computes the surface area and volume of a structure; the van der Waals, accessible-excluded, and contact-reentrant definitions are available</p>
<p><strong>SPECTRUM Program</strong></p>
<p>“spectrum” computes a power spectrum over a wavelength range from the velocity autocorrelation as a Function** of time</p>
<p><strong>SQUARE Subroutine</strong></p>
<p>“square” is a nonlinear least squares routine derived from the IMSL routine BCLSF and More’s Minpack routine LMDER; the Jacobian is estimated by finite differences and bounds can be specified for the variables to be refined</p>
<p><strong>SUFFIX Subroutine</strong></p>
<p>“suffix” checks a filename for the presence of an extension, and appends an extension if none is found</p>
<p><strong>SUPERPOSE Program</strong></p>
<p>“superpose” takes pairs of structures and superimposes them in the optimal least squares sense; it will attempt to match all atom pairs or only those specified by the user</p>
<p><strong>SURFACE Subroutine</strong></p>
<p>“surface” performs an analytical computation of the weighted solvent accessible surface area of each atom and the first derivatives of the area with respect to Cartesian coordinates</p>
<p><strong>SURFATOM Subroutine</strong></p>
<p>“surfatom” performs an analytical computation of the surface area of a specified atom; a simplified version of “surface”</p>
<p><strong>SWITCH Subroutine</strong></p>
<p>“switch” sets the coeffcients used by the fifth and seventh order polynomial switching Function**s for spherical cutoffs</p>
<p><strong>SYBYLXYZ Program</strong></p>
<p>“sybylxyz” takes as input a Sybyl MOL2 coordinates file, converts to and then writes out Cartesian coordinates</p>
<p><strong>SYMMETRY Subroutine</strong></p>
<p>“symmetry” applies symmetry operators to the fractional coordinates of the asymmetric unit in order to generate the symmetry related atoms of the full unit cell</p>
<p><strong>TANGENT Subroutine</strong></p>
<p>“tangent” finds the projected gradient on the synchronous transit path for a point along the transit pathway</p>
<p><strong>TEMPER Subroutine</strong></p>
<p>“temper” applies a velocity correction at the half time step as needed for the Nose-Hoover extended system thermostat</p>
<p><strong>TEMPER2 Subroutine</strong></p>
<p>“temper2” computes the instantaneous temperature and applies a thermostat via Berendsen velocity scaling, Andersen stochastic collisions, Langevin piston or Nose-Hoover extended systems</p>
<p><strong>TESTGRAD Program</strong></p>
<p>“testgrad” computes and compares the analytical and numerical gradient vectors of the potential energy Function** with respect to Cartesian coordinates</p>
<p><strong>TESTHESS Program</strong></p>
<p>“testhess” computes and compares the analytical and numerical Hessian matrices of the potential energy Function** with respect to Cartesian coordinates</p>
<p><strong>TESTLIGHT Program</strong></p>
<p>“testlight” performs a set of timing tests to compare the evaluation of potential energy and energy/gradient using the method of lights with a double loop over all atom pairs</p>
<p><strong>TESTROT Program</strong></p>
<p>“testrot” computes and compares the analytical and numerical gradient vectors of the potential energy Function** with respect to rotatable torsional angles</p>
<p><strong>TIMER Program</strong></p>
<p>“timer” measures the CPU time required for file reading and parameter assignment, potential energy computation, energy and gradient computation, and Hessian matrix evaluation</p>
<p><strong>TIMEROT Program</strong></p>
<p>“timerot” measures the CPU time required for file reading and parameter assignment, potential energy computation, energy and gradient over torsions, and torsional angle Hessian matrix evaluation</p>
<p><strong>TNCG Subroutine</strong></p>
<p>“tncg” implements a truncated Newton optimization algorithm in which a preconditioned linear conjugate gradient method is used to approximately solve Newton’s equations; special features include use of an explicit sparse Hessian or finite-difference gradient-Hessian products within the PCG iteration; the exact Newton search directions can be used optionally; by default the algorithm checks for negative curvature to prevent convergence to a stationary point having negative eigenvalues; if a saddle point is desired this test can be removed by disabling “negtest”</p>
<p><strong>TNSOLVE Subroutine</strong></p>
<p>“tnsolve” uses a linear conjugate gradient method to find an approximate solution to the set of linear equations represented in matrix form by Hp = -g (Newton’s equations)</p>
<p><strong>TORPHASE Subroutine</strong></p>
<p>“torphase” sets the n-fold amplitude and phase values for each torsion via sorting of the input parameters</p>
<p><strong>TORQUE Subroutine</strong></p>
<p>“torque” takes the torque values on sites defined by local coordinate frames and distributes thme to convert to forces on the original sites and sites specifying the local frames</p>
<p><strong>TORQUE1 Subroutine</strong></p>
<p>“torque1” takes the torque value on a site defined by a local coordinate frame and distributes it to convert to forces on the original site and sites specifying the local frame</p>
<p><strong>TORSER Function</strong></p>
<p>“torser” computes the torsional error Function** and its first derivatives with respect to the atomic Cartesian coordinates based on the deviation of specified torsional angles from desired values, the contained bond angles are also restrained to avoid a numerical instability</p>
<p><strong>TORSIONS Subroutine</strong></p>
<p>“torsions” finds the total number of dihedral angles and the numbers of the four atoms defining each dihedral angle</p>
<p><strong>TORUS Subroutine</strong></p>
<p>“torus” sets a list of all of the temporary torus positions by testing for a torus between each atom and its neighbors</p>
<p><strong>TOTERR Function</strong></p>
<p>“toterr” is the error Function** and derivatives for a distance geometry embedding; it includes components from the distance bounds, hard sphere contacts, local geometry, chirality and torsional restraint errors</p>
<p><strong>TRANSIT Function</strong></p>
<p>“transit” evaluates the synchronous transit Function** and gradient; linear and quadratic transit paths are available</p>
<p><strong>TRIANGLE Subroutine</strong></p>
<p>“triangle” smooths the upper and lower distance bounds via the triangle inequality using a full-matrix variant of the Floyd-Warshall shortest path algorithm; this routine is usually much slower than the sparse matrix shortest path methods in “geodesic” and “trifix”, and should be used only for comparison with answers generated by those routines</p>
<p><strong>TRIFIX Subroutine</strong></p>
<p>“trifix” rebuilds both the upper and lower distance bound matrices following tightening of one or both of the bounds between a specified pair of atoms, “p” and “q”, using a modification of Murchland’s shortest path update algorithm</p>
<p><strong>TRIMTEXT Function</strong></p>
<p>“trimtext” finds and returns the location of the last non-blank character before the first null character in an input text string; the Function** returns zero if no such character is found</p>
<p><strong>TRIPLE Function</strong></p>
<p>“triple” finds the triple product of three vectors; used as a service routine by the Connolly surface area and volume computation</p>
<p><strong>TRUST Subroutine</strong></p>
<p>“trust” updates the model trust region for a nonlinear least squares calculation; this version is based on the ideas found in NL2SOL and in Dennis and Schnabel’s book</p>
<p><strong>UDIRECT1 Subroutine</strong></p>
<p>“udirect1” computes the reciprocal space contribution of the permanent atomic multipole moments to the electrostatic field for use in finding the direct induced dipole moments via a regular Ewald summation</p>
<p><strong>UDIRECT2 Subroutine</strong></p>
<p>“udirect2” computes the real space contribution of the permanent atomic multipole moments to the electrostatic field for use in finding the direct induced dipole moments via a regular Ewald summation</p>
<p><strong>UFIELD Subroutine</strong></p>
<p>“ufield” finds the field at each polarizable site due to the induced dipoles at the other sites using Thole’s method to damp the field at close range</p>
<p><strong>UMUTUAL1 Subroutine</strong></p>
<p>“umutual1” computes the reciprocal space contribution of the induced atomic dipole moments to the electrostatic field for use in iterative calculation of induced dipole moments via a regular Ewald summation</p>
<p><strong>UMUTUAL2 Subroutine</strong></p>
<p>“umutual2” computes the real space contribution of the induced atomic dipole moments to the electrostatic field for use in iterative calculation of induced dipole moments via a regular Ewald summation</p>
<p><strong>UNITCELL Subroutine</strong></p>
<p>“unitcell” gets the periodic boundary box size and related values from an external keyword file</p>
<p><strong>UPCASE Subroutine</strong></p>
<p>“upcase” converts a text string to all upper case letters</p>
<p><strong>VAM Subroutine</strong></p>
<p>“vam” takes the analytical molecular surface defined as a collection of spherical and toroidal polygons and uses it to compute the volume and surface area</p>
<p><strong>VCROSS Subroutine</strong></p>
<p>“vcross” finds the cross product of two vectors</p>
<p><strong>VDWERR Function</strong></p>
<p>“vdwerr” is the hard sphere van der Waals bound error Function** and derivatives that penalizes close nonbonded contacts, pairwise neighbors are generated via the method of lights</p>
<p><strong>VECANG Function</strong></p>
<p>“vecang” finds the angle between two vectors handed with respect to a coordinate axis; returns an angle in the range [0,2*pi]</p>
<p><strong>VERLET Subroutine</strong></p>
<p>“verlet” performs a single molecular dynamics time step by means of the velocity Verlet multistep recursion formula</p>
<p><strong>VERSION Subroutine</strong></p>
<p>“version” checks the name of a file about to be opened; if if “old” status is passed, the name of the highest current version is returned; if “new” status is passed the filename of the next available unused version is generated</p>
<p><strong>VIBRATE Program</strong></p>
<p>“vibrate” performs a vibrational normal mode analysis; the Hessian matrix of second derivatives is determined and then diagonalized both directly and after mass weighting; output consists of the eigenvalues of the force constant matrix as well as the vibrational frequencies and displacements</p>
<p><strong>VIBRIGID Program</strong></p>
<p>“vibrigid” computes the eigenvalues and eigenvectors of the Hessian matrix over rigid body degrees of freedom</p>
<p><strong>VIBROT Program</strong></p>
<p>“vibrot” computes the eigenvalues and eigenvectors of the torsional Hessian matrix</p>
<p><strong>VNORM Subroutine</strong></p>
<p>“vnorm” normalizes a vector to unit length; used as a service routine by the Connolly surface area and volume computation</p>
<p><strong>VOLUME Subroutine</strong></p>
<p>“volume” calculates the excluded volume via the Connolly analytical volume and surface area algorithm</p>
<p><strong>VOLUME1 Subroutine</strong></p>
<p>“volume1” calculates first derivatives of the total excluded volume with respect to the Cartesian coordinates of each atom</p>
<p><strong>VOLUME2 Subroutine</strong></p>
<p>“volume2” calculates second derivatives of the total excluded volume with respect to the Cartesian coordinates of the atoms</p>
<p><strong>WATSON Subroutine</strong></p>
<p>“watson” uses a rigid body optimization to approximately align the paired strands of a nucleic acid double helix</p>
<p><strong>WATSON1 Function</strong></p>
<p>“watson1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization of rigid bodies</p>
<p><strong>XTALERR Subroutine</strong></p>
<p>“xtalerr” computes an error Function** value derived from derivatives with respect to lattice parameters, lattice energy and monomer dipole moments</p>
<p><strong>XTALFIT Program</strong></p>
<p>“xtalfit” computes an optimized set of potential energy parameters for user specified van der Waals and electrostatic interactions by fitting to crystal structure, lattice energy and monomer dipole moment data</p>
<p><strong>XTALLAT1 Function</strong></p>
<p>“xtalmol1” is a service routine that computes the energy and numerical gradient with respect to the six lattice lengths and angles for a crystal energy minimization</p>
<p><strong>XTALMIN Program</strong></p>
<p>“xtalmin” performs a full crystal energy minimization by alternating cycles of truncated Newton optimization over atomic coordinates with variable metric optimization over the six lattice dimensions and angles</p>
<p><strong>XTALMOL1 Function</strong></p>
<p>“xtalmol1” is a service routine that computes the energy and gradient with respect to the atomic Cartesian coordinates for a crystal energy minimization</p>
<p><strong>XTALMOL2 Subroutine</strong></p>
<p>“xtalmol2” is a service routine that computes the sparse matrix Hessian elements with respect to the atomic Cartesian coordinates for a crystal energy minimization</p>
<p><strong>XTALMOVE Subroutine</strong></p>
<p>“xtalmove” converts fractional to Cartesian coordinates for rigid molecules during fitting of force field parameters to crystal structure data</p>
<p><strong>XTALPRM Subroutine</strong></p>
<p>“xtalprm” stores or retrieves a crystal structure; used to make a previously stored structure the currently active structure, or to store a structure for later use; only provides for the intermolecular energy terms</p>
<p><strong>XTALWRT Subroutine</strong></p>
<p>“xtalwrt” is a utility that prints intermediate results during fitting of force field parameters to crystal data</p>
<p><strong>XYZATM Subroutine</strong></p>
<p>“xyzatm” computes the Cartesian coordinates of a single atom from its defining internal coordinate values</p>
<p><strong>XYZEDIT Program</strong></p>
<p>“xyzedit” provides for modification and manipulation of the contents of a Cartesian coordinates file</p>
<p><strong>XYZINT Program</strong></p>
<p>“xyzint” takes as input a Cartesian coordinates file, then converts to and writes out an internal coordinates file</p>
<p><strong>XYZPDB Program</strong></p>
<p>“xyzpdb” takes as input a Cartesian coordinates file, then converts to and writes out a Protein Data Bank file</p>
<p><strong>XYZRIGID Subroutine</strong></p>
<p>“xyzrigid” computes the center of mass and Euler angle rigid body coordinates for each atom group in the system</p>
<p><strong>XYZSYBYL Program</strong></p>
<p>“xyzsybyl” takes as input a Cartesian coordinates file, converts to and then writes out a Sybyl MOL2 file</p>
<p><strong>ZATOM Subroutine</strong></p>
<p>“zatom” adds an atom to the end of the current Z-matrix and then increments the atom counter; atom type, defining atoms and internal coordinates are passed as arguments</p>
<p><strong>ZHELP Subroutine</strong></p>
<p>“zhelp” prints the general information and instructions for the Z-matrix editing Program**</p>
<p><strong>ZVALUE Subroutine</strong></p>
<p>“zvalue” gets user supplied values for selected coordinates as needed by the internal coordinate editing Program**</p>
</div>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="../index.html">Tinker User's Guide</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="introduction.html">Introduction to the Software</a></li>
<li class="toctree-l1"><a class="reference internal" href="installation.html">Installation on Your Computer</a></li>
<li class="toctree-l1"><a class="reference internal" href="file-types.html">Types of Input &amp; Output Files</a></li>
<li class="toctree-l1"><a class="reference internal" href="energy-programs.html">Potential Energy Programs</a></li>
<li class="toctree-l1"><a class="reference internal" href="analysis-programs.html">Analysis &amp; Utility Programs &amp; Scripts</a></li>
<li class="toctree-l1"><a class="reference internal" href="parameters.html">Force Field Parameter Sets</a></li>
<li class="toctree-l1"><a class="reference internal" href="special-features.html">Special Features &amp; Methods</a></li>
<li class="toctree-l1"><a class="reference internal" href="keywords.html">Use of the Keyword Control File</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Descriptions of Tinker Routines</a></li>
<li class="toctree-l1"><a class="reference internal" href="modules.html">Description of Modules &amp; Global Variables</a></li>
<li class="toctree-l1"><a class="reference internal" href="test-cases.html">Test Cases &amp; Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="benchmarks.html">Benchmark Results</a></li>
<li class="toctree-l1"><a class="reference internal" href="acknowledgements.html">Acknowledgments</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="../index.html">Documentation overview</a><ul>
      <li>Previous: <a href="keywords.html" title="previous chapter">Use of the Keyword Control File</a></li>
      <li>Next: <a href="modules.html" title="next chapter">Description of Modules &amp; Global Variables</a></li>
  </ul></li>
</ul>
</div>
<div id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="../search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" />
      <input type="submit" value="Go" />
    </form>
    </div>
</div>
<script>$('#searchbox').show(0);</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2020, Jay Ponder.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 3.0.3</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.12</a>
      
      |
      <a href="../_sources/text/routines.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>