
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <title>Routines &amp; Functions &#8212; Tinker User&#39;s Guide  documentation</title>
    <link rel="stylesheet" href="../_static/alabaster.css" type="text/css" />
    <link rel="stylesheet" href="../_static/pygments.css" type="text/css" />
    <script id="documentation_options" data-url_root="../" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/language_data.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Modules &amp; Global Variables" href="modules.html" />
    <link rel="prev" title="Use of the Keyword Control File" href="keywords.html" />
   
  <link rel="stylesheet" href="../_static/custom.css" type="text/css" />
  
  
  <meta name="viewport" content="width=device-width, initial-scale=0.9, maximum-scale=0.9" />

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <div class="section" id="routines-functions">
<h1>Routines &amp; Functions<a class="headerlink" href="#routines-functions" title="Permalink to this headline">¶</a></h1>
<p>The distribution version of the Tinker package contains over 1000 separate programs, subroutines and functions. This section contains a brief description of the purpose of most of these code units. Further information can be found in the comments located at the top of each source code file.</p>
<p><strong>ACTIVE Subroutine</strong></p>
<p>“active” sets the list of atoms that are used during
each potential energy function calculation</p>
<p><strong>ADDBASE Subroutine</strong></p>
<p>“addbase” builds the Cartesian coordinates for a single nucleic
acid base; coordinates are read from the Protein Data Bank file
or found from internal coordinates, then atom types are assigned
and connectivity data generated</p>
<p><strong>ADDBOND Subroutine</strong></p>
<p>“addbond” adds entries to the attached atoms list in
order to generate a direct connection between two atoms</p>
<p><strong>ADDIONS Subroutine</strong></p>
<p>“addions” takes a currently defined solvated system and
places ions, with removal of solvent molecules</p>
<p><strong>ADDSIDE Subroutine</strong></p>
<p>“addside” builds the Cartesian coordinates for a single amino
acid side chain; coordinates are read from the Protein Data
Bank file or found from internal coordinates, then atom types
are assigned and connectivity data generated</p>
<p><strong>ADJACENT Function</strong></p>
<p>“adjacent” finds an atom connected to atom “i1” other than
atom “i2”; if no such atom exists, then the closest atom
in space is returned</p>
<p><strong>ADJUST Subroutine</strong></p>
<p>“adjust” modifies site bounds on the PME grid and returns
an offset into the B-spline coefficient arrays</p>
<p><strong>ALCHEMY Program</strong></p>
<p>“alchemy” computes the free energy difference corresponding
to a small perturbation by Boltzmann weighting the potential
energy difference over a number of sample states; current
version (incorrectly) considers the charge energy to be
intermolecular in finding the perturbation energies</p>
<p><strong>ALTELEC Subroutine</strong></p>
<p>“altelec” constructs mutated electrostatic parameters based
on the lambda mutation parameter “elambda”</p>
<p><strong>ALTERPOL Subroutine</strong></p>
<p>“alterpol” finds an output set of atomic multipole parameters
which when used with an intergroup polarization model will
give the same electrostatic potential around the molecule as
the input set of multipole parameters with all atoms in one
polarization group</p>
<p><strong>ALTTORS Subroutine</strong></p>
<p>“alttors” constructs mutated torsional parameters based
on the lambda mutation parameter “tlambda”</p>
<p><strong>AMBERYZE Subroutine</strong></p>
<p>“amberyze” prints the force field parameters in a format needed
by the Amber setup protocol for using AMOEBA within Amber</p>
<p><strong>ANALYSIS Subroutine</strong></p>
<p>“analysis” calls the series of routines needed to calculate
the potential energy and perform energy partitioning analysis
in terms of type of interaction or atom number</p>
<p><strong>ANALYZE Program</strong></p>
<p>“analyze” computes and displays the total potential energy;
options are provided to display system and force field info,
partition the energy by atom or by potential function type,
show force field parameters by atom; output the large energy
interactions and find electrostatic and inertial properties</p>
<p><strong>ANGGUESS Function</strong></p>
<p>“angguess” sets approximate angle bend force constants based
on atom type and connected atoms</p>
<p><strong>ANGLES Subroutine</strong></p>
<p>“angles” finds the total number of bond angles and stores
the atom numbers of the atoms defining each angle; for
each angle to a trivalent central atom, the third bonded
atom is stored for use in out-of-plane bending</p>
<p><strong>ANNEAL Program</strong></p>
<p>“anneal” performs a simulated annealing protocol by means of
variable temperature molecular dynamics using either linear,
exponential or sigmoidal cooling schedules</p>
<p><strong>ANORM Function</strong></p>
<p>“anorm” finds the norm (length) of a vector; used as a
service routine by the Connolly surface area and volume
computation</p>
<p><strong>APBSEMPOLE Subroutine</strong></p>
<p><strong>APBSFINAL Subroutine</strong></p>
<p><strong>APBSINDUCE Subroutine</strong></p>
<p><strong>APBSINITIAL Subroutine</strong></p>
<p><strong>APBSNLINDUCE Subroutine</strong></p>
<p><strong>ARCHIVE Program</strong></p>
<p>“archive” is a utility program for coordinate files which
concatenates multiple coordinate sets into a new archive or
performs any of several manipulations on an existing archive</p>
<p><strong>ASET Subroutine</strong></p>
<p>“aset” computes by recursion the A functions used in the
evaluation of Slater-type (STO) overlap integrals</p>
<p><strong>ATOMYZE Subroutine</strong></p>
<p>“atomyze” prints the potential energy components broken
down by atom and to a choice of precision</p>
<p><strong>ATTACH Subroutine</strong></p>
<p>“attach” generates lists of 1-3, 1-4 and 1-5 connectivities
starting from the previously determined list of attached
atoms (ie, 1-2 connectivity)</p>
<p><strong>AUXINIT Subroutine</strong></p>
<p>“auxinit” initializes auxiliary variables and settings for
inertial extended Lagrangian induced dipole prediction</p>
<p><strong>AVGPOLE Subroutine</strong></p>
<p>“avgpole” condenses the number of multipole atom types based
upon atoms with equivalent attachments and additional user
specified sets of equivalent atoms</p>
<p><strong>BAOAB Subroutine</strong></p>
<p>“baoab” implements a constrained stochastic dynamics time
step using the geodesic BAOAB scheme</p>
<p><strong>BAR Program</strong></p>
<p>“bar” computes the free energy, enthalpy and entropy difference
between two states via Zwanzig free energy perturbation (FEP)
and Bennett acceptance ratio (BAR) methods</p>
<p><strong>BARCALC Subroutine</strong></p>
<p><strong>BASEFILE Subroutine</strong></p>
<p>“basefile” extracts from an input filename the portion
consisting of any directory name and the base filename;
also reads any keyfile and sets information level values</p>
<p><strong>BCUCOF Subroutine</strong></p>
<p>“bcucof” determines the coefficient matrix needed for bicubic
interpolation of a function, gradients and cross derivatives</p>
<p><strong>BCUINT Subroutine</strong></p>
<p>“bcuint” performs a bicubic interpolation of the function
value on a 2D spline grid</p>
<p><strong>BCUINT1 Subroutine</strong></p>
<p>“bcuint1” performs a bicubic interpolation of the function
value and gradient along the directions of a 2D spline grid</p>
<p><strong>BCUINT2 Subroutine</strong></p>
<p>“bcuint2” performs a bicubic interpolation of the function value,
gradient and Hessian along the directions of a 2D spline grid</p>
<p><strong>BEEMAN Subroutine</strong></p>
<p>“beeman” performs a single molecular dynamics time step
via the Beeman multistep recursion formula; uses original
coefficients or Bernie Brooks’ “Better Beeman” values</p>
<p><strong>BETACF Function</strong></p>
<p>“betacf” computes a rapidly convergent continued fraction needed
by routine “betai” to evaluate the cumulative Beta distribution</p>
<p><strong>BETAI Function</strong></p>
<p>“betai” evaluates the cumulative Beta distribution function
as the probability that a random variable from a distribution
with Beta parameters “a” and “b” will be less than “x”</p>
<p><strong>BIGBLOCK Subroutine</strong></p>
<p>“bigblock” replicates the coordinates of a single unit cell
to give a larger unit cell as a block of repeated units</p>
<p><strong>BIOSORT Subroutine</strong></p>
<p>“biosort” renumbers and formats biotype parameters used to
convert biomolecular structure into force field atom types</p>
<p><strong>BITORS Subroutine</strong></p>
<p>“bitors” finds the total number of bitorsions as pairs
of adjacent torsional angles, and the numbers of the five
atoms defining each bitorsion</p>
<p><strong>BMAX Function</strong></p>
<p>“bmax” computes the maximum order of the B functions needed
for evaluation of Slater-type (STO) overlap integrals</p>
<p><strong>BNDERR Function</strong></p>
<p>“bnderr” is the distance bound error function and derivatives;
this version implements the original and Havel’s normalized
lower bound penalty, the normalized version is preferred when
lower bounds are small (as with NMR NOE restraints), the
original penalty is needed if large lower bounds are present</p>
<p><strong>BNDGUESS Function</strong></p>
<p>“bndguess” sets approximate bond stretch force constants based
on atom type and connected atoms</p>
<p><strong>BONDS Subroutine</strong></p>
<p>“bonds” finds the total number of covalent bonds and
stores the atom numbers of the atoms defining each bond</p>
<p><strong>BORN Subroutine</strong></p>
<p>“born” computes the Born radius of each atom for use with
the various implicit solvation models</p>
<p><strong>BORN1 Subroutine</strong></p>
<p>“born1” computes derivatives of the Born radii with respect
to atomic coordinates and increments total energy derivatives
and virial components for potentials involving Born radii</p>
<p><strong>BOUNDS Subroutine</strong></p>
<p>“bounds” finds the center of mass of each molecule and
translates any stray molecules back into the periodic box</p>
<p><strong>BOXMIN Subroutine</strong></p>
<p>“boxmin” uses minimization of valence and vdw potential energy
to expand and refine a collection of solvent molecules in a
periodic box</p>
<p><strong>BOXMIN1 Function</strong></p>
<p>“boxmin1” is a service routine that computes the energy and
gradient during refinement of a periodic box</p>
<p><strong>BSET Subroutine</strong></p>
<p>“bset” computes by downward recursion the B functions used
in the evaluation of Slater-type (STO) overlap integrals</p>
<p><strong>BSPLGEN Subroutine</strong></p>
<p>“bsplgen” gets B-spline coefficients and derivatives for
a single PME atomic site along a particular direction</p>
<p><strong>BSPLINE Subroutine</strong></p>
<p>“bspline” calculates the coefficients for an n-th order
B-spline approximation</p>
<p><strong>BSPLINE_FILL Subroutine</strong></p>
<p>“bspline_fill” finds B-spline coefficients and derivatives
for PME atomic sites along the fractional coordinate axes</p>
<p><strong>BSSTEP Subroutine</strong></p>
<p>“bsstep” takes a single Bulirsch-Stoer step with monitoring
of local truncation error to ensure accuracy</p>
<p><strong>BUSSI Subroutine</strong></p>
<p>“bussi” performs a single molecular dynamics time step via
the Bussi-Parrinello isothermal-isobaric algorithm</p>
<p><strong>CALENDAR Subroutine</strong></p>
<p>“calendar” returns the current time as a set of integer values
representing the year, month, day, hour, minute and second</p>
<p><strong>CART_TO_FRAC Subroutine</strong></p>
<p>“cart_to_frac” computes a transformation matrix to convert
a multipole object in Cartesian coordinates to fractional</p>
<p><strong>CBUILD Subroutine</strong></p>
<p>“cbuild” performs a complete rebuild of the partial charge
electrostatic neighbor list for all sites</p>
<p><strong>CELLANG Subroutine</strong></p>
<p>“cellang” computes atomic coordinates and unit cell parameters
from fractional coordinates and lattice vectors</p>
<p><strong>CELLATOM Subroutine</strong></p>
<p>“cellatom” completes the addition of a symmetry related atom
to a unit cell by updating the atom type and attachment arrays</p>
<p><strong>CENTER Subroutine</strong></p>
<p>“center” moves the weighted centroid of each coordinate
set to the origin during least squares superposition</p>
<p><strong>CERROR Subroutine</strong></p>
<p>“cerror” is the error handling routine for the Connolly
surface area and volume computation</p>
<p><strong>CFFTB Subroutine</strong></p>
<p>“cfftb” computes the backward complex discrete Fourier
transform, the Fourier synthesis</p>
<p><strong>CFFTB1 Subroutine</strong></p>
<p><strong>CFFTF Subroutine</strong></p>
<p>“cfftf” computes the forward complex discrete Fourier
transform, the Fourier analysis</p>
<p><strong>CFFTF1 Subroutine</strong></p>
<p><strong>CFFTI Subroutine</strong></p>
<p>“cffti” initializes arrays used in both forward and backward
transforms; “ifac” is the prime factorization of “n”, and
“wsave” contains a tabulation of trigonometric functions</p>
<p><strong>CFFTI1 Subroutine</strong></p>
<p><strong>CHIRER Function</strong></p>
<p>“chirer” computes the chirality error and its derivatives
with respect to atomic Cartesian coordinates as a sum the
squares of deviations of chiral volumes from target values</p>
<p><strong>CHKANGLE Subroutine</strong></p>
<p>“chkangle” tests angles to be constrained for their presence
in small rings and removes constraints that are redundant</p>
<p><strong>CHKAROM Function</strong></p>
<p>“chkatom” tests for the presence of a specified atom as a
member of an aromatic ring</p>
<p><strong>CHKPOLE Subroutine</strong></p>
<p>“chkpole” inverts atomic multipole moments as necessary
at sites with chiral local reference frame definitions</p>
<p><strong>CHKRING Subroutine</strong></p>
<p>“chkring” tests an atom or a set of connected atoms for
their presence within a single 3- to 6-membered ring</p>
<p><strong>CHKSIZE Subroutine</strong></p>
<p>“chksize” computes a measure of overall global structural
expansion or compaction from the number of excess upper
or lower bounds matrix violations</p>
<p><strong>CHKSOCKET Subroutine</strong></p>
<p><strong>CHKTREE Subroutine</strong></p>
<p>“chktree” tests a minimum energy structure to see if it
belongs to the correct progenitor in the existing map</p>
<p><strong>CHKTTOR Subroutine</strong></p>
<p>“chkttor” tests the attached atoms at a torsion-torsion central
site and inverts the angle values if the site is chiral</p>
<p><strong>CHKXYZ Subroutine</strong></p>
<p>“chkxyz” finds any pairs of atoms with identical Cartesian
coordinates, and prints a warning message</p>
<p><strong>CHOLESKY Subroutine</strong></p>
<p>“cholesky” uses a modified Cholesky method to solve the linear
system Ax = b, returning “x” in “b”; “A” is a real symmetric
positive definite matrix with its upper triangle (including the
diagonal) stored by rows</p>
<p><strong>CIRPLN Subroutine</strong></p>
<p>“cirpln” determines the points of intersection between a
specified circle and plane</p>
<p><strong>CJKM Function</strong></p>
<p>“cjkm” computes the coefficients of spherical harmonics
expressed in prolate spheroidal coordinates</p>
<p><strong>CLIGHT Subroutine</strong></p>
<p>“clight” performs a complete rebuild of the partial charge
pair neighbor list for all sites using the method of lights</p>
<p><strong>CLIMBER Subroutine</strong></p>
<p><strong>CLIMBRGD Subroutine</strong></p>
<p><strong>CLIMBROT Subroutine</strong></p>
<p><strong>CLIMBTOR Subroutine</strong></p>
<p><strong>CLIMBXYZ Subroutine</strong></p>
<p><strong>CLIST Subroutine</strong></p>
<p>“clist” performs an update or a complete rebuild of the
nonbonded neighbor lists for partial charges</p>
<p><strong>CLUSTER Subroutine</strong></p>
<p>“cluster” gets the partitioning of the system into groups
and stores a list of the group to which each atom belongs</p>
<p><strong>CMP_TO_FMP Subroutine</strong></p>
<p>“cmp_to_fmp” transforms the atomic multipoles from Cartesian
to fractional coordinates</p>
<p><strong>COLUMN Subroutine</strong></p>
<p>“column” takes the off-diagonal Hessian elements stored
as sparse rows and sets up indices to allow column access</p>
<p><strong>COMMAND Subroutine</strong></p>
<p>“command” uses the standard Unix-like iargc/getarg routines
to get the number and values of arguments specified on the
command line at program runtime</p>
<p><strong>COMPRESS Subroutine</strong></p>
<p>“compress” transfers only the non-buried tori from
the temporary tori arrays to the final tori arrays</p>
<p><strong>CONNECT Subroutine</strong></p>
<p>“connect” sets up the attached atom arrays
starting from a set of internal coordinates</p>
<p><strong>CONNOLLY Subroutine</strong></p>
<p>“connolly” uses the algorithms from the AMS/VAM programs of
Michael Connolly to compute the analytical molecular surface
area and volume of a collection of spherical atoms; thus
it implements Fred Richards’ molecular surface definition as
a set of analytically defined spherical and toroidal polygons</p>
<p><strong>CONNYZE Subroutine</strong></p>
<p>“connyze” prints information onconnected atoms as lists
of all atom pairs that are 1-2 through 1-5 interactions</p>
<p><strong>CONTACT Subroutine</strong></p>
<p>“contact” constructs the contact surface, cycles and convex faces</p>
<p><strong>CONTROL Subroutine</strong></p>
<p>“control” gets initial values for parameters that determine
the output style and information level provided by Tinker</p>
<p><strong>COORDS Subroutine</strong></p>
<p>“coords” converts the three principal eigenvalues/vectors from
the metric matrix into atomic coordinates, and calls a routine
to compute the rms deviation from the bounds</p>
<p><strong>CORRELATE Program</strong></p>
<p>“correlate” computes the time correlation function of some
user-supplied property from individual snapshot frames taken
from a molecular dynamics or other trajectory</p>
<p><strong>CREATEJVM Subroutine</strong></p>
<p><strong>CREATESERVER Subroutine</strong></p>
<p><strong>CREATESYSTEM Subroutine</strong></p>
<p><strong>CREATEUPDATE Subroutine</strong></p>
<p><strong>CRYSTAL Program</strong></p>
<p>“crystal” is a utility which converts between fractional and
Cartesian coordinates, and can generate full unit cells from
asymmetric units</p>
<p><strong>CSPLINE Subroutine</strong></p>
<p>“cspline” computes the coefficients for a periodic interpolating
cubic spline</p>
<p><strong>CUTOFFS Subroutine</strong></p>
<p>“cutoffs” initializes and stores spherical energy cutoff
distance windows, Hessian element and Ewald sum cutoffs,
and allocates pairwise neighbor lists</p>
<p><strong>CYTSY Subroutine</strong></p>
<p>“cytsy” solves a system of linear equations for a cyclically
tridiagonal, symmetric, positive definite matrix</p>
<p><strong>CYTSYP Subroutine</strong></p>
<p>“cytsyp” finds the Cholesky factors of a cyclically tridiagonal
symmetric, positive definite matrix given by two vectors</p>
<p><strong>CYTSYS Subroutine</strong></p>
<p>“cytsys” solves a cyclically tridiagonal linear system
given the Cholesky factors</p>
<p><strong>D1D2 Function</strong></p>
<p>“d1d2” is a utility function used in computation of the
reaction field recursive summation elements</p>
<p><strong>DAMPDIR Subroutine</strong></p>
<p>“dampdir” generates coefficients for the direct field damping
function for powers of the interatomic distance</p>
<p><strong>DAMPEWALD Subroutine</strong></p>
<p>“dampewald” generates coefficients for Ewald error function
damping for powers of the interatomic distance</p>
<p><strong>DAMPMUT Subroutine</strong></p>
<p>“dampmut” generates coefficients for the mutual field damping
function for powers of the interatomic distance</p>
<p><strong>DAMPPOLAR Subroutine</strong></p>
<p>“damppolar” generates coefficients for the charge penetration
damping function used for polarization interactions</p>
<p><strong>DAMPPOLE Subroutine</strong></p>
<p>“damppole” generates coefficients for the charge penetration
damping function for powers of the interatomic distance</p>
<p><strong>DAMPPOT Subroutine</strong></p>
<p>“damppot” generates coefficients for the charge penetration
damping function used for the electrostatic potential</p>
<p><strong>DAMPREP Subroutine</strong></p>
<p>“damprep” generates coefficients for the Pauli repulsion
damping function for powers of the interatomic distance</p>
<p><strong>DAMPTHOLE Subroutine</strong></p>
<p>“dampthole” generates coefficients for the Thole damping
function for powers of the interatomic distance</p>
<p><strong>DBUILD Subroutine</strong></p>
<p>“dbuild” performs a complete rebuild of the damped dispersion
neighbor list for all sites</p>
<p><strong>DEFLATE Subroutine</strong></p>
<p>“deflate” uses the power method with deflation to compute the
few largest eigenvalues and eigenvectors of a symmetric matrix</p>
<p><strong>DELETE Subroutine</strong></p>
<p>“delete” removes a specified atom from the Cartesian
coordinates list and shifts the remaining atoms</p>
<p><strong>DEPTH Function</strong></p>
<p><strong>DESTROYJVM Subroutine</strong></p>
<p><strong>DESTROYSERVER Subroutine</strong></p>
<p><strong>DFIELD0A Subroutine</strong></p>
<p>“dfield0a” computes the direct electrostatic field due to
permanent multipole moments via a double loop</p>
<p><strong>DFIELD0B Subroutine</strong></p>
<p>“dfield0b” computes the direct electrostatic field due to
permanent multipole moments via a pair list</p>
<p><strong>DFIELD0C Subroutine</strong></p>
<p>“dfield0c” computes the mutual electrostatic field due to
permanent multipole moments via Ewald summation</p>
<p><strong>DFIELD0D Subroutine</strong></p>
<p>“dfield0d” computes the direct electrostatic field due to
permanent multipole moments for use with with generalized
Kirkwood implicit solvation</p>
<p><strong>DFIELD0E Subroutine</strong></p>
<p>“dfield0e” computes the direct electrostatic field due to
permanent multipole moments for use with in Poisson-Boltzmann</p>
<p><strong>DFIELDI Subroutine</strong></p>
<p>“dfieldi” computes the electrostatic field due to permanent
multipole moments</p>
<p><strong>DFTMOD Subroutine</strong></p>
<p>“dftmod” computes the modulus of the discrete Fourier transform
of “bsarray” and stores it in “bsmod”</p>
<p><strong>DIAGBLK Subroutine</strong></p>
<p>“diagblk” performs diagonalization of the Hessian for a
block of atoms within a larger system</p>
<p><strong>DIAGQ Subroutine</strong></p>
<p>“diagq” is a matrix diagonalization routine which is derived
from the classical given, housec, and eigen algorithms with
several modifications to increase efficiency and accuracy</p>
<p><strong>DIFFEQ Subroutine</strong></p>
<p>“diffeq” performs the numerical integration of an ordinary
differential equation using an adaptive stepsize method to
solve the corresponding coupled first-order equations of the
general form dyi/dx = f(x,y1,…,yn) for yi = y1,…,yn</p>
<p><strong>DIFFUSE Program</strong></p>
<p>“diffuse” finds the self-diffusion constant for a homogeneous
liquid via the Einstein relation from a set of stored molecular
dynamics frames; molecular centers of mass are unfolded and mean
squared displacements are computed versus time separation</p>
<p><strong>DIST2 Function</strong></p>
<p>“dist2” finds the distance squared between two points; used
as a service routine by the Connolly surface area and volume
computation</p>
<p><strong>DISTGEOM Program</strong></p>
<p>“distgeom” uses a metric matrix distance geometry procedure to
generate structures with interpoint distances that lie within
specified bounds, with chiral centers that maintain chirality,
and with torsional angles restrained to desired values; the
user also has the ability to interactively inspect and alter
the triangle smoothed bounds matrix prior to embedding</p>
<p><strong>DLIGHT Subroutine</strong></p>
<p>“dlight” performs a complete rebuild of the damped dispersion
pair neighbor list for all sites using the method of lights</p>
<p><strong>DLIST Subroutine</strong></p>
<p>“dlist” performs an update or a complete rebuild of the
nonbonded neighbor lists for damped dispersion sites</p>
<p><strong>DMDUMP Subroutine</strong></p>
<p>“dmdump” puts the distance matrix of the final structure
into the upper half of a matrix, the distance of each atom
to the centroid on the diagonal, and the individual terms
of the bounds errors into the lower half of the matrix</p>
<p><strong>DOCUMENT Program</strong></p>
<p>“document” generates a formatted description of all the code
modules or common blocks, an index of routines called by each
source code module, a listing of all valid keywords, a list of
include file dependencies as needed by a Unix-style Makefile,
or a formatted force field parameter set summary</p>
<p><strong>DOT Function</strong></p>
<p>“dot” finds the dot product of two vectors</p>
<p><strong>DSTMAT Subroutine</strong></p>
<p>“dstmat” selects a distance matrix containing values between
the previously smoothed upper and lower bounds; the distance
values are chosen from uniform distributions, in a triangle
correlated fashion, or using random partial metrization</p>
<p><strong>DYNAMIC Program</strong></p>
<p>“dynamic” computes a molecular or stochastic dynamics trajectory
in one of the standard statistical mechanical ensembles and using
any of several possible integration methods</p>
<p><strong>EANGANG Subroutine</strong></p>
<p>“eangang” calculates the angle-angle potential energy</p>
<p><strong>EANGANG1 Subroutine</strong></p>
<p>“eangang1” calculates the angle-angle potential energy and
first derivatives with respect to Cartesian coordinates</p>
<p><strong>EANGANG2 Subroutine</strong></p>
<p>“eangang2” calculates the angle-angle potential energy
second derivatives with respect to Cartesian coordinates
using finite difference methods</p>
<p><strong>EANGANG2A Subroutine</strong></p>
<p>“eangang2a” calculates the angle-angle first derivatives for
a single interaction with respect to Cartesian coordinates;
used in computation of finite difference second derivatives</p>
<p><strong>EANGANG3 Subroutine</strong></p>
<p>“eangang3” calculates the angle-angle potential energy;
also partitions the energy among the atoms</p>
<p><strong>EANGLE Subroutine</strong></p>
<p>“eangle” calculates the angle bending potential energy;
projected in-plane angles at trigonal centers, special
linear or Fourier angle bending terms are optionally used</p>
<p><strong>EANGLE1 Subroutine</strong></p>
<p>“eangle1” calculates the angle bending potential energy and
the first derivatives with respect to Cartesian coordinates;
projected in-plane angles at trigonal centers, special linear
or Fourier angle bending terms are optionally used</p>
<p><strong>EANGLE2 Subroutine</strong></p>
<p>“eangle2” calculates second derivatives of the angle bending
energy for a single atom using a mixture of analytical and
finite difference methods; projected in-plane angles at trigonal
centers, special linear or Fourier angle bending terms are
optionally used</p>
<p><strong>EANGLE2A Subroutine</strong></p>
<p>“eangle2a” calculates bond angle bending potential energy
second derivatives with respect to Cartesian coordinates</p>
<p><strong>EANGLE2B Subroutine</strong></p>
<p>“eangle2b” computes projected in-plane bending first derivatives
for a single angle with respect to Cartesian coordinates;
used in computation of finite difference second derivatives</p>
<p><strong>EANGLE3 Subroutine</strong></p>
<p>“eangle3” calculates the angle bending potential energy, also
partitions the energy among the atoms; projected in-plane
angles at trigonal centers, spceial linear or Fourier angle
bending terms are optionally used</p>
<p><strong>EANGTOR Subroutine</strong></p>
<p>“eangtor” calculates the angle-torsion potential energy</p>
<p><strong>EANGTOR1 Subroutine</strong></p>
<p>“eangtor1” calculates the angle-torsion energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>EANGTOR2 Subroutine</strong></p>
<p>“eangtor2” calculates the angle-torsion potential energy
second derivatives with respect to Cartesian coordinates</p>
<p><strong>EANGTOR3 Subroutine</strong></p>
<p>“eangtor3” calculates the angle-torsion potential energy;
also partitions the energy terms among the atoms</p>
<p><strong>EBOND Subroutine</strong></p>
<p>“ebond” calculates the bond stretching energy</p>
<p><strong>EBOND1 Subroutine</strong></p>
<p>“ebond1” calculates the bond stretching energy and
first derivatives with respect to Cartesian coordinates</p>
<p><strong>EBOND2 Subroutine</strong></p>
<p>“ebond2” calculates second derivatives of the bond
stretching energy for a single atom at a time</p>
<p><strong>EBOND3 Subroutine</strong></p>
<p>“ebond3” calculates the bond stretching energy; also
partitions the energy among the atoms</p>
<p><strong>EBUCK Subroutine</strong></p>
<p>“ebuck” calculates the Buckingham exp-6 van der Waals energy</p>
<p><strong>EBUCK0A Subroutine</strong></p>
<p>“ebuck0a” calculates the Buckingham exp-6 van der Waals energy
using a pairwise double loop</p>
<p><strong>EBUCK0B Subroutine</strong></p>
<p>“ebuck0b” calculates the Buckingham exp-6 van der Waals energy
using the method of lights</p>
<p><strong>EBUCK0C Subroutine</strong></p>
<p>“ebuck0c” calculates the Buckingham exp-6 van der Waals energy
using a pairwise neighbor list</p>
<p><strong>EBUCK0D Subroutine</strong></p>
<p>“ebuck0d” calculates the Buckingham exp-6 van der Waals energy
via a Gaussian approximation for potential energy smoothing</p>
<p><strong>EBUCK1 Subroutine</strong></p>
<p>“ebuck1” calculates the Buckingham exp-6 van der Waals energy
and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EBUCK1A Subroutine</strong></p>
<p>“ebuck1a” calculates the Buckingham exp-6 van der Waals energy
and its first derivatives using a pairwise double loop</p>
<p><strong>EBUCK1B Subroutine</strong></p>
<p>“ebuck1b” calculates the Buckingham exp-6 van der Waals energy
and its first derivatives using the method of lights</p>
<p><strong>EBUCK1C Subroutine</strong></p>
<p>“ebuck1c” calculates the Buckingham exp-6 van der Waals energy
and its first derivatives using a pairwise neighbor list</p>
<p><strong>EBUCK1D Subroutine</strong></p>
<p>“ebuck1d” calculates the Buckingham exp-6 van der Waals energy
and its first derivatives via a Gaussian approximation for
potential energy smoothing</p>
<p><strong>EBUCK2 Subroutine</strong></p>
<p>“ebuck2” calculates the Buckingham exp-6 van der Waals
second derivatives for a single atom at a time</p>
<p><strong>EBUCK2A Subroutine</strong></p>
<p>“ebuck2a” calculates the Buckingham exp-6 van der Waals second
derivatives using a double loop over relevant atom pairs</p>
<p><strong>EBUCK2B Subroutine</strong></p>
<p>“ebuck2b” calculates the Buckingham exp-6 van der Waals second
derivatives via a Gaussian approximation for use with potential
energy smoothing</p>
<p><strong>EBUCK3 Subroutine</strong></p>
<p>“ebuck3” calculates the Buckingham exp-6 van der Waals energy
and partitions the energy among the atoms</p>
<p><strong>EBUCK3A Subroutine</strong></p>
<p>“ebuck3a” calculates the Buckingham exp-6 van der Waals
energy and partitions the energy among the atoms using
a pairwise double loop</p>
<p><strong>EBUCK3B Subroutine</strong></p>
<p>“ebuck3b” calculates the Buckingham exp-6 van der Waals
energy and also partitions the energy among the atoms using
the method of lights</p>
<p><strong>EBUCK3C Subroutine</strong></p>
<p>“ebuck3c” calculates the Buckingham exp-6 van der Waals energy
and also partitions the energy among the atoms using a pairwise
neighbor list</p>
<p><strong>EBUCK3D Subroutine</strong></p>
<p>“ebuck3d” calculates the Buckingham exp-6 van der Waals energy
via a Gaussian approximation for potential energy smoothing</p>
<p><strong>ECHARGE Subroutine</strong></p>
<p>“echarge” calculates the charge-charge interaction energy</p>
<p><strong>ECHARGE0A Subroutine</strong></p>
<p>“echarge0a” calculates the charge-charge interaction energy
using a pairwise double loop</p>
<p><strong>ECHARGE0B Subroutine</strong></p>
<p>“echarge0b” calculates the charge-charge interaction energy
using the method of lights</p>
<p><strong>ECHARGE0C Subroutine</strong></p>
<p>“echarge0c” calculates the charge-charge interaction energy
using a pairwise neighbor list</p>
<p><strong>ECHARGE0D Subroutine</strong></p>
<p>“echarge0d” calculates the charge-charge interaction energy
using a particle mesh Ewald summation</p>
<p><strong>ECHARGE0E Subroutine</strong></p>
<p>“echarge0e” calculates the charge-charge interaction energy
using a particle mesh Ewald summation and the method of lights</p>
<p><strong>ECHARGE0F Subroutine</strong></p>
<p>“echarge0f” calculates the charge-charge interaction energy
using a particle mesh Ewald summation and a neighbor list</p>
<p><strong>ECHARGE0G Subroutine</strong></p>
<p>“echarge0g” calculates the charge-charge interaction energy
for use with potential smoothing methods</p>
<p><strong>ECHARGE1 Subroutine</strong></p>
<p>“echarge1” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ECHARGE1A Subroutine</strong></p>
<p>“echarge1a” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
using a pairwise double loop</p>
<p><strong>ECHARGE1B Subroutine</strong></p>
<p>“echarge1b” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
using the method of lights</p>
<p><strong>ECHARGE1C Subroutine</strong></p>
<p>“echarge1c” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
using a pairwise neighbor list</p>
<p><strong>ECHARGE1D Subroutine</strong></p>
<p>“echarge1d” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
using a particle mesh Ewald summation</p>
<p><strong>ECHARGE1E Subroutine</strong></p>
<p>“echarge1e” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
using a particle mesh Ewald summation and the method of lights</p>
<p><strong>ECHARGE1F Subroutine</strong></p>
<p>“echarge1f” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
using a particle mesh Ewald summation and a neighbor list</p>
<p><strong>ECHARGE1G Subroutine</strong></p>
<p>“echarge1g” calculates the charge-charge interaction energy
and first derivatives with respect to Cartesian coordinates
for use with potential smoothing methods</p>
<p><strong>ECHARGE2 Subroutine</strong></p>
<p>“echarge2” calculates second derivatives of the
charge-charge interaction energy for a single atom</p>
<p><strong>ECHARGE2A Subroutine</strong></p>
<p>“echarge2a” calculates second derivatives of the charge-charge
interaction energy for a single atom using a pairwise loop</p>
<p><strong>ECHARGE2B Subroutine</strong></p>
<p>“echarge2b” calculates second derivatives of the charge-charge
interaction energy for a single atom using a neighbor list</p>
<p><strong>ECHARGE2C Subroutine</strong></p>
<p>“echarge2c” calculates second derivatives of the reciprocal
space charge-charge interaction energy for a single atom using
a particle mesh Ewald summation via numerical differentiation</p>
<p><strong>ECHARGE2D Subroutine</strong></p>
<p>“echarge2d” calculates second derivatives of the real space
charge-charge interaction energy for a single atom using a
pairwise loop</p>
<p><strong>ECHARGE2E Subroutine</strong></p>
<p>“echarge2e” calculates second derivatives of the real space
charge-charge interaction energy for a single atom using a
pairwise neighbor list</p>
<p><strong>ECHARGE2F Subroutine</strong></p>
<p>“echarge2f” calculates second derivatives of the charge-charge
interaction energy for a single atom for use with potential
smoothing methods</p>
<p><strong>ECHARGE2R Subroutine</strong></p>
<p>“echarge2r” computes reciprocal space charge-charge first
derivatives; used to get finite difference second derivatives</p>
<p><strong>ECHARGE3 Subroutine</strong></p>
<p>“echarge3” calculates the charge-charge interaction energy
and partitions the energy among the atoms</p>
<p><strong>ECHARGE3A Subroutine</strong></p>
<p>“echarge3a” calculates the charge-charge interaction energy
and partitions the energy among the atoms using a pairwise
double loop</p>
<p><strong>ECHARGE3B Subroutine</strong></p>
<p>“echarge3b” calculates the charge-charge interaction energy
and partitions the energy among the atoms using the method
of lights</p>
<p><strong>ECHARGE3C Subroutine</strong></p>
<p>“echarge3c” calculates the charge-charge interaction energy
and partitions the energy among the atoms using a pairwise
neighbor list</p>
<p><strong>ECHARGE3D Subroutine</strong></p>
<p>“echarge3d” calculates the charge-charge interaction energy
and partitions the energy among the atoms using a particle
mesh Ewald summation</p>
<p><strong>ECHARGE3E Subroutine</strong></p>
<p>“echarge3e” calculates the charge-charge interaction energy
and partitions the energy among the atoms using a particle
mesh Ewald summation and the method of lights</p>
<p><strong>ECHARGE3F Subroutine</strong></p>
<p>“echarge3f” calculates the charge-charge interaction energy
and partitions the energy among the atoms using a particle
mesh Ewald summation and a pairwise neighbor list</p>
<p><strong>ECHARGE3G Subroutine</strong></p>
<p>“echarge3g” calculates the charge-charge interaction energy
and partitions the energy among the atoms for use with
potential smoothing methods</p>
<p><strong>ECHGDPL Subroutine</strong></p>
<p>“echgdpl” calculates the charge-dipole interaction energy</p>
<p><strong>ECHGDPL1 Subroutine</strong></p>
<p>“echgdpl1” calculates the charge-dipole interaction energy
and first derivatives with respect to Cartesian coordinates</p>
<p><strong>ECHGDPL2 Subroutine</strong></p>
<p>“echgdpl2” calculates second derivatives of the
charge-dipole interaction energy for a single atom</p>
<p><strong>ECHGDPL3 Subroutine</strong></p>
<p>“echgdpl3” calculates the charge-dipole interaction energy;
also partitions the energy among the atoms</p>
<p><strong>ECHGTRN Subroutine</strong></p>
<p>“echgtrn” calculates the charge transfer potential energy</p>
<p><strong>ECHGTRN0A Subroutine</strong></p>
<p>“echgtrn0a” calculates the charge transfer interaction energy
using a double loop</p>
<p><strong>ECHGTRN0B Subroutine</strong></p>
<p>“echgtrn0b” calculates the charge transfer interaction energy
using the method of lights</p>
<p><strong>ECHGTRN0C Subroutine</strong></p>
<p>“echgtrn0c” calculates the charge transfer interaction energy
using a neighbor list</p>
<p><strong>ECHGTRN1 Subroutine</strong></p>
<p>“echgtrn1” calculates the charge transfer energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>ECHGTRN1A Subroutine</strong></p>
<p>“echgtrn1a” calculates the charge transfer interaction energy
and first derivatives using a double loop</p>
<p><strong>ECHGTRN1B Subroutine</strong></p>
<p>“echgtrn1b” calculates the charge transfer energy and first
derivatives using a pairwise neighbor list</p>
<p><strong>ECHGTRN2 Subroutine</strong></p>
<p>“echgtrn2” calculates the second derivatives of the charge
transfer energy using a double loop over relevant atom pairs</p>
<p><strong>ECHGTRN3 Subroutine</strong></p>
<p>“echgtrn3” calculates the charge transfer energy; also partitions
the energy among the atoms</p>
<p><strong>ECHGTRN3A Subroutine</strong></p>
<p>“echgtrn3a” calculates the charge transfer interaction energy
and also partitions the energy among the atoms using a pairwise
double loop</p>
<p><strong>ECHGTRN3B Subroutine</strong></p>
<p>“echgtrn3b” calculates the charge transfer interaction energy
and also partitions the energy among the atoms using the method
of lights</p>
<p><strong>ECHGTRN3C Subroutine</strong></p>
<p>“echgtrn3c” calculates the charge transfer interaction energy
and also partitions the energy among the atoms using a pairwise
neighbor list</p>
<p><strong>ECRECIP Subroutine</strong></p>
<p>“ecrecip” evaluates the reciprocal space portion of the particle
mesh Ewald energy due to partial charges</p>
<p><strong>ECRECIP1 Subroutine</strong></p>
<p>“ecrecip1” evaluates the reciprocal space portion of the particle
mesh Ewald summation energy and gradient due to partial charges</p>
<p><strong>EDIFF Subroutine</strong></p>
<p>“ediff” calculates the energy of polarizing the vacuum induced
dipoles to their SCRF polarized values</p>
<p><strong>EDIFF1A Subroutine</strong></p>
<p>“ediff1a” calculates the energy and derivatives of polarizing
the vacuum induced dipoles to their SCRF polarized values using
a double loop</p>
<p><strong>EDIFF1B Subroutine</strong></p>
<p>“ediff1b” calculates the energy and derivatives of polarizing
the vacuum induced dipoles to their SCRF polarized values using
a neighbor list</p>
<p><strong>EDIFF3 Subroutine</strong></p>
<p>“ediff3” calculates the energy of polarizing the vacuum induced
dipoles to their generalized Kirkwood values with energy analysis</p>
<p><strong>EDIPOLE Subroutine</strong></p>
<p>“edipole” calculates the dipole-dipole interaction energy</p>
<p><strong>EDIPOLE1 Subroutine</strong></p>
<p>“edipole1” calculates the dipole-dipole interaction energy
and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EDIPOLE2 Subroutine</strong></p>
<p>“edipole2” calculates second derivatives of the
dipole-dipole interaction energy for a single atom</p>
<p><strong>EDIPOLE3 Subroutine</strong></p>
<p>“edipole3” calculates the dipole-dipole interaction energy;
also partitions the energy among the atoms</p>
<p><strong>EDISP Subroutine</strong></p>
<p>“edisp” calculates the damped dispersion potential energy</p>
<p><strong>EDISP0A Subroutine</strong></p>
<p>“edisp0a” calculates the damped dispersion potential energy
using a pairwise double loop</p>
<p><strong>EDISP0B Subroutine</strong></p>
<p>“edisp0b” calculates the damped dispersion potential energy
using a pairwise neighbor list</p>
<p><strong>EDISP0C Subroutine</strong></p>
<p>“edisp0c” calculates the dispersion interaction energy using
particle mesh Ewald summation and a double loop</p>
<p><strong>EDISP0D Subroutine</strong></p>
<p>“edisp0d” calculates the dispersion interaction energy using
particle mesh Ewald summation and a neighbor list</p>
<p><strong>EDISP1 Subroutine</strong></p>
<p>“edisp1” calculates the damped dispersion energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>EDISP1A Subroutine</strong></p>
<p>“edisp1a” calculates the damped dispersion energy and
derivatives with respect to Cartesian coordinates using
a pairwise double loop</p>
<p><strong>EDISP1B Subroutine</strong></p>
<p>“edisp1b” calculates the damped dispersion energy and
derivatives with respect to Cartesian coordinates using
a pairwise neighbor list</p>
<p><strong>EDISP1C Subroutine</strong></p>
<p>“edisp1c” calculates the damped dispersion energy and
derivatives with respect to Cartesian coordinates using
particle mesh Ewald summation and a double loop</p>
<p><strong>EDISP1D Subroutine</strong></p>
<p>“edisp1d” calculates the damped dispersion energy and
derivatives with respect to Cartesian coordinates using
particle mesh Ewald summation and a neighbor list</p>
<p><strong>EDISP2 Subroutine</strong></p>
<p>“edisp2” calculates the damped dispersion second derivatives
for a single atom at a time</p>
<p><strong>EDISP3 Subroutine</strong></p>
<p>“edisp3” calculates the dispersion energy; also partitions
the energy among the atoms</p>
<p><strong>EDISP3A Subroutine</strong></p>
<p>“edisp3a” calculates the dispersion potential energy and
also partitions the energy among the atoms using a pairwise
double loop</p>
<p><strong>EDISP3B Subroutine</strong></p>
<p>“edisp3b” calculates the damped dispersion potential energy
and also partitions the energy among the atomsusing a pairwise
neighbor list</p>
<p><strong>EDISP3C Subroutine</strong></p>
<p>“edisp3c” calculates the dispersion interaction energy using
particle mesh Ewald summation and a double loop</p>
<p><strong>EDISP3D Subroutine</strong></p>
<p>“edisp3d” calculates the damped dispersion energy and analysis
using particle mesh Ewald summation and a neighbor list</p>
<p><strong>EDREAL0C Subroutine</strong></p>
<p>“edreal0c” calculates the damped dispersion potential energy
using a particle mesh Ewald sum and pairwise double loop</p>
<p><strong>EDREAL0D Subroutine</strong></p>
<p>“edreal0d” evaluated the real space portion of the damped
dispersion energy using a neighbor list</p>
<p><strong>EDREAL1C Subroutine</strong></p>
<p>“edreal1c” evaluates the real space portion of the Ewald
summation energy and gradient due to damped dispersion
interactions via a double loop</p>
<p><strong>EDREAL1D Subroutine</strong></p>
<p>“edreal1d” evaluates the real space portion of the Ewald
summation energy and gradient due to damped dispersion
interactions via a neighbor list</p>
<p><strong>EDREAL3C Subroutine</strong></p>
<p>“edreal3c” calculates the real space portion of the damped
dispersion energy and analysis using Ewald and a double loop</p>
<p><strong>EDREAL3D Subroutine</strong></p>
<p>“edreal3d” evaluated the real space portion of the damped
dispersion energy and analysis using Ewald and a neighbor list</p>
<p><strong>EDRECIP Subroutine</strong></p>
<p>“edrecip” evaluates the reciprocal space portion of the particle
mesh Ewald energy due to damped dispersion</p>
<p><strong>EDRECIP1 Subroutine</strong></p>
<p>“edrecip1” evaluates the reciprocal space portion of particle
mesh Ewald energy and gradient due to damped dispersion</p>
<p><strong>EGAUSS Subroutine</strong></p>
<p>“egauss” calculates the Gaussian expansion van der Waals energy</p>
<p><strong>EGAUSS0A Subroutine</strong></p>
<p>“egauss0a” calculates the Gaussian expansion van der Waals
energy using a pairwise double loop</p>
<p><strong>EGAUSS0B Subroutine</strong></p>
<p>“egauss0b” calculates the Gaussian expansion van der Waals energy
using the method of lights</p>
<p><strong>EGAUSS0C Subroutine</strong></p>
<p>“egauss0c” calculates the Gaussian expansion van der Waals
energy using a pairwise neighbor list</p>
<p><strong>EGAUSS0D Subroutine</strong></p>
<p>“egauss0d” calculates the Gaussian expansion van der Waals
energy for use with potential energy smoothing</p>
<p><strong>EGAUSS1 Subroutine</strong></p>
<p>“egauss1” calculates the Gaussian expansion van der Waals
interaction energy and its first derivatives with respect
to Cartesian coordinates</p>
<p><strong>EGAUSS1A Subroutine</strong></p>
<p>“egauss1a” calculates the Gaussian expansion van der Waals
interaction energy and its first derivatives using a pairwise
double loop</p>
<p><strong>EGAUSS1B Subroutine</strong></p>
<p>“egauss1b” calculates the Gaussian expansion van der Waals
energy and its first derivatives with respect to Cartesian
coordinates using the method of lights</p>
<p><strong>EGAUSS1C Subroutine</strong></p>
<p>“egauss1c” calculates the Gaussian expansion van der Waals
energy and its first derivatives with respect to Cartesian
coordinates using a pairwise neighbor list</p>
<p><strong>EGAUSS1D Subroutine</strong></p>
<p>“egauss1d” calculates the Gaussian expansion van der Waals
interaction energy and its first derivatives for use with
potential energy smoothing</p>
<p><strong>EGAUSS2 Subroutine</strong></p>
<p>“egauss2” calculates the Gaussian expansion van der Waals
second derivatives for a single atom at a time</p>
<p><strong>EGAUSS2A Subroutine</strong></p>
<p>“egauss2a” calculates the Gaussian expansion van der Waals
second derivatives using a pairwise double loop</p>
<p><strong>EGAUSS2B Subroutine</strong></p>
<p>“egauss2b” calculates the Gaussian expansion van der Waals
second derivatives for use with potential energy smoothing</p>
<p><strong>EGAUSS3 Subroutine</strong></p>
<p>“egauss3” calculates the Gaussian expansion van der Waals
interaction energy and partitions the energy among the atoms</p>
<p><strong>EGAUSS3A Subroutine</strong></p>
<p>“egauss3a” calculates the Gaussian expansion van der Waals
energy and partitions the energy among the atoms using a
pairwise double loop</p>
<p><strong>EGAUSS3B Subroutine</strong></p>
<p>“egauss3b” calculates the Gaussian expansion van der Waals
energy and partitions the energy among the atoms using the
method of lights</p>
<p><strong>EGAUSS3C Subroutine</strong></p>
<p>“egauss3c” calculates the Gaussian expansion van der Waals
energy and partitions the energy among the atoms using a
pairwise neighbor list</p>
<p><strong>EGAUSS3D Subroutine</strong></p>
<p>“egauss3d” calculates the Gaussian expansion van der Waals
interaction energy and partitions the energy among the atoms
for use with potential energy smoothing</p>
<p><strong>EGB0A Subroutine</strong></p>
<p>“egb0a” calculates the generalized Born polarization energy
for the GB/SA solvation models using a pairwise double loop</p>
<p><strong>EGB0B Subroutine</strong></p>
<p>“egb0b” calculates the generalized Born polarization energy
for the GB/SA solvation models using a pairwise neighbor list</p>
<p><strong>EGB0C Subroutine</strong></p>
<p>“egb0c” calculates the generalized Born polarization energy
for the GB/SA solvation models for use with potential smoothing
methods via analogy to the smoothing of Coulomb’s law</p>
<p><strong>EGB1A Subroutine</strong></p>
<p>“egb1a” calculates the generalized Born electrostatic energy
and first derivatives of the GB/SA solvation models using a
double loop</p>
<p><strong>EGB1B Subroutine</strong></p>
<p>“egb1b” calculates the generalized Born electrostatic energy
and first derivatives of the GB/SA solvation models using a
neighbor list</p>
<p><strong>EGB1C Subroutine</strong></p>
<p>“egb1c” calculates the generalized Born energy and first
derivatives of the GB/SA solvation models for use with
potential smoothing methods</p>
<p><strong>EGB2A Subroutine</strong></p>
<p>“egb2a” calculates second derivatives of the generalized
Born energy term for the GB/SA solvation models</p>
<p><strong>EGB2B Subroutine</strong></p>
<p>“egb2b” calculates second derivatives of the generalized
Born energy term for the GB/SA solvation models for use with
potential smoothing methods</p>
<p><strong>EGB3A Subroutine</strong></p>
<p>“egb3a” calculates the generalized Born electrostatic energy
for GB/SA solvation models using a pairwise double loop; also
partitions the energy among the atoms</p>
<p><strong>EGB3B Subroutine</strong></p>
<p>“egb3b” calculates the generalized Born electrostatic energy
for GB/SA solvation models using a pairwise neighbor list; also
partitions the energy among the atoms</p>
<p><strong>EGB3C Subroutine</strong></p>
<p>“egb3c” calculates the generalized Born electrostatic energy
for GB/SA solvation models for use with potential smoothing
methods via analogy to the smoothing of Coulomb’s law; also
partitions the energy among the atoms</p>
<p><strong>EGEOM Subroutine</strong></p>
<p>“egeom” calculates the energy due to restraints on positions,
distances, angles and torsions as well as Gaussian basin and
spherical droplet restraints</p>
<p><strong>EGEOM1 Subroutine</strong></p>
<p>“egeom1” calculates the energy and first derivatives
with respect to Cartesian coordinates due to restraints
on positions, distances, angles and torsions as well as
Gaussian basin and spherical droplet restraints</p>
<p><strong>EGEOM2 Subroutine</strong></p>
<p>“egeom2” calculates second derivatives of restraints
on positions, distances, angles and torsions as well
as Gaussian basin and spherical droplet restraints</p>
<p><strong>EGEOM3 Subroutine</strong></p>
<p>“egeom3” calculates the energy due to restraints on positions,
distances, angles and torsions as well as Gaussian basin and
droplet restraints; also partitions energy among the atoms</p>
<p><strong>EGK Subroutine</strong></p>
<p>“egk” calculates the generalized Kirkwood electrostatic
solvation free energy for the GK/NP implicit solvation model</p>
<p><strong>EGK0A Subroutine</strong></p>
<p>“egk0a” calculates the electrostatic portion of the implicit
solvation energy via the generalized Kirkwood model</p>
<p><strong>EGK1 Subroutine</strong></p>
<p>“egk1” calculates the implicit solvation energy and derivatives
via the generalized Kirkwood plus nonpolar implicit solvation</p>
<p><strong>EGK1A Subroutine</strong></p>
<p>“egk1a” calculates the electrostatic portion of the implicit
solvation energy and derivatives via the generalized Kirkwood
model</p>
<p><strong>EGK3 Subroutine</strong></p>
<p>“egk3” calculates the generalized Kirkwood electrostatic
energy for GK/NP solvation models; also partitions the
energy among the atoms</p>
<p><strong>EGK3A Subroutine</strong></p>
<p>“egk3a” calculates the electrostatic portion of the implicit
solvation energy via the generalized Kirkwood model; also
partitions the energy among the atoms</p>
<p><strong>EHAL Subroutine</strong></p>
<p>“ehal” calculates the buffered 14-7 van der Waals energy</p>
<p><strong>EHAL0A Subroutine</strong></p>
<p>“ehal0a” calculates the buffered 14-7 van der Waals energy
using a pairwise double loop</p>
<p><strong>EHAL0B Subroutine</strong></p>
<p>“ehal0b” calculates the buffered 14-7 van der Waals energy
using the method of lights</p>
<p><strong>EHAL0C Subroutine</strong></p>
<p>“ehal0c” calculates the buffered 14-7 van der Waals energy
using a pairwise neighbor list</p>
<p><strong>EHAL1 Subroutine</strong></p>
<p>“ehal1” calculates the buffered 14-7 van der Waals energy and
its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EHAL1A Subroutine</strong></p>
<p>“ehal1a” calculates the buffered 14-7 van der Waals energy and
its first derivatives with respect to Cartesian coordinates
using a pairwise double loop</p>
<p><strong>EHAL1B Subroutine</strong></p>
<p>“ehal1b” calculates the buffered 14-7 van der Waals energy and
its first derivatives with respect to Cartesian coordinates
using the method of lights</p>
<p><strong>EHAL1C Subroutine</strong></p>
<p>“ehal1c” calculates the buffered 14-7 van der Waals energy and
its first derivatives with respect to Cartesian coordinates
using a pairwise neighbor list</p>
<p><strong>EHAL2 Subroutine</strong></p>
<p>“ehal2” calculates the buffered 14-7 van der Waals second
derivatives for a single atom at a time</p>
<p><strong>EHAL3 Subroutine</strong></p>
<p>“ehal3” calculates the buffered 14-7 van der Waals energy
and partitions the energy among the atoms</p>
<p><strong>EHAL3A Subroutine</strong></p>
<p>“ehal3a” calculates the buffered 14-7 van der Waals energy
and partitions the energy among the atoms using a pairwise
double loop</p>
<p><strong>EHAL3B Subroutine</strong></p>
<p>“ehal3b” calculates the buffered 14-7 van der Waals energy
and also partitions the energy among the atoms using the
method of lights</p>
<p><strong>EHAL3C Subroutine</strong></p>
<p>“ehal3c” calculates the buffered 14-7 van der Waals energy
and also partitions the energy among the atoms using a
pairwise neighbor list</p>
<p><strong>EHPMF Subroutine</strong></p>
<p>“ehpmf” calculates the hydrophobic potential of mean force
energy using a pairwise double loop</p>
<p><strong>EHPMF1 Subroutine</strong></p>
<p>“ehpmf1” calculates the hydrophobic potential of mean force
energy and first derivatives using a pairwise double loop</p>
<p><strong>EHPMF3 Subroutine</strong></p>
<p>“ehpmf3” calculates the hydrophobic potential of mean force
nonpolar energy; also partitions the energy among the atoms</p>
<p><strong>EIGEN Subroutine</strong></p>
<p>“eigen” uses the power method to compute the largest eigenvalues
and eigenvectors of the metric matrix, “valid” is set true if the
first three eigenvalues are positive</p>
<p><strong>EIGENRGD Subroutine</strong></p>
<p><strong>EIGENROT Subroutine</strong></p>
<p><strong>EIGENROT Subroutine</strong></p>
<p><strong>EIGENTOR Subroutine</strong></p>
<p><strong>EIGENXYZ Subroutine</strong></p>
<p><strong>EIMPROP Subroutine</strong></p>
<p>“eimprop” calculates the improper dihedral potential energy</p>
<p><strong>EIMPROP1 Subroutine</strong></p>
<p>“eimprop1” calculates improper dihedral energy and its
first derivatives with respect to Cartesian coordinates</p>
<p><strong>EIMPROP2 Subroutine</strong></p>
<p>“eimprop2” calculates second derivatives of the improper
dihedral angle energy for a single atom</p>
<p><strong>EIMPROP3 Subroutine</strong></p>
<p>“eimprop3” calculates the improper dihedral potential
energy; also partitions the energy terms among the atoms</p>
<p><strong>EIMPTOR Subroutine</strong></p>
<p>“eimptor” calculates the improper torsion potential energy</p>
<p><strong>EIMPTOR1 Subroutine</strong></p>
<p>“eimptor1” calculates improper torsion energy and its
first derivatives with respect to Cartesian coordinates</p>
<p><strong>EIMPTOR2 Subroutine</strong></p>
<p>“eimptor2” calculates second derivatives of the improper
torsion energy for a single atom</p>
<p><strong>EIMPTOR3 Subroutine</strong></p>
<p>“eimptor3” calculates the improper torsion potential energy;
also partitions the energy terms among the atoms</p>
<p><strong>ELJ Subroutine</strong></p>
<p>“elj” calculates the Lennard-Jones 6-12 van der Waals energy</p>
<p><strong>ELJ0A Subroutine</strong></p>
<p>“elj0a” calculates the Lennard-Jones 6-12 van der Waals energy
using a pairwise double loop</p>
<p><strong>ELJ0B Subroutine</strong></p>
<p>“elj0b” calculates the Lennard-Jones 6-12 van der Waals energy
using the method of lights</p>
<p><strong>ELJ0C Subroutine</strong></p>
<p>“elj0c” calculates the Lennard-Jones 6-12 van der Waals energy
using a pairwise neighbor list</p>
<p><strong>ELJ0D Subroutine</strong></p>
<p>“elj0d” calculates the Lennard-Jones 6-12 van der Waals energy
via a Gaussian approximation for potential energy smoothing</p>
<p><strong>ELJ0E Subroutine</strong></p>
<p>“elj0e” calculates the Lennard-Jones 6-12 van der Waals energy
for use with stophat potential energy smoothing</p>
<p><strong>ELJ1 Subroutine</strong></p>
<p>“elj1” calculates the Lennard-Jones 6-12 van der Waals energy
and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>ELJ1A Subroutine</strong></p>
<p>“elj1a” calculates the Lennard-Jones 6-12 van der Waals energy
and its first derivatives using a pairwise double loop</p>
<p><strong>ELJ1B Subroutine</strong></p>
<p>“elj1b” calculates the Lennard-Jones 6-12 van der Waals energy
and its first derivatives using the method of lights</p>
<p><strong>ELJ1C Subroutine</strong></p>
<p>“elj1c” calculates the Lennard-Jones 12-6 van der Waals energy
and its first derivatives using a pairwise neighbor list</p>
<p><strong>ELJ1D Subroutine</strong></p>
<dl class="simple">
<dt>“elj1d” calculates the Lennard-Jones 6-12 van der Waals energy</dt><dd><p>and its first derivatives via a Gaussian approximation for
potential energy smoothing</p>
</dd>
</dl>
<p><strong>ELJ1E Subroutine</strong></p>
<p>“elj1e” calculates the van der Waals interaction energy and its
first derivatives for use with stophat potential energy smoothing</p>
<p><strong>ELJ2 Subroutine</strong></p>
<p>“elj2” calculates the Lennard-Jones 6-12 van der Waals second
derivatives for a single atom at a time</p>
<p><strong>ELJ2A Subroutine</strong></p>
<p>“elj2a” calculates the Lennard-Jones 6-12 van der Waals second
derivatives using a double loop over relevant atom pairs</p>
<p><strong>ELJ2B Subroutine</strong></p>
<p>“elj2b” calculates the Lennard-Jones 6-12 van der Waals second
derivatives via a Gaussian approximation for use with potential
energy smoothing</p>
<p><strong>ELJ2C Subroutine</strong></p>
<p>“elj2c” calculates the Lennard-Jones 6-12 van der Waals second
derivatives for use with stophat potential energy smoothing</p>
<p><strong>ELJ3 Subroutine</strong></p>
<p>“elj3” calculates the Lennard-Jones 6-12 van der Waals energy
and also partitions the energy among the atoms</p>
<p><strong>ELJ3A Subroutine</strong></p>
<p>“elj3a” calculates the Lennard-Jones 6-12 van der Waals
energy and also partitions the energy among the atoms using
a pairwise double loop</p>
<p><strong>ELJ3B Subroutine</strong></p>
<p>“elj3b” calculates the Lennard-Jones 6-12 van der Waals
energy and also partitions the energy among the atoms using
the method of lights</p>
<p><strong>ELJ3C Subroutine</strong></p>
<p>“elj3c” calculates the Lennard-Jones van der Waals energy
and also partitions the energy among the atoms using a
pairwise neighbor list</p>
<p><strong>ELJ3D Subroutine</strong></p>
<p>“elj3d” calculates the Lennard-Jones 6-12 van der Waals energy
and also partitions the energy among the atoms via a Gaussian
approximation for potential energy smoothing</p>
<p><strong>ELJ3E Subroutine</strong></p>
<p>“elj3e” calculates the Lennard-Jones 6-12 van der Waals energy
and also partitions the energy among the atoms for use with
stophat potential energy smoothing</p>
<p><strong>EMBED Subroutine</strong></p>
<p>“embed” is a distance geometry routine patterned after the
ideas of Gordon Crippen, Irwin Kuntz and Tim Havel; it takes
as input a set of upper and lower bounds on the interpoint
distances, chirality restraints and torsional restraints,
and attempts to generate a set of coordinates that satisfy
the input bounds and restraints</p>
<p><strong>EMETAL Subroutine</strong></p>
<p>“emetal” calculates the transition metal ligand field energy</p>
<p><strong>EMETAL1 Subroutine</strong></p>
<p>“emetal1” calculates the transition metal ligand field energy
and its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EMETAL2 Subroutine</strong></p>
<p>“emetal2” calculates the transition metal ligand field second
derivatives for a single atom at a time</p>
<p><strong>EMETAL3 Subroutine</strong></p>
<p>“emetal3” calculates the transition metal ligand field energy
and also partitions the energy among the atoms</p>
<p><strong>EMM3HB Subroutine</strong></p>
<p>“emm3hb” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding energy</p>
<p><strong>EMM3HB0A Subroutine</strong></p>
<p>“emm3hb0a” calculates the MM3 exp-6 van der Waals and
directional charge transfer hydrogen bonding energy using
a pairwise double loop</p>
<p><strong>EMM3HB0B Subroutine</strong></p>
<p>“emm3hb0b” calculates the MM3 exp-6 van der Waals and
directional charge transfer hydrogen bonding energy using
the method of lights</p>
<p><strong>EMM3HB0C Subroutine</strong></p>
<p>“emm3hb0c” calculates the MM3 exp-6 van der Waals and
directional charge transfer hydrogen bonding energy using
a pairwise neighbor list</p>
<p><strong>EMM3HB1 Subroutine</strong></p>
<p>“emm3hb1” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding energy with respect to Cartesian
coordinates</p>
<p><strong>EMM3HB1A Subroutine</strong></p>
<p>“emm3hb1a” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding energy with respect to Cartesian
coordinates using a pairwise double loop</p>
<p><strong>EMM3HB1B Subroutine</strong></p>
<p>“emm3hb1b” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding energy with respect to Cartesian
coordinates using the method of lights</p>
<p><strong>EMM3HB1C Subroutine</strong></p>
<p>“emm3hb1c” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding energy with respect to Cartesian
coordinates using a pairwise neighbor list</p>
<p><strong>EMM3HB2 Subroutine</strong></p>
<p>“emm3hb2” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding second derivatives for a single
atom at a time</p>
<p><strong>EMM3HB3 Subroutine</strong></p>
<p>“emm3hb3” calculates the MM3 exp-6 van der Waals and directional
charge transfer hydrogen bonding energy, and partitions the energy
among the atoms</p>
<p><strong>EMM3HB3A Subroutine</strong></p>
<p>“emm3hb3” calculates the MM3 exp-6 van der Waals and
directional charge transfer hydrogen bonding energy, and
partitions the energy among the atoms</p>
<p><strong>EMM3HB3B Subroutine</strong></p>
<p>“emm3hb3b” calculates the MM3 exp-6 van der Waals and
directional charge transfer hydrogen bonding energy using
the method of lights</p>
<p><strong>EMM3HB3C Subroutine</strong></p>
<p>“emm3hb3c” calculates the MM3 exp-6 van der Waals and
directional charge transfer hydrogen bonding energy using
a pairwise neighbor list</p>
<p><strong>EMPOLE Subroutine</strong></p>
<p>“empole” calculates the electrostatic energy due to atomic
multipole interactions</p>
<p><strong>EMPOLE0A Subroutine</strong></p>
<p>“empole0a” calculates the atomic multipole interaction energy
using a double loop</p>
<p><strong>EMPOLE0B Subroutine</strong></p>
<p>“empole0b” calculates the atomic multipole interaction energy
using a neighbor list</p>
<p><strong>EMPOLE0C Subroutine</strong></p>
<p>“empole0c” calculates the atomic multipole interaction energy
using particle mesh Ewald summation and a double loop</p>
<p><strong>EMPOLE0D Subroutine</strong></p>
<p>“empole0d” calculates the atomic multipole interaction energy
using particle mesh Ewald summation and a neighbor list</p>
<p><strong>EMPOLE1 Subroutine</strong></p>
<p>“empole1” calculates the atomic multipole energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>EMPOLE1A Subroutine</strong></p>
<p>“empole1a” calculates the multipole energy and derivatives with
respect to Cartesian coordinates using a pairwise double loop</p>
<p><strong>EMPOLE1B Subroutine</strong></p>
<p>“empole1b” calculates the multipole energy and derivatives
with respect to Cartesian coordinates using a neighbor list</p>
<p><strong>EMPOLE1C Subroutine</strong></p>
<p>“empole1c” calculates the multipole energy and derivatives
with respect to Cartesian coordinates using particle mesh
Ewald summation and a double loop</p>
<p><strong>EMPOLE1D Subroutine</strong></p>
<p>“empole1d” calculates the multipole energy and derivatives
with respect to Cartesian coordinates using particle mesh Ewald
summation and a neighbor list</p>
<p><strong>EMPOLE2 Subroutine</strong></p>
<p>“empole2” calculates second derivatives of the multipole energy
for a single atom at a time</p>
<p><strong>EMPOLE2A Subroutine</strong></p>
<p>“empole2a” computes multipole first derivatives for a single
atom; used to get finite difference second derivatives</p>
<p><strong>EMPOLE3 Subroutine</strong></p>
<p>“empole3” calculates the electrostatic energy due to atomic
multipole interactions, and partitions the energy among atoms</p>
<p><strong>EMPOLE3A Subroutine</strong></p>
<p>“empole3a” calculates the atomic multipole interaction energy
using a double loop, and partitions the energy among atoms</p>
<p><strong>EMPOLE3B Subroutine</strong></p>
<p>“empole3b” calculates the atomic multipole interaction energy
using a neighbor list, and partitions the energy among the atoms</p>
<p><strong>EMPOLE3C Subroutine</strong></p>
<p>“empole3c” calculates the atomic multipole interaction energy
using a particle mesh Ewald summation and double loop, and
partitions the energy among the atoms</p>
<p><strong>EMPOLE3D Subroutine</strong></p>
<p>“empole3d” calculates the atomic multipole interaction energy
using particle mesh Ewald summation and a neighbor list, and
partitions the energy among the atoms</p>
<p><strong>EMREAL0C Subroutine</strong></p>
<p>“emreal0c” evaluates the real space portion of the Ewald sum
energy due to atomic multipoles using a double loop</p>
<p><strong>EMREAL0D Subroutine</strong></p>
<p>“emreal0d” evaluates the real space portion of the Ewald sum
energy due to atomic multipoles using a neighbor list</p>
<p><strong>EMREAL1C Subroutine</strong></p>
<p>“emreal1c” evaluates the real space portion of the Ewald
summation energy and gradient due to multipole interactions
via a double loop</p>
<p><strong>EMREAL1D Subroutine</strong></p>
<p>“emreal1d” evaluates the real space portion of the Ewald
summation energy and gradient due to multipole interactions
via a neighbor list</p>
<p><strong>EMREAL3C Subroutine</strong></p>
<p>“emreal3c” evaluates the real space portion of the Ewald sum
energy due to atomic multipole interactions and partitions
the energy among the atoms</p>
<p><strong>EMREAL3D Subroutine</strong></p>
<p>“emreal3d” evaluates the real space portion of the Ewald sum
energy due to atomic multipole interactions, and partitions
the energy among the atoms using a pairwise neighbor list</p>
<p><strong>EMRECIP Subroutine</strong></p>
<p>“emrecip” evaluates the reciprocal space portion of the particle
mesh Ewald energy due to atomic multipole interactions</p>
<p><strong>EMRECIP1 Subroutine</strong></p>
<p>“emrecip1” evaluates the reciprocal space portion of particle
mesh Ewald summation energy and gradient due to multipoles</p>
<p><strong>ENERGY Function</strong></p>
<p>“energy” calls the subroutines to calculate the potential
energy terms and sums up to form the total energy</p>
<p><strong>ENP Subroutine</strong></p>
<p>“enp” calculates the nonpolar implicit solvation energy
as a sum of cavity and dispersion terms</p>
<p><strong>ENP1 Subroutine</strong></p>
<p>“enp1” calculates the nonpolar implicit solvation energy
and derivatives as a sum of cavity and dispersion terms</p>
<p><strong>ENP3 Subroutine</strong></p>
<p>“enp3” calculates the nonpolar implicit solvation energy as
a sum of cavity and dispersion terms; also partitions the
energy among the atoms</p>
<p><strong>ENRGYZE Subroutine</strong></p>
<p>“enrgyze” is an auxiliary routine for the analyze program
that performs the energy analysis and prints the total and
intermolecular energies</p>
<p><strong>EOPBEND Subroutine</strong></p>
<p>“eopbend” computes the out-of-plane bend potential energy at
trigonal centers via a Wilson-Decius-Cross or Allinger angle</p>
<p><strong>EOPBEND1 Subroutine</strong></p>
<p>“eopbend1” computes the out-of-plane bend potential energy and
first derivatives at trigonal centers via a Wilson-Decius-Cross
or Allinger angle</p>
<p><strong>EOPBEND2 Subroutine</strong></p>
<p>“eopbend2” calculates second derivatives of the out-of-plane
bend energy via a Wilson-Decius-Cross or Allinger angle for
a single atom using finite difference methods</p>
<p><strong>EOPBEND2A Subroutine</strong></p>
<p>“eopbend2a” calculates out-of-plane bend first derivatives at
a trigonal center via a Wilson-Decius-Cross or Allinger angle;
used in computation of finite difference second derivatives</p>
<p><strong>EOPBEND3 Subroutine</strong></p>
<p>“eopbend3” computes the out-of-plane bend potential energy at
trigonal centers via a Wilson-Decius-Cross or Allinger angle;
also partitions the energy among the atoms</p>
<p><strong>EOPDIST Subroutine</strong></p>
<p>“eopdist” computes the out-of-plane distance potential
energy at trigonal centers via the central atom height</p>
<p><strong>EOPDIST1 Subroutine</strong></p>
<p>“eopdist1” computes the out-of-plane distance potential
energy and first derivatives at trigonal centers via
the central atom height</p>
<p><strong>EOPDIST2 Subroutine</strong></p>
<p>“eopdist2” calculates second derivatives of the out-of-plane
distance energy for a single atom via the central atom height</p>
<p><strong>EOPDIST3 Subroutine</strong></p>
<p>“eopdist3” computes the out-of-plane distance potential energy
at trigonal centers via the central atom height; also partitions
the energy among the atoms</p>
<p><strong>EPB Subroutine</strong></p>
<p>“epb” calculates the implicit solvation energy via the
Poisson-Boltzmann plus nonpolar implicit solvation</p>
<p><strong>EPB1 Subroutine</strong></p>
<p>“epb1” calculates the implicit solvation energy and derivatives
via the Poisson-Boltzmann plus nonpolar implicit solvation</p>
<p><strong>EPB1A Subroutine</strong></p>
<p>“epb1a” calculates the solvation energy and gradients for the
PB/NP solvation model</p>
<p><strong>EPB3 Subroutine</strong></p>
<p>“epb3” calculates the implicit solvation energy via the
Poisson-Boltzmann model; also partitions the energy among
the atoms</p>
<p><strong>EPITORS Subroutine</strong></p>
<p>“epitors” calculates the pi-system torsion potential energy</p>
<p><strong>EPITORS1 Subroutine</strong></p>
<p>“epitors1” calculates the pi-system torsion potential energy
and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EPITORS2 Subroutine</strong></p>
<p>“epitors2” calculates the second derivatives of the pi-system
torsion energy for a single atom using finite difference methods</p>
<p><strong>EPITORS2A Subroutine</strong></p>
<p>“epitors2a” calculates the pi-system torsion first derivatives;
used in computation of finite difference second derivatives</p>
<p><strong>EPITORS3 Subroutine</strong></p>
<p>“epitors3” calculates the pi-system torsion potential energy;
also partitions the energy terms among the atoms</p>
<p><strong>EPOLAR Subroutine</strong></p>
<p>“epolar” calculates the polarization energy due to induced
dipole interactions</p>
<p><strong>EPOLAR0A Subroutine</strong></p>
<p>“epolar0a” calculates the induced dipole polarization energy
using a double loop, and partitions the energy among atoms</p>
<p><strong>EPOLAR0B Subroutine</strong></p>
<p>“epolar0b” calculates the induced dipole polarization energy
using a neighbor list</p>
<p><strong>EPOLAR0C Subroutine</strong></p>
<p>“epolar0c” calculates the dipole polarization energy with respect
to Cartesian coordinates using particle mesh Ewald summation and
a double loop</p>
<p><strong>EPOLAR0D Subroutine</strong></p>
<p>“epolar0d” calculates the dipole polarization energy with respect
to Cartesian coordinates using particle mesh Ewald summation and
a neighbor list</p>
<p><strong>EPOLAR0E Subroutine</strong></p>
<p>“epolar0e” calculates the dipole polarizability interaction
from the induced dipoles times the electric field</p>
<p><strong>EPOLAR1 Subroutine</strong></p>
<p>“epolar1” calculates the induced dipole polarization energy
and first derivatives with respect to Cartesian coordinates</p>
<p><strong>EPOLAR1A Subroutine</strong></p>
<p>“epolar1a” calculates the dipole polarization energy and
derivatives with respect to Cartesian coordinates using a
pairwise double loop</p>
<p><strong>EPOLAR1B Subroutine</strong></p>
<p>“epolar1b” calculates the dipole polarization energy and
derivatives with respect to Cartesian coordinates using a
neighbor list</p>
<p><strong>EPOLAR1C Subroutine</strong></p>
<p>“epolar1c” calculates the dipole polarization energy and
derivatives with respect to Cartesian coordinates using
particle mesh Ewald summation and a double loop</p>
<p><strong>EPOLAR1D Subroutine</strong></p>
<p>“epolar1d” calculates the dipole polarization energy and
derivatives with respect to Cartesian coordinates using
particle mesh Ewald summation and a neighbor list</p>
<p><strong>EPOLAR1E Subroutine</strong></p>
<p>“epolar1e” calculates the dipole polarizability interaction
from the induced dipoles times the electric field</p>
<p><strong>EPOLAR2 Subroutine</strong></p>
<p>“epolar2” calculates second derivatives of the dipole polarization
energy for a single atom at a time</p>
<p><strong>EPOLAR2A Subroutine</strong></p>
<p>“epolar2a” computes polarization first derivatives for a single
atom with respect to Cartesian coordinates; used to get finite
difference second derivatives</p>
<p><strong>EPOLAR3 Subroutine</strong></p>
<p>“epolar3” calculates the induced dipole polarization energy,
and partitions the energy among atoms</p>
<p><strong>EPOLAR3A Subroutine</strong></p>
<p>“epolar3a” calculates the induced dipole polarization energy
using a double loop, and partitions the energy among atoms</p>
<p><strong>EPOLAR3B Subroutine</strong></p>
<p>“epolar3b” calculates the induced dipole polarization energy
using a neighbor list, and partitions the energy among atoms</p>
<p><strong>EPOLAR3C Subroutine</strong></p>
<p>“epolar3c” calculates the polarization energy and analysis with
respect to Cartesian coordinates using particle mesh Ewald and
a double loop</p>
<p><strong>EPOLAR3D Subroutine</strong></p>
<p>“epolar3d” calculates the polarization energy and analysis with
respect to Cartesian coordinates using particle mesh Ewald and
a neighbor list</p>
<p><strong>EPOLAR3E Subroutine</strong></p>
<p>“epolar3e” calculates the dipole polarizability interaction
from the induced dipoles times the electric field</p>
<p><strong>EPREAL0C Subroutine</strong></p>
<p>“epreal0c” calculates the induced dipole polarization energy
using particle mesh Ewald summation and a double loop</p>
<p><strong>EPREAL0D Subroutine</strong></p>
<p>“epreal0d” calculates the induced dipole polarization energy
using particle mesh Ewald summation and a neighbor list</p>
<p><strong>EPREAL1C Subroutine</strong></p>
<p>“epreal1c” evaluates the real space portion of the Ewald
summation energy and gradient due to dipole polarization
via a double loop</p>
<p><strong>EPREAL1D Subroutine</strong></p>
<p>“epreal1d” evaluates the real space portion of the Ewald
summation energy and gradient due to dipole polarization
via a neighbor list</p>
<p><strong>EPREAL3C Subroutine</strong></p>
<p>“epreal3c” calculates the induced dipole polarization energy and
analysis using particle mesh Ewald summation and a double loop</p>
<p><strong>EPREAL3D Subroutine</strong></p>
<p>“epreal3d” calculates the induced dipole polarization energy
and analysis using particle mesh Ewald and a neighbor list</p>
<p><strong>EPRECIP Subroutine</strong></p>
<p>“eprecip” evaluates the reciprocal space portion of particle
mesh Ewald summation energy due to dipole polarization</p>
<p><strong>EPRECIP1 Subroutine</strong></p>
<p>“eprecip1” evaluates the reciprocal space portion of the particle
mesh Ewald summation energy and gradient due to dipole polarization</p>
<p><strong>EQUCLC Subroutine</strong></p>
<p><strong>EREPEL Subroutine</strong></p>
<p>“erepel” calculates the Pauli exchange repulsion energy</p>
<p><strong>EREPEL0A Subroutine</strong></p>
<p>“erepel0a” calculates the Pauli repulsion interaction energy
using a double loop</p>
<p><strong>EREPEL0B Subroutine</strong></p>
<p>“erepel0b” calculates the Pauli repulsion interaction energy
using a pairwise neighbor list</p>
<p><strong>EREPEL1 Subroutine</strong></p>
<p>“erepel1” calculates the Pauli repulsion energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>EREPEL1A Subroutine</strong></p>
<p>“erepel1a” calculates the Pauli repulsion energy and first
derivatives with respect to Cartesian coordinates using a
pairwise double loop</p>
<p><strong>EREPEL1B Subroutine</strong></p>
<p>“erepel1b” calculates the Pauli repulsion energy and first
derivatives with respect to Cartesian coordinates using a
pariwise neighbor list</p>
<p><strong>EREPEL2 Subroutine</strong></p>
<p>“erepel2” calculates the second derivatives of the Pauli
repulsion energy</p>
<p><strong>EREPEL2A Subroutine</strong></p>
<p>“erepel2a” computes Pauli repulsion first derivatives for a
single atom via a double loop; used to get finite difference
second derivatives</p>
<p><strong>EREPEL3 Subroutine</strong></p>
<p>“erepel3” calculates the Pauli repulsion energy and partitions
the energy among the atoms</p>
<p><strong>EREPEL3A Subroutine</strong></p>
<p>“erepel3a” calculates the Pauli repulsion energy and also
partitions the energy among the atoms using a double loop</p>
<p><strong>EREPEL3B Subroutine</strong></p>
<p>“erepel3b” calculates the Pauli repulsion energy and also
partitions the energy among the atoms using a neighbor list</p>
<p><strong>ERF Function</strong></p>
<p>“erf” computes a numerical approximation to the value of
the error function via a Chebyshev approximation</p>
<p><strong>ERFC Function</strong></p>
<p>“erfc” computes a numerical approximation to the value of the
complementary error function via a Chebyshev approximation</p>
<p><strong>ERFCORE Subroutine</strong></p>
<p>“erfcore” evaluates erf(x) or erfc(x) for a real argument x;
when called with mode set to 0 it returns erf, a mode of 1
returns erfc; uses rational functions that approximate erf(x)
and erfc(x) to at least 18 significant decimal digits</p>
<p><strong>ERFIK Subroutine</strong></p>
<p>“erfik” compute the reaction field energy due to a single pair
of atomic multipoles</p>
<p><strong>ERFINV Function</strong></p>
<p>“erfinv” evaluates the inverse of the error function for
an argument in the range (-1,1) using a rational function
approximation followed by cycles of Newton-Raphson correction</p>
<p><strong>ERXNFLD Subroutine</strong></p>
<p>“erxnfld” calculates the macroscopic reaction field energy
arising from a set of atomic multipoles</p>
<p><strong>ERXNFLD1 Subroutine</strong></p>
<p>“erxnfld1” calculates the macroscopic reaction field energy
and derivatives with respect to Cartesian coordinates</p>
<p><strong>ERXNFLD2 Subroutine</strong></p>
<p>“erxnfld2” calculates second derivatives of the macroscopic
reaction field energy for a single atom at a time</p>
<p><strong>ERXNFLD3 Subroutine</strong></p>
<p>“erxnfld3” calculates the macroscopic reaction field energy,
and also partitions the energy among the atoms</p>
<p><strong>ESOLV Subroutine</strong></p>
<p>“esolv” calculates the implicit solvation energy for surface area,
generalized Born, generalized Kirkwood and Poisson-Boltzmann
solvation models</p>
<p><strong>ESOLV1 Subroutine</strong></p>
<p>“esolv1” calculates the implicit solvation energy and
first derivatives with respect to Cartesian coordinates
for surface area, generalized Born, generalized Kirkwood
and Poisson-Boltzmann solvation models</p>
<p><strong>ESOLV2 Subroutine</strong></p>
<p>“esolv2” calculates second derivatives of the implicit
solvation energy for surface area, generalized Born,
generalized Kirkwood and Poisson-Boltzmann solvation models</p>
<p><strong>ESOLV2A Subroutine</strong></p>
<p>“esolv2a” calculates second derivatives of the implicit solvation
potential energy by finite differences</p>
<p><strong>ESOLV2B Subroutine</strong></p>
<p>“esolv2b” finds implicit solvation gradients needed for
calculation of the Hessian matrix by finite differences</p>
<p><strong>ESOLV3 Subroutine</strong></p>
<p>“esolv3” calculates the implicit solvation energy for
surface area, generalized Born, generalized Kirkwood
and Poisson-Boltzmann solvation models; also partitions
the energy among the atoms</p>
<p><strong>ESTRBND Subroutine</strong></p>
<p>“estrbnd” calculates the stretch-bend potential energy</p>
<p><strong>ESTRBND1 Subroutine</strong></p>
<p>“estrbnd1” calculates the stretch-bend potential energy and
first derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRBND2 Subroutine</strong></p>
<p>“estrbnd2” calculates the stretch-bend potential energy
second derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRBND3 Subroutine</strong></p>
<p>“estrbnd3” calculates the stretch-bend potential energy;
also partitions the energy among the atoms</p>
<p><strong>ESTRTOR Subroutine</strong></p>
<p>“estrtor” calculates the stretch-torsion potential energy</p>
<p><strong>ESTRTOR1 Subroutine</strong></p>
<p>“estrtor1” calculates the stretch-torsion energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRTOR2 Subroutine</strong></p>
<p>“estrtor2” calculates the stretch-torsion potential energy
second derivatives with respect to Cartesian coordinates</p>
<p><strong>ESTRTOR3 Subroutine</strong></p>
<p>“estrtor3” calculates the stretch-torsion potential energy;
also partitions the energy terms among the atoms</p>
<p><strong>ETORS Subroutine</strong></p>
<p>“etors” calculates the torsional potential energy</p>
<p><strong>ETORS0A Subroutine</strong></p>
<p>“etors0a” calculates the torsional potential energy
using a standard sum of Fourier terms</p>
<p><strong>ETORS0B Subroutine</strong></p>
<p>“etors0b” calculates the torsional potential energy
for use with potential energy smoothing methods</p>
<p><strong>ETORS1 Subroutine</strong></p>
<p>“etors1” calculates the torsional potential energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>ETORS1A Subroutine</strong></p>
<p>“etors1a” calculates the torsional potential energy and first
derivatives with respect to Cartesian coordinates using a
standard sum of Fourier terms</p>
<p><strong>ETORS1B Subroutine</strong></p>
<p>“etors1b” calculates the torsional potential energy and first
derivatives with respect to Cartesian coordinates for use with
potential energy smoothing methods</p>
<p><strong>ETORS2 Subroutine</strong></p>
<p>“etors2” calculates the second derivatives of the torsional
energy for a single atom</p>
<p><strong>ETORS2A Subroutine</strong></p>
<p>“etors2a” calculates the second derivatives of the torsional
energy for a single atom using a standard sum of Fourier terms</p>
<p><strong>ETORS2B Subroutine</strong></p>
<p>“etors2b” calculates the second derivatives of the torsional
energy for a single atom for use with potential energy
smoothing methods</p>
<p><strong>ETORS3 Subroutine</strong></p>
<p>“etors3” calculates the torsional potential energy; also
partitions the energy among the atoms</p>
<p><strong>ETORS3A Subroutine</strong></p>
<p>“etors3a” calculates the torsional potential energy using
a standard sum of Fourier terms and partitions the energy
among the atoms</p>
<p><strong>ETORS3B Subroutine</strong></p>
<p>“etors3b” calculates the torsional potential energy for use
with potential energy smoothing methods and partitions the
energy among the atoms</p>
<p><strong>ETORTOR Subroutine</strong></p>
<p>“etortor” calculates the torsion-torsion potential energy</p>
<p><strong>ETORTOR1 Subroutine</strong></p>
<p>“etortor1” calculates the torsion-torsion energy and first
derivatives with respect to Cartesian coordinates</p>
<p><strong>ETORTOR2 Subroutine</strong></p>
<p>“etortor2” calculates the torsion-torsion potential energy
second derivatives with respect to Cartesian coordinates</p>
<p><strong>ETORTOR3 Subroutine</strong></p>
<p>“etortor3” calculates the torsion-torsion potential energy;
also partitions the energy terms among the atoms</p>
<p><strong>EUREY Subroutine</strong></p>
<p>“eurey” calculates the Urey-Bradley 1-3 interaction energy</p>
<p><strong>EUREY1 Subroutine</strong></p>
<p>“eurey1” calculates the Urey-Bradley interaction energy and
its first derivatives with respect to Cartesian coordinates</p>
<p><strong>EUREY2 Subroutine</strong></p>
<p>“eurey2” calculates second derivatives of the Urey-Bradley
interaction energy for a single atom at a time</p>
<p><strong>EUREY3 Subroutine</strong></p>
<p>“eurey3” calculates the Urey-Bradley energy; also
partitions the energy among the atoms</p>
<p><strong>EVCORR Subroutine</strong></p>
<p>“evcorr” computes the long range van der Waals correction
to the energy via numerical integration</p>
<p><strong>EVCORR1 Subroutine</strong></p>
<p>“evcorr1” computes the long range van der Waals correction
to the energy and virial via numerical integration</p>
<p><strong>EWALDCOF Subroutine</strong></p>
<p>“ewaldcof” finds an Ewald coefficient such that all terms
beyond the specified cutoff distance will have a value less
than a specified tolerance</p>
<p><strong>EWCA Subroutine</strong></p>
<p>“ewca” find the Weeks-Chandler-Andersen dispersion energy
of a solute using an HCT-like method</p>
<p><strong>EWCA1 Subroutine</strong></p>
<p>“ewca1” finds the Weeks-Chandler-Anderson dispersion energy
and derivatives of a solute</p>
<p><strong>EWCA3 Subroutine</strong></p>
<p>“ewca3” find the Weeks-Chandler-Andersen dispersion energy
of a solute; also partitions the energy among the atoms</p>
<p><strong>EWCA3X Subroutine</strong></p>
<p>“ewca3x” finds the Weeks-Chandler-Anderson dispersion energy
of a solute using a numerical “onion shell” method; also
partitions the energy among the atoms</p>
<p><strong>EWCAX Subroutine</strong></p>
<p>“ewcax” finds the Weeks-Chandler-Anderson dispersion energy
of a solute using a numerical “onion shell” method</p>
<p><strong>EXPLORE Subroutine</strong></p>
<p>“explore” uses simulated annealing on an initial crude
embedded distance geoemtry structure to refine versus the
bound, chirality, planarity and torsional error functions</p>
<p><strong>EXTENT Subroutine</strong></p>
<p>“extent” finds the largest interatomic distance in a system</p>
<p><strong>EXTRA Subroutine</strong></p>
<p>“extra” calculates any additional user defined potential
energy contribution</p>
<p><strong>EXTRA1 Subroutine</strong></p>
<p>“extra1” calculates any additional user defined potential
energy contribution and its first derivatives</p>
<p><strong>EXTRA2 Subroutine</strong></p>
<p>“extra2” calculates second derivatives of any additional
user defined potential energy contribution for a single
atom at a time</p>
<p><strong>EXTRA3 Subroutine</strong></p>
<p>“extra3” calculates any additional user defined potential
contribution and also partitions the energy among the atoms</p>
<p><strong>FATAL Subroutine</strong></p>
<p>“fatal” terminates execution due to a user request, a severe
error or some other nonstandard condition</p>
<p><strong>FFTBACK Subroutine</strong></p>
<p>“fftback” performs a 3-D FFT backward transform via a single
3-D transform or three separate 1-D transforms</p>
<p><strong>FFTCLOSE Subroutine</strong></p>
<p>“fftclose” does cleanup after performing a 3-D FFT by destroying
the FFTW plans for the forward and backward transforms</p>
<p><strong>FFTFRONT Subroutine</strong></p>
<p>“fftfront” performs a 3-D FFT forward transform via a single
3-D transform or three separate 1-D transforms</p>
<p><strong>FFTSETUP Subroutine</strong></p>
<p>“fftsetup” does initialization for a 3-D FFT to be computed
via either the FFTPACK or FFTW libraries</p>
<p><strong>FIELD Subroutine</strong></p>
<p>“field” sets the force field potential energy functions from
a parameter file and modifications specified in a keyfile</p>
<p><strong>FINAL Subroutine</strong></p>
<p>“final” performs any final program actions such as deallocation
of global memory, prints a status message, and then pauses if
necessary to avoid closing the execution window</p>
<p><strong>FINDATM Subroutine</strong></p>
<p>“findatm” locates a specific PDB atom name type within a
range of atoms from the PDB file, returns zero if the name
type was not found</p>
<p><strong>FITRSD Subroutine</strong></p>
<p>“fitrsd” computes residuals for electrostatic potential fitting
including total charge restraints, dipole and quadrupole moment
targets, and restraints to initial parameter values</p>
<p><strong>FITTORS Subroutine</strong></p>
<p>“fittors” refines torsion parameters based on a quantum
mechanical optimized energy surface</p>
<p><strong>FIXFRAME Subroutine</strong></p>
<p>“fixframe” is a service routine that alters the local frame
definition for specified atoms</p>
<p><strong>FIXPDB Subroutine</strong></p>
<p>“fixpdb” corrects problems with PDB files by converting residue
and atom names to the standard forms used by Tinker</p>
<p><strong>FIXPOLE Subroutine</strong></p>
<p>“fixpole” performs unit conversion of the multipole components,
rounds moments to desired precision, and enforces integer net
charge and traceless quadrupoles</p>
<p><strong>FLATTEN Subroutine</strong></p>
<p>“flatten” sets the type of smoothing method and the extent of
surface deformation for use with potential energy smoothing</p>
<p><strong>FPHI_MPOLE Subroutine</strong></p>
<p>“fphi_mpole” extracts the permanent multipole potential from
the particle mesh Ewald grid</p>
<p><strong>FPHI_TO_CPHI Subroutine</strong></p>
<p>“fphi_to_cphi” transforms the reciprocal space potential from
fractional to Cartesian coordinates</p>
<p><strong>FPHI_UIND Subroutine</strong></p>
<p>“fphi_uind” extracts the induced dipole potential from
the particle mesh Ewald grid</p>
<p><strong>FRACDIST Subroutine</strong></p>
<p>“fracdist” computes a normalized distribution of the pairwise
fractional distances between the smoothed upper and lower bounds</p>
<p><strong>FRAC_TO_CART Subroutine</strong></p>
<p>“frac_to_cart” computes a transformation matrix to convert
a multipole object in fraction coordinates to Cartesian</p>
<p><strong>FRAME13 Subroutine</strong></p>
<p>“frame13” finds local coordinate frame defining atoms in cases
where the use of 1-3 connected atoms is required</p>
<p><strong>FREEUNIT Function</strong></p>
<p>“freeunit” finds an unopened Fortran I/O unit and returns
its numerical value from 1 to 99; the units already assigned
to “input” and “iout” (usually 5 and 6) are skipped since
they have special meaning as the default I/O units</p>
<p><strong>GAMMLN Function</strong></p>
<p>“gammln” uses a series expansion due to Lanczos to compute
the natural logarithm of the Gamma function at “x” in [0,1]</p>
<p><strong>GAUSSJORDAN Subroutine</strong></p>
<p>“gaussjordan” solves a system of linear equations by using
the method of Gaussian elimination with partial pivoting</p>
<p><strong>GDA Program</strong></p>
<p>“gda” implements Gaussian Density Annealing (GDA) algorithm
for global optimization via simulated annealing</p>
<p><strong>GDA1 Subroutine</strong></p>
<p><strong>GDA2 Function</strong></p>
<p><strong>GDA3 Subroutine</strong></p>
<p><strong>GDASTAT Subroutine</strong></p>
<p>for a GDA integration step; also saves the coordinates</p>
<p><strong>GENDOT Subroutine</strong></p>
<p>“gendot” finds the coordinates of a specified number of surface
points for a sphere with the input radius and coordinate center</p>
<p><strong>GEODESIC Subroutine</strong></p>
<p>“geodesic” smooths the upper and lower distance bounds via
the triangle inequality using a sparse matrix version of a
shortest path algorithm</p>
<p><strong>GEOMETRY Function</strong></p>
<p>“geometry” finds the value of the interatomic distance, angle
or dihedral angle defined by two to four input atoms</p>
<p><strong>GETARC Subroutine</strong></p>
<p>“getarc” asks for a coordinate archive or trajectory file name,
then reads in the initial set of coordinates</p>
<p><strong>GETBASE Subroutine</strong></p>
<p>“getbase” finds the base heavy atoms for a single nucleotide
residue and copies the names and coordinates to the Protein
Data Bank file</p>
<p><strong>GETCHUNK Subroutine</strong></p>
<p>“getchunk” determines the number of grid point “chunks” used
along each axis of the PME grid for parallelization</p>
<p><strong>GETINT Subroutine</strong></p>
<p>“getint” asks for an internal coordinate file name, then reads
the internal coordinates and computes Cartesian coordinates</p>
<p><strong>GETKEY Subroutine</strong></p>
<p>“getkey” finds a valid keyfile and stores its contents as
line images for subsequent keyword parameter searching</p>
<p><strong>GETMOL Subroutine</strong></p>
<p>“getmol” asks for a MDL MOL molecule file name,
then reads the coordinates from the file</p>
<p><strong>GETMOL2 Subroutine</strong></p>
<p>“getmol2” asks for a Tripos MOL2 molecule file name,
then reads the coordinates from the file</p>
<p><strong>GETMONITOR Subroutine</strong></p>
<p><strong>GETNUCH Subroutine</strong></p>
<p>“getnuch” finds the nucleotide hydrogen atoms for a single
residue and copies the names and coordinates to the Protein
Data Bank file</p>
<p><strong>GETNUMB Subroutine</strong></p>
<p>“getnumb” searches an input string from left to right for an
integer and puts the numeric value in “number”; returns zero
with “next” unchanged if no integer value is found</p>
<p><strong>GETPDB Subroutine</strong></p>
<p>“getpdb” asks for a Protein Data Bank file name,
then reads in the coordinates file</p>
<p><strong>GETPRB Subroutine</strong></p>
<p>“getprb” tests for a possible probe position at the interface
between three neighboring atoms</p>
<p><strong>GETPRM Subroutine</strong></p>
<p>“getprm” finds the potential energy parameter file
and then opens and reads the parameters</p>
<p><strong>GETPROH Subroutine</strong></p>
<p>“getproh” finds the hydrogen atoms for a single amino acid
residue and copies the names and coordinates to the Protein
Data Bank file</p>
<p><strong>GETREF Subroutine</strong></p>
<p>“getref” copies structure information from the reference area
into the standard variables for the current system structure</p>
<p><strong>GETSEQ Subroutine</strong></p>
<p>“getseq” asks the user for the amino acid sequence
and torsional angle values needed to define a peptide</p>
<p><strong>GETSEQN Subroutine</strong></p>
<p>“getseqn” asks the user for the nucleotide sequence and
torsional angle values needed to define a nucleic acid</p>
<p><strong>GETSIDE Subroutine</strong></p>
<p>“getside” finds the side chain heavy atoms for a single amino
acid residue and copies the names and coordinates to the Protein
Data Bank file</p>
<p><strong>GETSTRING Subroutine</strong></p>
<p>“getstring” searches for a quoted text string within an input
character string; the region between the first and second
double quote is returned as the “text”; if the actual text is
too long, only the first part is returned</p>
<p><strong>GETTEXT Subroutine</strong></p>
<p>“gettext” searches an input string for the first string of
non-blank characters; the region from a non-blank character
to the first space or tab is returned as “text”; if the
actual text is too long, only the first part is returned</p>
<p><strong>GETTIME Subroutine</strong></p>
<p>“gettime” finds the elapsed wall clock and CPU times in seconds
since the last call to “settime”</p>
<p><strong>GETTOR Subroutine</strong></p>
<p>“gettor” tests for a possible torus position at the interface
between two atoms, and finds the torus radius, center and axis</p>
<p><strong>GETWORD Subroutine</strong></p>
<p>“getword” searches an input string for the first alphabetic
character (A-Z or a-z); the region from this first character
to the first blank space or separator is returned as a “word”;
if the actual word is too long, only the first part is returned</p>
<p><strong>GETXYZ Subroutine</strong></p>
<p>“getxyz” asks for a Cartesian coordinate file name,
then reads in the coordinates file</p>
<p><strong>GHMCSTEP Subroutine</strong></p>
<p>“ghmcstep” performs a single stochastic dynamics time step via
the generalized hybrid Monte Carlo (GHMC) algorithm to ensure
exact sampling from the Boltzmann density</p>
<p><strong>GHMCTERM Subroutine</strong></p>
<p>“ghmcterm” finds the friction and fluctuation terms needed
to update velocities during GHMC stochastic dynamics</p>
<p><strong>GRADFAST Subroutine</strong></p>
<p>“gradfast” calculates the potential energy and first derivatives
for the fast-evolving local valence potential energy terms</p>
<p><strong>GRADIENT Subroutine</strong></p>
<p>“gradient” calls subroutines to calculate the potential energy
and first derivatives with respect to Cartesian coordinates</p>
<p><strong>GRADRGD Subroutine</strong></p>
<p>“gradrgd” calls subroutines to calculate the potential energy
and first derivatives with respect to rigid body coordinates</p>
<p><strong>GRADROT Subroutine</strong></p>
<p>“gradrot” calls subroutines to calculate the potential
energy and its torsional first derivatives</p>
<p><strong>GRADSLOW Subroutine</strong></p>
<p>“gradslow” calculates the potential energy and first derivatives
for the slow-evolving nonbonded potential energy terms</p>
<p><strong>GRAFIC Subroutine</strong></p>
<p>“grafic” outputs the upper &amp; lower triangles and diagonal
of a square matrix in a schematic form for visual inspection</p>
<p><strong>GRID_DISP Subroutine</strong></p>
<p>“grid_disp” places the damped dispersion coefficients onto
the particle mesh Ewald grid</p>
<p><strong>GRID_MPOLE Subroutine</strong></p>
<p>“grid_mpole” places the fractional atomic multipoles onto
the particle mesh Ewald grid</p>
<p><strong>GRID_PCHG Subroutine</strong></p>
<p>“grid_pchg” places the fractional atomic partial charges onto
the particle mesh Ewald grid</p>
<p><strong>GRID_UIND Subroutine</strong></p>
<p>“grid_uind” places the fractional induced dipoles onto the
particle mesh Ewald grid</p>
<p><strong>GROUPS Subroutine</strong></p>
<p>“groups” tests a set of atoms to see if all are members of a
single atom group or a pair of atom groups; if so, then the
correct intra- or intergroup weight is assigned</p>
<p><strong>GRPLINE Subroutine</strong></p>
<p>“grpline” tests each atom group for linearity of the sites
contained in the group</p>
<p><strong>GSORT Subroutine</strong></p>
<p>“gsort” uses the Gram-Schmidt algorithm to build orthogonal
vectors for sliding block interative matrix diagonalization</p>
<p><strong>GYRATE Subroutine</strong></p>
<p>“gyrate” computes the radius of gyration of a molecular system
from its atomic coordinates; only active atoms are included</p>
<p><strong>HANGLE Subroutine</strong></p>
<p>“hangle” constructs hybrid angle bending parameters given
an initial state, final state and “lambda” value</p>
<p><strong>HATOM Subroutine</strong></p>
<p>“hatom” assigns a new atom type to each hybrid site</p>
<p><strong>HBOND Subroutine</strong></p>
<p>“hbond” constructs hybrid bond stretch parameters given
an initial state, final state and “lambda” value</p>
<p><strong>HCHARGE Subroutine</strong></p>
<p>“hcharge” constructs hybrid charge interaction parameters
given an initial state, final state and “lambda” value</p>
<p><strong>HDIPOLE Subroutine</strong></p>
<p>“hdipole” constructs hybrid dipole interaction parameters
given an initial state, final state and “lambda” value</p>
<p><strong>HESSBLK Subroutine</strong></p>
<p>“hessblk” calls subroutines to calculate the Hessian elements
for each atom in turn with respect to Cartesian coordinates</p>
<p><strong>HESSIAN Subroutine</strong></p>
<p>“hessian” calls subroutines to calculate the Hessian elements
for each atom in turn with respect to Cartesian coordinates</p>
<p><strong>HESSRGD Subroutine</strong></p>
<p>“hessrgd” computes the numerical Hessian elements with
respect to rigid body coordinates via 6*ngroup+1 gradient
evaluations</p>
<p><strong>HESSROT Subroutine</strong></p>
<p>“hessrot” computes numerical Hessian elements with respect
to torsional angles; either the diagonal or the full matrix
can be calculated; the full matrix needs nomega+1 gradient
evaluations while the diagonal needs just two evaluations</p>
<p><strong>HETATOM Subroutine</strong></p>
<p>“hetatom” translates water molecules and ions in Protein Data
Bank format to a Cartesian coordinate file and sequence file</p>
<p><strong>HIMPTOR Subroutine</strong></p>
<p>“himptor” constructs hybrid improper torsional parameters
given an initial state, final state and “lambda” value</p>
<p><strong>HOOVER Subroutine</strong></p>
<p>“hoover” applies a combined thermostat and barostat via a
Nose-Hoover chain algorithm</p>
<p><strong>HSTRBND Subroutine</strong></p>
<p>“hstrbnd” constructs hybrid stretch-bend parameters given
an initial state, final state and “lambda” value</p>
<p><strong>HSTRTOR Subroutine</strong></p>
<p>“hstrtor” constructs hybrid stretch-torsion parameters
given an initial state, final state and “lambda” value</p>
<p><strong>HTORS Subroutine</strong></p>
<p>“htors” constructs hybrid torsional parameters for a given
initial state, final state and “lambda” value</p>
<p><strong>HVDW Subroutine</strong></p>
<p>“hvdw” constructs hybrid van der Waals  parameters given
an initial state, final state and “lambda” value</p>
<p><strong>HYBRID Subroutine</strong></p>
<p>“hybrid” constructs the hybrid hamiltonian for a specified
initial state, final state and mutation parameter “lambda”</p>
<p><strong>IJKPTS Subroutine</strong></p>
<p>“ijkpts” stores a set of indices used during calculation
of macroscopic reaction field energetics</p>
<p><strong>IMAGE Subroutine</strong></p>
<p>“image” takes the components of pairwise distance between
two points in a periodic box and converts to the components
of the minimum image distance</p>
<p><strong>IMAGEN Subroutine</strong></p>
<p>“imagen” takes the components of pairwise distance between
two points and converts to the components of the minimum
image distance</p>
<p><strong>IMAGER Subroutine</strong></p>
<p>“imager” takes the components of pairwise distance between
two points in the same or neighboring periodic boxes and
converts to the components of the minimum image distance</p>
<p><strong>IMPOSE Subroutine</strong></p>
<p>“impose” performs the least squares best superposition
of two atomic coordinate sets via a quaternion method;
upon return, the first coordinate set is unchanged while
the second set is translated and rotated to give best fit;
the final root mean square fit is returned in “rmsvalue”</p>
<p><strong>INDTCGA Subroutine</strong></p>
<p>“indtcga” computes the induced dipoles and intermediates used
in polarization force calculation for the TCG method with dp
cross terms = true, initial guess mu0 = 0 and using a diagonal
preconditioner</p>
<p><strong>INDTCGB Subroutine</strong></p>
<p>“indtcgb” computes the induced dipoles and intermediates used
in polarization force calculation for the TCG method with dp
cross terms = true, initial guess mu0 = direct and using diagonal
preconditioner</p>
<p><strong>INDUCE Subroutine</strong></p>
<p>“induce” computes the induced dipole moments at polarizable
sites due to direct or mutual polarization</p>
<p><strong>INDUCE0A Subroutine</strong></p>
<p>“induce0a” computes the induced dipole moments at polarizable
sites using a preconditioned conjugate gradient solver</p>
<p><strong>INDUCE0B Subroutine</strong></p>
<p>“induce0b” computes and stores the induced dipoles via
the truncated conjugate gradient (TCG) method</p>
<p><strong>INDUCE0C Subroutine</strong></p>
<p>“induce0c” computes the induced dipole moments at polarizable
sites for generalized Kirkwood SCRF and vacuum environments</p>
<p><strong>INDUCE0D Subroutine</strong></p>
<p>“induce0d” computes the induced dipole moments at polarizable
sites for Poisson-Boltzmann SCRF and vacuum environments</p>
<p><strong>INEDGE Subroutine</strong></p>
<p>“inedge” inserts a concave edge into the
linked list for its temporary torus</p>
<p><strong>INERTIA Subroutine</strong></p>
<p>“inertia” computes the principal moments of inertia for the
system, and optionally translates the center of mass to the
origin and rotates the principal axes onto the global axes</p>
<p><strong>INITATOM Subroutine</strong></p>
<p>“initatom” sets the atomic symbol, standard atomic weight,
van der Waals radius and covalent radius for each element in
the periodic table</p>
<p><strong>INITERR Function</strong></p>
<p>“initerr” is the initial error function and derivatives for
a distance geometry embedding; it includes components from
the local geometry and torsional restraint errors</p>
<p><strong>INITIAL Subroutine</strong></p>
<p>“initial” sets up original values for some parameters and
variables that might not otherwise get initialized</p>
<p><strong>INITMMFF Subroutine</strong></p>
<p>“initmmff” initializes some parameter values for the Merck
Molecular force field</p>
<p><strong>INITPRM Subroutine</strong></p>
<p>“initprm” completely initializes a force field by setting all
parameters to zero and using defaults for control values</p>
<p><strong>INITRES Subroutine</strong></p>
<p>“initres” sets biopolymer residue names and biotype codes used
in PDB file conversion and automated generation of structures</p>
<p><strong>INITROT Subroutine</strong></p>
<p>“initrot” sets the torsional angles which are to be rotated
in subsequent computation, by default automatically selects
all rotatable single bonds; optionally makes atoms inactive
when they are not moved by any torsional rotation</p>
<p><strong>INSERT Subroutine</strong></p>
<p>“insert” adds the specified atom to the Cartesian
coordinates list and shifts the remaining atoms</p>
<p><strong>INTEDIT Program</strong></p>
<p>“intedit” allows the user to extract information from
or alter the values within an internal coordinates file</p>
<p><strong>INTERPOL Subroutine</strong></p>
<p>“interpol” computes intergroup induced dipole moments for use
during removal of intergroup polarization</p>
<p><strong>INTXYZ Program</strong></p>
<p>“intxyz” takes as input an internal coordinates file,
converts to and then writes out Cartesian coordinates</p>
<p><strong>INVBETA Function</strong></p>
<p>“invbeta” computes the inverse Beta distribution function
via a combination of Newton iteration and bisection search</p>
<p><strong>INVERT Subroutine</strong></p>
<p>“invert” inverts a matrix using the Gauss-Jordan method</p>
<p><strong>IPEDGE Subroutine</strong></p>
<p>“ipedge” inserts convex edge into linked list for atom</p>
<p><strong>JACOBI Subroutine</strong></p>
<p>“jacobi” performs a matrix diagonalization of a real
symmetric matrix by the method of Jacobi rotations</p>
<p><strong>JUSTIFY Subroutine</strong></p>
<p>“justify” converts a text string to right justified format
with leading blank spaces</p>
<p><strong>KANGANG Subroutine</strong></p>
<p>“kangang” assigns the parameters for angle-angle cross term
interactions and processes new or changed parameter values</p>
<p><strong>KANGLE Subroutine</strong></p>
<p>“kangle” assigns the force constants and ideal angles for
the bond angles; also processes new or changed parameters</p>
<p><strong>KANGLEM Subroutine</strong></p>
<p>“kanglem” assigns the force constants and ideal angles for
bond angles according to the Merck Molecular Force Field (MMFF)</p>
<p><strong>KANGTOR Subroutine</strong></p>
<p>“kangtor” assigns parameters for angle-torsion interactions
and processes new or changed parameter values</p>
<p><strong>KATOM Subroutine</strong></p>
<p>“katom” assigns an atom type definitions to each atom in
the structure and processes any new or changed values</p>
<p><strong>KBOND Subroutine</strong></p>
<p>“kbond” assigns a force constant and ideal bond length
to each bond in the structure and processes any new or
changed parameter values</p>
<p><strong>KBONDM Subroutine</strong></p>
<p>“kbondm” assigns a force constant and ideal bond length to
each bond according to the Merck Molecular Force Field (MMFF)</p>
<p><strong>KCHARGE Subroutine</strong></p>
<p>“kcharge” assigns partial charges to the atoms within
the structure and processes any new or changed values</p>
<p><strong>KCHARGEM Subroutine</strong></p>
<p>“kchargem” assigns partial charges to the atoms according to
the Merck Molecular Force Field (MMFF)</p>
<p><strong>KCHGTRN Subroutine</strong></p>
<p>“kchgtrn” assigns charge magnitude and damping parameters for
charge transfer interactions and processes any new or changed
values for these parameters</p>
<p><strong>KCHIRAL Subroutine</strong></p>
<p>“kchiral” determines the target value for each chirality
and planarity restraint as the signed volume of the
parallelpiped spanned by vectors from a common atom to
each of three other atoms</p>
<p><strong>KDIPOLE Subroutine</strong></p>
<p>“kdipole” assigns bond dipoles to the bonds within
the structure and processes any new or changed values</p>
<p><strong>KDISP Subroutine</strong></p>
<p>“kdisp” assigns C6 coefficients and damping parameters for
dispersion interactions and processes any new or changed
values for these parameters</p>
<p><strong>KENEG Subroutine</strong></p>
<p>“keneg” applies primary and secondary electronegativity bond
length corrections to applicable bond parameters</p>
<p><strong>KEWALD Subroutine</strong></p>
<p>“kewald” assigns particle mesh Ewald parameters and options
for a periodic system</p>
<p><strong>KEXTRA Subroutine</strong></p>
<p>“kextra” assigns parameters to any additional user defined
potential energy contribution</p>
<p><strong>KGB Subroutine</strong></p>
<p>“kgb” initializes parameters needed for the generalized
Born implicit solvation models</p>
<p><strong>KGEOM Subroutine</strong></p>
<p>“kgeom” asisgns parameters for geometric restraint terms
to be included in the potential energy calculation</p>
<p><strong>KGK Subroutine</strong></p>
<p>“kgk” initializes parameters needed for the generalized
Kirkwood implicit solvation model</p>
<p><strong>KHPMF Subroutine</strong></p>
<p>“khpmf” initializes parameters needed for the hydrophobic
potential of mean force nonpolar implicit solvation model</p>
<p><strong>KIMPROP Subroutine</strong></p>
<p>“kimprop” assigns potential parameters to each improper
dihedral in the structure and processes any changed values</p>
<p><strong>KIMPTOR Subroutine</strong></p>
<p>“kimptor” assigns torsional parameters to each improper
torsion in the structure and processes any changed values</p>
<p><strong>KINAUX Subroutine</strong></p>
<p>“kinaux” computes the total kinetic energy and temperature
for auxiliary dipole variables used in iEL polarization</p>
<p><strong>KINETIC Subroutine</strong></p>
<p>“kinetic” computes the total kinetic energy and kinetic energy
contributions to the pressure tensor by summing over velocities</p>
<p><strong>KMETAL Subroutine</strong></p>
<p>“kmetal” assigns ligand field parameters to transition metal
atoms and processes any new or changed parameter values</p>
<p><strong>KMPOLE Subroutine</strong></p>
<p>“kmpole” assigns atomic multipole moments to the atoms of
the structure and processes any new or changed values</p>
<p><strong>KNP Subroutine</strong></p>
<p>“knp” initializes parameters needed for the cavity-plus-
dispersion nonpolar implicit solvation model</p>
<p><strong>KONVEC Subroutine</strong></p>
<p>“konvec” finds a Hessian-vector product via finite-difference
evaluation of the gradient based on atomic displacements</p>
<p><strong>KOPBEND Subroutine</strong></p>
<p>“kopbend” assigns the force constants for out-of-plane bends
at trigonal centers via Wilson-Decius-Cross or Allinger angles;
also processes any new or changed parameter values</p>
<p><strong>KOPBENDM Subroutine</strong></p>
<p>“kopbendm” assigns the force constants for out-of-plane bends
according to the Merck Molecular Force Field (MMFF)</p>
<p><strong>KOPDIST Subroutine</strong></p>
<p>“kopdist” assigns the force constants for out-of-plane
distance at trigonal centers via the central atom height;
also processes any new or changed parameter values</p>
<p><strong>KORBIT Subroutine</strong></p>
<p>“korbit” assigns pi-orbital parameters to conjugated systems
and processes any new or changed parameters</p>
<p><strong>KPB Subroutine</strong></p>
<p>“kpb” assigns parameters needed for the Poisson-Boltzmann
implicit solvation model implemented via APBS</p>
<p><strong>KPITORS Subroutine</strong></p>
<p>“kpitors” assigns pi-system torsion parameters to torsions
needing them, and processes any new or changed values</p>
<p><strong>KPOLAR Subroutine</strong></p>
<p>“kpolar” assigns atomic dipole polarizabilities to the atoms
within the structure and processes any new or changed values</p>
<p><strong>KREPEL Subroutine</strong></p>
<p>“krepel” assigns the size values, exponential parameter and
number of valence electrons for Pauli repulsion interactions
and processes any new or changed values for these parameters</p>
<p><strong>KSA Subroutine</strong></p>
<p>“ksa” initializes parameters needed for surface area-based
implicit solvation models including ASP and SASA</p>
<p><strong>KSOLV Subroutine</strong></p>
<p>“ksolv” assigns implicit solvation energy parameters for
the surface area, generalized Born, generalized Kirkwood,
Poisson-Boltzmann, cavity-dispersion and HPMF models</p>
<p><strong>KSTRBND Subroutine</strong></p>
<p>“kstrbnd” assigns parameters for stretch-bend interactions
and processes new or changed parameter values</p>
<p><strong>KSTRBNDM Subroutine</strong></p>
<p>“kstrbndm” assigns parameters for stretch-bend interactions
according to the Merck Molecular Force Field (MMFF)</p>
<p><strong>KSTRTOR Subroutine</strong></p>
<p>“kstrtor” assigns stretch-torsion parameters to torsions
needing them, and processes any new or changed values</p>
<p><strong>KTORS Subroutine</strong></p>
<p>“ktors” assigns torsional parameters to each torsion in
the structure and processes any new or changed values</p>
<p><strong>KTORSM Subroutine</strong></p>
<p>“ktorsm” assigns torsional parameters to each torsion according
to the Merck Molecular Force Field (MMFF)</p>
<p><strong>KTORTOR Subroutine</strong></p>
<p>“ktortor” assigns torsion-torsion parameters to adjacent
torsion pairs and processes any new or changed values</p>
<p><strong>KUREY Subroutine</strong></p>
<p>“kurey” assigns the force constants and ideal distances
for the Urey-Bradley 1-3 interactions; also processes any
new or changed parameter values</p>
<p><strong>KVDW Subroutine</strong></p>
<p>“kvdw” assigns the parameters to be used in computing the
van der Waals interactions and processes any new or changed
values for these parameters</p>
<p><strong>LATTICE Subroutine</strong></p>
<p>“lattice” stores the periodic box dimensions and sets angle
values to be used in computing fractional coordinates</p>
<p><strong>LBFGS Subroutine</strong></p>
<p>“lbfgs” is a limited memory BFGS quasi-newton nonlinear
optimization routine</p>
<p><strong>LIGASE Subroutine</strong></p>
<p>“ligase” translates a nucleic acid structure in Protein Data
Bank format to a Cartesian coordinate file and sequence file</p>
<p><strong>LIGHTS Subroutine</strong></p>
<p>“lights” computes the set of nearest neighbor interactions
using the method of lights algorithm</p>
<p><strong>LINBODY Subroutine</strong></p>
<p>“linbody” finds the angular velocity of a linear rigid body
given the inertia tensor and angular momentum</p>
<p><strong>LMSTEP Subroutine</strong></p>
<p>“lmstep” computes a Levenberg-Marquardt step during a nonlinear
least squares calculation using ideas from the MINPACK LMPAR
routine and the internal doubling strategy of Dennis and Schnabel</p>
<p><strong>LOCALMIN Subroutine</strong></p>
<p>“localmin” is used during normal mode local search to
perform a Cartesian coordinate energy minimization</p>
<p><strong>LOCALRGD Subroutine</strong></p>
<p>“localrgd” is used during the PSS local search procedure
to perform a rigid body energy minimization</p>
<p><strong>LOCALROT Subroutine</strong></p>
<p>“localrot” is used during the PSS local search procedure
to perform a torsional space energy minimization</p>
<p><strong>LOCALXYZ Subroutine</strong></p>
<p>“localxyz” is used during the potential smoothing and search
procedure to perform a local optimization at the current
smoothing level</p>
<p><strong>LOCERR Function</strong></p>
<p>“locerr” is the local geometry error function and derivatives
including the 1-2, 1-3 and 1-4 distance bound restraints</p>
<p><strong>LOWCASE Subroutine</strong></p>
<p>“lowcase” converts a text string to all lower case letters</p>
<p><strong>MAJORIZE Subroutine</strong></p>
<p>“majorize” refines the projected coordinates by attempting to
minimize the least square residual between the trial distance
matrix and the distances computed from the coordinates</p>
<p><strong>MAKEBAR Subroutine</strong></p>
<p><strong>MAKEBOX Subroutine</strong></p>
<p>“makebox” builds a periodic box of a desired size by randomly
copying a specified number of monomers into a target box size,
followed by optional excluded volume refinement</p>
<p><strong>MAKEINT Subroutine</strong></p>
<p>“makeint” converts Cartesian to internal coordinates where
selection of internal coordinates is controlled by “mode”</p>
<p><strong>MAKEPDB Subroutine</strong></p>
<p>“makepdb” cconstructs a Protein Data Bank file from a set
of Cartesian coordinates with special handling for systems
consisting of biopolymer chains, ligands and water molecules</p>
<p><strong>MAKEREF Subroutine</strong></p>
<p>“makeref” copies the information contained in the “xyz” file
of the current structure into corresponding reference areas</p>
<p><strong>MAKEXYZ Subroutine</strong></p>
<p>“makexyz” generates a complete set of Cartesian coordinates
for a full structure from the internal coordinate values</p>
<p><strong>MAPCHECK Subroutine</strong></p>
<p>“mapcheck” checks the current minimum energy structure
for possible addition to the master list of local minima</p>
<p><strong>MATCH1 Subroutine</strong></p>
<p>“match1” finds and stores the first multipole component found
on a line of output from Stone’s GDMA program</p>
<p><strong>MATCH2 Subroutine</strong></p>
<p>“match2” finds and stores the second multipole component found
on a line of output from Stone’s GDMA program</p>
<p><strong>MATCH3 Subroutine</strong></p>
<p>“match3” finds and stores the third multipole component found
on a line of output from Stone’s GDMA program</p>
<p><strong>MAXWELL Function</strong></p>
<p>“maxwell” returns a speed in Angstroms/picosecond randomly
selected from a 3-D Maxwell-Boltzmann distribution for the
specified particle mass and system temperature</p>
<p><strong>MBUILD Subroutine</strong></p>
<p>“mbuild” performs a complete rebuild of the atomic multipole
electrostatic neighbor list for all sites</p>
<p><strong>MCM1 Function</strong></p>
<p>“mcm1” is a service routine that computes the energy and
gradient for truncated Newton optimization in Cartesian
coordinate space</p>
<p><strong>MCM2 Subroutine</strong></p>
<p>“mcm2” is a service routine that computes the sparse matrix
Hessian elements for truncated Newton optimization in Cartesian
coordinate space</p>
<p><strong>MCMSTEP Function</strong></p>
<p>“mcmstep” implements the minimization phase of an MCM step
via Cartesian minimization following a Monte Carlo step</p>
<p><strong>MDINIT Subroutine</strong></p>
<p>“mdinit” initializes the velocities and accelerations
for a molecular dynamics trajectory, including restarts</p>
<p><strong>MDREST Subroutine</strong></p>
<p>“mdrest” finds and removes any translational or rotational
kinetic energy of the overall system center of mass</p>
<p><strong>MDSAVE Subroutine</strong></p>
<p>“mdsave” writes molecular dynamics trajectory snapshots and
auxiliary files with velocity, force or induced dipole data;
also checks for user requested termination of a simulation</p>
<p><strong>MDSTAT Subroutine</strong></p>
<p>“mdstat” is called at each molecular dynamics time step to
form statistics on various average values and fluctuations,
and to periodically save the state of the trajectory</p>
<p><strong>MEASFN Subroutine</strong></p>
<p><strong>MEASFQ Subroutine</strong></p>
<p><strong>MEASFS Subroutine</strong></p>
<p><strong>MEASPM Subroutine</strong></p>
<p>“measpm” computes the volume of a single prism section of
the full interior polyhedron</p>
<p><strong>MECHANIC Subroutine</strong></p>
<p>“mechanic” sets up needed parameters for the potential energy
calculation and reads in many of the user selectable options</p>
<p><strong>MERGE Subroutine</strong></p>
<p>“merge” combines the reference and current structures into
a single new “current” structure containing the reference
atoms followed by the atoms of the current structure</p>
<p><strong>METRIC Subroutine</strong></p>
<p>“metric” takes as input the trial distance matrix and computes
the metric matrix of all possible dot products between the atomic
vectors and the center of mass using the law of cosines and the
following formula for the distances to the center of mass:</p>
<p><strong>MIDERR Function</strong></p>
<p>“miderr” is the secondary error function and derivatives
for a distance geometry embedding; it includes components
from the distance bounds, local geometry, chirality and
torsional restraint errors</p>
<p><strong>MINIMIZ1 Function</strong></p>
<p>“minimiz1” is a service routine that computes the energy and
gradient for a low storage BFGS optimization in Cartesian
coordinate space</p>
<p><strong>MINIMIZE Program</strong></p>
<p>“minimize” performs energy minimization in Cartesian coordinate
space using a low storage BFGS nonlinear optimization</p>
<p><strong>MINIROT Program</strong></p>
<p>“minirot” performs an energy minimization in torsional
angle space using a low storage BFGS nonlinear optimization</p>
<p><strong>MINIROT1 Function</strong></p>
<p>“minirot1” is a service routine that computes the energy
and gradient for a low storage BFGS nonlinear optimization
in torsional angle space</p>
<p><strong>MINPATH Subroutine</strong></p>
<p>“minpath” is a routine for finding the triangle smoothed upper
and lower bounds of each atom to a specified root atom using a
sparse variant of the Bellman-Ford shortest path algorithm</p>
<p><strong>MINRIGID Program</strong></p>
<p>“minrigid” performs an energy minimization of rigid body atom
groups using a low storage BFGS nonlinear optimization</p>
<p><strong>MINRIGID1 Function</strong></p>
<p>“minrigid1” is a service routine that computes the energy
and gradient for a low storage BFGS nonlinear optimization
of rigid bodies</p>
<p><strong>MLIGHT Subroutine</strong></p>
<p>“mlight” performs a complete rebuild of the atomic multipole
pair neighbor list for all sites using the method of lights</p>
<p><strong>MLIST Subroutine</strong></p>
<p>“mlist” performs an update or a complete rebuild of the
nonbonded neighbor lists for atomic multipoles</p>
<p><strong>MMID Subroutine</strong></p>
<p>“mmid” implements a modified midpoint method to advance the
integration of a set of first order differential equations</p>
<p><strong>MODECART Subroutine</strong></p>
<p><strong>MODERGD Subroutine</strong></p>
<p><strong>MODEROT Subroutine</strong></p>
<p><strong>MODESRCH Subroutine</strong></p>
<p><strong>MODETORS Subroutine</strong></p>
<p><strong>MODULI Subroutine</strong></p>
<p>“moduli” sets the moduli of the inverse discrete Fourier
transform of the B-splines</p>
<p><strong>MOL2XYZ Program</strong></p>
<p>“mol2xyz” takes as input a Tripos MOL2 coordinates file,
converts to and then writes out Cartesian coordinates</p>
<p><strong>MOLECULE Subroutine</strong></p>
<p>“molecule” counts the molecules, assigns each atom to
its molecule and computes the mass of each molecule</p>
<p><strong>MOLMERGE Subroutine</strong></p>
<p>“molmerge” connects fragments and removes duplicate atoms
during generation of a unit cell from an asymmetric unit</p>
<p><strong>MOLSETUP Subroutine</strong></p>
<p>“molsetup” generates trial parameters needed to perform
polarizable multipole calculations on a structure read
from distributed multipole analysis output</p>
<p><strong>MOLUIND Subroutine</strong></p>
<p>“moluind” computes the molecular induced dipole components
in the presence of an external electric field</p>
<p><strong>MOLXYZ Program</strong></p>
<p>“molxyz” takes as input a MDL MOL coordinates file,
converts to and then writes out Cartesian coordinates</p>
<p><strong>MOMENTS Subroutine</strong></p>
<p>“moments” computes the total electric charge, dipole and
quadrupole moments for the active atoms as a sum over the
partial charges, bond dipoles and atomic multipole moments</p>
<p><strong>MOMFULL Subroutine</strong></p>
<p>“momfull” computes the electric moments for the full system
as a sum over the partial charges, bond dipoles and atomic
multipole moments</p>
<p><strong>MOMYZE Subroutine</strong></p>
<p>“momyze” finds and prints the total charge, dipole moment
components, radius of gyration and moments of inertia</p>
<p><strong>MONTE Program</strong></p>
<p>“monte” performs a Monte Carlo-Minimization conformational
search using Cartesian single atom or torsional move sets</p>
<p><strong>MUTATE Subroutine</strong></p>
<p>“mutate” constructs the hybrid hamiltonian for a specified
initial state, final state and mutation parameter “lambda”</p>
<p><strong>NBLIST Subroutine</strong></p>
<p>“nblist” builds and maintains nonbonded pair neighbor lists
for vdw, dispersion, electrostatic and polarization terms</p>
<p><strong>NEARBY Subroutine</strong></p>
<p>“nearby” finds all of the through-space neighbors of each
atom for use in surface area and volume calculations</p>
<p><strong>NEEDUPDATE Subroutine</strong></p>
<p><strong>NEWATM Subroutine</strong></p>
<p>“newatm” creates and defines an atom needed for the
Cartesian coordinates file, but which may not present
in the original Protein Data Bank file</p>
<p><strong>NEWTON Program</strong></p>
<p>“newton” performs an energy minimization in Cartesian
coordinate space using a truncated Newton method</p>
<p><strong>NEWTON1 Function</strong></p>
<p>“newton1” is a service routine that computes the energy
and gradient for truncated Newton optimization in Cartesian
coordinate space</p>
<p><strong>NEWTON2 Subroutine</strong></p>
<p>“newton2” is a service routine that computes the sparse
matrix Hessian elements for truncated Newton optimization
in Cartesian coordinate space</p>
<p><strong>NEWTROT Program</strong></p>
<p>“newtrot” performs an energy minimization in torsional angle
space using a truncated Newton conjugate gradient method</p>
<p><strong>NEWTROT1 Function</strong></p>
<p>“newtrot1” is a service routine that computes the energy
and gradient for truncated Newton conjugate gradient
optimization in torsional angle space</p>
<p><strong>NEWTROT2 Subroutine</strong></p>
<p>“newtrot2” is a service routine that computes the sparse
matrix Hessian elements for truncated Newton optimization
in torsional angle space</p>
<p><strong>NEXTARG Subroutine</strong></p>
<p>“nextarg” finds the next unused command line argument
and returns it in the input character string</p>
<p><strong>NEXTTEXT Function</strong></p>
<p>“nexttext” finds and returns the location of the first
non-blank character within an input text string; zero
is returned if no such character is found</p>
<p><strong>NORMAL Function</strong></p>
<p>“normal” generates a random number from a normal Gaussian
distribution with a mean of zero and a variance of one</p>
<p><strong>NOSE Subroutine</strong></p>
<p>“nose” performs a single molecular dynamics time step via
a Nose-Hoover extended system isothermal-isobaric algorithm</p>
<p><strong>NSPLINE Subroutine</strong></p>
<p>“nspline” computes coefficients for an nonperiodic cubic spline
with natural boundary conditions where the first and last second
derivatives are already known</p>
<p><strong>NUCBASE Subroutine</strong></p>
<p>“nucbase” builds the side chain for a single nucleotide base
in terms of internal coordinates</p>
<p><strong>NUCCHAIN Subroutine</strong></p>
<p>“nucchain” builds up the internal coordinates for a nucleic
acid sequence from the sugar type, backbone and glycosidic
torsional values</p>
<p><strong>NUCLEIC Program</strong></p>
<p>“nucleic” builds the internal and Cartesian coordinates
of a polynucleotide from nucleic acid sequence and torsional
angle values for the nucleic acid backbone and side chains</p>
<p><strong>NUMBER Function</strong></p>
<p>“number” converts a text numeral into an integer value;
the input string must contain only numeric characters</p>
<p><strong>NUMERAL Subroutine</strong></p>
<p>“numeral” converts an input integer number into the
corresponding right- or left-justified text numeral</p>
<p><strong>NUMGRAD Subroutine</strong></p>
<p>“numgrad” computes the gradient of the objective function
“fvalue” with respect to Cartesian coordinates of the atoms
via a one-sided or two-sided numerical differentiation</p>
<p><strong>OCVM Subroutine</strong></p>
<p>“ocvm” is an optimally conditioned variable metric nonlinear
optimization routine without line searches</p>
<p><strong>OLDATM Subroutine</strong></p>
<p>“oldatm” get the Cartesian coordinates for an atom from
the Protein Data Bank file, then assigns the atom type
and atomic connectivities</p>
<p><strong>OPBGUESS Function</strong></p>
<p>“opbguess” sets approximate out-of-plane bend force constants
based on atom type and connected atoms</p>
<p><strong>OPENEND Subroutine</strong></p>
<p>“openend” opens a file on a Fortran unit such that the position
is set to the bottom for appending to the end of the file</p>
<p><strong>OPREP Subroutine</strong></p>
<p>“oprep” sets up the frictional and random terms needed to
update positions and velocities for the BAOAB integrator</p>
<p><strong>OPTFIT Function</strong></p>
<p><strong>OPTIMIZ1 Function</strong></p>
<p>“optimiz1” is a service routine that computes the energy and
gradient for optimally conditioned variable metric optimization
in Cartesian coordinate space</p>
<p><strong>OPTIMIZE Program</strong></p>
<p>“optimize” performs energy minimization in Cartesian coordinate
space using an optimally conditioned variable metric method</p>
<p><strong>OPTINIT Subroutine</strong></p>
<p>“optinit” initializes values and keywords used by multiple
structure optimization methods</p>
<p><strong>OPTIROT Program</strong></p>
<p>“optirot” performs an energy minimization in torsional angle
space using an optimally conditioned variable metric method</p>
<p><strong>OPTIROT1 Function</strong></p>
<p>“optirot1” is a service routine that computes the energy and
gradient for optimally conditioned variable metric optimization
in torsional angle space</p>
<p><strong>OPTRIGID Program</strong></p>
<p>“optrigid” performs an energy minimization of rigid body atom
groups using an optimally conditioned variable metric method</p>
<p><strong>OPTRIGID1 Function</strong></p>
<p>“optrigid1” is a service routine that computes the energy
and gradient for optimally conditioned variable metric
optimization of rigid bodies</p>
<p><strong>OPTSAVE Subroutine</strong></p>
<p>“optsave” is used by the optimizers to write imtermediate
coordinates and other relevant information; also checks for
user requested termination of an optimization</p>
<p><strong>ORBITAL Subroutine</strong></p>
<p>“orbital” finds and organizes lists of atoms in a pisystem,
bonds connecting pisystem atoms and torsions whose central
atoms are both pisystem atoms</p>
<p><strong>ORIENT Subroutine</strong></p>
<p>“orient” computes a set of reference Cartesian coordinates
in standard orientation for each rigid body atom group</p>
<p><strong>ORTHOG Subroutine</strong></p>
<p>“orthog” performs an orthogonalization of an input matrix
via the modified Gram-Schmidt algorithm</p>
<p><strong>OVERLAP Subroutine</strong></p>
<p>“overlap” computes the overlap for two parallel p-orbitals
given the atomic numbers and distance of separation</p>
<p><strong>PARAMYZE Subroutine</strong></p>
<p>“paramyze” prints the force field parameters used in the
computation of each of the potential energy terms</p>
<p><strong>PARTYZE Subroutine</strong></p>
<p>“partyze” prints the energy component and number of
interactions for each of the potential energy terms</p>
<p><strong>PASSB Subroutine</strong></p>
<p><strong>PASSB2 Subroutine</strong></p>
<p><strong>PASSB3 Subroutine</strong></p>
<p><strong>PASSB4 Subroutine</strong></p>
<p><strong>PASSB5 Subroutine</strong></p>
<p><strong>PASSF Subroutine</strong></p>
<p><strong>PASSF2 Subroutine</strong></p>
<p><strong>PASSF3 Subroutine</strong></p>
<p><strong>PASSF4 Subroutine</strong></p>
<p><strong>PASSF5 Subroutine</strong></p>
<p><strong>PATH Program</strong></p>
<p>“path” locates a series of structures equally spaced along
a conformational pathway connecting the input reactant and
product structures; a series of constrained optimizations
orthogonal to the path is done via Lagrangian multipliers</p>
<p><strong>PATH1 Function</strong></p>
<p><strong>PATHPNT Subroutine</strong></p>
<p>“pathpnt” finds a structure on the synchronous transit path
with the specified path value “tpath”</p>
<p><strong>PATHSCAN Subroutine</strong></p>
<p>“pathscan” makes a scan of a synchronous transit pathway by
computing structures and energies for specific path values</p>
<p><strong>PATHVAL Subroutine</strong></p>
<p>“pathval” computes the synchronous transit path value for
the specified structure</p>
<p><strong>PAULING Subroutine</strong></p>
<p>“pauling” uses a rigid body optimization to approximately
pack multiple polypeptide chains</p>
<p><strong>PAULING1 Function</strong></p>
<p>“pauling1” is a service routine that computes the energy
and gradient for optimally conditioned variable metric
optimization of rigid bodies</p>
<p><strong>PBDIRECTPOLFORCE Subroutine</strong></p>
<p><strong>PBEMPOLE Subroutine</strong></p>
<p>“pbempole” calculates the permanent multipole PB energy,
field, forces and torques</p>
<p><strong>PBMUTUALPOLFORCE Subroutine</strong></p>
<p><strong>PDBATOM Subroutine</strong></p>
<p>“pdbatom” adds an atom to the Protein Data Bank file</p>
<p><strong>PDBXYZ Program</strong></p>
<p>“pdbxyz” takes as input a Protein Data Bank file and then
converts to and writes out a Cartesian coordinates file and,
for biopolymers, a sequence file</p>
<p><strong>PIALTER Subroutine</strong></p>
<p>“pialter” modifies bond lengths and force constants according
to the “planar” P-P-P bond order values; also alters 2-fold
torsional parameters based on the “nonplanar” bond orders</p>
<p><strong>PICALC Subroutine</strong></p>
<p>“picalc” performs a modified Pariser-Parr-Pople molecular
orbital calculation for each conjugated pisystem</p>
<p><strong>PIMOVE Subroutine</strong></p>
<p>“pimove” rotates the vector between atoms “list(1)” and
“list(2)” so that atom 1 is at the origin and atom 2 along
the x-axis; the atoms defining the respective planes are
also moved and their bond lengths normalized</p>
<p><strong>PIPLANE Subroutine</strong></p>
<p>“piplane” selects the three atoms which specify the plane
perpendicular to each p-orbital; the current version will
fail in certain situations, including ketenes, allenes,
and isolated or adjacent triple bonds</p>
<p><strong>PISCF Subroutine</strong></p>
<p>“piscf” performs an SCF molecular orbital calculation for a
pisystem to determine bond orders used in parameter scaling</p>
<p><strong>PITILT Subroutine</strong></p>
<p>“pitilt” calculates for each pibond the ratio of the
actual p-orbital overlap integral to the ideal overlap
if the same orbitals were perfectly parallel</p>
<p><strong>PLACE Subroutine</strong></p>
<p>“place” finds the probe sites by putting the probe sphere
tangent to each triple of neighboring atoms</p>
<p><strong>PMONTE Subroutine</strong></p>
<p>“pmonte” implements a Monte Carlo barostat via random trial
changes in the periodic box volume and shape</p>
<p><strong>POLARGRP Subroutine</strong></p>
<p>“polargrp” generates members of the polarization group of
each atom and separate lists of the 1-2, 1-3 and 1-4 group
connectivities</p>
<p><strong>POLARIZE Program</strong></p>
<p>“polarize” computes the molecular polarizability by applying
an external field along each axis followed by diagonalization
of the resulting polarizability tensor</p>
<p><strong>POLEDIT Program</strong></p>
<p>“poledit” provides for the modification and manipulation
of polarizable atomic multipole electrostatic models</p>
<p><strong>POLESORT Subroutine</strong></p>
<p>“polesort” sorts a set of atomic multipole parameters based
on the atom types of centers involved</p>
<p><strong>POLYMER Subroutine</strong></p>
<p>“polymer” tests for the presence of an infinite polymer
extending across periodic boundaries</p>
<p><strong>POLYP Subroutine</strong></p>
<p>“polyp” is a polynomial product routine that multiplies two
algebraic forms</p>
<p><strong>POTENTIAL Program</strong></p>
<p>“potential” calculates the electrostatic potential for a
molecule at a set of grid points; optionally compares to a
target potential or optimizes electrostatic parameters</p>
<p><strong>POTGRID Subroutine</strong></p>
<p>“potgrid” generates electrostatic potential grid points in
radially distributed shells based on the molecular surface</p>
<p><strong>POTNRG Function</strong></p>
<p><strong>POTOFF Subroutine</strong></p>
<p>“potoff” clears the forcefield definition by turning off
the use of each of the potential energy functions</p>
<p><strong>POTPOINT Subroutine</strong></p>
<p>“potpoint” calculates the electrostatic potential at a grid
point “i” as the total electrostatic interaction energy of
the system with a positive charge located at the grid point</p>
<p><strong>POTSTAT Subroutine</strong></p>
<p>“potstat” computes and prints statistics for the electrostatic
potential over a set of grid points</p>
<p><strong>POTWRT Subroutine</strong></p>
<p><strong>PRECONBLK Subroutine</strong></p>
<p>“preconblk” applies a preconditioner to an atom block section
of the Hessian matrix</p>
<p><strong>PRECOND Subroutine</strong></p>
<p>“precond” solves a simplified version of the Newton equations
Ms = r, and uses the result to precondition linear conjugate
gradient iterations on the full Newton equations in “tnsolve”</p>
<p><strong>PRESSURE Subroutine</strong></p>
<p>“pressure” uses the internal virial to find the pressure
in a periodic box and maintains a constant desired pressure
via a barostat method</p>
<p><strong>PRESSURE2 Subroutine</strong></p>
<p>“pressure2” applies a box size and velocity correction at
the half time step as needed for the Monte Carlo barostat</p>
<p><strong>PRIORITY Function</strong></p>
<p>“priority” decides which of a set of connected atoms should
have highest priority in construction of a local coordinate
frame and returns its atom number; if all atoms are of equal
priority then zero is returned</p>
<p><strong>PRMEDIT Program</strong></p>
<p>“prmedit” reformats an existing parameter file, and revises
type and class numbers based on the “atom” parameter ordering</p>
<p><strong>PRMFORM Subroutine</strong></p>
<p>“prmform” formats each individual parameter record to conform
to a consistent text layout</p>
<p><strong>PRMKEY Subroutine</strong></p>
<p>“prmkey” parses a text string to extract keywords related to
force field potential energy functional forms and constants</p>
<p><strong>PRMORDER Subroutine</strong></p>
<p>“prmorder” places a list of atom type or class numbers into
canonical order for potential energy parameter definitions</p>
<p><strong>PRMSORT Subroutine</strong></p>
<p>“prmsort” places a list of atom type or class numbers into
canonical order for potential energy parameter definitions</p>
<p><strong>PRMVAR Subroutine</strong></p>
<p>“prmvar” determines the optimization values from the
corresponding electrostatic potential energy parameters</p>
<p><strong>PRMVAR Subroutine</strong></p>
<p>“prmvar” determines the optimization values from the
corresponding valence potential energy parameters</p>
<p><strong>PROCHAIN Subroutine</strong></p>
<p>“prochain” builds up the internal coordinates for an amino
acid sequence from the phi, psi, omega and chi values</p>
<p><strong>PROJCT Subroutine</strong></p>
<p><strong>PROJECT Subroutine</strong></p>
<p>“project” reads locked vectors from a binary file and projects
them out of the components of the set of trial eigenvectors
using the relation Y = X - U * U^T * X</p>
<p><strong>PROJECTK Subroutine</strong></p>
<p>“projectk” reads locked vectors from a binary file and projects
them out of the components of the set of trial eigenvectors
using the relation Y = X - U * U^T * X</p>
<p><strong>PROMO Subroutine</strong></p>
<p>“promo” writes a banner message containing information
about the Tinker version, release date and copyright notice</p>
<p><strong>PROPERTY Function</strong></p>
<p>“property” takes two input snapshot frames and computes the
value of the property for which the correlation function is
being accumulated</p>
<p><strong>PROSIDE Subroutine</strong></p>
<p>“proside” builds the side chain for a single amino acid
residue in terms of internal coordinates</p>
<p><strong>PROTEIN Program</strong></p>
<p>“protein” builds the internal and Cartesian coordinates
of a polypeptide from amino acid sequence and torsional
angle values for the peptide backbone and side chains</p>
<p><strong>PRTARC Subroutine</strong></p>
<p>“prtarc” writes out a set of Cartesian coordinates for
all active atoms in the Tinker XYZ archive format</p>
<p><strong>PRTDYN Subroutine</strong></p>
<p>“prtdyn” writes out the information needed to restart a
molecular dynamics trajectory to an external disk file</p>
<p><strong>PRTERR Subroutine</strong></p>
<p>“prterr” writes out a set of coordinates to a disk
file prior to aborting on a serious error</p>
<p><strong>PRTFIT Subroutine</strong></p>
<p>“prtfit” makes a key file containing results from fitting a
charge or multipole model to an electrostatic potential grid</p>
<p><strong>PRTINT Subroutine</strong></p>
<p>“prtint” writes out a set of Z-matrix internal
coordinates to an external disk file</p>
<p><strong>PRTMOD Subroutine</strong></p>
<p>“prtmod” writes out a set of modified Cartesian coordinates
with an optional atom number offset to an external disk file</p>
<p><strong>PRTMOL2 Program</strong></p>
<p>“prtmol2” writes out a set of coordinates in Tripos MOL2
format to an external disk file</p>
<p><strong>PRTPDB Subroutine</strong></p>
<p>“prtpdb” writes out a set of Protein Data Bank coordinates
to an external disk file</p>
<p><strong>PRTPOLE Subroutine</strong></p>
<p>“prtpole” creates a coordinates file, and a key file with
atomic multipoles corrected for intergroup polarization</p>
<p><strong>PRTPRM Subroutine</strong></p>
<p>“prtprm” writes out a formatted listing of the default
set of potential energy parameters for a force field</p>
<p><strong>PRTSEQ Subroutine</strong></p>
<p>“prtseq” writes out a biopolymer sequence to an external
disk file with 15 residues per line and distinct chains
separated by blank lines</p>
<p><strong>PRTVAL Subroutine</strong></p>
<p>“prtval” writes the final valence parameter results to the
standard output and appends the values to a key file</p>
<p><strong>PRTVIB Subroutine</strong></p>
<p>“prtvib” writes to an external disk file a series of
coordinate sets representing motion along a vibrational
normal mode</p>
<p><strong>PRTXYZ Subroutine</strong></p>
<p>“prtxyz” writes out a set of Cartesian coordinates
to an external disk file</p>
<p><strong>PSCALE Subroutine</strong></p>
<p>“pscale” implements a Berendsen barostat by scaling the
coordinates and box dimensions via coupling to an external
constant pressure bath</p>
<p><strong>PSS Program</strong></p>
<p>“pss” implements the potential smoothing plus search method
for global optimization in Cartesian coordinate space with
local searches performed in Cartesian or torsional space</p>
<p><strong>PSS1 Function</strong></p>
<p>“pss1” is a service routine that computes the energy
and gradient during PSS global optimization in Cartesian
coordinate space</p>
<p><strong>PSS2 Subroutine</strong></p>
<p>“pss2” is a service routine that computes the sparse
matrix Hessian elements during PSS global optimization
in Cartesian coordinate space</p>
<p><strong>PSSRGD1 Function</strong></p>
<p>“pssrgd1” is a service routine that computes the energy and
gradient during PSS global optimization over rigid bodies</p>
<p><strong>PSSRIGID Program</strong></p>
<p>“pssrigid” implements the potential smoothing plus search method
for global optimization for a set of rigid bodies</p>
<p><strong>PSSROT Program</strong></p>
<p>“pssrot” implements the potential smoothing plus search method
for global optimization in torsional space</p>
<p><strong>PSSROT1 Function</strong></p>
<p>“pssrot1” is a service routine that computes the energy and
gradient during PSS global optimization in torsional space</p>
<p><strong>PSSWRITE Subroutine</strong></p>
<p><strong>PTEST Subroutine</strong></p>
<p>“ptest” determines the numerical virial tensor, and compares
analytical to numerical values for dE/dV and isotropic pressure</p>
<p><strong>PTINCY Function</strong></p>
<p><strong>PZEXTR Subroutine</strong></p>
<p>“pzextr” is a polynomial extrapolation routine used during
Bulirsch-Stoer integration of ordinary differential equations</p>
<p><strong>QIROTMAT Subroutine</strong></p>
<p>“qirotmat” finds a rotation matrix that describes the
interatomic vector</p>
<p><strong>QONVEC Subroutine</strong></p>
<p>“qonvec” is a vector utility routine used during sliding
block iterative matrix diagonalization</p>
<p><strong>QRFACT Subroutine</strong></p>
<p>“qrfact” computes the QR factorization of an m by n matrix a
via Householder transformations with optional column pivoting;
the routine determines an orthogonal matrix q, a permutation
matrix p, and an upper trapezoidal matrix r with diagonal
elements of nonincreasing magnitude, such that a*p = q*r; the
Householder transformation for column k, k = 1,2,…,min(m,n),
is of the form:</p>
<p><strong>QRSOLVE Subroutine</strong></p>
<p>“qrsolve” solves a*x = b and d*x = 0 in the least squares sense;
used with routine “qrfact” to solve least squares problems</p>
<p><strong>QUATFIT Subroutine</strong></p>
<p>“quatfit” uses a quaternion-based method to achieve the best
fit superposition of two sets of coordinates</p>
<p><strong>RADIAL Program</strong></p>
<p>“radial” finds the radial distribution function for a specified
pair of atom types via analysis of a set of coordinate frames</p>
<p><strong>RANDOM Function</strong></p>
<p>“random” generates a random number on [0,1] via a long
period generator due to L’Ecuyer with Bays-Durham shuffle</p>
<p><strong>RANVEC Subroutine</strong></p>
<p>“ranvec” generates a unit vector in 3-dimensional
space with uniformly distributed random orientation</p>
<p><strong>RATTLE Subroutine</strong></p>
<p>“rattle” implements the first portion of the RATTLE algorithm
by correcting atomic positions and half-step velocities to
maintain interatomic distance and absolute spatial constraints</p>
<p><strong>RATTLE2 Subroutine</strong></p>
<p>“rattle2” implements the second portion of the RATTLE algorithm
by correcting the full-step velocities in order to maintain
interatomic distance constraints</p>
<p><strong>READBLK Subroutine</strong></p>
<p>“readblk” reads in a set of snapshot frames and transfers
the values to internal arrays for use in the computation
of time correlation functions</p>
<p><strong>READDYN Subroutine</strong></p>
<p>“readdyn” get the positions, velocities and accelerations
for a molecular dynamics restart from an external disk file</p>
<p><strong>READGARC Subroutine</strong></p>
<p>“readgarc” reads data from Gaussian archive section; each
entry is terminated with a backslash symbol</p>
<p><strong>READGAU Subroutine</strong></p>
<p>“readgau” reads an ab initio optimized structure, forces,
Hessian and frequencies from a Gaussian 09 output file</p>
<p><strong>READGDMA Subroutine</strong></p>
<p>“readgdma” takes the DMA output in spherical harmonics from
the GDMA program and converts to Cartesian multipoles in
the global coordinate frame</p>
<p><strong>READINT Subroutine</strong></p>
<p>“readint” gets a set of Z-matrix internal coordinates
from an external file</p>
<p><strong>READMOL Subroutine</strong></p>
<p>“readmol” gets a set of MDL MOL coordinates from
an external disk file</p>
<p><strong>READMOL2 Subroutine</strong></p>
<p>“readmol2” gets a set of Tripos MOL2 coordinates from an
external disk file</p>
<p><strong>READPDB Subroutine</strong></p>
<p>“readpdb” gets a set of Protein Data Bank coordinates
from an external disk file</p>
<p><strong>READPOT Subroutine</strong></p>
<p>“readpot” gets a set of grid points and target electrostatic
potential values from an external disk file</p>
<p><strong>READPRM Subroutine</strong></p>
<p>“readprm” processes the potential energy parameter file
in order to define the default force field parameters</p>
<p><strong>READSEQ Subroutine</strong></p>
<p>“readseq” gets a biopolymer sequence containing one or more
separate chains from an external file; all lines containing
sequence must begin with the starting sequence number, the
actual sequence is read from subsequent nonblank characters</p>
<p><strong>READXYZ Subroutine</strong></p>
<p>“readxyz” gets a set of Cartesian coordinates from
an external disk file</p>
<p><strong>REFINE Subroutine</strong></p>
<p>“refine” performs minimization of the atomic coordinates
of an initial crude embedded distance geometry structure versus
the bound, chirality, planarity and torsional error functions</p>
<p><strong>RELEASEMONITOR Subroutine</strong></p>
<p><strong>REPLICA Subroutine</strong></p>
<p>“replica” decides between images and replicates for generation
of periodic boundary conditions, and sets the cell replicate
list if the replicates method is to be used</p>
<p><strong>RESPA Subroutine</strong></p>
<p>“respa” performs a single multiple time step molecular dynamics
step using the reversible reference system propagation algorithm
(r-RESPA) via a Verlet core with the potential split into fast-
and slow-evolving portions</p>
<p><strong>RFINDEX Subroutine</strong></p>
<p>“rfindex” finds indices for each multipole site for use
in computing reaction field energetics</p>
<p><strong>RGDSTEP Subroutine</strong></p>
<p>“rgdstep” performs a single molecular dynamics time step
via a rigid body integration algorithm</p>
<p><strong>RIBOSOME Subroutine</strong></p>
<p>“ribosome” translates a polypeptide structure in Protein Data
Bank format to a Cartesian coordinate file and sequence file</p>
<p><strong>RIGIDXYZ Subroutine</strong></p>
<p>“rigidxyz” computes Cartesian coordinates for a rigid body
group via rotation and translation of reference coordinates</p>
<p><strong>RINGS Subroutine</strong></p>
<p>“rings” searches the structure for small rings and stores
their constituent atoms, and optionally reduces large rings
into their component smaller rings</p>
<p><strong>RMSERROR Subroutine</strong></p>
<p>“rmserror” computes the maximum absolute deviation and the
rms deviation from the distance bounds, and the number and
rms value of the distance restraint violations</p>
<p><strong>RMSFIT Function</strong></p>
<p>“rmsfit” computes the rms fit of two coordinate sets</p>
<p><strong>ROTANG Function</strong></p>
<p><strong>ROTCHECK Function</strong></p>
<p>“rotcheck” tests a specified candidate rotatable bond for
the disallowed case where inactive atoms are found on both
sides of the candidate bond</p>
<p><strong>ROTEULER Subroutine</strong></p>
<p>“roteuler” computes a set of Euler angle values consistent
with an input rotation matrix</p>
<p><strong>ROTFRAME Subroutine</strong></p>
<p>“rotframe” takes the global multipole moments and rotates them
into the local coordinate frame defined at each atomic site</p>
<p><strong>ROTLIST Subroutine</strong></p>
<p>“rotlist” generates the minimum list of all the atoms lying
to one side of a pair of directly bonded atoms; optionally
finds the minimal list by choosing the side with fewer atoms</p>
<p><strong>ROTMAT Subroutine</strong></p>
<p>“rotmat” finds the rotation matrix that rotates the local
coordinate system into the global frame at a multipole site</p>
<p><strong>ROTPOLE Subroutine</strong></p>
<p>“rotpole” constructs the set of atomic multipoles in the global
frame by applying the correct rotation matrix for each site</p>
<p><strong>ROTRGD Subroutine</strong></p>
<p>“rotrgd” finds the rotation matrix for a rigid body due
to a single step of dynamics</p>
<p><strong>ROTSITE Subroutine</strong></p>
<p>“rotsite” rotates the local frame atomic multipoles at a
specified site into the global coordinate frame by applying
a rotation matrix</p>
<p><strong>SADDLE Program</strong></p>
<p>“saddle” finds a transition state between two conformational
minima using a combination of ideas from the synchronous transit
(Halgren-Lipscomb) and quadratic path (Bell-Crighton) methods</p>
<p><strong>SADDLE1 Function</strong></p>
<p>“saddle1” is a service routine that computes the energy and
gradient for transition state optimization</p>
<p><strong>SADDLES Subroutine</strong></p>
<p>“saddles” constructs circles, convex edges and saddle faces</p>
<p><strong>SAVEYZE Subroutine</strong></p>
<p>“saveyze” prints the atomic forces and/or the induced dipoles
to separate external disk files</p>
<p><strong>SBGUESS Subroutine</strong></p>
<p>“sbguess” sets approximate stretch-bend force constants based
on atom type and connected atoms</p>
<p><strong>SCAN Program</strong></p>
<p>“scan” attempts to find all the local minima on a potential
energy surface via an iterative series of local searches along
normal mode directions</p>
<p><strong>SCAN1 Function</strong></p>
<p>“scan1” is a service routine that computes the energy and
gradient during exploration of a potential energy surface
via iterative local search</p>
<p><strong>SCAN2 Subroutine</strong></p>
<p>“scan2” is a service routine that computes the sparse matrix
Hessian elements during exploration of a potential energy
surface via iterative local search</p>
<p><strong>SCANPDB Subroutine</strong></p>
<p>“scanpdb” reads the first model in a Protein Data Bank file and
sets chains, alternate sites and insertion records to be used</p>
<p><strong>SDAREA Subroutine</strong></p>
<p>“sdarea” optionally scales the atomic friction coefficient
of each atom based on its accessible surface area</p>
<p><strong>SDSTEP Subroutine</strong></p>
<p>“sdstep” performs a single stochastic dynamics time step
via the velocity Verlet integration algorithm</p>
<p><strong>SDTERM Subroutine</strong></p>
<p>“sdterm” finds the frictional and random terms needed to
update positions and velocities during stochastic dynamics</p>
<p><strong>SEARCH Subroutine</strong></p>
<p>“search” is a unidimensional line search based upon parabolic
extrapolation and cubic interpolation using both function and
gradient values</p>
<p><strong>SETACCELERATION Subroutine</strong></p>
<p><strong>SETATOMIC Subroutine</strong></p>
<p><strong>SETATOMTYPES Subroutine</strong></p>
<p><strong>SETCHARGE Subroutine</strong></p>
<p><strong>SETCHUNK Subroutine</strong></p>
<p>“setchunk” marks a chunk in the PME spatial table which is
overlapped by the B-splines for a site</p>
<p><strong>SETCONNECTIVITY Subroutine</strong></p>
<p><strong>SETCOORDINATES Subroutine</strong></p>
<p><strong>SETELECT Subroutine</strong></p>
<p>“setelect” assigns partial charge, bond dipole and atomic
multipole parameters for the current structure, as needed
for computation of the electrostatic potential</p>
<p><strong>SETENERGY Subroutine</strong></p>
<p><strong>SETFILE Subroutine</strong></p>
<p><strong>SETFORCEFIELD Subroutine</strong></p>
<p><strong>SETFRAME Subroutine</strong></p>
<p>“setframe” assigns a local coordinate frame at each atomic
multipole site using high priority connected atoms along axes</p>
<p><strong>SETGRADIENTS Subroutine</strong></p>
<p><strong>SETINDUCED Subroutine</strong></p>
<p><strong>SETKEYWORD Subroutine</strong></p>
<p><strong>SETMASS Subroutine</strong></p>
<p><strong>SETMDTIME Subroutine</strong></p>
<p><strong>SETMOL2 Program</strong></p>
<p>“setmol2” assigns MOL2 atom names/types/charges and bond types
based upon atomic numbers and connectivity</p>
<p><strong>SETNAME Subroutine</strong></p>
<p><strong>SETPAIR Program</strong></p>
<p>“setpair” is a service routine that assigns flags, sets cutoffs
and allocates arrays used by different pairwise neighbor methods</p>
<p><strong>SETPOLAR Subroutine</strong></p>
<p>“setpolar” assigns atomic polarizabilities, Thole damping or
charge penetration parameters, and polarization groups with
user modification of these values</p>
<p><strong>SETSTEP Subroutine</strong></p>
<p><strong>SETSTORY Subroutine</strong></p>
<p><strong>SETTIME Subroutine</strong></p>
<p>“settime” initializes the wall clock and elapsed CPU times</p>
<p><strong>SETUPDATED Subroutine</strong></p>
<p><strong>SETVELOCITY Subroutine</strong></p>
<p><strong>SHAKE Subroutine</strong></p>
<p>“shake” implements the SHAKE algorithm by correcting atomic
positions to maintain interatomic distance and absolute spatial
constraints</p>
<p><strong>SHAKEF Subroutine</strong></p>
<p>“shakef” modifies the gradient to remove components along any
holonomic distance contraints using a variant of SHAKE</p>
<p><strong>SHAKEUP Subroutine</strong></p>
<p>“shakeup” initializes any holonomic constraints for use with
the SHAKE and RATTLE algorithms</p>
<p><strong>SHROTMAT Subroutine</strong></p>
<p>“shrotmat” finds the rotation matrix that converts spherical
harmonic quadrupoles from the local to the global frame given
the required dipole rotation matrix</p>
<p><strong>SHROTSITE Subroutine</strong></p>
<p>“shrotsite” converts spherical harmonic multipoles from the
local to the global frame given required rotation matrices</p>
<p><strong>SIGMOID Function</strong></p>
<p>“sigmoid” implements a normalized sigmoidal function on the
interval [0,1]; the curves connect (0,0) to (1,1) and have
a cooperativity controlled by beta, they approach a straight
line as beta -&gt; 0 and get more nonlinear as beta increases</p>
<p><strong>SIMPLEX Subroutine</strong></p>
<p>“simplex” is a general multidimensional Nelder-Mead simplex
optimization routine requiring only repeated evaluations of
the objective function</p>
<p><strong>SIMPLEX1 Function</strong></p>
<p>“simplex1” is a service routine used only by the Nelder-Mead
simplex optimization method</p>
<p><strong>SKTDYN Subroutine</strong></p>
<p>“sktdyn” sends the current dynamics info via a socket</p>
<p><strong>SKTINIT Subroutine</strong></p>
<p>“sktinit” sets up socket communication with the graphical
user interface by starting a Java virtual machine, initiating
a server, and loading an object with system information</p>
<p><strong>SKTKILL Subroutine</strong></p>
<p>“sktkill” closes the server and Java virtual machine</p>
<p><strong>SKTOPT Subroutine</strong></p>
<p>“sktopt” sends the current optimization info via a socket</p>
<p><strong>SLATER Subroutine</strong></p>
<p>“slater” is a general routine for computing the overlap
integrals between two Slater-type orbitals</p>
<p><strong>SNIFFER Program</strong></p>
<p>“sniffer” performs a global energy minimization using a
discrete version of Griewank’s global search trajectory</p>
<p><strong>SNIFFER1 Function</strong></p>
<p>“sniffer1” is a service routine that computes the energy
and gradient for the Sniffer global optimization method</p>
<p><strong>SOAK Subroutine</strong></p>
<p>“soak” takes a currently defined solute system and places
it into a solvent box, with removal of any solvent molecules
that overlap the solute</p>
<p><strong>SORT Subroutine</strong></p>
<p>“sort” takes an input list of integers and sorts it
into ascending order using the Heapsort algorithm</p>
<p><strong>SORT10 Subroutine</strong></p>
<p>“sort10” takes an input list of character strings and sorts
it into alphabetical order using the Heapsort algorithm,
duplicate values are removed from the final sorted list</p>
<p><strong>SORT2 Subroutine</strong></p>
<p>“sort2” takes an input list of reals and sorts it
into ascending order using the Heapsort algorithm;
it also returns a key into the original ordering</p>
<p><strong>SORT3 Subroutine</strong></p>
<p>“sort3” takes an input list of integers and sorts it
into ascending order using the Heapsort algorithm;
it also returns a key into the original ordering</p>
<p><strong>SORT4 Subroutine</strong></p>
<p>“sort4” takes an input list of integers and sorts it into
ascending absolute value using the Heapsort algorithm</p>
<p><strong>SORT5 Subroutine</strong></p>
<p>“sort5” takes an input list of integers and sorts it
into ascending order based on each value modulo “m”</p>
<p><strong>SORT6 Subroutine</strong></p>
<p>“sort6” takes an input list of character strings and sorts
it into alphabetical order using the Heapsort algorithm</p>
<p><strong>SORT7 Subroutine</strong></p>
<p>“sort7” takes an input list of character strings and sorts it
into alphabetical order using the Heapsort algorithm; it also
returns a key into the original ordering</p>
<p><strong>SORT8 Subroutine</strong></p>
<p>“sort8” takes an input list of integers and sorts it into
ascending order using the Heapsort algorithm, duplicate
values are removed from the final sorted list</p>
<p><strong>SORT9 Subroutine</strong></p>
<p>“sort9” takes an input list of reals and sorts it into
ascending order using the Heapsort algorithm, duplicate
values are removed from the final sorted list</p>
<p><strong>SPACEFILL Program</strong></p>
<p>“spacefill” computes the surface area and volume of
a structure; the van der Waals, accessible-excluded,
and contact-reentrant definitions are available</p>
<p><strong>SPECTRUM Program</strong></p>
<p>“spectrum” computes a power spectrum over a wavelength range
from the velocity autocorrelation as a function of time</p>
<p><strong>SPHERE Subroutine</strong></p>
<p>“sphere” finds a specified number of uniformly distributed
points on a sphere of unit radius centered at the origin</p>
<p><strong>SQUARE Subroutine</strong></p>
<p>“square” is a nonlinear least squares routine derived from the
IMSL BCLSF routine and the MINPACK LMDER routine; the Jacobian
is estimated by finite differences and bounds can be specified
for the variables to be refined</p>
<p><strong>SUFFIX Subroutine</strong></p>
<p>“suffix” checks a filename for the presence of an extension,
and appends an extension and version if none is found</p>
<p><strong>SUPERPOSE Program</strong></p>
<p>“superpose” takes pairs of structures and superimposes them
in the optimal least squares sense; it will attempt to match
all atom pairs or only those specified by the user</p>
<p><strong>SURFACE Subroutine</strong></p>
<p>“surface” performs an analytical computation of the weighted
solvent accessible surface area of each atom and the first
derivatives of the area with respect to Cartesian coordinates</p>
<p><strong>SURFACE1 Subroutine</strong></p>
<p>“surface1” performs an analytical computation of the weighted
solvent accessible surface area of each atom and the first
derivatives of the area with respect to Cartesian coordinates</p>
<p><strong>SURFATOM Subroutine</strong></p>
<p>“surfatom” performs an analytical computation of the surface
area of a specified atom; a simplified version of “surface”</p>
<p><strong>SURFATOM1 Subroutine</strong></p>
<p>“surfatom1” performs an analytical computation of the surface
area and first derivatives with respect to Cartesian coordinates
of a specified atom</p>
<p><strong>SWITCH Subroutine</strong></p>
<p>“switch” sets the coeffcients used by the fifth and seventh
order polynomial switching functions for spherical cutoffs</p>
<p><strong>SYMMETRY Subroutine</strong></p>
<p>“symmetry” applies symmetry operators to the fractional
coordinates of the asymmetric unit in order to generate
the symmetry related atoms of the full unit cell</p>
<p><strong>SYSTYZE Subroutine</strong></p>
<p>“systyze” is an auxiliary routine for the analyze program
that prints general information about the molecular system
and the force field model</p>
<p><strong>TABLE_FILL Subroutine</strong></p>
<p>“table_fill” constructs an array which stores the spatial
regions of the particle mesh Ewald grid with contributions
from each site</p>
<p><strong>TANGENT Subroutine</strong></p>
<p>“tangent” finds the projected gradient on the synchronous
transit path for a point along the transit pathway</p>
<p><strong>TCGSWAP Subroutine</strong></p>
<p>“tcgswap” switches two sets of induced dipole quantities for
use with the TCG induced dipole solver</p>
<p><strong>TCG_ALPHA12 Subroutine</strong></p>
<p>“tcg_alpha12” computes source1 = alpha*source1 and
source2 = alpha*source2</p>
<p><strong>TCG_ALPHA22 Subroutine</strong></p>
<p>“tcg_alpha22” computes result1 = alpha*source1 and
result2 = alpha*source2</p>
<p><strong>TCG_ALPHAQUAD Subroutine</strong></p>
<p>“tcg_alphaquad” computes the quadratic form, &lt;a*alpha*b&gt;,
where alpha is the diagonal atomic polarizability matrix</p>
<p><strong>TCG_DOTPROD Subroutine</strong></p>
<p>“tcg_dotprod” computes the dot product of two vectors
of length n elements</p>
<p><strong>TCG_RESOURCE Subroutine</strong></p>
<p>“tcg_resource” sets the number of mutual induced dipole
pairs based on the passed argument</p>
<p><strong>TCG_T0 Subroutine</strong></p>
<p>“tcg_t0” applies T matrix to ind/p, and returns v3d/p
T = 1/alpha + Tu</p>
<p><strong>TCG_UFIELD Subroutine</strong></p>
<p>“tcg_ufield” applies -Tu to ind/p and returns v3d/p</p>
<p><strong>TCG_UPDATE Subroutine</strong></p>
<p>“tcg_update” computes pvec = alpha*rvec + beta*pvec;
if the preconditioner is not used, then alpha = identity</p>
<p><strong>TEMPER Subroutine</strong></p>
<p>“temper” computes the instantaneous temperature and applies a
thermostat via Berendsen or Bussi-Parrinello velocity scaling,
Andersen stochastic collisions or Nose-Hoover chains; also uses
Berendsen scaling for any iEL induced dipole variables</p>
<p><strong>TEMPER2 Subroutine</strong></p>
<p>“temper2” applies a velocity correction at the half time step
as needed for the Nose-Hoover thermostat</p>
<p><strong>TESTGRAD Program</strong></p>
<p>“testgrad” computes and compares the analytical and numerical
gradient vectors of the potential energy function with respect
to Cartesian coordinates</p>
<p><strong>TESTHESS Program</strong></p>
<p>“testhess” computes and compares the analytical and numerical
Hessian matrices of the potential energy function with respect
to Cartesian coordinates</p>
<p><strong>TESTPAIR Program</strong></p>
<p>“testpair” performs a set of timing tests to compare the
evaluation of potential energy and energy/gradient using
different methods for finding pairwise neighbors</p>
<p><strong>TESTPOL Program</strong></p>
<p>“testpol” compares the induced dipoles from direct polarization,
mutual SCF iterations, perturbation theory extrapolation (OPT),
and truncated conjugate gradient (TCG) solvers</p>
<p><strong>TESTROT Program</strong></p>
<p>“testrot” computes and compares the analytical and numerical
gradient vectors of the potential energy function with respect
to rotatable torsional angles</p>
<p><strong>TESTVIR Program</strong></p>
<p>“testvir” computes the analytical internal virial and compares
it to a numerical virial derived from the finite difference
derivative of the energy with respect to lattice vectors</p>
<p><strong>TIMER Program</strong></p>
<p>“timer” measures the CPU time required for file reading and
parameter assignment, potential energy computation, energy
and gradient computation, and Hessian matrix evaluation</p>
<p><strong>TIMEROT Program</strong></p>
<p>“timerot” measures the CPU time required for file reading
and parameter assignment, potential energy computation,
energy and gradient over torsions, and torsional angle
Hessian matrix evaluation</p>
<p><strong>TNCG Subroutine</strong></p>
<p>“tncg” implements a truncated Newton optimization algorithm
in which a preconditioned linear conjugate gradient method is
used to approximately solve Newton’s equations; special features
include use of an explicit sparse Hessian or finite-difference
gradient-Hessian products within the PCG iteration; the exact
Newton search directions can be used optionally; by default the
algorithm checks for negative curvature to prevent convergence
to a stationary point having negative eigenvalues; if a saddle
point is desired this test can be removed by disabling “negtest”</p>
<p><strong>TNSOLVE Subroutine</strong></p>
<p>“tnsolve” uses a linear conjugate gradient method to find
an approximate solution to the set of linear equations
represented in matrix form by Hp = -g (Newton’s equations)</p>
<p><strong>TORFIT1 Function</strong></p>
<p>“torfit1” is a service routine that computes the energy and
gradient for a low storage BFGS optimization in Cartesian
coordinate space</p>
<p><strong>TORGUESS Subroutine</strong></p>
<p>“torguess” set approximate torsion amplitude parameters based
on atom type and connected atoms</p>
<p><strong>TORPHASE Subroutine</strong></p>
<p>“torphase” sets the n-fold amplitude and phase values
for each torsion via sorting of the input parameters</p>
<p><strong>TORQUE Subroutine</strong></p>
<p>“torque” takes the torque values on a single site defined by
a local coordinate frame and converts to Cartesian forces on
the original site and sites specifying the local frame, also
gives the x,y,z-force components needed for virial computation</p>
<p><strong>TORSER Function</strong></p>
<p>“torser” computes the torsional error function and its first
derivatives with respect to the atomic Cartesian coordinates
based on the deviation of specified torsional angles from
desired values, the contained bond angles are also restrained
to avoid a numerical instability</p>
<p><strong>TORSFIT Program</strong></p>
<p>“torsfit” refines torsional force field parameters based on
a quantum mechanical potential surface and analytical gradient</p>
<p><strong>TORSIONS Subroutine</strong></p>
<p>“torsions” finds the total number of torsional angles and
the numbers of the four atoms defining each torsional angle</p>
<p><strong>TORUS Subroutine</strong></p>
<p>“torus” sets a list of all of the temporary torus positions
by testing for a torus between each atom and its neighbors</p>
<p><strong>TOTERR Function</strong></p>
<p>“toterr” is the error function and derivatives for a distance
geometry embedding; it includes components from the distance
bounds, hard sphere contacts, local geometry, chirality and
torsional restraint errors</p>
<p><strong>TRANSFORM Subroutine</strong></p>
<p>“transform” diagonalizes the current basis vectors to produce
trial roots for sliding block iterative matrix diagonalization</p>
<p><strong>TRANSIT Function</strong></p>
<p>“transit” evaluates the synchronous transit function and
gradient; linear and quadratic transit paths are available</p>
<p><strong>TRBASIS Subroutine</strong></p>
<p>“trbasis” forms translation and rotation basis vectors used
during vibrational analysis via block iterative diagonalization</p>
<p><strong>TRIANGLE Subroutine</strong></p>
<p>“triangle” smooths the upper and lower distance bounds via
the triangle inequality using a full-matrix variant of the
Floyd-Warshall shortest path algorithm; this routine is
usually much slower than the sparse matrix shortest path
methods in “geodesic” and “trifix”, and should be used only
for comparison with answers generated by those routines</p>
<p><strong>TRIFIX Subroutine</strong></p>
<p>“trifix” rebuilds both the upper and lower distance bound
matrices following tightening of one or both of the bounds
between a specified pair of atoms, “p” and “q”, using a
modification of Murchland’s shortest path update algorithm</p>
<p><strong>TRIGGER Subroutine</strong></p>
<p>“trigger” constructs a set of initial trial vectors for
use during sliding block iterative matrix diagonalization</p>
<p><strong>TRIMHEAD Subroutine</strong></p>
<p>“trimhead” removes blank spaces before the first non-blank
character in a text string by shifting the string to the left</p>
<p><strong>TRIMTEXT Function</strong></p>
<p>“trimtext” finds and returns the location of the last
non-blank character before the first null character in
an input text string; the function returns zero if no
such character is found</p>
<p><strong>TRIPLE Function</strong></p>
<p>“triple” finds the triple product of three vectors; used as
a service routine by the Connolly surface area and volume
computation</p>
<p><strong>TRUST Subroutine</strong></p>
<p>“trust” updates the model trust region for a nonlinear least
squares calculation based on ideas found in NL2SOL and Dennis
and Schnabel’s book</p>
<p><strong>UBUILD Subroutine</strong></p>
<p>“ubuild” performs a complete rebuild of the polarization
preconditioner neighbor list for all sites</p>
<p><strong>UDIRECT1 Subroutine</strong></p>
<p>“udirect1” computes the reciprocal space contribution of the
permanent atomic multipole moments to the field</p>
<p><strong>UDIRECT2A Subroutine</strong></p>
<p>“udirect2a” computes the real space contribution of the permanent
atomic multipole moments to the field via a double loop</p>
<p><strong>UDIRECT2B Subroutine</strong></p>
<p>“udirect2b” computes the real space contribution of the permanent
atomic multipole moments to the field via a neighbor list</p>
<p><strong>UFIELD0A Subroutine</strong></p>
<p>“ufield0a” computes the mutual electrostatic field due to
induced dipole moments via a double loop</p>
<p><strong>UFIELD0B Subroutine</strong></p>
<p>“ufield0b” computes the mutual electrostatic field due to
induced dipole moments via a pair list</p>
<p><strong>UFIELD0C Subroutine</strong></p>
<p>“ufield0c” computes the mutual electrostatic field due to
induced dipole moments via Ewald summation</p>
<p><strong>UFIELD0D Subroutine</strong></p>
<p>“ufield0d” computes the mutual electrostatic field due to
induced dipole moments for use with with generalized Kirkwood
implicit solvation</p>
<p><strong>UFIELD0E Subroutine</strong></p>
<p>“ufield0e” computes the mutual electrostatic field due to
induced dipole moments via a Poisson-Boltzmann solver</p>
<p><strong>UFIELDI Subroutine</strong></p>
<p>“ufieldi” computes the electrostatic field due to intergroup
induced dipole moments</p>
<p><strong>ULIGHT Subroutine</strong></p>
<p>“ulight” performs a complete rebuild of the polarization
preconditioner pair neighbor list for all sites using the
method of lights</p>
<p><strong>ULIST Subroutine</strong></p>
<p>“ulist” performs an update or a complete rebuild of the
neighbor lists for the polarization preconditioner</p>
<p><strong>ULSPRED Subroutine</strong></p>
<p>“ulspred” uses standard extrapolation or a least squares fit
to set coefficients of an induced dipole predictor polynomial</p>
<p><strong>UMUTUAL1 Subroutine</strong></p>
<p>“umutual1” computes the reciprocal space contribution of the
induced atomic dipole moments to the field</p>
<p><strong>UMUTUAL2A Subroutine</strong></p>
<p>“umutual2a” computes the real space contribution of the induced
atomic dipole moments to the field via a double loop</p>
<p><strong>UMUTUAL2B Subroutine</strong></p>
<p>“umutual2b” computes the real space contribution of the induced
atomic dipole moments to the field via a neighbor list</p>
<p><strong>UNITCELL Subroutine</strong></p>
<p>“unitcell” gets the periodic boundary box size and related
values from an external keyword file</p>
<p><strong>UPCASE Subroutine</strong></p>
<p>“upcase” converts a text string to all upper case letters</p>
<p><strong>URYGUESS Function</strong></p>
<p>“uryguess” sets approximate Urey-Bradley force constants
based on atom type and connected atoms</p>
<p><strong>USCALE0A Subroutine</strong></p>
<p>“uscale0a” builds and applies a preconditioner for the conjugate
gradient induced dipole solver using a double loop</p>
<p><strong>USCALE0B Subroutine</strong></p>
<p>“uscale0b” builds and applies a preconditioner for the conjugate
gradient induced dipole solver using a neighbor pair list</p>
<p><strong>VALENCE Program</strong></p>
<p>“valence” refines force field parameters for valence terms based
on a quantum mechanical optimized structure and frequencies</p>
<p><strong>VALENCE1 Function</strong></p>
<p>“valence1” is a service routine that computes the energy and
gradient for a structure during valence parameter fitting</p>
<p><strong>VALFIT1 Function</strong></p>
<p>“valfit1” is a service routine that computes the RMS error
and gradient for valence parameters fit to QM results</p>
<p><strong>VALGUESS Subroutine</strong></p>
<p>“valguess” sets approximate valence parameter values based on
quantum mechanical structure and frequency data</p>
<p><strong>VALRMS Function</strong></p>
<p>“valrms” evaluates a valence parameter goodness-of-fit error
function based on comparison of forces, frequencies, bond
lengths and angles to QM results</p>
<p><strong>VAM Subroutine</strong></p>
<p>“vam” takes the analytical molecular surface defined
as a collection of spherical and toroidal polygons
and uses it to compute the volume and surface area</p>
<p><strong>VARPRM Subroutine</strong></p>
<p>“varprm” copies the current optimization values into the
corresponding electrostatic potential energy parameters</p>
<p><strong>VARPRM Subroutine</strong></p>
<p>“varprm” copies the current optimization values into the
corresponding valence potential energy parameters</p>
<p><strong>VBUILD Subroutine</strong></p>
<p>“vbuild” performs a complete rebuild of the van der Waals
pair neighbor list for all sites</p>
<p><strong>VCROSS Subroutine</strong></p>
<p>“vcross” finds the cross product of two vectors</p>
<p><strong>VDWERR Function</strong></p>
<p>“vdwerr” is the hard sphere van der Waals bound error function
and derivatives that penalizes close nonbonded contacts,
pairwise neighbors are generated via the method of lights</p>
<p><strong>VDWGUESS Subroutine</strong></p>
<p>“vdwguess” sets initial VDW parameters based on atom type
and connected atoms</p>
<p><strong>VECANG Function</strong></p>
<p>“vecang” finds the angle between two vectors handed with respect
to a coordinate axis; returns an angle in the range [0,2*pi]</p>
<p><strong>VERLET Subroutine</strong></p>
<p>“verlet” performs a single molecular dynamics time step
via the velocity Verlet multistep recursion formula</p>
<p><strong>VERSION Subroutine</strong></p>
<p>“version” checks the name of a file about to be opened; if
if “old” status is passed, the name of the highest current
version is returned; if “new” status is passed the filename
of the next available unused version is generated</p>
<p><strong>VIBBIG Program</strong></p>
<p>“vibbig” performs large-scale vibrational mode analysis using
only vector storage and gradient evaluations; preconditioning
is via an approximate inverse from a block diagonal Hessian,
and a sliding block method is used to converge any number of
eigenvectors starting from either lowest or highest frequency</p>
<p><strong>VIBRATE Program</strong></p>
<p>“vibrate” performs a vibrational normal mode analysis; the
Hessian matrix of second derivatives is determined and then
diagonalized both directly and after mass weighting; output
consists of the eigenvalues of the force constant matrix as
well as the vibrational frequencies and displacements</p>
<p><strong>VIBROT Program</strong></p>
<p>“vibrot” computes the eigenvalues and eigenvectors of the
torsional Hessian matrix</p>
<p><strong>VIRIYZE Subroutine</strong></p>
<p>“propyze” finds and prints the internal virial, the dE/dV value
and an estimate of the pressure</p>
<p><strong>VLIGHT Subroutine</strong></p>
<p>“vlight” performs a complete rebuild of the van der Waals
pair neighbor list for all sites using the method of lights</p>
<p><strong>VLIST Subroutine</strong></p>
<p>“vlist” performs an update or a complete rebuild of the
nonbonded neighbor lists for vdw sites</p>
<p><strong>VNORM Subroutine</strong></p>
<p>“vnorm” normalizes a vector to unit length; used as a
service routine by the Connolly surface area and volume
computation</p>
<p><strong>VOLUME Subroutine</strong></p>
<p>“volume” calculates the excluded volume via the Connolly
analytical volume and surface area algorithm</p>
<p><strong>VOLUME1 Subroutine</strong></p>
<p>“volume1” calculates first derivatives of the total excluded
volume with respect to the Cartesian coordinates of each atom</p>
<p><strong>VOLUME2 Subroutine</strong></p>
<p>“volume2” calculates second derivatives of the total excluded
volume with respect to the Cartesian coordinates of the atoms</p>
<p><strong>WATSON Subroutine</strong></p>
<p>“watson” uses a rigid body optimization to approximately
align the paired strands of a nucleic acid double helix</p>
<p><strong>WATSON1 Function</strong></p>
<p>“watson1” is a service routine that computes the energy
and gradient for optimally conditioned variable metric
optimization of rigid bodies</p>
<p><strong>WIGGLE Subroutine</strong></p>
<p>“wiggle” applies a random perturbation to the atomic coordinates
to avoid numerical instabilities for various linear, planar and
symmetric structures</p>
<p><strong>XTALERR Subroutine</strong></p>
<p>“xtalerr” computes an error function value derived from
lattice energies, dimer intermolecular energies and the
gradient with respect to structural parameters</p>
<p><strong>XTALFIT Program</strong></p>
<p>“xtalfit” determines optimized van der Waals and electrostatic
parameters by fitting to crystal structures, lattice energies,
and dimer structures and interaction energies</p>
<p><strong>XTALMIN Program</strong></p>
<p>“xtalmin” performs a full crystal energy minimization by
optimizing over fractional atomic coordinates and the six
lattice lengths and angles</p>
<p><strong>XTALMIN1 Function</strong></p>
<p>“xtalmin1” is a service routine that computes the energy and
gradient with respect to fractional coordinates and lattice
dimensions for a crystal energy minimization</p>
<p><strong>XTALMOVE Subroutine</strong></p>
<p>“xtalmove” converts fractional to Cartesian coordinates for
rigid molecules during optimization of force field parameters</p>
<p><strong>XTALPRM Subroutine</strong></p>
<p>“xtalprm” stores or retrieves a molecular structure; used to
make a previously stored structure the active structure, or to
store a structure for later use</p>
<p><strong>XTALWRT Subroutine</strong></p>
<p>“xtalwrt” prints intermediate results during fitting of
force field parameters to structures and energies</p>
<p><strong>XYZATM Subroutine</strong></p>
<p>“xyzatm” computes the Cartesian coordinates of a single
atom from its defining internal coordinate values</p>
<p><strong>XYZEDIT Program</strong></p>
<p>“xyzedit” provides for modification and manipulation
of the contents of Cartesian coordinates files</p>
<p><strong>XYZINT Program</strong></p>
<p>“xyzint” takes as input a Cartesian coordinates file, then
converts to and writes out an internal coordinates file</p>
<p><strong>XYZMOL2 Program</strong></p>
<p>“xyzmol2” takes as input a Cartesian coordinates file,
converts to and then writes out a Tripos MOL2 file</p>
<p><strong>XYZPDB Program</strong></p>
<p>“xyzpdb” takes as input a Cartesian coordinates file,
then converts to and writes out a Protein Data Bank file</p>
<p><strong>XYZRIGID Subroutine</strong></p>
<p>“xyzrigid” computes the center of mass and Euler angle rigid
body coordinates for each atom group in the system</p>
<p><strong>ZATOM Subroutine</strong></p>
<p>“zatom” adds an atom to the end of the current Z-matrix
and then increments the atom counter; atom type, defining
atoms and internal coordinates are passed as arguments</p>
<p><strong>ZHELP Subroutine</strong></p>
<p>“zhelp” prints the general information and instructions
for the Z-matrix editing program</p>
<p><strong>ZVALUE Subroutine</strong></p>
<p>“zvalue” gets user supplied values for selected coordinates
as needed by the internal coordinate editing program</p>
</div>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="main navigation">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="../index.html">Tinker User's Guide</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="introduction.html">Introduction to the Software</a></li>
<li class="toctree-l1"><a class="reference internal" href="installation.html">Installation on Your Computer</a></li>
<li class="toctree-l1"><a class="reference internal" href="file-types.html">Types of Input &amp; Output Files</a></li>
<li class="toctree-l1"><a class="reference internal" href="energy-programs.html">Potential Energy Programs</a></li>
<li class="toctree-l1"><a class="reference internal" href="analysis-programs.html">Analysis &amp; Utility Programs &amp; Scripts</a></li>
<li class="toctree-l1"><a class="reference internal" href="parameters.html">Force Field Parameter Sets</a></li>
<li class="toctree-l1"><a class="reference internal" href="special-features.html">Special Features &amp; Methods</a></li>
<li class="toctree-l1"><a class="reference internal" href="keywords.html">Use of the Keyword Control File</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Routines &amp; Functions</a></li>
<li class="toctree-l1"><a class="reference internal" href="modules.html">Modules &amp; Global Variables</a></li>
<li class="toctree-l1"><a class="reference internal" href="test-cases.html">Test Cases &amp; Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="benchmarks.html">Benchmark Results</a></li>
<li class="toctree-l1"><a class="reference internal" href="acknowledgements.html">Acknowledgments</a></li>
<li class="toctree-l1"><a class="reference internal" href="references.html">References</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="../index.html">Documentation overview</a><ul>
      <li>Previous: <a href="keywords.html" title="previous chapter">Use of the Keyword Control File</a></li>
      <li>Next: <a href="modules.html" title="next chapter">Modules &amp; Global Variables</a></li>
  </ul></li>
</ul>
</div>
<div id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="../search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" />
      <input type="submit" value="Go" />
    </form>
    </div>
</div>
<script>$('#searchbox').show(0);</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &copy;2020, Jay Ponder.
      
      |
      Powered by <a href="http://sphinx-doc.org/">Sphinx 3.0.3</a>
      &amp; <a href="https://github.com/bitprophet/alabaster">Alabaster 0.7.12</a>
      
      |
      <a href="../_sources/text/routines.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>