%% Generated by Sphinx.
\def\sphinxdocclass{report}
\documentclass[letterpaper,11pt,english]{sphinxmanual}
\ifdefined\pdfpxdimen
   \let\sphinxpxdimen\pdfpxdimen\else\newdimen\sphinxpxdimen
\fi \sphinxpxdimen=.75bp\relax

\PassOptionsToPackage{warn}{textcomp}
\usepackage[utf8]{inputenc}
\ifdefined\DeclareUnicodeCharacter
% support both utf8 and utf8x syntaxes
  \ifdefined\DeclareUnicodeCharacterAsOptional
    \def\sphinxDUC#1{\DeclareUnicodeCharacter{"#1}}
  \else
    \let\sphinxDUC\DeclareUnicodeCharacter
  \fi
  \sphinxDUC{00A0}{\nobreakspace}
  \sphinxDUC{2500}{\sphinxunichar{2500}}
  \sphinxDUC{2502}{\sphinxunichar{2502}}
  \sphinxDUC{2514}{\sphinxunichar{2514}}
  \sphinxDUC{251C}{\sphinxunichar{251C}}
  \sphinxDUC{2572}{\textbackslash}
\fi
\usepackage{cmap}
\usepackage[T1]{fontenc}
\usepackage{amsmath,amssymb,amstext}
\usepackage{babel}



\usepackage{times}
\expandafter\ifx\csname T@LGR\endcsname\relax
\else
% LGR was declared as font encoding
  \substitutefont{LGR}{\rmdefault}{cmr}
  \substitutefont{LGR}{\sfdefault}{cmss}
  \substitutefont{LGR}{\ttdefault}{cmtt}
\fi
\expandafter\ifx\csname T@X2\endcsname\relax
  \expandafter\ifx\csname T@T2A\endcsname\relax
  \else
  % T2A was declared as font encoding
    \substitutefont{T2A}{\rmdefault}{cmr}
    \substitutefont{T2A}{\sfdefault}{cmss}
    \substitutefont{T2A}{\ttdefault}{cmtt}
  \fi
\else
% X2 was declared as font encoding
  \substitutefont{X2}{\rmdefault}{cmr}
  \substitutefont{X2}{\sfdefault}{cmss}
  \substitutefont{X2}{\ttdefault}{cmtt}
\fi


\usepackage[Bjarne]{fncychap}
\usepackage{sphinx}

\fvset{fontsize=\small}
\usepackage{geometry}


% Include hyperref last.
\usepackage{hyperref}
% Fix anchor placement for figures with captions.
\usepackage{hypcap}% it must be loaded after hyperref.
% Set up styles of URL: it should be placed after hyperref.
\urlstyle{same}

\usepackage{sphinxmessages}
\setcounter{tocdepth}{1}


        \usepackage{charter}
        \usepackage[defaultsans]{lato}
        \usepackage{inconsolata}
    

\title{Tinker User\textquotesingle{}s Guide}
\date{May 17, 2020}
\release{}
\author{TinkerTools Organization}
\newcommand{\sphinxlogo}{\vbox{}}
\renewcommand{\releasename}{}
\makeindex
\begin{document}

\pagestyle{empty}
\sphinxmaketitle
\pagestyle{plain}
\sphinxtableofcontents
\pagestyle{normal}
\phantomsection\label{\detokenize{index::doc}}



\chapter{Introduction to the Software}
\label{\detokenize{text/introduction:introduction-to-the-software}}\label{\detokenize{text/introduction::doc}}

\section{What is the Tinker Software?}
\label{\detokenize{text/introduction:what-is-the-tinker-software}}
Welcome to the Tinker molecular modeling package! Tinker is designed to be an easily used and flexible system of programs and routines for molecular mechanics and dynamics as well as other energy\sphinxhyphen{}based and structural manipulation calculations. It is intended to be modular enough to enable development of new computational methods and efficient enough to meet most production calculation needs. Rather than incorporating all the functionality in one monolithic program, Tinker provides a set of relatively small programs that interoperate to perform complex computations. New programs can be easily added by modelers with only limited programming experience.


\section{Features and Capabilities}
\label{\detokenize{text/introduction:features-and-capabilities}}
The series of major programs included in the distribution system perform the following core tasks:
\begin{enumerate}
\sphinxsetlistlabels{\arabic}{enumi}{enumii}{(}{)}%
\item {} 
building protein and nucleic acid models from sequence

\item {} 
energy minimization and structural optimization

\item {} 
analysis of energy distribution within a structure

\item {} 
molecular dynamics and stochastic dynamics

\item {} 
simulated annealing with a choice of cooling schedules

\item {} 
normal modes and vibrational frequencies

\item {} 
conformational search and global optimization

\item {} 
transition state location and conformational pathways

\item {} 
fitting of energy parameters to crystal data

\item {} 
distance geometry with pairwise metrization

\item {} 
molecular volumes and surface areas

\item {} 
free energy changes for structural mutations

\item {} 
advanced algorithms based on potential smoothing

\end{enumerate}

Many of the various energy minimization and molecular dynamics computations can be performed on full or partial structures, over Cartesian, internal or rigid body coordinates, and including a variety of boundary conditions and crystal cell types. Other programs are available to generate timing data and allow checking of potential function derivatives for coding errors. Special features are available to facilitate input and output of protein and nucleic acid structures. However, the basic core routines have no knowledge of biopolymer structure and can be used for general molecular systems.

Due to its emphasis on ease of modification, Tinker differs from many other currently available molecular modeling packages in that the user is expected to be willing to write simple {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}front\sphinxhyphen{}end’’ programs and make some alterations at the source code level. The main programs provided should be considered as templates for the users to change according to their wishes. All subroutines are internally documented and structured programming practices are adhered to throughout. The result, it is hoped, will be a calculational system which can be tailored to local needs and desires.

The core Tinker system consists of over 240,000 lines of source written entirely in a portable Fortran superset. Use is made of only some very common extensions that aid in writing highly structured code. The current version of the package has been ported to a wide range of computers with no or extremely minimal changes. Tested systems include: Ubuntu, CentOS and Red Hat Linux, Microsoft Windows 10 and earlier, Apple MacOS, and various older Unix\sphinxhyphen{}based workstations under vendor supplied Unix. At present, our new code is written on various Linux platforms, and occasionally tested for compatibility on various of the other machine and OS combinations listed above. At present, our primary source code development efforts are in Fortran, using a portable subset of Fortran90 with some common extensions. A machine\sphinxhyphen{}translated C version of Tinker is currently available, and a hand\sphinxhyphen{}translated optimized C version of a previous Tinker release is available for inspection. Conversion to C or C++ is under consideration, but not being actively pursued at this time.

The basic design of the energy function engine used by the Tinker system allows usage of several different parameter sets. At present we are distributing parameters that implement several Amber and CHARMM potentials, MM2, MM3, OPLS\sphinxhyphen{}UA, OPLS\sphinxhyphen{}AA, MMFF, Liam Dang’s polarizable potentials, and our own AMOEBA (Atomic Multipole Optimized Energetics for Biomolecular Applications), AMOEBA+, and HIPPO (Hydrogen\sphinxhyphen{}like Intermolecular Polarizable Potential) force fields. In most cases, the source code separates the geometric manipulations needed for energy derivatives from the actual form of the energy function itself. Several other literature parameter sets are being considered for possible future development (later versions of CHARMM and Amber, as well as GROMOS, ENCAD, MM4, UFF, etc.), and many of the alternative potential function forms reported in the literature can be implemented directly or after minor code changes.

Much of the software in the Tinker package has been heavily used and well tested, but some modules are still in a fairly early stage of development. Further work on the Tinker system is planned in three main areas: (1) extension and improvement of the potential energy parameters including additional parameterization and testing of our polarizable multipole AMOEBA force field, (2) coding of new computational algorithms including additional methods for free energy determination, torsional Monte Carlo and molecular dynamics sampling, advanced methods for long range interactions, better transition state location, and further application of the potential smoothing paradigm, and (3) further development of Force Field Explorer, a Java\sphinxhyphen{}based GUI front\sphinxhyphen{}end to the Tinker programs that provides for calculation setup, launch and control as well as basic molecular visualization.


\section{Contact Information}
\label{\detokenize{text/introduction:contact-information}}
Questions and comments regarding the Tinker package, including suggestions for improvements and changes should be made to the author:
\begin{quote}

Professor Jay William Ponder
Department of Chemistry, Box 1134
Washington University in Saint Louis
One Brookings Hall
Saint Louis, MO 63130 U.S.A.

office: Louderman Hall, Room 453
phone:  (314) 935\sphinxhyphen{}4275
fax:    (314) 935\sphinxhyphen{}4481
email:  \sphinxhref{mailto:ponder@dasher.wustl.edu}{ponder@dasher.wustl.edu}
\end{quote}

In addition, an Internet web site containing an online version of this User’s Guide, the most recent distribution version of the full Tinker package and other useful information can be found at \sphinxurl{https://dasher.wustl.edu/tinker/}, the Home Page for the Tinker Molecular Modeling Package. Tinker and related software packages are also available from GitHub at the site \sphinxurl{https://github.com/TinkerTools/Tinker.git/}.


\chapter{Installation on Your Computer}
\label{\detokenize{text/installation:installation-on-your-computer}}\label{\detokenize{text/installation::doc}}

\section{How to Obtain a Copy of Tinker}
\label{\detokenize{text/installation:how-to-obtain-a-copy-of-tinker}}
The Tinker package is distributed on the Internet via either the web site or the anonymous ftp account on dasher.wustl.edu with an IP number of 128.252.208.48. This node is a web and file server located in the Ponder lab at Washington University School of Medicine. The package is available via the web and standard browsers from the Tinker home page at \sphinxurl{http://dasher.wustl.edu/Tinker/}. Alternatively Tinker can be downloaded by logging into dasher.wustl.edu via anonymous ftp (Username: anonymous, Password: “your email address”) and downloading the software from the /pub/Tinker subdirectory. The complete Tinker distributions as well as individual files can be downloaded from this site.

The easiest way to get Tinker running on your machine is to use the self\sphinxhyphen{}extracting installation kit for either Linux, Windows, or Macintosh OS X 10.3. The installer will guide you through complete setup of Tinker and the Force Field Explorer (FFE) GUI, and perform all required configuration chores. The installer kits for the three supported systems are Tinker4.2\sphinxhyphen{}linux.sh, Tinker4.2\sphinxhyphen{}windows.exe and Tinker4.2\sphinxhyphen{}macosx.sit. The Linux and Windows kits each contain a private copy of a Java and Java3D run\sphinxhyphen{}time environment for use with the package. The Macintosh version requires an OS X 10.3 (Panther) system for installation. The native Java implementation is used on Macs, and the Java3D package must be downloaded from Apple and installed prior to using Tinker with Force Field Explorer.


\section{Prebuilt Tinker Executables}
\label{\detokenize{text/installation:prebuilt-tinker-executables}}
The Tinker package is also available as compressed Unix tar archives, Windows zip files, and as a complete set of uncompressed source and data files. Binaries are provided for machines running Windows 9X/ME/NT/2000/XP, Linux, and Apple MacOS. All of these executables are present in standard compressed formats as individual programs or as complete sets of executables. It is expected that other Unix users and PC users who need specially customized versions, will build binaries for their specific system. Sites with access to the Unix tar, compress and uncompress commands should simply obtain the archive file Tinker.tar.Z. Alternatively, Tinker.tar.gz and Tinker.zip containing identical distributions compressed to GNU gzip and Windows ZIP format are also provided. If you choose to download individual files, you will need at a minimum the contents of the /doc, /source and /params subdirectories. Also required are the compile/build scripts from the subdirectory named for your machine type. Other areas contain test cases and examples, benchmark results, machine\sphinxhyphen{}translated C code, and the Force Field Explorer Java GUI for Tinker. The entire Tinker package, after building the executables, will require from about 40 to over 150 megabytes of disk space depending on the components installed and the use of shared libraries in the executables.


\section{Building your Own Executables}
\label{\detokenize{text/installation:building-your-own-executables}}
The compilation and building of the Tinker executables should be easy for most of the common workstation and PC class computers. We provide in the /make area a Unix\sphinxhyphen{}style Makefile that with some modification can be used to build Tinker on most Unix machines. As a simpler alternative to Makefiles for the Unix versions, we also provide machine\sphinxhyphen{}specific directories with three separate shell scripts to compile the source, build an object library, and link binary executables. Three similar command files are provided for Windows, Macintosh and Open VMS systems. Compilation on Unix workstations should use the vendor supplied Fortran compiler, if available. The public domain GNU g77 Fortran compiler available from \sphinxurl{http://gcc.gnu.org/} is also capable of building Tinker on Linux and other Unix\sphinxhyphen{}based machines. The Linux executables we provide are built with the Intel Fortran for Linux 8.0 compiler. The Portland Group (PGI) and Absoft ProFortran compilers have also been tested under Linux, both of which generate executables roughly comparable in speed to the Intel compiler. On Linux, the g77 executables tend to exhibit degraded performance compared with executables from commercial compilers. Some benchmark results are provided in a later section of this User’s Guide For the Macintosh we distribute executables built under Apple OS X 10.3 with the GNU g77 compiler. Tinker also builds on the Macintosh using the Absoft ProFortran compiler. For PCs running Windows 9X/NT/2000/XP, the distributed Tinker executables are built under the Intel Fortran for Windows 8.0 compiler. Alternative Windows compilers such as Compaq Visual Fortran, Lahey/Fujitsu and The Portland Group compilers, and GNU g77 under Cygwin have been tested and shown to build Tinker correctly. Please see the README files in each of the machine\sphinxhyphen{}specific areas for further information.

The first step in building Tinker using the script files is to run the appropriate compile.make script for your operating system and compiler version. Next you must use a library.make script to create an archive of object code modules. Finally, run a link.make script to produce the complete set of Tinker executables. The executables can be renamed and moved to wherever you like by editing and running the {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}rename’’ script. These steps will produce executables that can run from the command line, but without the capability to interact with the FFE GUI. Building FFE\sphinxhyphen{}enabled Tinker executables involves replacing the sockets.f source file with sockets.c, and included the object from the C code in the Tinker object library. Then executables must be linked against Java libraries in addition to the usual resources. Sample compgui.make and linkgui.make scripts are provided for systems capable of building GUI\sphinxhyphen{}enabled executables.

Regardless of your target machine, only a few small pieces of code can possibly require attention prior to building. The first two are the system dependent time and date routines found in clock.f and calendar.f respectively. Next is the openend.f routine that facilitates appending data to the end of an existing disk file. Please uncomment the sections of these routines needed for your computer type. Version of these system dependent routines suitable for each system are also provided in the directory for each machine/OS type. The final set of possible source alterations are to the master array dimensions found in the include file sizes.i. The most basic limit is on the number of atoms allowed, {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}maxatm’’. This parameter can be set to 10000 or more on most workstations. Personal computers with minimal memory may need a lower limit, perhaps 1000 atoms, depending on available memory, swap space and other resources. A description of the other parameter values is contained in the header of the file. Note that in order to keep the code completely transparent, Tinker does not implement any sort of dynamic memory allocation or heap data structure. This requires that sizes.i dimensioning values be set at least as large as the biggest problem you intend to run. Obviously, you should not set the array sizes to unnecessarily large values, since this can tax your compute resources and may result in performance degradation or overt failure of the executables.


\section{Documentation and Other Information}
\label{\detokenize{text/installation:documentation-and-other-information}}
The documentation for the Tinker programs, including the guide you are currently reading, is located in the /pub/Tinker/doc subdirectory. The documentation was prepared using the Applixware Words and Graphics programs. Portable versions of the documentation are provided as ascii text in .txt files and in Adobe Acrobat .pdf file formats. Please read and return by mail the Tinker license. In particular, we note that Tinker is not “Open Source” as users are prohibited from redistribution of original or modified Tinker source code or binaries to other parties. While our intent is to distribute the Tinker code to anyone who wants it, the Ponder Lab would like to remain the sole distribution site and keep track of researchers using the package. The returned license forms also help us justify further development of Tinker. When new modules and capabilities become available, and when the almost inevitable bugs are uncovered, we will attempt to notify those who have returned a license form. Finally, we remind you that this software is copyrighted, and ask that it not be redistributed in any form.


\section{Where to Direct Questions}
\label{\detokenize{text/installation:where-to-direct-questions}}
Specific questions about the building or use of the Tinker package should be directed to \sphinxhref{mailto:ponder@dasher.wustl.edu}{ponder@dasher.wustl.edu}. Tinker related questions or comments of more general interest can be sent to the Computational Chemistry List (\sphinxurl{http://www.ccl.net/}). The Tinker developers monitor this list and will respond to the list or the individual poster as appropriate.


\chapter{Types of Input \& Output Files}
\label{\detokenize{text/file-types:types-of-input-output-files}}\label{\detokenize{text/file-types::doc}}
This section describes the basic file types used by the Tinker package. Let’s say you wish to perform a calculation on a particular small organic molecule. Assume that the file name chosen for our input and output files is sample. Then all of the Tinker files will reside on the computer under the name sample.xxx where .xxx is any of the several extension types to be described below.

\sphinxstylestrong{SAMPLE.XYZ}

The .xyz file is the basic Tinker Cartesian coordinates file type. It contains a title line followed by one line for each atom in the structure. Each line contains: the sequential number within the structure, an atomic symbol or name, X\sphinxhyphen{}, Y\sphinxhyphen{}, and Z\sphinxhyphen{}coordinates, the force field atom type number of the atom, and a list of the atoms connected to the current atom. Except for programs whose basic operation is in torsional space, all Tinker calculations are done from some version of the .xyz format.

\sphinxstylestrong{SAMPLE.INT}

The .int file contains an internal coordinates representation of the molecular structure. It consists of a title line followed by one line for each atom in the structure. Each line contains: the sequential number within the structure, an atomic symbol or name, the force field atom type number of the atom, and internal coordinates in the usual Z\sphinxhyphen{}matrix format. For each atom the internal coordinates consist of a distance to some previously defined atom, and either two bond angles or a bond angle and a dihedral angle to previous atoms. The length, angle and dihedral definitions do not have to represent real bonded interactions. Following the last atom definition are two optional blank line separated sets of atom number pairs. The first list contains pairs of atoms that are covalently bonded, but whose bond length was not used as part of the atom definitions. These pairs are typically used to close ring structures. The second list contains {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}bonds’’ that are to be broken, i.e., pairs of atoms that are not covalently bonded, but which were used to define a distance in the atom definitions.

\sphinxstylestrong{SAMPLE.KEY}

The keyword parameter file always has the extension .key and is optionally present during Tinker calculations. It contains values for any of a wide variety of switches and parameters that are used to change the course of the computation from the default. The detailed contents of this file is explained in a latter section of this User’s Guide. If a molecular system specific keyfile, in this case sample.key, is not present, the the Tinker program will look in the same directory for a generic file named Tinker.key.

\sphinxstylestrong{SAMPLE.DYN}

The .dyn file contains values needed to restart a molecular or stochastic dynamics computation. It stores the current position, current velocity and current and previous accelerations for each atom, as well as the size and shape of any periodic box or crystal unit cell. This information can be used to start a new dynamics run from the final state of a previous run. Upon startup, the dynamics programs always check for the presence of a .dyn file and make use of it whenever possible. The .dyn file is updated concurrent with the saving of a new dynamics trajectory snapshot.

\sphinxstylestrong{SAMPLE.END}

The .end file type provides a mechanism to gracefully stop a running Tinker calculation. At appropriate checkpoints during a calculation, Tinker will test for the presence of a sample.end file, and if found will terminate the calculation after updating the output. The .end file can be created at any time during a computation, and will be detected when the next checkpoint is reached. The file may be of zero size, and its contents are unimportant. In the current version of Tinker, the .end mechanism is only available within dynamics\sphinxhyphen{}based programs.

\sphinxstylestrong{SAMPLE.001, SAMPLE.002, ….}

Several types of computations produce files containing a three or more digit extension (.001 as shown; or .002, .137, .5678, etc.). These are referred to as cycle files, and are used to store various types of output structures. The cycle files from a given computation are identical in internal structure to either the .xyz or .int files described above. For example, the vibrational analysis program can save the tenth normal mode in sample.010. A molecular dynamics\sphinxhyphen{}based program might save its tenth 0.1 picosecond frame (or an energy minimizer its tenth partially minimized intermediate) in a file of the same name.

\sphinxstylestrong{SAMPLE.LOG}

The Force Field Explorer interface to Tinker saves results of all calculations launched from the GUI to a log file with the .log suffix. Any output that would normally be directed to the screen after starting a program from the command line is appended to this log file by Force Field Explorer.

\sphinxstylestrong{SAMPLE.ARC}

A Tinker archive file is simply a series of .xyz Cartesian coordinate files appended together one after another. This file can be used to condense the results from intermediate stages of an optimization, frames from a molecular dynamics trajectory, or set of normal mode vibrations into a single file for storage. Tinker archive files can be displayed as sequential frame “movies” by the Force Field Explorer modeling program.

\sphinxstylestrong{SAMPLE.PDB}

This file type contains coordinate information in the PDB format developed by the Brookhaven Protein Data Bank for deposition of model structures based on macromolecular X\sphinxhyphen{}ray diffraction and NMR data. Although Tinker itself does not use .pdb files directly for input/output, auxiliary programs are provided with the system for interconverting .pdb files with the .xyz format described above.

\sphinxstylestrong{SAMPLE.SEQ}

This file type contains the primary sequence of a biopolymer in the standard one\sphinxhyphen{}letter code with 50 residues per line. The .seq file for a biopolymer is generated automatically when a PDB file is converted to Tinker .xyz format or when using the PROTEIN or NUCLEIC programs to build a structure from sequence It is required for the reverse conversion of a Tinker file back to PDB format..

\sphinxstylestrong{SAMPLE.FRAC}

The fractional coordinates corresponding to the asymmetric unit of a crystal unit cell are stored in the .frac file. The internal format of this file is identical to the .xyz file; except that the coordinates are fractional instead of in Angstrom units.

\sphinxstylestrong{SAMPLE.MOL2}

File conversion to and from the Tripos Sybyl MOL2 file format is supported by Tinker. The utility programs XYZMOL2 and MOL2XYZ transform a Tinker XYZ file to MOL2 format, and the reverse.

\sphinxstylestrong{PARAMETER FILES (*.PRM)}

The potential energy parameter files distributed with the Tinker package all end in the extension .prm, although this is not required by the programs themselves. Each of these files contains a definition of the potential energy functional forms for that force field as well as values for individual energy parameters. For example, the mm3pro.prm file contains the energy parameters and definitions needed for a protein\sphinxhyphen{}specific version of the MM3 force field.


\chapter{Potential Energy Programs}
\label{\detokenize{text/energy-programs:potential-energy-programs}}\label{\detokenize{text/energy-programs::doc}}
This section of the manual contains a brief description of each of the Tinker potential energy programs. A detailed example showing how to run each program is included in a later section. The programs listed below are all part of the main, supported distribution. Additional source code for various unsupported programs can be found in the /other directory of the Tinker distribution.

\sphinxstylestrong{ALCHEMY}

A simple program to perform very basic free energy perturbation calculations. This program is provided mostly for demonstration purposes.  For example, we use ALCHEMY in a molecular modeling course laboratory exercise to perform such classic mutations as chloride to bromide and ethane to methanol in water. The present version uses the perturbation formula and windowing with an explicit mapping of atoms involved in the mutation (“Amber”\sphinxhyphen{}style), instead of thermodynamic integration and independent freely propagating groups of mutated atoms (“CHARMM”\sphinxhyphen{}style). Some of the code specific to this program is limited to the Amber and OPLS potential functional forms, but could be easily generalized to handle other potentials. A more general and sophisticated version is currently under development.

\sphinxstylestrong{ANALYZE}

Provides information about a specific molecular structure. The program will ask for the name of a structure file, which must be in the Tinker XYZ file format, and the type of analysis desired. Options allow output of:  (1) total potential energy of the system, (2) breakdown of the energy by potential function type or over individual atoms, (3) computation of the total dipole moment and its components, moments of inertia and radius of gyration, (4) listing of the parameters used to compute selected interaction energies, (5) energies associated with specified individual interactions.

\sphinxstylestrong{ANNEAL}

Performs a molecular dynamics simulated annealing computation. The program starts from a specified input molecular structure in Tinker XYZ format. The trajectory is updated using either a modified Beeman or a velocity Verlet integration method. The annealing protocol is implemented by allowing smooth changes between starting and final values of the system temperature via the Groningen method of coupling to an external bath. The scaling can be linear or sigmoidal in nature. In addition, parameters such as cutoff distance can be transformed along with the temperature. The user must input the desired number of dynamics steps for both the equilibration and cooling phases, a time interval for the dynamics steps, and an interval between coordinate/trajectory saves. All saved coordinate sets along the trajectory are placed in sequentially numbered cycle files.

\sphinxstylestrong{DYNAMIC}

Performs a molecular dynamics (MD) or stochastic dynamics (SD) computation. Starts either from a specified input molecular structure (an XYZ file) or from a structure\sphinxhyphen{}velocity\sphinxhyphen{}acceleration set saved from a previous dynamics trajectory (a restart from a DYN file). MD trajectories are propagated using either a modified Beeman or a velocity Verlet integration method. SD is implemented via our own derivation of a velocity Verlet\sphinxhyphen{}based algorithm. In addition the program can perform full crystal calculations, and can operate in constant energy mode or with maintenance of a desired temperature and/or pressure using the Berendsen method of coupling to external baths. The user must input the desired number of dynamics steps, a time interval for the dynamics steps, and an interval between coordinate/trajectory saves. Coordinate sets along the trajectory can be saved as sequentially numbered cycle files or directly to a Tinker archive (ARC) file. At the same time that a point along the trajectory is saved, the complete information needed to restart the trajectory from that point is updated and stored in the DYN file.

\sphinxstylestrong{GDA}

A program to implement Straub’s Gaussian Density Annealing algorithm over an effective series of analytically smoothed potential energy surfaces. This method can be viewed as an extended stochastic version of the diffusion equation method of Scheraga, et al., and also has many similar features to the Tinker Potential Smoothing and Search (PSS) series of programs. The current version of GDA is similar to but does not exactly reproduce Straub’s published method and is limited to argon clusters and other simple systems involving only van der Waals interactions; further modification and development of this code is currently underway in the Ponder research group. As with other programs involving potential smoothing, GDA currently requires use of the smooth.prm force field parameters.

\sphinxstylestrong{MINIMIZE}

The MINIMIZE program performs a limited memory L\sphinxhyphen{}BFGS minimization of an input structure over Cartesian coordinates using a modified version of the algorithm of Jorge Nocedal. The method requires only the potential energy and gradient at each step along the minimization pathway. It requires storage space proportional to the number of atoms in the structure. The MINIMIZE procedure is recommended for preliminary minimization of trial structures to an RMS gradient of 1.0 to 0.1 kcal/mole/Ang. It has a relatively fast cycle time and is tolerant of poor initial structures, but converges in a slow, linear fashion near the minimum. The user supplies the name of the Tinker XYZ coordinates file and a target rms gradient value at which the minimization will terminate. Output consists of minimization statistics written to the screen or redirected to an output file, and the new coordinates written to updated XYZ files or to cycle files.

\sphinxstylestrong{MINIROT}

The MINIROT program uses the same limited memory L\sphinxhyphen{}BFGS method as MINIMIZE, but performs the computation in terms of dihedral angles instead of Cartesian coordinates. Output is saved in an updated .int file or in cycle files.

\sphinxstylestrong{MINRIGID}

The MINRIGID program is similar to MINIMIZE except that it operates on rigid bodies starting from a Tinker XYZ coordinate file and the rigid body group definitions found in the corresponding KEY file. Output is saved in an updated XYZ file or in cycle files.

\sphinxstylestrong{MONTE}

The MONTE program implements the Monte Carlo Minimization algorithm developed by Harold Scheraga’s group and others. The procedure takes Monte Carlo steps for either a single atom or a single torsional angle, then performs a minimization before application of the Metropolis sampling method. This results in effective sampling of a modified potential surface where the only possible energy levels are those of local minima on the original surface. The program can be easily modified to elaborate on the available move set.

\sphinxstylestrong{NEWTON}

A truncated Newton minimization method which requires potential energy, gradient and Hessian information. This procedure has significant advantages over standard Newton methods, and is able to minimize very large structures completely. Several options are provided with respect to minimization method and preconditioning of the Newton equations. The default options are recommended unless the user is familiar with the math involved. This program operates in Cartesian coordinate space and is fairly tolerant of poor input structures. Typical algorithm iteration times are longer than with nonlinear conjugate gradient or variable metric methods, but many fewer iterations are required for complete minimization. NEWTON is usually the best choice for minimizations to the 0.01 to 0.000001 kcal/mole/Ang level of RMS gradient convergence. Tests for directions of negative curvature can be removed, allowing NEWTON to be used for optimization to conformational transition state structures (this only works if the starting point is very close to the transition state). Input consists of a Tinker XYZ coordinates file; output is an updated set of minimized coordinates and minimization statistics.

\sphinxstylestrong{NEWTROT}

The NEWTROT program is similar to NEWTON except that it requires a .int file as input and then operates in terms of dihedral angles as the minimization variables. Since the dihedral space Hessian matrix of an arbitrary structure is often indefinite, this method will often not perform as well as the other, simpler dihedral angle based minimizers.

\sphinxstylestrong{OPTIMIZE}

The OPTIMIZE program performs a optimally conditioned variable metric minimization of an input structure over Cartesian coordinates using an algorithm due to William Davidon. The method does not perform line searches, but requires computation of energies and gradients as well as storage for an estimate of the inverse Hessian matrix. The program operates on Cartesian coordinates from a Tinker XYZ file. OPTIMIZE will typically converge somewhat faster and more completely than MINIMIZE. However, the need to store and manipulate a full inverse Hessian estimate limits its use to structures containing less than a few hundred atoms on workstation class machines. As with the other minimizers, OPTIMIZE needs input coordinates and an rms gradient cutoff criterion. The output coordinates are saved in updated .xyz files or as cycle files.

\sphinxstylestrong{OPTIROT}

The OPTIROT program is similar to OPTIMIZE except that it operates on dihedral angles starting from a Tinker INT internal coordinate file. This program is usually the preferred method for most dihedral angle optimization problems since Truncated Newton methods appear, in our hands, to lose some of their efficacy in moving from Cartesian to torsional coordinates.

\sphinxstylestrong{OPTRIGID}

The OPTRIGID program is similar to OPTIMIZE except that it operates on rigid bodies starting from a Tinker XYZ coordinate file and the rigid body atom group definitions found in the corresponding KEY file. Output is saved in an updated XYZ file or in cycle files.

\sphinxstylestrong{PATH}

A program that implements a variant of Elber’s Lagrangian multiplier\sphinxhyphen{}based reaction path following algorithm. The program takes as input a pair of structural minima as Tinker XYZ files, and then generates a user specified number of points along a path through conformational space connecting the input structures. The intermediate structures are output as Tinker cycle files, and the higher energy intermediates can be used as input to a Newton\sphinxhyphen{}based optimization to locate conformational transition states.

\sphinxstylestrong{PSS}

Implements our version of a potential smoothing and search algorithm for the global optimization of molecular conformation. An initial structure in .xyz format is first minimized in Cartesian coordinates on a series of increasingly smoothed potential energy surfaces. Then the smoothing procedure is reversed with minimization on each successive surface starting from the coordinates of the minimum on the previous surface. A local search procedure is used during the backtracking to explore for alternative minima better than the one found during the current minimization. The final result is usually a very low energy conformation or, in favorable cases, the global energy minimum conformation. The minimum energy coordinate sets found on each surface during both the forward smoothing and backtracking procedures are placed in sequentially numbered cycle files.

\sphinxstylestrong{PSSRIGID}

This program implements the potential smoothing and search method as described above for the PSS program, but performs the computation in terms of keyfile\sphinxhyphen{}defined rigid body atom groups instead of Cartesian coordinates. Output is saved in numbered cycle files with the XYZ file format.

\sphinxstylestrong{PSSROT}

This program implements the potential smoothing and search method as described above for the PSS program, but performs the computation in terms of a set of user\sphinxhyphen{}specified dihedral angles instead of Cartesian coordinates. Output is saved in numbered cycle files with the INT file format.

\sphinxstylestrong{SADDLE}

A program for the location of a conformational transition state between two potential energy minima. SADDLE uses a conglomeration of ideas from the Bell\sphinxhyphen{}Crighton quadratic path and the Halgren\sphinxhyphen{}Lipscomb synchronous transit methods. The basic idea is to perform a nonlinear conjugate gradient optimization in a subspace orthogonal to a suitably defined reaction coordinate. The program requires as input the coordinates, as Tinker XYZ files, of the two minima and an rms gradient convergence criterion for the optimization. The current estimate of the transition state structure is written to the file TSTATE.XYZ. Crude transition state structures generated by SADDLE can sometimes be refined using the NEWTON program. Optionally, a scan of the interconversion pathway can be made at each major iteration.

\sphinxstylestrong{SCAN}

A program for general conformational search of an entire potential energy surface via a basin hopping method. The program takes as input a Tinker XYZ coordinates file which is then minimized to find the first local minimum for a search list. A series of activations along various normal modes from this initial minimum are used as seed points for additional minimizations. Whenever a previously unknown local minimum is located it is added to the search list. When all minima on the search list have been subjected to the normal mode activation without locating additional new minima, the program terminates. The individual local minima are written to cycle files as they are discovered. While the SCAN program can be used on standard undeformed potential energy surfaces, we have found it to be most useful for quickly “scanning” a smoothed energy surface to enumerate the major basins of attraction spaning the entire surface.

\sphinxstylestrong{SNIFFER}

A program that implements the Sniffer global optimization algorithm of Butler and Slaminka, a discrete version of Griewank’s global search trajectory method. The program takes an input Tinker XYZ coordinates file and shakes it vigorously via a modified dynamics trajectory before, hopefully, settling into a low lying minimum. Some trial and error is often required as the current implementation is sensitive to various parameters and tolerances that govern the computation. At present, these parameters are not user accessible, and must be altered in the source code. However, this method can do a good job of quickly optimizing conformation within a limited range of convergence.

\sphinxstylestrong{TESTGRAD}

The TESTGRAD program computes and compares the analytical and numerical first derivatives (i.e., the gradient vector) of the potential energy for a Cartesian coordinate input structure. The output can be used to test or debug the current potential or any added user defined energy terms.

\sphinxstylestrong{TESTHESS}

The TESTHESS program computes and compares the analytical and numerical second derivatives (i.e., the Hessian matrix) of the potential energy for a Cartesian coordinate input structure. The output can be used to test or debug the current potential or any added user defined energy terms.

\sphinxstylestrong{TESTLIGHT}

A program to compare the efficiency of different nonbonded neighbor methods for the current molecular system. The program times the computation of energy and gradient for the van der Waals and charge\sphinxhyphen{}charge electrostatic potential terms using a simple double loop over all interactions and using the Method of Lights algorithm to select neighbors. The results can be used to decide whether the Method of Lights has any CPU time advantage for the current structure. Both methods should give exactly the same answer in all cases, since the identical individual interactions are computed by both methods. The default double loop method is faster when cutoffs are not used, or when the cutoff sphere contains about half or more of the total system of unit cell. In cases where the cutoff sphere is much smaller than the system size, the Method of Lights can be much faster since it avoids unnecessary calculation of distances beyond the cutoff range.

\sphinxstylestrong{TESTROT}

The TESTROT program computes and compares the analytical and numerical first derivatives (i.e., the gradient vector) of the potential energy with respect to dihedral angles. Input is a Tinker INT internal coordinate file. The output can be used to test or debug the current potential functions or any added user defined energy terms.

\sphinxstylestrong{TIMER}

A simple program to provide timing statistics for energy function calls within the Tinker package. TIMER requires an input XYZ file and outputs the CPU time (or wall clock time, on some machine types) needed to perform a specified number of energy, gradient and Hessian evaluations.

\sphinxstylestrong{TIMEROT}

This program is similar to TIMER, only it operates over dihedral angles via input of a Tinker INT internal coordinate file. In the current version, the torsional Hessian is computed numerically from the analytical torsional gradient.

\sphinxstylestrong{VIBRATE}

A program to perform vibrational analysis by computing and diagonalizing the full Hessian matrix (i.e., the second partial derivatives) for an input structure (a Tinker XYZ file). Eigenvalues and eigenvectors of the mass weighted Hessian (i.e., the vibrational frequencies and normal modes) are also calculated. Structures corresponding to individual normal mode motions can be saved in cycle files.

\sphinxstylestrong{VIBROT}

The program VIBROT forms the torsional Hessian matrix via numerical differentiation of the analytical torsional gradient. The Hessian is then diagonalized and the eigenvalues are output. The present version does not compute the kinetic energy matrix elements needed to convert the Hessian into the torsional normal modes; this will be added in a later version. The required input is a Tinker INT internal coordinate file.

\sphinxstylestrong{XTALFIT}

The XTALFIT program is of use in the automated fitting of potential parameters to crystal structure and thermodynamic data. XTALFIT takes as input several crystal structures (Tinker XYZ files with unit cell parameters in corresponding KEY files) as well as information on lattice energies and dipole moments of monomers. The current version uses a nonlinear least squares optimization to fit van der Waals and electrostatic parameters to the input data. Bounds can be placed on the values of the optimization parameters.

\sphinxstylestrong{XTALMIN}

A program to perform full crystal minimizations. The program takes as input the structure coordinates and unit cell lattice parameters. It then alternates cycles of Newton\sphinxhyphen{}style optimization of the structure and conjugate gradient optimization of the crystal lattice parameters. This alternating minimization is slower than more direct optimization of all parameters at once, but is somewhat more robust in our hands. The symmetry of the original crystal is not enforced, so interconversion of crystal forms may be observed in some cases.


\chapter{Analysis \& Utility Programs \& Scripts}
\label{\detokenize{text/analysis-programs:analysis-utility-programs-scripts}}\label{\detokenize{text/analysis-programs::doc}}
This section of the manual contains a brief description of each of the Tinker structure manipulation, geometric calculation and auxiliary programs. A detailed example showing how to run each program is included in a later section. The programs listed below are all part of the main, supported distribution. Additional source code for various unsupported programs can be found in the /other directory of the Tinker distribution.

\sphinxstylestrong{ARCHIVE}

A program for concatenating Tinker cycle files into a single archive file; useful for storing the intermediate results of minimizations, dynamics trajectories, and so on. The program can also extract individual cycle files from a Tinker archive.

\sphinxstylestrong{CORRELATE}

A program to compute time correlation functions from collections of Tinker cycle files. Its use requires a user supplied function property that computes the value of the property for which a time correlation is desired for two input structures. A sample routine is supplied that computes either a velocity autocorrelation function or an rms structural superposition as a function of time. The main body of the program organizes the overall computation in an efficient manner and outputs the final time correlation function.

\sphinxstylestrong{CRYSTAL}

A program for the manipulation of crystal structures including interconversion of fractional and Cartesian coordinates, generation of the unit cell from an asymmetric unit, and building of a crystalline block of specified size via replication of a single unit cell. The present version can handle about 25 of the most common space groups, others can easily be added as needed by modification of the routine symmetry.

\sphinxstylestrong{DIFFUSE}

A program to compute the self\sphinxhyphen{}diffusion constant for a homogeneous liquid via the Einstein equation. A previously saved dynamics trajectory is read in and {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}unfolded’’ to reverse translation of molecules due to use of periodic boundary conditions. The average motion over all molecules is then used to compute the self\sphinxhyphen{}diffusion constant. While the current program assumes a homogeneous system, it should be easy to modify the code to handle diffusion of individual molecules or other desired effects.

\sphinxstylestrong{DISTGEOM}

A program to perform distance geometry calculations using variations on the classic metric matrix method. A user specified number of structures consistent with keyfile input distance and dihedral restraints is generated. Bond length and angle restraints are derived from the input structure. Trial distances between the triangle smoothed lower and upper bounds can be chosen via any of several metrization methods, including a very effective partial random pairwise scheme. The correct radius of gyration of the structure is automatically maintained by choosing trial distances from Gaussian distributions of appropriate mean and width. The initial embedded structures can be further refined against a geometric restraint\sphinxhyphen{}only potential using either a sequential minimization protocol or simulated annealing.

\sphinxstylestrong{DOCUMENT}

The DOCUMENT program is provided as a minimal listing and documentation tool. It operates on the Tinker source code, either individual files or the complete source listing produced by the command script listing.make, to generate lists of routines, common blocks or valid keywords. In addition, the program has the ability to output a formatted parameter listing from the standard Tinker parameter files.

\sphinxstylestrong{INTEDIT}

A program to allow interactive inspection and alteration of the internal coordinate definitions and values of a Tinker structure. If the structure is altered, the user has the option to write out a new internal coordinates file upon exit.

\sphinxstylestrong{INTXYZ}

A program to convert a Tinker .int internal coordinates formatted file into a Tinker .xyz Cartesian coordinates formatted file.

\sphinxstylestrong{MOL2XYZ}

A program for converting a Tripos Sybyl MOL2 file into a Tinker XYZ Cartesian coordinate file. The current version of the program does not attempt to convert the Sybyl atoms types into the active Tinker force field types, i.e., all atoms types are simply set to zero.

\sphinxstylestrong{NUCLEIC}

A program for automated building of nucleic acid structures. Upon interactive input of a nucleotide sequence with optional phosphate backbone angles, the program builds internal and Cartesian coordinates. Standard bond lengths and angles are used. Both DNA and RNA sequences are supported as are A\sphinxhyphen{}, B\sphinxhyphen{} and Z\sphinxhyphen{}form structures. Double helixes of complementary sequence can be automatically constructed via a rigid docking of individual strands.

\sphinxstylestrong{PDBXYZ}

A program for converting a Brookhaven Protein Data Bank file (a PDB file) into a Tinker .xyz Cartesian coordinate file. If the PDB file contains only protein/peptide amino acid residues, then standard protein connectivity is assumed, and transferred to the .xyz file. For non\sphinxhyphen{}protein portions of the PDB file, atom connectivity is determined by the program based on interatomic distances. The program also has the ability to add or remove hydrogen atoms from a protein as required by the force field specified during the computation.

\sphinxstylestrong{POLARIZE}

A program for computing molecular polarizability from an atom\sphinxhyphen{}based distributed model of polarizability. A damped interaction model due to Thole is optionally via keyfile settings. A Tinker .xyz file is required as input. The output consists of the overall polarizability tensor in the global coordinates and its eigenvalues.

\sphinxstylestrong{PRMEDIT}

A program for formatting and renumbering Tinker force field parameter files. When atom types or classes are added to a parameter file, this utility program has the ability to renumber all the atom records sequentially, and alter type and class numbers in all other parameter entries to maintain consistency.

\sphinxstylestrong{PROTEIN}

A program for automated building of peptide and protein structures. Upon interactive input of an amino acid sequence with optional phi/psi/omega/chi angles, D/L chirality, etc., the program builds internal and Cartesian coordinates. Standard bond lengths and angles are assumed for the peptide. The program will optionally convert the structure to a cyclic peptide, or add either or both N\sphinxhyphen{} and C\sphinxhyphen{}terminal capping groups. Atom type numbers are automatically assigned for the specified force field. The final coordinates and a sequence file are produced as the output.

\sphinxstylestrong{RADIAL}

A program to compute the pair radial distribution function between two atom types. The user supplies the two atom names for which the distribution function is to be computed, and the width of the distance bins for data analysis. A previously saved dynamics trajectory is read as input. The raw radial distribution and a spline smoothed version are then output from zero to a distance equal to half the minimum periodic box dimension. The atom names are matched to the atom name column of the Tinker .xyz file, independent of atom type.

\sphinxstylestrong{SPACEFILL}

A program to compute the volume and surface areas of molecules. Using a modified version of Connolly’s original analytical description of the molecular surface, the program determines either the van der Waals, accessible or molecular (contact/reentrant) volume and surface area. Both surface area and volume are broken down into their geometric components, and surface area is decomposed into the convex contribution for each individual atom. The probe radius is input as a user option, and atomic radii can be set via the keyword file. If Tinker archive files are used as input, the program will compute the volume and surface area of each structure in the input file.

\sphinxstylestrong{SPECTRUM}

A program to compute a power spectrum from velocity autocorrelation data. As input, this program requires a velocity autocorrelation function as produced by the CORRELATE program. This data, along with a user input time step, are Fourier transformed to generate the spectral intensities over a wavelength range. The result is a power spectrum, and the positions of the bands are those predicted for an infrared or Raman spectrum. However, the data is not weighted by molecular dipole moment derivatives as would be required to produce correct IR intensities.

\sphinxstylestrong{SUPERPOSE}

A program to superimpose two molecular structures in 3\sphinxhyphen{}dimensions. A variety of options for input of the atom sets to be used during the superposition are presented interactively to the user. The superposition can be mass\sphinxhyphen{}weighted if desired, and the coordinates of the second structure superimposed on the first structure are optionally output. If Tinker archive files are used as input, the program will compute all pairwise superpositions between structures in the input files.

\sphinxstylestrong{XYZEDIT}

A program that performs and of a variety of manipulations on an input Tinker .xyz Cartesian coordinates formatted file. The present version of the program has the following interactively selectable options: (1) Offset the Numbers of the Current Atoms, (2) Deletion of Individual Specified Atoms, (3) Deletion of Specified Types of Atoms, (4) Deletion of Atoms outside Cutoff Range, (5) Insertion of Individual Specified Atoms, (6) Replace Old Atom Type with a New Type, (7) Assign Connectivities based on Distance, (8) Convert Units from Bohrs to Angstroms, (9) Invert thru Origin to give Mirror Image, (10) Translate Center of Mass to the Origin, (11) Translate a Specified Atom to the Origin, (12) Translate and Rotate to Inertial Frame, (13) Move to Specified Rigid Body Coordinates, (14) Create and Fill a Periodic Boundary Box, (15) Soak Current Molecule in Box of Solvent, (16) Append another XYZ file to Current One. In most cases, multiply options can be applied sequentially to an input file. At the end of the editing process, a new version of the original .xyz file is written as output.

\sphinxstylestrong{XYZINT}

A program for converting a Tinker .xyz Cartesian coordinate formatted file into a Tinker .int internal coordinates formatted file. This program can optionally use an existing internal coordinates file as a template for the connectivity information.

\sphinxstylestrong{XYZMOL2}

A program to convert a Tinker .xyz Cartesian coordinates file into a Tripos Sybyl MOL2 file. The conversion generates only the MOLECULE, ATOM, BOND and SUBSTRUCTURE record type in the MOL2 file. Generic Sybyl atom types are used in most cases; while these atom types may need to be altered in some cases, Sybyl is usually able to correctly display the resulting MOL2 file.

\sphinxstylestrong{XYZPDB}

A program for converting a Tinker .xyz Cartesian coordinate file into a Brookhaven Protein Data Bank file (a PDB file).


\chapter{Force Field Parameter Sets}
\label{\detokenize{text/parameters:force-field-parameter-sets}}\label{\detokenize{text/parameters::doc}}
The Tinker package is distributed with several force field parameter sets, implementing a selection of widely used literature force fields as well as the Tinker force field currently under construction in the Ponder lab. We try to exactly reproduce the intent of the original authors of our distributed, third\sphinxhyphen{}party force fields. In all cases the parameter sets have been validated against literature reports, results provided by the original developers, or calculations made with the authentic programs. With the few exceptions noted below, Tinker calculations can be treated as authentic results from the genuine force fields. A brief description of each parameter set, including some still in preparation and not distributed with the current version, is provided below with lead literature references for the force field:

\sphinxstylestrong{AMOEBA.PRM}

Parameters for the AMOEBA polarizable atomic multipole force field. As of the current Tinker release, we have completed parametrization for a number of ions and small organic molecules. For further information, or if you are interested in developing or testing parameters for other small molecules, please contact the Ponder lab.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, A Consistent Treatment of Inter\sphinxhyphen{} and Intramolecular Polarization in Molecular Mechanics Calculations, J. Comput. Chem., 23, 1497\sphinxhyphen{}1506 (2002)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, Polarizable Atomic Multipole Water Model for Molecular Mechanics Simulation, J. Phys. Chem. B, 107, 5933\sphinxhyphen{}5947 (2003)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, Ion Solvation Thermodynamics from Simulation with a Polarizable Force Field, A. Grossfield, J. Am. Chem. Soc., 125, 15671\sphinxhyphen{}15682 (2003)

\end{enumerate}

\sphinxstylestrong{AMOEBAPRO.PRM}

Preliminary protein parameters for the AMOEBA polarizable atomic multipole force field. While the distributed parameters are still subject to minor alteration as we continue validation, they are now stable enough for other groups to begin using them. For further information, or if you are interested in testing the protein parameter set, please contact the Ponder lab.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{22}
\item {} 
Ponder and D. A. Case, Force Fields for Protein Simulation, Adv. Prot. Chem., 66, 27\sphinxhyphen{}85 (2003)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, Polarizable Atomic Multipole\sphinxhyphen{}based Potential for Proteins: Model and Parameterization, in preparation

\end{enumerate}

\sphinxstylestrong{AMBER94.PRM}

AMBER ff94 parameters for proteins and nucleic acids. Note that with their {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}Cornell’’ force field, the Kollman group has devised separate, fully independent partial charge values for each of the N\sphinxhyphen{} and C\sphinxhyphen{}terminal amino acid residues. At present, the terminal residue charges for Tinker’s version maintain the correct formal charge, but redistributed somewhat at the alpha carbon atoms from the original Kollman group values. The total magnitude of the redistribution is less than 0.01 electrons in most cases.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Cornell, P. Cieplak, C. I. Bayly, I. R. Gould, K. M. Merz, Jr., D. M. Ferguson, D. C. Spellmeyer, T. Fox, J. W. Caldwell and P. A. Kollman, A Second Generation Force Field for the Simulation of Proteins, Nucleic Acids, and Organic Molecules, J. Am. Chem. Soc., 117, 5179\sphinxhyphen{}5197 (1995)  {[}ff94{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} 
Moyna, H. J. Williams, R. J. Nachman and A. I. Scott, Conformation in Solution and Dynamics of a Structurally Constrained Linear Insect Kinin Pentapeptide Analogue, Biopolymers, 49, 403\sphinxhyphen{}413 (1999)  {[}AIB charges{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Ross and C. C. Hardin, Ion\sphinxhyphen{}Induced Stabilization of the G\sphinxhyphen{}DNA Quadruplex: Free Energy Perturbation Studies, J. Am. Chem. Soc., 116, 4363\sphinxhyphen{}4366 (1994)   {[}alkali metal ions{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Aqvist, Ion\sphinxhyphen{}Water Interaction Potentials Derived from Free Energy Perturbation Simulations, J. Phys. Chem., 94, 8021\sphinxhyphen{}8024, 1990  {[}alkaline earth Ions, radii adapted for Amber combining rule{]}

\end{enumerate}

Current force field parameter values and suggested procedures for development of parameters for additional molecules are available from the Amber web site in the Case lab at Scripps, \sphinxurl{http://amber.scripps.edu/}

\sphinxstylestrong{AMBER96.PRM}

AMBER ff96 parameters for proteins and nucleic acids. The only change from the ff94 parameter set is in the torsional parameters for the protein phi/psi angles. These values were altered to give better agreement with  changes of ff96 with LMP2 QM results from the Friesner lab on alanine dipeptide and tetrapeptide.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Kollman, R. Dixon, W. Cornell, T. Fox, C. Chipot and A. Pohorille, The Development/ Application of a ‘Minimalist’ Organic/Biochemical Molecular Mechanic Force Field using a Combination of ab Initio Calculations and Experimental Data, in Computer Simulation of Biomolecular Systems, W. F. van Gunsteren, P. K. Weiner, A. J. Wilkinson, eds., Volume 3, 83\sphinxhyphen{}96 (1997)  {[}ff96{]}

\end{enumerate}

Current force field parameter values and suggested procedures for development of parameters for additional molecules are available from the Amber web site in the Case lab at Scripps, \sphinxurl{http://amber.scripps.edu/}

\sphinxstylestrong{AMBER98.PRM}

AMBER ff98 parameters for proteins and nucleic acids. The only change from the ff94 parameter set is in the glycosidic torsional parameters that control sugar pucker.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Cheatham III, P. Cieplak and P. A. Kollman, A Modified Version of the Cornell et al. Force Field with Improved Sugar Pucker Phases and Helical Repeat, J. Biomol. Struct. Dyn., 16, 845\sphinxhyphen{}862 (1999)

\end{enumerate}

\end{enumerate}

Current force field parameter values and suggested procedures for development of parameters for additional molecules are available from the Amber web site in the Case lab at Scripps, \sphinxurl{http://amber.scripps.edu/}

\sphinxstylestrong{AMBER99.PRM}

AMBER ff99 parameters for proteins and nucleic acids. The original partial charges from the ff94 parameter set are retained, but many of the bond, angle and torsional parameters have been revised to provide better general agreement with experiment.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Wang, P. Cieplak and P. A. Kollman, How Well Does a Restrained Electrostatic Potential (RESP) Model Perform in Calcluating Conformational Energies of Organic and Biological Molecules?, J. Comput. Chem., 21, 1049\sphinxhyphen{}1074 (2000)

\end{enumerate}

Current force field parameter values and suggested procedures for development of parameters for additional molecules are available from the Amber web site in the Case lab at Scripps, \sphinxurl{http://amber.scripps.edu/}

\sphinxstylestrong{CHARMM19.PRM}

CHARMM19 united\sphinxhyphen{}atom parameters for proteins. The nucleic acid parameter are not yet implemented. There are some differences between authentic CHARMM19 and the Tinker version due to replacement of CHARMM impropers by torsions for cases that involve atoms not bonded to the trigonal atom and Tinker’s use of all possible torsions across a bond instead of a single torsion per bond.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{4}
\item {} 
Neria, S. Fischer and M. Karplus, Simulation of Activation Free Energies in Molecular Systems, J. Chem. Phys., 105, 1902\sphinxhyphen{}1921 (1996)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} 
Nilsson and M. Karplus, Empirical Energy Functions for Energy Minimizations and Dynamics of Nucleic Acids, J. Comput. Chem., 7, 591\sphinxhyphen{}616 (1986)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Reiher III, Theoretical Studies of Hydrogen Bonding, Ph.D. Thesis, Department of Chemistry, Harvard University, Cambridge, MA, 1985

\end{enumerate}

\end{enumerate}

\sphinxstylestrong{CHARMM22.PRM}

CHARMM27 all\sphinxhyphen{}atom parameters for proteins and lipids. Most of the nucleic acid and small model compound parameters are not yet implemented. We plan to provide these additional parameters in due course.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} 
Foloppe and A. D. MacKerell, Jr., All\sphinxhyphen{}Atom Empirical Force Field for Nucleic Acids: 1) Parameter Optimization Based on Small Molecule and Condensed Phase Macromolecular Target Data, J. Comput. Chem., 21, 86\sphinxhyphen{}104 (2000)  {[}CHARMM27{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} 
Banavali and A. D. MacKerell, Jr., All\sphinxhyphen{}Atom Empirical Force Field for Nucleic Acids: 2) Application to Molecular Dynamics Simulations of DNA and RNA in Solution, J. Comput. Chem., 21, 105\sphinxhyphen{}120 (2000)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
MacKerrell, Jr., et al., All\sphinxhyphen{}Atom Empirical Potential for Molecular Modeling and Dynamics Studies of Proteins, J. Phys. Chem. B, 102, 3586\sphinxhyphen{}3616 (1998)  {[}CHARMM22{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
MacKerell, Jr., J. Wiorkeiwicz\sphinxhyphen{}Kuczera and M. Karplus, An All\sphinxhyphen{}Atom Empirical Energy Function for the Simulation of Nucleic Acids, J. Am. Chem. Soc., 117, 11946\sphinxhyphen{}11975 (1995)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Feller, D. Yin, R. W. Pastor and A. D. MacKerell, Jr., Molecular Dynamics Simulation of Unsaturated Lipids at Low Hydration: Parametrization and Comparison with Diffraction Studies, Biophysical Journal, 73, 2269\sphinxhyphen{}2279 (1997)  {[}alkenes{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{7}
\item {} 
Stote and M. Karplus, Zinc Binding in Proteins and Solution \sphinxhyphen{} A Simple but Accurate Nonbonded Representation, Proteins, 23, 12\sphinxhyphen{}31 (1995)  {[}zinc ion{]}

\end{enumerate}

\end{enumerate}

Current and legacy parameter values are available from the CHARMM force field web site on Alex MacKerell’s  Research Interests page at the University of Maryland School of Pharmacy, \sphinxurl{https://rxsecure.umaryland.edu/research/amackere/research.html/}

\sphinxstylestrong{DUDEK.PRM}

Protein\sphinxhyphen{}only parameters for the early 1990’s Tinker force field with multipole values of Dudek and Ponder. The current file contains only the multipole values from the 1995 paper by Dudek and Ponder. This set is now superceeded by the more recent Tinker force field developed by Pengyu Ren (see WATER.PRM, below).
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Dudek and J. W. Ponder, Accurate Electrostatic Modelling of the Intramolecular Energy of Proteins, J. Comput. Chem., 16, 791\sphinxhyphen{}816 (1995)

\end{enumerate}

\end{enumerate}

\sphinxstylestrong{ENCAD.PRM}

ENCAD parameters for proteins and nucleic acids.  (in preparation)
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Levitt, M. Hirshberg, R. Sharon and V. Daggett, Potential Energy Function and Parameters for Simulations of the Molecular Dynamics of Protein and Nucleic Acids in Solution, Comp. Phys. Commun., 91, 215\sphinxhyphen{}231 (1995)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Levitt, M. Hirshberg, R. Sharon, K. E. Laidig and V. Daggett, Calibration and Testing of a Water Model for Simulation of the Molecular Dynamics of Protein and Nucleic Acids in Solution, J. Phys. Chem. B, 101, 5051\sphinxhyphen{}5061 (1997)  {[}F3C water{]}

\end{enumerate}

\sphinxstylestrong{HOCH.PRM}

Simple NMR\sphinxhyphen{}NOE force field of Hoch and Stern.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Hoch and A. S. Stern, A Method for Determining Overall Protein Fold from NMR Distance Restraints, J. Biomol. NMR, 2, 535\sphinxhyphen{}543 (1992)

\end{enumerate}

\end{enumerate}

\sphinxstylestrong{MM2.PRM}

Full MM2(1991) parameters including ?\sphinxhyphen{}systems. The anomeric and electronegativity correction terms included in some later versions of MM2 are not implemented.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, Conformational Analysis. 130. MM2. A Hydrocarbon Force Field Utilizing V1 and V2 Torsional Terms, J. Am. Chem. Soc., 99, 8127\sphinxhyphen{}8134 (1977)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Sprague, J. C. Tai, Y. Yuh and N. L. Allinger, The MMP2 Calculational Method, J. Comput. Chem., 8, 581\sphinxhyphen{}603 (1987)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Tai and N. L. Allinger, Molecular Mechanics Calculations on Conjugated Nitrogen\sphinxhyphen{}Containing Heterocycles, J. Am. Chem. Soc., 110, 2050\sphinxhyphen{}2055 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Tai, J.\sphinxhyphen{}H. Lii and N. L. Allinger, A Molecular Mechanics (MM2) Study of Furan, Thiophene, and Related Compounds, J. Comput. Chem., 10, 635\sphinxhyphen{}647 (1989)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, R. A. Kok and M. R. Imam, Hydrogen Bonding in MM2, J. Comput. Chem., 9, 591\sphinxhyphen{}595 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} 
Norskov\sphinxhyphen{}Lauritsen and N. L. Allinger, A Molecular Mechanics Treatment of the Anomeric Effect, J. Comput. Chem., 5, 326\sphinxhyphen{}335 (1984)

\end{enumerate}

All parameters distributed with Tinker are from the “MM2 (1991) Parameter Set”, as provided by N. L. Allinger, University of Georgia

\sphinxstylestrong{MM3.PRM}

Full MM3(2000) parameters including pi\sphinxhyphen{}systems. The directional hydrogen bonding term and electronegativity bond length corrections are implemented, but the anomeric and Bohlmann correction terms are not implemented.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, Y. H. Yuh and J.\sphinxhyphen{}H. Lii, Molecular Mechanics. The MM3 Force Field for Hydrocarbons. 1, J. Am. Chem. Soc., 111, 8551\sphinxhyphen{}8566 (1989)

\end{enumerate}

\end{enumerate}

J.\sphinxhyphen{}H. Lii and N. L. Allinger, Molecular Mechanics. The MM3 Force Field for Hydrocarbons. 2. Vibrational Frequencies and Thermodynamics, J. Am. Chem. Soc., 111, 8566\sphinxhyphen{}8575 (1989)

J.\sphinxhyphen{}H. Lii and N. L. Allinger, Molecular Mechanics. The MM3 Force Field for Hydrocarbons. 3. The van der Waals’ Potentials and Crystal Data for Aliphatic and Aromatic Hydrocarbons, J. Am. Chem. Soc., 111, 8576\sphinxhyphen{}8582 (1989)
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, H. J. Geise, W. Pyckhout, L. A. Paquette and J. C. Gallucci, Structures of Norbornane and Dodecahedrane by Molecular Mechanics Calculations (MM3), X\sphinxhyphen{}ray Crystallography, and Electron Diffraction, J. Am. Chem. Soc., 111, 1106\sphinxhyphen{}1114 (1989)  {[}stretch\sphinxhyphen{}torsion cross term{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, F. Li and L. Yan, Molecular Mechanics. The MM3 Force Field for Alkenes, J. Comput. Chem., 11, 848\sphinxhyphen{}867 (1990)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, F. Li, L. Yan and J. C. Tai, Molecular Mechanics (MM3) Calculations on Conjugated Hydrocarbons, J. Comput. Chem., 11, 868\sphinxhyphen{}895 (1990)

\end{enumerate}

\end{enumerate}

J.\sphinxhyphen{}H. Lii and N. L. Allinger, Directional Hydrogen Bonding in the MM3 Force Field. I, J. Phys. Org. Chem., 7, 591\sphinxhyphen{}609 (1994)

J.\sphinxhyphen{}H. Lii and N. L. Allinger, Directional Hydrogen Bonding in the MM3 Force Field. II, J. Comput. Chem., 19, 1001\sphinxhyphen{}1016 (1998)

All parameters distributed with Tinker are from the “MM3 (2000) Parameter Set”, as provided by N. L. Allinger, University of Georgia, August 2000

\sphinxstylestrong{MM3PRO.PRM}

Protein\sphinxhyphen{}only version of the MM3 parameters.

J.\sphinxhyphen{}H. Lii and N. L. Allinger, The MM3 Force Field for Amides, Polypeptides and Proteins, J. Comput. Chem., 12, 186\sphinxhyphen{}199 (1991)

\sphinxstylestrong{OPLSUA.PRM}

Complete OPLS\sphinxhyphen{}UA with united\sphinxhyphen{}atom parameters for proteins and many classes of organic molecules. Explicit hydrogens on polar atoms and aromatic carbons.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen and J. Tirado\sphinxhyphen{}Rives, The OPLS Potential Functions for Proteins. Energy Minimizations for Crystals of Cyclic Peptides and Crambin, J. Am. Chem. Soc., 110, 1657\sphinxhyphen{}1666 (1988)  {[}peptide and proteins{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen and D. L. Severance, Aromatic\sphinxhyphen{}Aromatic Interactions: Free Energy Profiles for the Benzene Dimer in Water, Chloroform, and Liquid Benzene, J. Am. Chem. Soc., 112, 4768\sphinxhyphen{}4774 (1990)  {[}aromatic hydrogens{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Weiner, P. A. Kollman, D. A. Case, U. C. Singh, C. Ghio, G. Alagona, S. Profeta, Jr. and P. Weiner, A New Force Field for Molecular Mechanical Simulation of Nucleic Acids and Proteins, J. Am. Chem. Soc., 106, 765\sphinxhyphen{}784 (1984)  {[}united\sphinxhyphen{}atom {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}AMBER/OPLS’’ local geometry{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Weiner, P. A. Kollman, D. T. Nguyen and D. A. Case, An All Atom Force Field for Simulations of Proteins and Nucleic Acids, J. Comput. Chem., 7, 230\sphinxhyphen{}252 (1986)  {[}all\sphinxhyphen{}atom “AMBER/OPLS” local geometry{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{23}
\item {} 
Dang and B. M. Pettitt, Simple Intramolecular Model Potentials for Water, J. Phys. Chem., 91, 3349\sphinxhyphen{}3354 (1987)  {[}flexible TIP3P and SPC water{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen, J. D. Madura and C. J. Swenson, Optimized Intermolecular Potential Functions for Liquid Hydrocarbons, J. Am. Chem. Soc., 106, 6638\sphinxhyphen{}6646 (1984)  {[}hydrocarbons{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen, E. R. Laird, T. B. Nguyen and J. Tirado\sphinxhyphen{}Rives, Monte Carlo Simulations of Pure Liquid Substituted Benzenes with OPLS Potential Functions, J. Comput. Chem., 14, 206\sphinxhyphen{}215 (1993)  {[}substituted benzenes{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{4}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{12}
\item {} 
Duffy, P. J. Kowalczyk and W. L. Jorgensen, Do Denaturants Interact with Aromatic Hydrocarbons in Water?, J. Am. Chem. Soc., 115, 9271\sphinxhyphen{}9275 (1993)  {[}benzene, naphthalene, urea, guanidinium, tetramethyl ammonium{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen and C. J. Swenson, Optimized Intermolecular Potential Functions for Amides and Peptides. Structure and Properties of Liquid Amides, J. Am. Chem. Soc., 106, 765\sphinxhyphen{}784 (1984)  {[}amides{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen, J. M. Briggs and M. L. Contreras, Relative Partition Coefficients for Organic Solutes form Fluid Simulations, J. Phys. Chem., 94, 1683\sphinxhyphen{}1686 (1990)  {[}chloroform, pyridine, pyrazine, pyrimidine{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{12}
\item {} 
Briggs, T. B. Nguyen and W. L. Jorgensen, Monte Carlo Simulations of Liquid Acetic Acid and Methyl Acetate with the OPLS Potential Functions, J. Phys. Chem., 95, 3315\sphinxhyphen{}3322 (1991)  {[}acetic acid, methyl acetate{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{7}
\item {} 
Liu, F. Muller\sphinxhyphen{}Plathe and W. F. van Gunsteren, A Force Field for Liquid Dimethyl Sulfoxide and Physical Properties of Liquid Dimethyl Sulfoxide Calculated Using Molecular Dynamics Simulation, J. Am. Chem. Soc., 117, 4363\sphinxhyphen{}4366 (1995)  {[}dimethyl sulfoxide{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Gao, X. Xia and T. F. George, Importance of Bimolecular Interactions in Developing Empirical Potential Functions for Liquid Ammonia, J. Phys. Chem., 97, 9241\sphinxhyphen{}9246 (1993)  {[}ammonia{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Aqvist, Ion\sphinxhyphen{}Water Interaction Potentials Derived from Free Energy Perturbation Simulations, J. Phys. Chem., 94, 8021\sphinxhyphen{}8024 (1990)  {[}metal ions{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Ross and C. C. Hardin, Ion\sphinxhyphen{}Induced Stabilization of the G\sphinxhyphen{}DNA Quadruplex: Free Energy Perturbation Studies, J. Am. Chem. Soc., 116, 4363\sphinxhyphen{}4366 (1994)  {[}alkali metal ions{]}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Chandrasekhar, D. C. Spellmeyer and W. L. Jorgensen, Energy Component Analysis for Dilute Aqueous Solutions of Li+, Na+, F\sphinxhyphen{}, and Cl\sphinxhyphen{} Ions, J. Am. Chem. Soc., 106, 903\sphinxhyphen{}910 (1984)  {[}halide ions{]}

\end{enumerate}

Most parameters distributed with Tinker are from {\color{red}\bfseries{}\textasciigrave{}\textasciigrave{}}OPLS and OPLS\sphinxhyphen{}AA Parameters for Organic Molecules, Ions, and Nucleic Acids’’ as provided by W. L. Jorgensen, Yale University, October 1997

\sphinxstylestrong{OPLSAA.PRM}

OPLS\sphinxhyphen{}AA force field with all\sphinxhyphen{}atom parameters for proteins and many general classes of organic molecules.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen, D. S. Maxwell and J. Tirado\sphinxhyphen{}Rives, Development and Testing of the OPLS All\sphinxhyphen{}Atom Force Field on Conformational Energetics and Properties of Organic Liquids, J. Am. Chem. Soc., 117, 11225\sphinxhyphen{}11236 (1996)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Maxwell, J. Tirado\sphinxhyphen{}Rives and W. L. Jorgensen, A Comprehensive Study of the Rotational Energy Profiles of Organic Systems by Ab Initio MO Theory, Forming a Basis for Peptide Torsional Parameters, J. Comput. Chem., 16, 984\sphinxhyphen{}1010 (1995)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen and N. A. McDonald, Development of an All\sphinxhyphen{}Atom Force Field for Heterocycles. Properties of Liquid Pyridine and Diazenes, THEOCHEM\sphinxhyphen{}J. Mol. Struct., 424, 145\sphinxhyphen{}155 (1998)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
McDonald and W. L. Jorgensen, Development of an All\sphinxhyphen{}Atom Force Field for Heterocycles. Properties of Liquid Pyrrole, Furan, Diazoles, and Oxazoles, J. Phys. Chem. B, 102, 8049\sphinxhyphen{}8059 (1998)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Rizzo and W. L. Jorgensen, OPLS All\sphinxhyphen{}Atom Model for Amines: Resolution of the Amine Hydration Problem, J. Am. Chem. Soc., 121, 4827\sphinxhyphen{}4836 (1999)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumiii}{enumiv}{}{.}%
\setcounter{enumiii}{15}
\item {} 
Price, D. Ostrovsky and W. L. Jorgensen, Gas\sphinxhyphen{}Phase and Liquid\sphinxhyphen{}State Properties of Esters, Nitriles, and Nitro Compounds with the OPLS\sphinxhyphen{}AA Force Field, J. Comput. Chem., 22, 1340\sphinxhyphen{}1352 (2001)

\end{enumerate}

\end{enumerate}

\end{enumerate}

All parameters distributed with Tinker are from “OPLS and OPLS\sphinxhyphen{}AA Parameters for Organic Molecules, Ions, and Nucleic Acids” as provided by W. L. Jorgensen, Yale University, October 1997

\sphinxstylestrong{OPLSAAL.PRM}

An improved OPLS\sphinxhyphen{}AA parameter set for proteins in which the only change is a reworking of many of the backbone and sidechain torsional parameters to give better agreement with LMP2 QM calculations. This parameter set is also known as OPLS(2000).
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
Kaminsky, R. A. Friesner, J. Tirado\sphinxhyphen{}Rives and W. L. Jorgensen, Evaluation and Reparametrization of the OPLS\sphinxhyphen{}AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides, J. Phys. Chem. B, 105, 6474\sphinxhyphen{}6487 (2001)

\end{enumerate}

\end{enumerate}

\sphinxstylestrong{SMOOTH.PRM}

Version of OPLS\sphinxhyphen{}UA for use with potential smoothing. Largely adapted largely from standard OPLS\sphinxhyphen{}UA parameters with modifications to the vdw and improper torsion terms.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{21}
\item {} 
Pappu, R. K. Hart and J. W. Ponder, Analysis and Application of Potential Energy Smoothing and Search Methods for Global Optimization, J. Phys, Chem. B, 102, 9725\sphinxhyphen{}9742 (1998)  {[}smoothing modifications{]}

\end{enumerate}

\end{enumerate}

\sphinxstylestrong{SMOOTHAA.PRM}

Version of OPLS\sphinxhyphen{}AA for use with potential smoothing. Largely adapted largely from standard OPLS\sphinxhyphen{}AA parameters with modifications to the vdw and improper torsion terms.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{21}
\item {} 
Pappu, R. K. Hart and J. W. Ponder, Analysis and Application of Potential Energy Smoothing and Search Methods for Global Optimization, J. Phys, Chem. B, 102, 9725\sphinxhyphen{}9742 (1998)  {[}smoothing modifications{]}

\end{enumerate}

\end{enumerate}

\sphinxstylestrong{WATER.PRM}

The AMOEBA water parameters for a polarizable atomic multipole electrostatics model. This model is equal or better to the best available water models for many bulk and cluster properties.
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, A Polarizable Atomic Multipole Water Model for Molecular Mechanics Simulation, J. Phys. Chem. B, 107, 5933\sphinxhyphen{}5947 (2003)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, Ion Solvation Thermodynamics from Simulation with a Polarizable Force Field, A. Grossfield, J. Am. Chem. Soc., 125, 15671\sphinxhyphen{}15682 (2003)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Ren and J. W. Ponder, Temperature and Pressure Dependence of the AMOEBA Water Model, J. Phys. Chem. B, 108, xxxx\sphinxhyphen{}xxxx (2004)

\end{enumerate}

An earlier version the AMOEBA water model is described in: Yong Kong, Multipole Electrostatic Methods for Protein Modeling with Reaction Field Treatment, Biochemistry \& Molecular Biophysics, Washington University, St. Louis, August, 1997 {[}available from \sphinxurl{http://dasher.wustl.edu/ponder/}{]}


\chapter{Special Features \& Methods}
\label{\detokenize{text/special-features:special-features-methods}}\label{\detokenize{text/special-features::doc}}
This section contains several short notes with further information about Tinker methodology, algorithms and special features. The discussion is not intended to be exhaustive, but rather to explain features and capabilities so that users can make more complete use of the package.


\section{File  Version Numbers}
\label{\detokenize{text/special-features:file-version-numbers}}
All of the input and output file types routinely used by the Tinker package are capable of existing as multiple versions of a base file name. For example, if the program XYZINT is run on the input file molecule.xyz, the output internal coordinates file will be written to molecule.int. If a file named molecule.int is already present prior to running XYZINT, then the output will be written instead to the next available version, in this case to molecule.int\_2. In fact the output is generally written to the lowest available, previously unused version number (molecule.int\_3, molecule.int\_4, etc., as high as needed). Input file names are handled similarly. If simply molecule or molecule.xyz is entered as the input file name upon running XYZINT, then the highest version of molecule.xyz will be used as the actual input file. If an explicit version number is entered as part of the input file name, then the specified version will be used as the input file.

The version number scheme will be recognized by many older users as a holdover from the VMS origins of the first version of the Tinker software. It has been maintained to make it easier to chain together multiple calculations that may create several new versions of a given file, and to make it more difficult to accidently overwrite a needed result. The version scheme applies to most uses of many common Tinker file types such as .xyz, .int, .key, .arc. It is not used when an overwritten file update is obviously the correct action, for example, the .dyn molecular dynamics restart files. For those users who prefer a more Unix\sphinxhyphen{}like operation, and do not desire use of file versions, this feature can be turned off by adding the NOVERSION keyword to the applicable Tinker keyfile.

The version scheme as implemented in Tinker does have two known quirks. First, it becomes impossible to directly use the original unversioned copy of a file if higher version numbers are present. For example, if the files molecule.xyz and molecule.xyz\_2 both exist, then molecule.xyz cannot be accessed as input by XYZINT. If molecule.xyz is entered in response to the input file name question, molecule.xyz\_2 (or the highest present version number) will be used as input. The only workaround is to copy or rename molecule.xyz to something else, say molecule.new, and use that name for the input file. Secondly, missing version numbers always end the search for the highest available version number; i.e., version numbers are assumed to be consecutive and without gaps. For example, if molecule.xyz, molecule.xyz\_2 and molecule.xyz\_4 are present, but not molecule.xyz\_3, then molecule.xyz\_2 will be used as input to XYZINT if molecule is given as the input file name. Similarly, output files will fill in gaps in an already existing set of file versions.


\section{Command Line Options}
\label{\detokenize{text/special-features:command-line-options}}
Many operating systems or compiler supplied\sphinxhyphen{}libraries make available something like the standard Unix iargc and getarg routines for capturing command line arguments. On these machines most of the Tinker programs support a selection of command line arguments and options. The name of the keyfile to be used for a calculation is read from the argument following a \sphinxhyphen{}k (equivalent to either \sphinxhyphen{}key or \sphinxhyphen{}keyfile, case insensitive) command line argument. Note that the \sphinxhyphen{}k options can appear anywhere on the command line following the executable name. All other command line arguments, excepting the name of the executable program itself, are treated as input arguments. These input arguments are read from left to right and interpreted in order as the answers to questions that would be asked by an interactive invocation of the same Tinker program. For example, the following command line:

newton molecule \sphinxhyphen{}k test a a 0.01

will invoke the NEWTON program on the structure file molecule.xyz using the keyfile test.key, automatic mode {[}a{]} for both the method and preconditioning, and 0.01 for the RMS gradient per atom termination criterion in kcal/mole/Ang. Provided that the force field parameter set, etc. is provided in test.key, the above compuation will procede directly from the command line invocation without further interactive input.


\section{Use on Windows Systems}
\label{\detokenize{text/special-features:use-on-windows-systems}}
Tinker executables for Microsoft PC systems should be run from the DOS or Command Prompt window available under the various versions of Windows. The Tinker executable directory should be added to your path via the autoexec.bat file or similar. If a Command Prompt window, set the number of scrollable lines to a very large number, so that you will be able to inspect screen output after it moves by. Alternatively, Tinker programs which generate large amounts of screen output should be run such that output will be redirected to a file. This can be accomplished by running the Tinker program in batch mode or by using the build\sphinxhyphen{}in Unix\sphinxhyphen{}like output redirection. For example, the command:

dynamic \textless{} molecule.inp \textgreater{} molecule.log

will run the Tinker dynamic program taking input from the file molecule.inp and sending output to molecule.log. Also note that command line options as described above are available with the distributed Tinker executables.

If the distributed Tinker executables are run directly from Windows by double clicking on the program icon, then the program will run in its own window. However, upon completion of the program the window will close and screen output will be lost. Any output files written by the program will, of course, still be available. The Windows behavior can be changed by adding the EXIT\sphinxhyphen{}PAUSE keyword to the keyfile. This keyword causes the executation window to remain open after completion until the “Return/Enter” key is pressed.

An alternative to Command Prompt windows is to use the PowerShell window available on Windows 10 systems, which provides a better emulation of many of the standard features of Linux shells and MacOS Terminal.

Yet another alternative, particularly attractive to those already familiar with Linux or Unix systems, is to download the Cygwin package currently available under GPL license from the site \sphinxurl{http://source.redhat.com/cygwin/}. The cygwin tools provide many of the GNU tools, including a bash shell window from which Tinker programs can be run.

Finally on Windows 10 systems, it is possible to download and install the Windows Subsystem for Linux (WSL), and then run the Tinker Linux executables from within WSL.


\section{Use on MacOS Systems}
\label{\detokenize{text/special-features:use-on-macos-systems}}
The command line versions of the Tinker executables are best run on MacOS in a “Terminal” application window where behavior is essentially identical to that in a Linux terminal.


\section{Atom Types vs. Atom Classes}
\label{\detokenize{text/special-features:atom-types-vs-atom-classes}}
Manipulation of atom types and the proliferation of parameters as atoms are further subdivided into new types is the bane of force field calculation. For example, if each topologically distinct atom arising from the 20 natural amino acids is given a different atom type, then about 300 separate type are required (this ignores the different N\sphinxhyphen{} and C\sphinxhyphen{}terminal forms of the residues, diastereotopic hydrogens, etc.). However, all these types lead to literally thousands of different force field parameters. In fact, there are many thousands of distinct torsional parameters alone. It is impossible at present to fully optimize each of these parameters; and even if we could, a great many of the parameters would be nearly identical. Two somewhat complimentary solutions are available to handle the proliferation of parameters. The first is to specify the molecular fragments to which a given parameter can be applied in terms of a chemical structure language, SMILES strings for example.

A second general approach is to use hierarchical cascades of parameter groups. Tinker uses a simple version of this scheme. Each Tinker force field atom has both an atom type number and an atom class number. The types are subsets of the atom classes, i.e., several different atom types can belong to the same atom class. Force field parameters that are somewhat less sensitive to local environment, such as local geometry terms, are then provided and assigned based on atom class. Other energy parameters, such as electrostatic parameters, that are very environment dependent are assigned over the atom types. This greatly reduces the number of independent multiple\sphinxhyphen{}atom parameters like the four\sphinxhyphen{}atom torsional parameters.


\section{Calculations on Partial Structures}
\label{\detokenize{text/special-features:calculations-on-partial-structures}}
Two methods are available for performing energetic calculations on portions or substructures within a full molecular system. Tinker allows division of the entire system into active and inactive parts which can be defined via keywords. In subsequent calculations, such as minimization or dynamics, only the active portions of the system are allowed to move. The force field engine responds to the active/inactive division by computing all energetic interactions involving at least one active atom; i.e., any interaction whose energy can change with the motion of one or more active atoms is computed.

The second method for partial structure computation involves dividing the original system into a set of atom groups. As before, the groups can be specified via appropriate keywords. The current Tinker implementation allows specification of up to a maximum number of groups as given in the sizes.i dimensioning file. The groups must be disjoint in that no atom can belong to more than one group. Further keywords allow the user to specify which intra\sphinxhyphen{} and intergroup sets of energetic interactions will contribute to the total force field energy. Weights for each set of interactions in the total energy can also be input. A specific energetic interaction is assigned to a particular intra\sphinxhyphen{} or intergroup set if all the atoms involved in the interaction belong to the group (intra\sphinxhyphen{}) or pair of groups (inter\sphinxhyphen{}). Interactions involving atoms from more than two groups are not computed.

Note that the groups method and active/inactive method use different assignment procedures for individual interactions. The active/inactive scheme is intended for situations where only a portion of a system is allowed to move, but the total energy needs to reflect the presence of the remaining inactive portion of the structure. The groups method is intended for use in rigid body calculations, and is needed for certain kinds of free energy perturbation calculations.


\section{Metal Complexes and Hypervalent Species}
\label{\detokenize{text/special-features:metal-complexes-and-hypervalent-species}}
The distribution version of Tinker comes dimensioned for a maximum atomic coordination number of four as needed for standard organic compounds. In order to use Tinker for calculations on species containing higher coordination numbers, simply change the value of the parameter maxval in the master dimensioning file sizes.i and rebuilt the package. Note that this parameter value should not be set larger than necessary since large values can slow the execution of portions of some Tinker programs.

Many molecular mechanics approaches to inorganic and metal structures use an angle bending term which is softer than the usual harmonic bending potential. Tinker implements a Fourier bending term similar to that used by the Landis group’s SHAPES force field. The parameters for specific Fourier angle terms are supplied via the ANGLEF parameter and keyword format. Note that a Fourier term will only be used for a particular angle if a corresponding harmonic angle term is not present in the parameter file.

We previously worked with the Anders Carlsson group at Washington University in St. Louis to add their transition metal ligand field term to Tinker. Support for this additional potential functional form is present in the distributed Tinker source code. We plan to develop energy routines and parameterization around alternative forms for handling transition metals, including the ligand field formulation proposed by Rob Deeth and coworkers.


\section{Neighbor Methods for Nonbonded Terms}
\label{\detokenize{text/special-features:neighbor-methods-for-nonbonded-terms}}
In addition to standard double loop methods, the Method of Lights is available to speed neighbor searching. This method based on taking intersections of sorted atom lists can be much faster for problems where the cutoff distance is significantly smaller than half the maximal cell dimension. The current version of Tinker does not implement the “neighbor list” schemes common to many other simulation packages.


\section{Periodic Boundary Conditions}
\label{\detokenize{text/special-features:periodic-boundary-conditions}}
Both spherical cutoff images or replicates of a cell are supported by all Tinker programs that implement periodic boundary conditions. Whenever the cutoff distance is too large for the minimum image to be the only relevant neighbor (i.e., half the minimum box dimension for orthogonal cells), Tinker will automatically switch from the image formalism to use of replicated cells.


\section{Distance Cutoffs for Energy Functions}
\label{\detokenize{text/special-features:distance-cutoffs-for-energy-functions}}
Polynomial energy switching over a window is used for terms whose energy is small near the cutoff distance. For monopole electrostatic interactions, which are quite large in typical cutoff ranges, a two polynomial multiplicative\sphinxhyphen{}additive shifted energy switch unique to Tinker is applied. The Tinker method is similar in spirit to the force switching methods of Steinbach and Brooks, J. Comput. Chem., 15, 667\sphinxhyphen{}683 (1994). While the particle mesh Ewald method is preferred when periodic boundary conditions are present, Tinker’s shifted energy switch with reasonable switching windows is quite satisfactory for most routine modeling problems. The shifted energy switch minimizes the perturbation of the energy and the gradient at the cutoff to acceptable levels. Problems should arise only if the property you wish to monitor is known to require explicit inclusion of long range components (i.e., calculation of the dielectric constant, etc.).


\section{Ewald Summations Methods}
\label{\detokenize{text/special-features:ewald-summations-methods}}
Tinker contains a versions of the Ewald summation technique for inclusion of long range electrostatic interactions via periodic boundaries. The particle mesh Ewald (PME) method is available for simple charge\sphinxhyphen{}charge potentials, while regular Ewald is provided for polarizable atomic multipole interactions. The accuracy and speed of the regular and PME calculations is dependent on several interrelated parameters. For both methods, the Ewald coefficient and real\sphinxhyphen{}space cutoff distance must be set to reasonable and complementary values. Additional control variables for regular Ewald are the fractional coverage and number of vectors used in reciprocal space. For PME the additional control values are the B\sphinxhyphen{}spline order and charge grid dimensions. Complete control over all of these parameters is available via the Tinker keyfile mechanism. By default Tinker will select a set of parameters which provide a reasonable compromise between accuracy and speed, but these should be checked and modified as necessary for each individual system.


\section{Continuum Solvation Models}
\label{\detokenize{text/special-features:continuum-solvation-models}}
Several alternative continuum solvation algorithms are contained within Tinker. All of these are accessed via the SOLVATE keyword and its modifiers. Two simple surface area methods are implemented: the ASP method of Eisenberg and McLachlan, and the SASA method from Scheraga’s group. These methods are applicable to any of the standard Tinker force fields. Various schemes based on the generalized Born formalism are also available: the original 1990 numerical “Onion\sphinxhyphen{}shell” GB/SA method from Still’s group, the 1997 analytical GB/SA method also due to Still, a pairwise descreening algorithm originally proposed by Hawkins, Cramer and Truhlar, and the analytical continuum solvation (ACE) method of Schaefer and Karplus. At present, the generalized Born methods should only be used with force fields having simple partial charge electrostatic interactions.

Some further comments are in order regarding the GB/SA\sphinxhyphen{}style solvation models. The Onion\sphinxhyphen{}shell model is provided mostly for comparison purposes. It uses an exact, analytical surface area calculation for the cavity term and the numerical scheme described in the original paper for the polarization term. This method is very slow, especially for large systems, and does not contain the contribution of the Born radii chain rule term to the first derivatives. We recommend its use only for single\sphinxhyphen{}point energy calculations. The other GB/SA methods (“analytical” Still, H\sphinxhyphen{}C\sphinxhyphen{}T pairwise descreening, and ACE) use an approximate cavity term based on Born radii, and do contain fully correct derivatives including the Born radii chain rule contribution. These methods all scale in CPU time with the square of the size of the system, and can be used with minimization, molecular dynamics and large molecules.

Finally, we note that the ACE solvation model should not be used with the current version of Tinker. The algorithm is fully implemented in the source code, but parameterization is not complete. As of late 2000, parameter values are only available in the literature for use of ACE with the older CHARMM19 force field. We plan to develop values for use with more modern all\sphinxhyphen{}atom force fields, and these will be incorporated into Tinker sometime in the future.


\section{Polarizable Multipole Electrostatics}
\label{\detokenize{text/special-features:polarizable-multipole-electrostatics}}
Atomic multipole electrostatics through the quadrupole moment is supported by the current version of Tinker, as is either mutual or direct dipole polarization. Ewald summation is available for inclusion of long range interactions. Calculations are implemented via a mixture of the CCP5 algorithms of W. Smith and the Applequist\sphinxhyphen{}Dykstra Cartesian polytensor method. At present analytical energy and Cartesian gradient code is provided.

The Tinker package allows intramolecular polarization to be treated via a version of the interaction damping scheme of Thole. To implement the Thole scheme, it is necessary to set all the mutual\sphinxhyphen{}1x\sphinxhyphen{}scale keywords to a value of one. The other polarization scaling keyword series, direct\sphinxhyphen{}1x\sphinxhyphen{}scale and polar\sphinxhyphen{}1x\sphinxhyphen{}scale, can be set independently to enable a wide variety of polarization models. In order to use an Applequist\sphinxhyphen{}style model without polarization damping, simply set the polar\sphinxhyphen{}damp keyword to zero.


\section{Potential Energy Smoothing}
\label{\detokenize{text/special-features:potential-energy-smoothing}}
Versions of our Potential Smoothing and Search (PSS) methodology have been implemented within Tinker. This methods belong to the same general family as Scheraga’s Diffusion Equation Method, Straub’s Gaussian Density Annealing, Shalloway’s Packet Annealing and Verschelde’s Effective Diffused Potential, but our algorithms reflect our own ongoing research in this area. In many ways the Tinker potential smoothing methods are the deterministic analog of stochastic simulated annealing. The PSS algorithms are very powerful, but are relatively new and are still undergoing modification, testing and calibration within our research group. This version of Tinker also includes a basin\sphinxhyphen{}hopping conformational scanning algorithm in the program SCAN which is particularly effective on smoothed potential surfaces.


\section{Distance Geometry Metrization}
\label{\detokenize{text/special-features:distance-geometry-metrization}}
A much improved and very fast random pairwise metrization scheme is available which allows good sampling during trial distance matrix generation without the usual structural anomalies and CPU constraints of other metrization procedures. An outline of the methodology and its application to NMR NOE\sphinxhyphen{}based structure refinement is described in the paper by Hodsdon, et al. in Journal of Molecular Biology, 264, 585\sphinxhyphen{}602 (1996). We have obtained good results with something like the keyword phrase trial\sphinxhyphen{}distribution pairwise 5, which performs 5\% partial random pairwise metrization. For structures over several hundred atoms, a value less than 5 for the percentage of metrization should be fine.


\chapter{Use of the Keyword Control File}
\label{\detokenize{text/keywords:use-of-the-keyword-control-file}}\label{\detokenize{text/keywords::doc}}

\section{Using Keywords to Control Tinker Calculations}
\label{\detokenize{text/keywords:using-keywords-to-control-tinker-calculations}}
This section contains detailed descriptions of the keyword parameters used to define or alter the course of a Tinker calculation. The keyword control file is optional in the sense that all of the Tinker programs will run in the absence of a keyfile and will simply use default values or query the user for needed information. However, the keywords allow use of a wide variety of algorithmic and procedural options, many of which are unavailable interactively.

Keywords are read from the keyword control file. All programs look first for a keyfile with the same base name as the input molecular system and ending in the extension .key. If this file does not exist, then Tinker tries to use a generic keyfile with the name Tinker.key and located in the same directory as the input system. If neither a system\sphinxhyphen{}specific nor a generic keyfile is present, Tinker will continue by using default values for keyword options and asking interactive questions as necessary.

Tinker searches the keyfile during the course of a calculation for relevant keywords that may be present. All keywords must appear as the first word on the line. Any blank space to the left of the keyword is ignored, and all contents of the keyfiles are case insensitive. Some keywords take modifiers; i.e., Tinker looks further on the same line for additional information, such as the value of some parameter related to the keyword. Modifier information is read in free format, but must be completely contained on the same line as the original keyword. Any lines contained in the keyfile which do not qualify as valid keyword lines are treated as comments and are simply ignored.

Several keywords take a list of integer values (atom numbers, for example) as modifiers. For these keywords the integers can simply be listed explicitly and separated by spaces, commas or tabs. If a range of numbers is desired, it can be specified by listing the negative of the first number of the range, followed by a separator and the last number of the range. For example, the keyword line ACTIVE 4 \sphinxhyphen{}9 17 23 could be used to add atoms 4, 9 through 17, and 23 to the set of active atoms during a Tinker calculation.


\section{Keywords Grouped by Functionality}
\label{\detokenize{text/keywords:keywords-grouped-by-functionality}}
Listed below are the available Tinker keywords sorted into groups by general function. The section ends with an alphabetical list containing each individual keyword, along with a brief description of its action, possible keyword modifiers, and usage examples.


\subsection{OUTPUT CONTROL KEYWORDS}
\label{\detokenize{text/keywords:output-control-keywords}}
ARCHIVE DEBUG   DIGITS
ECHO    EXIT\sphinxhyphen{}PAUSE      NOVERSION
OVERWRITE       PRINTOUT        SAVE\sphinxhyphen{}CYCLE
SAVE\sphinxhyphen{}FORCE      SAVE\sphinxhyphen{}INDUCED    SAVE\sphinxhyphen{}VELOCITY
VERBOSE WRITEOUT


\subsection{FORCE FIELD SELECTION KEYWORDS}
\label{\detokenize{text/keywords:force-field-selection-keywords}}
FORCEFIELD      PARAMETERS


\subsection{POTENTIAL FUNCTION SELECTION KEYWORDS}
\label{\detokenize{text/keywords:potential-function-selection-keywords}}
ANGANGTERM      ANGLETERM       BONDTERM
CHARGETERM      CHGDPLTERM      DIPOLETERM
EXTRATERM       IMPROPTERM      IMPTORSTERM
METALTERM       MPOLETERM       OPBENDTERM
OPDISTTERM      PITORSTERM      POLARIZETERM
RESTRAINTERM    RXNFIELDTERM    SOLVATETERM
STRBNDTERM      STRTORTERM      TORSIONTERM
TORTORTERM      UREYTERM        VDWTERM


\subsection{POTENTIAL FUNCTION PARAMETER KEYWORDS}
\label{\detokenize{text/keywords:potential-function-parameter-keywords}}
ANGANG  ANGLE   ANGLE3
ANGLE4  ANGLE5  ANGLEF
ATOM    BIOTYPE BOND
BOND3   BOND4   BOND5
CHARGE  DIPOLE  DIPOLE3
DIPOLE4 DIPOLE5 ELECTNEG
HBOND   IMPROPER        IMPTORS
METAL   MULTIPOLE       OPBEND
OPDIST  PIATOM  PIBOND
PITORS  POLARIZE        SOLVATE
STRBND  STRTORS TORSION
TORSION4        TORSION5        TORTOR
UREYBRAD        VDW     VDW14
VDWPR


\subsection{ENERGY UNIT CONVERSION KEYWORDS}
\label{\detokenize{text/keywords:energy-unit-conversion-keywords}}
ANGLEUNIT       ANGANGUNIT      BONDUNIT
ELECTRIC        IMPROPUNIT      IMPTORUNIT
OPBENDUNIT      OPDISTUNIT      PITORSUNIT
STRBNDUNIT      STRTORUNIT      TORSIONUNIT
TORTORUNIT      UREYUNIT


\subsection{LOCAL GEOMETRY FUNCTIONAL FORM KEYWORDS}
\label{\detokenize{text/keywords:local-geometry-functional-form-keywords}}
ANGLE\sphinxhyphen{}CUBIC     ANGLE\sphinxhyphen{}QUARTIC   ANGLE\sphinxhyphen{}PENTIC
ANGLE\sphinxhyphen{}SEXTIC    BOND\sphinxhyphen{}CUBIC      BOND\sphinxhyphen{}QUARTIC
BONDTYPE        MM2\sphinxhyphen{}STRBND      PISYSTEM
UREY\sphinxhyphen{}CUBIC      UREY\sphinxhyphen{}QUARTIC


\subsection{VAN DER WAALS FUNCTIONAL FORM KEYWORDS}
\label{\detokenize{text/keywords:van-der-waals-functional-form-keywords}}
A\sphinxhyphen{}EXPTERM       B\sphinxhyphen{}EXPTERM       C\sphinxhyphen{}EXPTERM
DELTA\sphinxhyphen{}HALGREN   EPSILONRULE     GAMMA\sphinxhyphen{}HALGREN
GAUSSTYPE       RADIUSRULE      RADIUSSIZE
RADIUSTYPE      VDW\sphinxhyphen{}12\sphinxhyphen{}SCALE    VDW\sphinxhyphen{}13\sphinxhyphen{}SCALE
VDW\sphinxhyphen{}14\sphinxhyphen{}SCALE    VDW\sphinxhyphen{}15\sphinxhyphen{}SCALE    VDW\sphinxhyphen{}CORRECTION
VDWINDEX        VDWTYPE


\subsection{ELECTROSTATICS FUNCTIONAL FORM KEYWORDS}
\label{\detokenize{text/keywords:electrostatics-functional-form-keywords}}
CHG\sphinxhyphen{}12\sphinxhyphen{}SCALE    CHG\sphinxhyphen{}13\sphinxhyphen{}SCALE    CHG\sphinxhyphen{}14\sphinxhyphen{}SCALE
CHG\sphinxhyphen{}15\sphinxhyphen{}SCALE    CHG\sphinxhyphen{}BUFFER      DIELECTRIC
DIRECT\sphinxhyphen{}11\sphinxhyphen{}SCALE DIRECT\sphinxhyphen{}12\sphinxhyphen{}SCALE DIRECT\sphinxhyphen{}13\sphinxhyphen{}SCALE
DIRECT\sphinxhyphen{}14\sphinxhyphen{}SCALE MPOLE\sphinxhyphen{}12\sphinxhyphen{}SCALE  MPOLE\sphinxhyphen{}13\sphinxhyphen{}SCALE
MPOLE\sphinxhyphen{}14\sphinxhyphen{}SCALE  MPOLE\sphinxhyphen{}15\sphinxhyphen{}SCALE  MUTUAL\sphinxhyphen{}11\sphinxhyphen{}SCALE
MUTUAL\sphinxhyphen{}12\sphinxhyphen{}SCALE MUTUAL\sphinxhyphen{}13\sphinxhyphen{}SCALE MUTUAL\sphinxhyphen{}14\sphinxhyphen{}SCALE
POLAR\sphinxhyphen{}12\sphinxhyphen{}SCALE  POLAR\sphinxhyphen{}13\sphinxhyphen{}SCALE  POLAR\sphinxhyphen{}14\sphinxhyphen{}SCALE
POLAR\sphinxhyphen{}15\sphinxhyphen{}SCALE  POLAR\sphinxhyphen{}ASPC      POLAR\sphinxhyphen{}EPS
POLAR\sphinxhyphen{}SOR       POLARIZATION    REACTIONFIELD


\subsection{NONBONDED CUTOFF KEYWORDS}
\label{\detokenize{text/keywords:nonbonded-cutoff-keywords}}
CHG\sphinxhyphen{}CUTOFF      CHG\sphinxhyphen{}TAPER       CUTOFF
DPL\sphinxhyphen{}CUTOFF      DPL\sphinxhyphen{}TAPER       HESS\sphinxhyphen{}CUTOFF
LIGHTS  MPOLE\sphinxhyphen{}CUTOFF    MPOLE\sphinxhyphen{}TAPER
NEIGHBOR\sphinxhyphen{}GROUPS NEUTRAL\sphinxhyphen{}GROUPS  POLYMER\sphinxhyphen{}CUTOFF
TAPER   TRUNCATE        VDW\sphinxhyphen{}CUTOFF
VDW\sphinxhyphen{}TAPER


\subsection{EWALD SUMMATION KEYWORDS}
\label{\detokenize{text/keywords:ewald-summation-keywords}}
EWALD   EWALD\sphinxhyphen{}ALPHA     EWALD\sphinxhyphen{}BOUNDARY
EWALD\sphinxhyphen{}CUTOFF    PME\sphinxhyphen{}GRID        PME\sphinxhyphen{}ORDER


\subsection{CRYSTAL LATTICE \& PERIODIC BOUNDARY KEYWORDS}
\label{\detokenize{text/keywords:crystal-lattice-periodic-boundary-keywords}}
A\sphinxhyphen{}AXIS  B\sphinxhyphen{}AXIS  C\sphinxhyphen{}AXIS
ALPHA   BETA    GAMMA
NO\sphinxhyphen{}SYMMETRY     OCTAHEDRON      SPACEGROUP
X\sphinxhyphen{}AXIS  Y\sphinxhyphen{}AXIS  Z\sphinxhyphen{}AXIS


\subsection{NEIGHBOR LIST KEYWORDS}
\label{\detokenize{text/keywords:neighbor-list-keywords}}
CHG\sphinxhyphen{}LIST        LIST\sphinxhyphen{}BUFFER     MPOLE\sphinxhyphen{}LIST
NEIGHBOR\sphinxhyphen{}LIST   VDW\sphinxhyphen{}LIST


\subsection{OPTIMIZATION KEYWORDS}
\label{\detokenize{text/keywords:optimization-keywords}}
ANGMAX  CAPPA   FCTMIN
HGUESS  INTMAX  LBFGS\sphinxhyphen{}VECTORS
MAXITER NEWHESS NEXTITER
SLOPEMAX        STEEPEST\sphinxhyphen{}DESCENT        STEPMAX
STEPMIN


\subsection{MOLECULAR DYNAMICS KEYWORDS}
\label{\detokenize{text/keywords:molecular-dynamics-keywords}}
BEEMAN\sphinxhyphen{}MIXING   DEGREES\sphinxhyphen{}FREEDOM INTEGRATOR
REMOVE\sphinxhyphen{}INERTIA


\subsection{THERMOSTAT \& BAROSTAT KEYWORDS}
\label{\detokenize{text/keywords:thermostat-barostat-keywords}}
ANISO\sphinxhyphen{}PRESSURE  BAROSTAT        COLLISION
COMPRESS        FRICTION        FRICTION\sphinxhyphen{}SCALING
TAU\sphinxhyphen{}PRESSURE    TAU\sphinxhyphen{}TEMPERATURE THERMOSTAT
VOLUME\sphinxhyphen{}MOVE     VOLUME\sphinxhyphen{}SCALE    VOLUME\sphinxhyphen{}TRIAL


\subsection{TRANSITION STATE KEYWORDS}
\label{\detokenize{text/keywords:transition-state-keywords}}
DIVERGE GAMMAMIN        REDUCE
SADDLEPOINT


\subsection{DISTANCE GEOMETRY KEYWORDS}
\label{\detokenize{text/keywords:distance-geometry-keywords}}
TRIAL\sphinxhyphen{}DISTANCE  TRIAL\sphinxhyphen{}DISTRIBUTION


\subsection{VIBRATIONAL ANALYSIS KEYWORDS}
\label{\detokenize{text/keywords:vibrational-analysis-keywords}}
IDUMP   VIB\sphinxhyphen{}ROOTS       VIB\sphinxhyphen{}TOLERANCE


\subsection{IMPLICIT SOLVATION KEYWORDS}
\label{\detokenize{text/keywords:implicit-solvation-keywords}}
BORN\sphinxhyphen{}RADIUS     GK\sphinxhyphen{}RADIUS       GKC
GKR     SOLVENT\sphinxhyphen{}PRESSURE        SURFACE\sphinxhyphen{}TENSION


\subsection{POISSON\sphinxhyphen{}BOLTZMANN KEYWORDS}
\label{\detokenize{text/keywords:poisson-boltzmann-keywords}}
AGRID   APBS\sphinxhyphen{}GRID       BCFL
CGCENT  CGRID   FGCENT
FGRID   ION     MG\sphinxhyphen{}AUTO
MG\sphinxhyphen{}MANUAL       PB\sphinxhyphen{}RADIUS       PDIE
SDENS   SDIE    SMIN
SRAD    SRFM    SWIN


\subsection{MATHEMATICAL ALGORITHM KEYWORDS}
\label{\detokenize{text/keywords:mathematical-algorithm-keywords}}
FFT\sphinxhyphen{}PACKAGE     RANDOMSEED


\subsection{PARALLELIZATION KEYWORDS}
\label{\detokenize{text/keywords:parallelization-keywords}}
OPENMP\sphinxhyphen{}THREADS


\subsection{FREE ENERGY PERTURBATION KEYWORDS}
\label{\detokenize{text/keywords:free-energy-perturbation-keywords}}
CHG\sphinxhyphen{}LAMBDA      DPL\sphinxhyphen{}LAMBDA      LAMBDA
LIGAND  MPOLE\sphinxhyphen{}LAMBDA    MUTATE
POLAR\sphinxhyphen{}LAMBDA    VDW\sphinxhyphen{}LAMBDA


\subsection{PARTIAL STRUCTURE KEYWORDS}
\label{\detokenize{text/keywords:partial-structure-keywords}}
ACTIVE  GROUP   GROUP\sphinxhyphen{}INTER
GROUP\sphinxhyphen{}INTRA     GROUP\sphinxhyphen{}MOLECULE  GROUP\sphinxhyphen{}SELECT
INACTIVE


\subsection{CONSTRAINT \& RESTRAINT KEYWORDS}
\label{\detokenize{text/keywords:constraint-restraint-keywords}}
BASIN   ENFORCE\sphinxhyphen{}CHIRALITY       RATTLE
RATTLE\sphinxhyphen{}DISTANCE RATTLE\sphinxhyphen{}EPS      RATTLE\sphinxhyphen{}LINE
RATTLE\sphinxhyphen{}ORIGIN   RATTLE\sphinxhyphen{}PLANE    RESTRAIN\sphinxhyphen{}ANGLE
RESTRAIN\sphinxhyphen{}DISTANCE       RESTRAIN\sphinxhyphen{}GROUPS RESTRAIN\sphinxhyphen{}POSITION
RESTRAIN\sphinxhyphen{}TORSION        SPHERE  WALL


\subsection{PARAMETER FITTING KEYWORDS}
\label{\detokenize{text/keywords:parameter-fitting-keywords}}
FIT\sphinxhyphen{}ANGLE       FIT\sphinxhyphen{}BOND        FIT\sphinxhyphen{}OPBEND
FIT\sphinxhyphen{}STRBND      FIT\sphinxhyphen{}TORSION     FIT\sphinxhyphen{}UREY
FIX\sphinxhyphen{}ANGLE       FIX\sphinxhyphen{}BOND        FIX\sphinxhyphen{}DIPOLE
FIX\sphinxhyphen{}MONOPOLE    FIX\sphinxhyphen{}OPBEND      FIX\sphinxhyphen{}QUADRUPOLE
FIX\sphinxhyphen{}STRBND      FIX\sphinxhyphen{}TORSION     FIX\sphinxhyphen{}UREY
POTENTIAL\sphinxhyphen{}ATOMS POTENTIAL\sphinxhyphen{}FIT   POTENTIAL\sphinxhyphen{}OFFSET
POTENTIAL\sphinxhyphen{}SHELLS        POTENTIAL\sphinxhyphen{}SPACING       TARGET\sphinxhyphen{}DIPOLE
TARGET\sphinxhyphen{}QUADRUPOLE


\subsection{POTENTIAL SMOOTHING KEYWORDS}
\label{\detokenize{text/keywords:potential-smoothing-keywords}}
DEFORM  DIFFUSE\sphinxhyphen{}CHARGE  DIFFUSE\sphinxhyphen{}TORSION
DIFFUSE\sphinxhyphen{}VDW     SMOOTHING


\section{Description of Individual Keywords}
\label{\detokenize{text/keywords:description-of-individual-keywords}}
The following is an alphabetical list of the Tinker keywords along with a brief description of the action of each keyword and required or optional parameters that can be used to extend or modify each keyword. The format of possible modifiers, if any, is shown in brackets following each keyword.

\sphinxstylestrong{A\sphinxhyphen{}AXIS {[}real{]}}  Sets the value of the a\sphinxhyphen{}axis length for a crystal unit cell, or, equivalently, the X\sphinxhyphen{}axis length for a periodic box. The length value in Angstroms is listed after the keyword.

\sphinxstylestrong{A\sphinxhyphen{}EXPTERM {[}real{]}}  Sets the value of the “A” premultiplier term in the Buckingham van der Waals function, i.e., the value of A in the formula Evdw = epsilon * \{ A exp{[}\sphinxhyphen{}B(Ro/R){]} \sphinxhyphen{} C (Ro/R)6 \}.

\sphinxstylestrong{ACTIVE {[}integer list{]}}  Sets the list of active atoms during a Tinker computation. Individual potential energy terms are computed when at least one atom involved in the term is active. For Cartesian space calculations, active atoms are those allowed to move. For torsional space calculations, rotations are allowed when all atoms on one side of the rotated bond are active. Multiple ACTIVE lines can be present in the keyfile and are treated cumulatively.  On each line the keyword can be followed by one or more atom numbers or atom ranges. The presence of any ACTIVE keyword overrides any INACTIVE keywords in the keyfile.

\sphinxstylestrong{ALPHA {[}real{]}}  Sets the value of the alpha angle of a crystal unit cell, i.e., the angle between the b\sphinxhyphen{}axis and c\sphinxhyphen{}axis of a unit cell, or, equivalently, the angle between the Y\sphinxhyphen{}axis and Z\sphinxhyphen{}axis of a periodic box. The default value in the absence of the ALPHA keyword is 90 degrees.

\sphinxstylestrong{ANGANG {[}1 integer \& 3 reals{]}}  This keyword provides the values for a single angle\sphinxhyphen{}angle cross term potential parameter.

\sphinxstylestrong{ANGANGTERM {[}NONE/ONLY{]}}  This keyword controls use of the angle\sphinxhyphen{}angle cross term potential energy. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

\sphinxstylestrong{ANGANGUNIT {[}real{]}}  Sets the scale factor needed to convert the energy value computed by the angle\sphinxhyphen{}angle cross term potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default of (Pi/180)\textasciicircum{}2 = 0.0003046 is used, if the ANGANGUNIT keyword is not given in the force field parameter file or the keyfile.

\sphinxstylestrong{ANGLE {[}3 integers \& 4 reals{]}}  This keyword provides the values for a single bond angle bending parameter. The integer modifiers give the atom class numbers for the three kinds of atoms involved in the angle which is to be defined. The real number modifiers give the force constant value for the angle and up to three ideal bond angles in degrees. In most cases only one ideal bond angle is given, and that value is used for all occurrences of the specified bond angle. If all three ideal angles are given, the values apply when the central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms, respectively. This “hydrogen environment” option is provided to implement the corresponding feature of Allinger’s MM force fields. The default units for the force constant are kcal/mole/radian2, but this can be controlled via the ANGLEUNIT keyword.

\sphinxstylestrong{ANGLE\sphinxhyphen{}CUBIC {[}real{]}}  Sets the value of the cubic term in the Taylor series expansion form of the bond angle bending potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. This term multiplied by the angle bending energy unit conversion factor, the force constant, and the cube of the deviation of the bond angle from its ideal value gives the cubic contribution to the angle bending energy. The default value in the absence of the ANGLE\sphinxhyphen{}CUBIC keyword is zero; i.e., the cubic angle bending term is omitted.

\sphinxstylestrong{ANGLE\sphinxhyphen{}PENTIC {[}real{]}}  Sets the value of the fifth power term in the Taylor series expansion form of the bond angle bending potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. This term multiplied by the angle bending energy unit conversion factor, the force constant, and the fifth power of the deviation of the bond angle from its ideal value gives the pentic contribution to the angle bending energy. The default value in the absence of the ANGLE\sphinxhyphen{}PENTIC keyword is zero; i.e., the pentic angle bending term is omitted.

\sphinxstylestrong{ANGLE\sphinxhyphen{}QUARTIC {[}real{]}}  Sets the value of the quartic term in the Taylor series expansion form of the bond angle bending potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. This term multiplied by the angle bending energy unit conversion factor, the force constant, and the forth power of the deviation of the bond angle from its ideal value gives the quartic contribution to the angle bending energy. The default value in the absence of the ANGLE\sphinxhyphen{}QUARTIC keyword is zero; i.e., the quartic angle bending term is omitted.

\sphinxstylestrong{ANGLE\sphinxhyphen{}SEXTIC {[}real{]}}  Sets the value of the sixth power term in the Taylor series expansion form of the bond angle bending potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. This term multiplied by the angle bending energy unit conversion factor, the force constant, and the sixth power of the deviation of the bond angle from its ideal value gives the sextic contribution to the angle bending energy. The default value in the absence of the ANGLE\sphinxhyphen{}SEXTIC keyword is zero; i.e., the sextic angle bending term is omitted.

\sphinxstylestrong{ANGLE3 {[}3 integers \& 4 reals{]}}  This keyword provides the values for a single bond angle bending parameter specific to atoms in 3\sphinxhyphen{}membered rings. The integer modifiers give the atom class numbers for the three kinds of atoms involved in the angle which is to be defined. The real number modifiers give the force constant value for the angle and up to three ideal bond angles in degrees. If all three ideal angles are given, the values apply when the central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms, respectively. The default units for the force constant are kcal/mole/radian\textasciicircum{}2, but this can be controlled via the ANGLEUNIT keyword. If any ANGLE3 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special ANGLE3 parameters be given for all angles in 3\sphinxhyphen{}membered rings. In the absence of any ANGLE3 keywords, standard ANGLE parameters will be used for bonds in 3\sphinxhyphen{}membered rings.

\sphinxstylestrong{ANGLE4 {[}3 integers \& 4 reals{]}}  This keyword provides the values for a single bond angle bending parameter specific to atoms in 4\sphinxhyphen{}membered rings. The integer modifiers give the atom class numbers for the three kinds of atoms involved in the angle which is to be defined. The real number modifiers give the force constant value for the angle and up to three ideal bond angles in degrees. If all three ideal angles are given, the values apply when the central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms, respectively. The default units for the force constant are kcal/mole/radian\textasciicircum{}2, but this can be controlled via the ANGLEUNIT keyword. If any ANGLE4 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special ANGLE4 parameters be given for all angles in 4\sphinxhyphen{}membered rings. In the absence of any ANGLE4 keywords, standard ANGLE parameters will be used for bonds in 4\sphinxhyphen{}membered rings.

\sphinxstylestrong{ANGLE5 {[}3 integers \& 4 reals{]}}  This keyword provides the values for a single bond angle bending parameter specific to atoms in 5\sphinxhyphen{}membered rings. The integer modifiers give the atom class numbers for the three kinds of atoms involved in the angle which is to be defined. The real number modifiers give the force constant value for the angle and up to three ideal bond angles in degrees. If all three ideal angles are given, the values apply when the central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms, respectively. The default units for the force constant are kcal/mole/radian\textasciicircum{}2, but this can be controlled via the ANGLEUNIT keyword. If any ANGLE5 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special ANGLE5 parameters be given for all angles in 5\sphinxhyphen{}membered rings. In the absence of any ANGLE5 keywords, standard ANGLE parameters will be used for bonds in 5\sphinxhyphen{}membered rings.

\sphinxstylestrong{ANGLEF {[}3 integers \& 3 reals{]}}  This keyword provides the values for a single bond angle bending parameter for a SHAPES\sphinxhyphen{}style Fourier potential function. The integer modifiers give the atom class numbers for the three kinds of atoms involved in the angle which is to be defined. The real number modifiers give the force constant value for the angle, the angle shift in degrees, and the periodicity value. Note that the force constant should be given as the “harmonic” value and not the native Fourier value. The default units for the force constant are kcal/mole/radian\textasciicircum{}2, but this can be controlled via the ANGLEUNIT keyword.

\sphinxstylestrong{ANGLETERM {[}NONE/ONLY{]}}  This keyword controls use of the bond angle bending potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

\sphinxstylestrong{ANGLEUNIT {[}real{]}}  Sets the scale factor needed to convert the energy value computed by the bond angle bending potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of (Pi/180)\textasciicircum{}2 = 0.0003046 is used, if the ANGLEUNIT keyword is not given in the force field parameter file or the keyfile.

\sphinxstylestrong{ANGMAX {[}real{]}}  Set the maximum permissible angle between the current optimization search direction and the negative of the gradient direction. If this maximum angle value is exceeded, the optimization routine will note an error condition and may restart from the steepest descent direction. The default value in the absence of the ANGMAX keyword is usually 88 degrees for conjugate gradient methods and 180 degrees (i.e., disabled) for variable metric optimizations.

\sphinxstylestrong{ANISO\sphinxhyphen{}PRESSURE}  This keyword invokes use of full anisotropic pressure during dynamics simulations. When using this option, the three axis lengths and axis angles vary separately in response to the pressure tensor. The default, in the absence of the keyword, is isotropic pressure based on the average of the diagonal of the pressure tensor.

\sphinxstylestrong{ARCHIVE}  This keyword causes Tinker molecular dynamics\sphinxhyphen{}based programs to write trajectories directly to a single plain\sphinxhyphen{}text archive file with the .arc format. If an archive file already exists at the start of the calculation, then the newly generated trajectory is appended to the end of the existing file. The default in the absence of this keyword is to write the trajectory snapshots to consecutively numbered cycle files.

\sphinxstylestrong{ATOM {[}2 integers, name, quoted string, integer, real \& integer{]}}  This keyword provides the values needed to define a single force field atom type.

B\sphinxhyphen{}AXIS {[}real{]}     Sets the value of the b\sphinxhyphen{}axis length for a crystal unit cell, or, equivalently,  the Y\sphinxhyphen{}axis length for a periodic box. The length value in Angstroms is listed after the keyword. If the keyword is absent, the b\sphinxhyphen{}axis length is set equal to the a\sphinxhyphen{}axis length.

B\sphinxhyphen{}EXPTERM {[}real{]}     Sets the value of the “B” exponential factor in the Buckingham van der Waals function, i.e., the value of B in the formula Evdw = epsilon * \{ A exp{[}\sphinxhyphen{}B(Ro/R){]} \sphinxhyphen{} C (Ro/R)6 \}.

BAROSTAT {[}BERENDSEN{]}     This keyword selects a barostat algorithm for use during molecular dynamics. At present only one modifier is available, a Berendsen bath coupling method. The default in the absence of the BAROSTAT keyword is to use the BERENDSEN algorithm.

BASIN {[}2 reals{]}     Presence of this keyword turns on a “basin” restraint potential function that serves to drive the system toward a compact structure. The actual function is a Gaussian of the form Ebasin = epsilon * A exp{[}\sphinxhyphen{}B R\textasciicircum{}2{]}, summed over all pairs of atoms where R is the distance between atoms. The A and B values are the depth and width parameters given as modifiers to the BASIN keyword. This potential is currently used to control the degree of expansion during potential energy smooth procedures through the use of shallow, broad basins.

BETA {[}real{]}     Sets the value of the ? angle of a crystal unit cell, i.e., the angle between the a\sphinxhyphen{}axis and c\sphinxhyphen{}axis of a unit cell, or, equivalently, the angle between the X\sphinxhyphen{}axis and Z\sphinxhyphen{}axis of a periodic box. The default value in the absence of the BETA keyword is to set the beta angle equal to the alpha angle as given by the keyword ALPHA.

BIOTYPE {[}integer, name, quoted string \& integer{]}     This keyword provides the values to define the correspondence between a single biopolymer atom type and its force field atom type.

BOND {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond stretching parameter. The integer modifiers give the atom class numbers for the two kinds of atoms involved in the bond which is to be defined. The real number modifiers give the force constant value for the bond and the ideal bond length in Angstroms. The default units for the force constant are kcal/mole/Ang\textasciicircum{}2, but this can be controlled via the BONDUNIT keyword.

BOND\sphinxhyphen{}CUBIC {[}real{]}     Sets the value of the cubic term in the Taylor series expansion form of the bond stretching potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. This term multiplied by the bond stretching energy unit conversion factor, the force constant, and the cube of the deviation of the bond length from its ideal value gives the cubic contribution to the bond stretching energy. The default value in the absence of the BOND\sphinxhyphen{}CUBIC keyword is zero; i.e., the cubic bond stretching term is omitted.

BOND\sphinxhyphen{}QUARTIC {[}real{]}     Sets the value of the quartic term in the Taylor series expansion form of the bond stretching potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. This term multiplied by the bond stretching energy unit conversion factor, the force constant, and the forth power of the deviation of the bond length from its ideal value gives the quartic contribution to the bond stretching energy. The default value in the absence of the BOND\sphinxhyphen{}QUARTIC keyword is zero; i.e., the quartic bond stretching term is omitted.

BOND3 {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond stretching parameter specific to atoms in 3\sphinxhyphen{}membered rings. The integer modifiers give the atom class numbers for the two kinds of atoms involved in the bond which is to be defined. The real number modifiers give the force constant value for the bond and the ideal bond length in Angstroms. The default units for the force constant are kcal/mole/Ang\textasciicircum{}2, but this can be controlled via the BONDUNIT keyword. If any BOND3 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special BOND3 parameters be given for all bonds in 3\sphinxhyphen{}membered rings. In the absence of any BOND3 keywords, standard BOND parameters will be used for bonds in 3\sphinxhyphen{}membered rings.

BOND4 {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond stretching parameter specific to atoms in 4\sphinxhyphen{}membered rings. The integer modifiers give the atom class numbers for the two kinds of atoms involved in the bond which is to be defined. The real number modifiers give the force constant value for the bond and the ideal bond length in Angstroms. The default units for the force constant are kcal/mole/Ang\textasciicircum{}2, but this can be controlled via the BONDUNIT keyword. If any BOND4 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special BOND4 parameters be given for all bonds in 4\sphinxhyphen{}membered rings. In the absence of any BOND4 keywords, standard BOND parameters will be used for bonds in 4\sphinxhyphen{}membered rings

BOND5 {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond stretching parameter specific to atoms in 5\sphinxhyphen{}membered rings. The integer modifiers give the atom class numbers for the two kinds of atoms involved in the bond which is to be defined. The real number modifiers give the force constant value for the bond and the ideal bond length in Angstroms. The default units for the force constant are kcal/mole/Ang\textasciicircum{}2, but this can be controlled via the BONDUNIT keyword. If any BOND5 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special BOND5 parameters be given for all bonds in 5\sphinxhyphen{}membered rings. In the absence of any BOND5 keywords, standard BOND parameters will be used for bonds in 5\sphinxhyphen{}membered rings

BONDTERM {[}NONE/ONLY{]}     This keyword controls use of the bond stretching potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

BONDTYPE {[}TAYLOR/MORSE/GAUSSIAN{]}     Chooses the functional form of the bond stretching potential. The TAYLOR option selects a Taylor series expansion containing terms from harmonic through quartic. The MORSE option selects a Morse potential fit to the ideal bond length and stretching force constant parameter values. The GAUSSIAN option uses an inverted Gaussian with amplitude equal to the Morse bond dissociation energy and width set to reproduce the vibrational frequency of a harmonic potential. The default is to use the TAYLOR potential.

BONDUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the bond stretching potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the BONDUNIT keyword is not given in the force field parameter file or the keyfile.

C\sphinxhyphen{}AXIS {[}real{]}     Sets the value of the C\sphinxhyphen{}axis length for a crystal unit cell, or, equivalently, the Z\sphinxhyphen{}axis length for a periodic box. The length value in Angstroms is listed after the keyword. If the keyword is absent, the C\sphinxhyphen{}axis length is set equal to the A\sphinxhyphen{}axis length.

C\sphinxhyphen{}EXPTERM {[}real{]}     Sets the value of the “C” dispersion multiplier in the Buckingham van der Waals function, i.e., the value of C in the formula Evdw = epsilon * \{ A exp{[}\sphinxhyphen{}B(Ro/R){]} \sphinxhyphen{} C (Ro/R)6 \}.

CAPPA {[}real{]}     This keyword is used to set the normal termination criterion for the line search phase of Tinker optimization routines. The line search exits successfully if the ratio of the current gradient projection on the line to the projection at the start of the line search falls below the value of CAPPA. A default value of 0.1 is used in the absence of the CAPPA keyword.

CHARGE {[}1 integer \& 1 real{]}     This keyword provides a value for a single atomic partial charge electrostatic parameter. The integer modifier, if positive, gives the atom type number for which the charge parameter is to be defined. Note that charge parameters are given for atom types, not atom classes. If the integer modifier is negative, then the parameter value to follow applies only to the individual atom whose atom number is the negative of the modifier. The real number modifier gives the values of the atomic partial charge in electrons.

CHARGETERM {[}NONE/ONLY{]}     This keyword controls use of the charge\sphinxhyphen{}charge potential energy term between pairs of atomic partial charges. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

CHG\sphinxhyphen{}12\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to charge\sphinxhyphen{}charge electrostatic interactions between 1\sphinxhyphen{}2 connected atoms, i.e., atoms that are directly bonded. The default value of 0.0 is used, if the CHG\sphinxhyphen{}12\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

CHG\sphinxhyphen{}13\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to charge\sphinxhyphen{}charge electrostatic interactions between 1\sphinxhyphen{}3 connected atoms, i.e., atoms separated by two covalent bonds. The default value of 0.0 is used, if the CHG\sphinxhyphen{}13\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

CHG\sphinxhyphen{}14\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to charge\sphinxhyphen{}charge electrostatic interactions between 1\sphinxhyphen{}4 connected atoms, i.e., atoms separated by three covalent bonds. The default value of 1.0 is used, if the CHG\sphinxhyphen{}14\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

CHG\sphinxhyphen{}15\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to charge\sphinxhyphen{}charge electrostatic interactions between 1\sphinxhyphen{}5 connected atoms, i.e., atoms separated by four covalent bonds. The default value of 1.0 is used, if the CHG\sphinxhyphen{}15\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

CHG\sphinxhyphen{}CUTOFF {[}real{]}     Sets the cutoff distance value in Angstroms for charge\sphinxhyphen{}charge electrostatic potential energy interactions. The energy for any pair of sites beyond the cutoff distance will be set to zero. Other keywords can be used to select a smoothing scheme near the cutoff distance. The default cutoff distance in the absence of the CHG\sphinxhyphen{}CUTOFF keyword is infinite for nonperiodic systems and 9.0 for periodic systems.

CHG\sphinxhyphen{}TAPER {[}real{]}     This keyword allows modification of the cutoff window for charge\sphinxhyphen{}charge electrostatic potential energy interactions. It is similar in form and action to the TAPER keyword, except that its value applies only to the charge\sphinxhyphen{}charge potential. The default value in the absence of the CHG\sphinxhyphen{}TAPER keyword is to begin the cutoff window at 0.65 of the corresponding cutoff distance.

CHGDPLTERM {[}NONE/ONLY{]}     This keyword controls use of the charge\sphinxhyphen{}dipole potential energy term between atomic partial charges and bond dipoles. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

COLLISION {[}real{]}     Sets the value of the random collision frequency used in the Andersen stochastic collision dynamics thermostat. The supplied value has units of fs\sphinxhyphen{}1 atom\sphinxhyphen{}1 and is multiplied internal to Tinker by the time step in fs and N\textasciicircum{}2/3 where N is the number of atoms. The default value used in the absence of the COLLISION keyword is 0.1 which is appropriate for many systems but may need adjustment to achieve adequate temperature control without perturbing the dynamics.

COMPRESS {[}real{]}     Sets the value of the bulk solvent isothermal compressibility in 1/Atm for use during pressure computation and scaling in molecular dynamics computations. The default value used in the absence of the COMPRESS keyword is 0.000046, appropriate for water. This parameter serves as a scale factor for the Groningen\sphinxhyphen{}style pressure bath coupling time, and its exact value should not be of critical importance.

CUTOFF {[}real{]}     Sets the cutoff distance value for all nonbonded potential energy interactions. The energy for any of the nonbonded potentials of a pair of sites beyond the cutoff distance will be set to zero. Other keywords can be used to select a smoothing scheme near the cutoff distance, or to apply different cutoff distances to various nonbonded energy terms.

DEBUG     Turns on printing of detailed information and intermediate values throughout the progress of a Tinker computation; not recommended for use with large structures or full potential energy functions since a summary of every individual interaction will usually be output.

DEFORM {[}real{]}     Sets the amount of diffusion equation\sphinxhyphen{}style smoothing that will be applied to the potential energy surface when using the SMOOTH force field. The real number option is equivalent to the “time” value in the original Piela, et al. formalism; the larger the value, the greater the smoothing. The default value is zero, meaning that no smoothing will be applied.

DEGREES\sphinxhyphen{}FREEDOM {[}integer{]}     This keyword allows manual setting of the number of degrees of freedom during a dynamics calculation. The integer modifier is used by thermostating methods and in other places as the number of degrees of freedom, overriding the value determined by the Tinker code at dynamics startup. In the absence of the keyword, the programs will automatically compute the correct value based on the number of atoms active during dynamics, bond or other constrains, and use of periodic boundary conditions.

DELTA\sphinxhyphen{}HALGREN {[}real{]}     Sets the value of the delta parameter in Halgren’s buffered 14\sphinxhyphen{}7 vdw potential energy functional form. In the absence of the DELTA\sphinxhyphen{}HALGREN keyword, a default value of 0.07 is used.

DIELECTRIC {[}real{]}     Sets the value of the bulk dielectric constant used to damp all electrostatic interaction energies for any of the Tinker electrostatic potential functions. The default value is force field dependent, but is usually equal to 1.0 (for Allinger’s MM force fields the default is 1.5).

DIFFUSE\sphinxhyphen{}CHARGE {[}real{]}     This keyword is used during potential function smoothing procedures to specify the effective diffusion coefficient to be applied to the smoothed form of the Coulomb’s Law charge\sphinxhyphen{}charge potential function. In the absence of the DIFFUSE\sphinxhyphen{}CHARGE keyword, a default value of 3.5 is used.

DIFFUSE\sphinxhyphen{}TORSION {[}real{]}     This keyword is used during potential function smoothing procedures to specify the effective diffusion coefficient to be applied to the smoothed form of the torsion angle potential function. In the absence of the DIFFUSE\sphinxhyphen{}TORSION keyword, a default value of 0.0225 is used.

DIFFUSE\sphinxhyphen{}VDW {[}real{]}     This keyword is used during potential function smoothing procedures to specify the effective diffusion coefficient to be applied to the smoothed Gaussian approximation to the Lennard\sphinxhyphen{}Jones van der Waals potential function. In the absence of the DIFFUSE\sphinxhyphen{}VDW keyword, a default value of 1.0 is used.

DIGITS {[}integer{]}     This keyword controls the number of digits of precision  output by Tinker in reporting potential energies and atomic coordinates. The allowed values for the integer modifier are 4, 6 and 8. Input values less than 4 will be set to 4, and those greater than 8 will be set to 8. Final energy values reported by most Tinker programs will contain the specified number of digits to the right of the decimal point. The number of decimal places to be output for atomic coordinates is generally two larger than the value of DIGITS. In the absence of the DIGITS keyword a default value of 4 is used, and  energies will be reported to 4 decimal places with coordinates to 6 decimal places.

DIPOLE {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond dipole electrostatic parameter. The integer modifiers give the atom type numbers for the two kinds of atoms involved in the bond dipole which is to be defined. The real number modifiers give the value of the bond dipole in Debyes and the position of the dipole site along the bond. If the bond dipole value is positive, then the first of the two atom types is the positive end of the dipole. For a negative bond dipole value, the first atom type listed is negative. The position along the bond is an optional modifier that gives the postion of the dipole site as a fraction between the first atom type (position=0) and the second atom type (position=1). The default for the dipole position in the absence of a specified value is 0.5, placing the dipole at the midpoint of the bond.

DIPOLE3 {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond dipole electrostatic parameter specific to atoms in 3\sphinxhyphen{}membered rings. The integer modifiers give the atom type numbers for the two kinds of atoms involved in the bond dipole which is to be defined. The real number modifiers give the value of the bond dipole in Debyes and the position of the dipole site along the bond. The default for the dipole position in the absence of a specified value is 0.5, placing the dipole at the midpoint of the bond. If any DIPOLE3 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special DIPOLE3 parameters be given for all bond dipoles in 3\sphinxhyphen{}membered rings. In the absence of any DIPOLE3 keywords, standard DIPOLE parameters will be used for bonds in 3\sphinxhyphen{}membered rings.

DIPOLE4 {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond dipole electrostatic parameter specific to atoms in 4\sphinxhyphen{}membered rings. The integer modifiers give the atom type numbers for the two kinds of atoms involved in the bond dipole which is to be defined. The real number modifiers give the value of the bond dipole in Debyes and the position of the dipole site along the bond. The default for the dipole position in the absence of a specified value is 0.5, placing the dipole at the midpoint of the bond. If any DIPOLE4 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special DIPOLE4 parameters be given for all bond dipoles in 4\sphinxhyphen{}membered rings. In the absence of any DIPOLE4 keywords, standard DIPOLE parameters will be used for bonds in 4\sphinxhyphen{}membered rings.

DIPOLE5 {[}2 integers \& 2 reals{]}     This keyword provides the values for a single bond dipole electrostatic parameter specific to atoms in 5\sphinxhyphen{}membered rings. The integer modifiers give the atom type numbers for the two kinds of atoms involved in the bond dipole which is to be defined. The real number modifiers give the value of the bond dipole in Debyes and the position of the dipole site along the bond. The default for the dipole position in the absence of a specified value is 0.5, placing the dipole at the midpoint of the bond. If any DIPOLE5 keywords are present, either in the master force field parameter file or the keyfile, then Tinker requires that special DIPOLE5 parameters be given for all bond dipoles in 5\sphinxhyphen{}membered rings. In the absence of any DIPOLE5 keywords, standard DIPOLE parameters will be used for bonds in 5\sphinxhyphen{}membered rings.

DIPOLETERM {[}NONE/ONLY{]}     This keyword controls use of the dipole\sphinxhyphen{}dipole potential energy term between pairs of bond dipoles. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

DIRECT\sphinxhyphen{}11\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the permanent (direct) field due to atoms within a polarization group during an induced dipole calculation, i.e., atoms that are in the same polarization group as the atom being polarized. The default value of 0.0 is used, if the DIRECT\sphinxhyphen{}11\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

DIRECT\sphinxhyphen{}12\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the permanent (direct) field due to atoms in 1\sphinxhyphen{}2 polarization groups during an induced dipole calculation, i.e., atoms that are in polarization groups directly connected to the group containing the atom being polarized. The default value of 0.0 is used, if the DIRECT\sphinxhyphen{}12\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

DIRECT\sphinxhyphen{}13\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the permanent (direct) field due to atoms in 1\sphinxhyphen{}3 polarization groups during an induced dipole calculation, i.e., atoms that are in polarization groups separated by one group from the group containing the atom being polarized. The default value of 0.0 is used, if the DIRECT\sphinxhyphen{}13\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

DIRECT\sphinxhyphen{}14\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the permanent (direct) field due to atoms in 1\sphinxhyphen{}4 polarization groups during an induced dipole calculation, i.e., atoms that are in polarization groups separated by two groups from the group containing the atom being polarized. The default value of 1.0 is used, if the DIRECT\sphinxhyphen{}14\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

DIVERGE {[}real{]}     This keyword is used by the SADDLE program to set the maximum allowed value of the ratio of the gradient length along the path to the total gradient norm at the end of a cycle of minimization perpendicular to the path. If the value provided by the DIVERGE keyword is exceeded, then another cycle of maximization along the path is required. A default value of 0.005 is used in the absence of the DIVERGE keyword.

DPL\sphinxhyphen{}CUTOFF {[}real{]}     Sets the cutoff distance value in Angstroms for bond dipole\sphinxhyphen{}bond dipole electrostatic potential energy interactions. The energy for any pair of bond dipole sites beyond the cutoff distance will be set to zero. Other keywords can be used to select a smoothing scheme near the cutoff distance. The default cutoff distance in the absence of the DPL\sphinxhyphen{}CUTOFF keyword is essentially infinite for nonperiodic systems and 10.0 for periodic systems.

DPL\sphinxhyphen{}TAPER {[}real{]}     This keyword allows modification of the cutoff windows for bond dipole\sphinxhyphen{}bond dipole electrostatic potential energy interactions. It is similar in form and action to the TAPER keyword, except that its value applies only to the vdw potential. The default value in the absence of the DPL\sphinxhyphen{}TAPER keyword is to begin the cutoff window at 0.75 of the dipole cutoff distance.

ECHO {[}text string{]}     The presence of this keyword causes whatever text follows it on the line to be copied directly to the output file. This keyword is also active in parameter files. It has no default value; if no text follows the ECHO keyword, a blank line is placed in the output file.

ELECTNEG {[}3 integers \& 1 real{]}     This keyword provides the values for a single electronegativity bond length correction parameter. The first two integer modifiers give the atom class numbers of the atoms involved in the bond to be corrected. The third integer modifier is the atom class of an electronegative atom. In the case of a primary correction, an atom of this third class must be directly bonded to an atom of the second atom class. For a secondary correction, the third class is one atom removed from an atom of the second class. The real number modifier is the value in Angstroms by which the original ideal bond length is to be corrected.

ENFORCE\sphinxhyphen{}CHIRALITY     This keyword causes the chirality found at chiral tetravalent centers in the input structure to be maintained during Tinker calculations. The test for chirality is not exhaustive; two identical monovalent atoms connected to a center cause it to be marked as non\sphinxhyphen{}chiral, but large equivalent substituents are not detected. Trivalent “chiral” centers, for example the alpha carbon in united\sphinxhyphen{}atom protein structures, are not enforced as chiral.

EPSILONRULE {[}GEOMETRIC/ARITHMETIC/HARMONIC/HHG{]}     This keyword selects the combining rule used to derive the ? value for van der Waals interactions. The default in the absence of the EPSILONRULE keyword is to use the GEOMETRIC mean of the individual epsilon values of the two atoms involved in the van der Waals interaction.

EWALD     This keyword turns on the use of Ewald summation during computation of electrostatic interactions in periodic systems. In the current version of Tinker, regular Ewald is used for polarizable atomic multipoles, and smooth particle mesh Ewald (PME) is used for charge\sphinxhyphen{}charge interactions. Ewald summation is not available for interactions involving bond\sphinxhyphen{}centered dipoles. By default, in the absence of the EWALD keyword, distance\sphinxhyphen{}based cutoffs are used for electrostatic interactions.

EWALD\sphinxhyphen{}ALPHA {[}real{]}     Sets the value of the Ewald coefficient which controls the width of the Gaussian screening charges during particle mesh Ewald summation. In the absence of the EWALD\sphinxhyphen{}ALPHA keyword, a value is chosen which causes interactions outside the real\sphinxhyphen{}space cutoff to be below a fixed tolerance. For most standard applications of Ewald summation, the program default should be used.

EWALD\sphinxhyphen{}BOUNDARY     This keyword invokes the use of insulating (ie, vacuum) boundary conditions during Ewald summation, corresponding to the media surrounding the system having a dielectric value of 1. The default in the absence of the EWALD\sphinxhyphen{}BOUNDARY keyword is to use conducting (ie, tinfoil) boundary conditions where the surrounding media is assumed to have an infinite dielectric value.

EWALD\sphinxhyphen{}CUTOFF {[}real{]}     Sets the value in Angstroms of the real\sphinxhyphen{}space distance cutoff for use during Ewald summation. By default, in the absence of the EWALD\sphinxhyphen{}CUTOFF keyword, a value of 9.0 is used.

EXIT\sphinxhyphen{}PAUSE     This keyword causes Tinker programs to pause and wait for a carriage return at the end of executation prior to returning control to the operating system. This is useful to keep the execution window open following termination on machines running Microsoft Windows or Apple MacOS. The default in the absence of the EXIT\sphinxhyphen{}PAUSE keyword, is to return control to the operating system immediately at program termination.

EXTRATERM {[}NONE/ONLY{]}     This keyword controls use of the user defined extra potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

FCTMIN {[}real{]}     This keyword sets a convergence criterion for successful completion of a Tinker optimization. If the value of the optimization objective function, typically the potential energy, falls below the value set by FCTMIN, then the optimization is deemed to have converged. The default value in the absence of the FCTMIN keyword is \sphinxhyphen{}1000000, effectively removing this criterion as a possible agent for termination.

FORCEFIELD {[}name{]}     This keyword provides a name for the force field to be used in the current calculation. Its value is usually set in the master force field parameter file for the calculation (see the PARAMETERS keyword) instead of in the keyfile.

FRICTION {[}real{]}     Sets the value of the frictional coefficient in 1/ps for use with stochastic dynamics. The default value used in the absence of the FRICTION keyword is 91.0, which is generally appropriate for water.

FRICTION\sphinxhyphen{}SCALING     This keyword turns on the use of atomic surface area\sphinxhyphen{}based scaling of the frictional coefficient during stochastic dynamics. When in use, the coefficient for each atom is multiplied by that atom’s fraction of exposed surface area. The default in the absence of the keyword is to omit the scaling and use the full coefficient value for each atom.

GAMMA {[}real{]}     Sets the value of the gamma angle of a crystal unit cell, i.e., the angle between the a\sphinxhyphen{}axis and b\sphinxhyphen{}axis of a unit cell, or, equivalently, the angle between the X\sphinxhyphen{}axis and Y\sphinxhyphen{}axis of a periodic box. The default value in the absence of the GAMMA keyword is to set the gamma angle equal to the gamma angle as given by the keyword ALPHA.

GAMMA\sphinxhyphen{}HALGREN {[}real{]}     Sets the value of the gamma parameter in Halgren’s buffered 14\sphinxhyphen{}7 vdw potential energy functional form. In the absence of the GAMMA\sphinxhyphen{}HALGREN keyword, a default value of 0.12 is used.

GAMMAMIN {[}real{]}     Sets the convergence target value for gamma during searches for maxima along the quadratic synchronous transit used by the SADDLE program. The value of gamma is the square of the ratio of the gradient projection along the path to the total gradient. A default value of 0.00001 is used in the absence of the GAMMAMIN keyword.

GAUSSTYPE {[}LJ\sphinxhyphen{}2/LJ\sphinxhyphen{}4/MM2\sphinxhyphen{}2/MM3\sphinxhyphen{}2/IN\sphinxhyphen{}PLACE{]}     This keyword specifies the underlying vdw form that a Gaussian vdw approximation will attempt to fit as the number of terms to be used in a Gaussian approximation of the Lennard\sphinxhyphen{}Jones van der Waals potential. The text modifier gives the name of the functional form to be used. Thus LJ\sphinxhyphen{}2 as a modifier will result in a 2\sphinxhyphen{}Gaussian fit to a Lennard\sphinxhyphen{}Jones vdw potential. The GAUSSTYPE keyword only takes effect when VDWTYPE is set to GAUSSIAN. This keyword has no default value.

GROUP {[}integer, integer list{]}     This keyword defines an atom group as a substructure within the full input molecular structure. The value of the first integer is the group number which must be in the range from 1 to the maximum number of allowed groups. The remaining intergers give the atom or atoms contained in this group as one or more atom numbers or ranges. Multiple keyword lines can be used to specify additional atoms in the same group. Note that an atom can only be in one group, the last group to which it is assigned is the one used.

GROUP\sphinxhyphen{}INTER     This keyword assigns a value of 1.0 to all inter\sphinxhyphen{}group interactions and a value of 0.0 to all intra\sphinxhyphen{}group interactions. For example, combination with the GROUP\sphinxhyphen{}MOLECULE keyword provides for rigid\sphinxhyphen{}body calculations.

GROUP\sphinxhyphen{}INTRA     This keyword assigns a value of 1.0 to all intra\sphinxhyphen{}group interactions and a value of 0.0 to all inter\sphinxhyphen{}group interactions.

GROUP\sphinxhyphen{}MOLECULE     This keyword sets each individual molecule in the system to be a separate atom group, but does not assign weights to group\sphinxhyphen{}group interactions.

GROUP\sphinxhyphen{}SELECT {[}2 integers, real{]}     This keyword gives the weight in the final potential energy of a specified set of intra\sphinxhyphen{} or intergroup interactions. The integer modifiers give the group numbers of the groups involved. If the two numbers are the same, then an intragroup set of interactions is specified. The real modifier gives the weight by which all energetic interactions in this set will be multiplied before incorporation into the final potential energy. If omitted as a keyword modifier, the weight will be set to 1.0 by default. If any SELECT\sphinxhyphen{}GROUP keywords are present, then any set of interactions not specified in a SELECT\sphinxhyphen{}GROUP keyword is given a zero weight. The default when no SELECT\sphinxhyphen{}GROUP keywords are specified is to use all intergroup interactions with a weight of 1.0 and to set all intragroup interactions to zero.

HBOND {[}2 integers \& 2 reals{]}     This keyword provides the values for the MM3\sphinxhyphen{}style directional hydrogen bonding parameters for a single pair of atoms. The integer modifiers give the pair of atom class numbers for which hydrogen bonding parameters are to be defined. The two real number modifiers give the values of the minimum energy contact distance in Angstroms and the well depth at the minimum distance in kcal/mole.

HESS\sphinxhyphen{}CUTOFF {[}real{]}     This keyword defines a lower limit for significant Hessian matrix elements. During computation of the Hessian matrix of partial second derivatives, any matrix elements with absolute value below HESS\sphinxhyphen{}CUTOFF will be set to zero and omitted from the sparse matrix Hessian storage scheme used by Tinker. For most calculations, the default in the absence of this keyword is zero, i.e., all elements will be stored. For most Truncated Newton optimizations the Hessian cutoff will be chosen dynamically by the optimizer.

HGUESS {[}real{]}     Sets an initial guess for the average value of the diagonal elements of the scaled inverse Hessian matrix used by the optimally conditioned variable metric optimization routine. A default value of 0.4 is used in the absence of the HGUESS keyword.

IMPROPER {[}4 integers \& 2 reals{]}     This keyword provides the values for a single CHARMM\sphinxhyphen{}style improper dihedral angle parameter.

IMPROPTERM {[}NONE/ONLY{]}     This keyword controls use of the CHARMM\sphinxhyphen{}style improper dihedral angle potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

IMPROPUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the CHARMM\sphinxhyphen{}style improper dihedral angle potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the IMPROPUNIT keyword is not given in the force field parameter file or the keyfile.

IMPTORS {[}4 integers \& up to 3 real/real/integer triples{]}     This keyword provides the values for a single AMBER\sphinxhyphen{}style improper torsional angle parameter. The first four integer modifiers give the atom class numbers for the atoms involved in the improper torsional angle to be defined. By convention, the third atom class of the four is the trigonal atom on which the improper torsion is centered. The torsional angle computed is literally that defined by the four atom classes in the order specified by the keyword. Each of the remaining triples of real/real/integer modifiers give the half\sphinxhyphen{}amplitude, phase offset in degrees and periodicity of a particular improper torsional term, respectively. Periodicities through 3\sphinxhyphen{}fold are allowed for improper torsional parameters.

IMPTORSTERM {[}NONE/ONLY{]}     This keyword controls use of the AMBER\sphinxhyphen{}style improper torsional angle potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

IMPTORSUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the AMBER\sphinxhyphen{}style improper torsional angle potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the IMPTORSUNIT keyword is not given in the force field parameter file or the keyfile.

INACTIVE {[}integer list{]}     Sets the list of inactive atoms during a Tinker computation. Individual potential energy terms are not computed when all atoms involved in the term are inactive. For Cartesian space calculations, inactive atoms are not allowed to move. For torsional space calculations, rotations are not allowed when there are inactive atoms on both sides of the rotated bond. Multiple INACTIVE lines can be present in the keyfile, and on each line the keyword can be followed by one or more atom numbers or ranges. If any INACTIVE keys are found, all atoms are set to active except those listed on the INACTIVE lines. The ACTIVE keyword overrides all INACTIVE keywords found in the keyfile.

INTEGRATE {[}VERLET/BEEMAN/STOCHASTIC/RIGIDBODY{]}     Chooses the integration method for propagation of dynamics trajectories. The keyword is followed on the same line by the name of the option. Standard Newtonian MD can be run using either VERLET for the Velocity Verlet method, or BEEMAN for the velocity form of Bernie Brook’s “Better Beeman” method. A Velocity Verlet\sphinxhyphen{}based stochastic dynamics trajectory is selected by the STOCHASTIC modifier. A rigid\sphinxhyphen{}body dynamics method is selected by the RIGIDBODY modifier. The default integration scheme is MD using the BEEMAN method.

INTMAX {[}integer{]}     Sets the maximum number of interpolation cycles that will be allowed during the line search phase of an optimization. All gradient\sphinxhyphen{}based Tinker optimization routines use a common line search routine involving quadratic extrapolation and cubic interpolation. If the value of INTMAX is reached, an error status is set for the line search and the search is repeated with a much smaller initial step size. The default value in the absence of this keyword is optimization routine dependent, but is usually in the range 5 to 10.

LAMBDA {[}real{]}     This keyword sets the value of the lambda path parameter for free energy perturbation calculations. The real number modifier specifies the position along the mutation path and must be a number in the range from 0 (initial state) to 1 (final state). The actual atoms involved in the mutation are given separately in individual MUTATE keyword lines.

LBFGS\sphinxhyphen{}VECTORS {[}integer{]}     Sets the number of correction vectors used by the limited\sphinxhyphen{}memory L\sphinxhyphen{}BFGS optimization routine. The current maximum allowable value, and the default in the absence of the LBFGS\sphinxhyphen{}VECTORS keyword is 15.

LIGHTS     This keyword turns on Method of Lights neighbor generation for the partial charge electrostatics and any of the van der Waals potentials. This method will yield identical energetic results to the standard double loop method. Method of Lights will be faster when the volume of a sphere with radius equal to the nonbond cutoff distance is significantly less than half the volume of the total system (i.e., the full molecular system, the crystal unit cell or the periodic box). It requires less storage than pairwise neighbor lists.

LIST\sphinxhyphen{}BUFFER {[}real{]}     Sets the size of the neighbor list buffer in Angstroms. This value is added to the actual cutoff distance to determine which pairs will be kept on the neighbor list. The same buffer value is used for all neighbor lists. The default value in the absence of 2.0 is used in the absence of the LIST\sphinxhyphen{}BUFFER keyword.

MAXITER {[}integer{]}     Sets the maximum number of minimization iterations that will be allowed for any Tinker program that uses any of the nonlinear optimization routines. The default value in the absence of this keyword is program dependent, but is always set to a very large number.

METAL     This keyword provides the values for a single transition metal ligand field parameter. Note this keyword is present in the code, but not active in the current version of Tinker.

METALTERM {[}NONE/ONLY{]}     This keyword controls use of the transition metal ligand field potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

MM2\sphinxhyphen{}STRBND     This keyword switches the behavior of the stretch\sphinxhyphen{}bend potential function to match the formulation used by the MM2 force field. In MM2, stretching of bonds to attached hydrogen atoms is not including in computing the stretch\sphinxhyphen{}bend cross term energy. The default behavior in the absence of this keyword is to include stretching of attached hydrogen atoms as in the MM3 force field.

MPOLE\sphinxhyphen{}12\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to permanent atomic multipole electrostatic interactions between 1\sphinxhyphen{}2 connected atoms, i.e., atoms that are directly bonded. The default value of 0.0 is used, if the MPOLE\sphinxhyphen{}12\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MPOLE\sphinxhyphen{}13\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to permanent atomic multipole  electrostatic interactions between 1\sphinxhyphen{}3 connected atoms, i.e., atoms separated by two covalent bonds. The default value of 0.0 is used, if the MPOLE\sphinxhyphen{}13\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MPOLE\sphinxhyphen{}14\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to permanent atomic multipole  electrostatic interactions between 1\sphinxhyphen{}4 connected atoms, i.e., atoms separated by three covalent bonds. The default value of 1.0 is used, if the MPOLE\sphinxhyphen{}14\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MPOLE\sphinxhyphen{}15\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to permanent atomic multipole  electrostatic interactions between 1\sphinxhyphen{}5 connected atoms, i.e., atoms separated by four covalent bonds. The default value of 1.0 is used, if the MPOLE\sphinxhyphen{}15\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MPOLE\sphinxhyphen{}CUTOFF {[}real{]}     Sets the cutoff distance value in Angstroms for atomic multipole potential energy interactions. The energy for any pair of sites beyond the cutoff distance will be set to zero. Other keywords can be used to select a smoothing scheme near the cutoff distance. The default cutoff distance in the absence of the MPOLE\sphinxhyphen{}CUTOFF keyword is infinite for nonperiodic systems and 9.0 for periodic systems.

MPOLE\sphinxhyphen{}TAPER {[}real{]}     This keyword allows modification of the cutoff window for atomic multipole potential energy interactions. It is similar in form and action to the TAPER keyword, except that its value applies only to the atomic multipole potential. The default value in the absence of the MPOLE\sphinxhyphen{}TAPER keyword is to begin the cutoff window at 0.65 of the corresponding cutoff distance.

MPOLETERM {[}NONE/ONLY{]}     This keyword controls use of the atomic multipole electrostatics potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

MULTIPOLE {[}5 lines with: 3 or 4 integers \& 1 real; 3 reals; 1 real; 2 reals; 3 reals{]}     This keyword provides the values for a set of atomic multipole parameters at a single site. A complete keyword entry consists of three consequtive lines, the first line containing the MULTIPOLE keyword and the two following lines. The first line contains three integers which define the atom type on which the multipoles are centered, and the Z\sphinxhyphen{}axis and X\sphinxhyphen{}axis defining atom types for this center. The optional fourth integer contains the Y\sphinxhyphen{}axis defining atom type, and is only required for locally chiral multipole sites. The real number on the first line gives the monopole (atomic charge) in electrons. The second line contains three real numbers which give the X\sphinxhyphen{}, Y\sphinxhyphen{} and Z\sphinxhyphen{}components of the atomic dipole in electron\sphinxhyphen{}Ang. The final three lines, consisting of one, two and three real numbers give the upper triangle of the traceless atomic quadrupole tensor in electron\sphinxhyphen{}Ang\textasciicircum{}2.

MUTATE {[}3 integers{]}     This keyword is used to specify atoms to be mutated during free energy perturbation calculations. The first integer modifier gives the atom number of an atom in the current system. The final two modifier values give the atom types corresponding the the lambda=0 and lambda=1 states of the specified atom.

MUTUAL\sphinxhyphen{}11\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the induced (mutual) field due to atoms within a polarization group during an induced dipole calculation, i.e., atoms that are in the same polarization group as the atom being polarized. The default value of 1.0 is used, if the MUTUAL\sphinxhyphen{}11\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MUTUAL\sphinxhyphen{}12\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the induced (mutual) field due to atoms in 1\sphinxhyphen{}2 polarization groups during an induced dipole calculation, i.e., atoms that are in polarization groups directly connected to the group containing the atom being polarized. The default value of 1.0 is used, if the MUTUAL\sphinxhyphen{}12\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MUTUAL\sphinxhyphen{}13\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the induced (mutual) field due to atoms in 1\sphinxhyphen{}3 polarization groups during an induced dipole calculation, i.e., atoms that are in polarization groups separated by one group from the group containing the atom being polarized. The default value of 1.0 is used, if the MUTUAL\sphinxhyphen{}13\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

MUTUAL\sphinxhyphen{}14\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to the induced (mutual) field due to atoms in 1\sphinxhyphen{}4 polarization groups during an induced dipole calculation, i.e., atoms that are in polarization groups separated by two groups from the group containing the atom being polarized. The default value of 1.0 is used, if the MUTUAL\sphinxhyphen{}14\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

NEIGHBOR\sphinxhyphen{}GROUPS     This keyword causes the attached atom to be used in determining the charge\sphinxhyphen{}charge neighbor distance for all monovalent atoms in the molecular system. Its use causes all monovalent atoms to be treated the same as their attached atoms for purposes of including or scaling 1\sphinxhyphen{}2, 1\sphinxhyphen{}3 and 1\sphinxhyphen{}4 interactions. This option works only for the simple charge\sphinxhyphen{}charge electrostatic potential; it does not affect bond dipole or atomic multipole potentials. The NEIGHBOR\sphinxhyphen{}GROUPS scheme is similar to that used by some common force fields such as ENCAD.

NEIGHBOR\sphinxhyphen{}LIST     This keyword turns on pairwise neighbor lists for partial charge electrostatics, polarize multipole electrostatics and any of the van der Waals potentials. This method will yield identical energetic results to the standard double loop method.

NEUTRAL\sphinxhyphen{}GROUPS     This keyword causes the attached atom to be used in determining the charge\sphinxhyphen{}charge interaction cutoff distance for all monovalent atoms in the molecular system. Its use reduces cutoff discontinuities by avoiding splitting many of the largest charge separations found in typical molecules. Note that this keyword does not rigorously implement the usual concept of a “neutral group” as used in the literature with Amber/OPLS and other force fields. This option works only for the simple charge\sphinxhyphen{}charge electrostatic potential; it does not affect bond dipole or atomic multipole potentials.

NEWHESS {[}integer{]}     Sets the number of algorithmic iterations between recomputation of the Hessian matrix. At present this keyword applies exclusively to optimizations using the Truncated Newton method. The default value in the absence of this keyword is 1, i.e., the Hessian is computed on every iteration.

NEXTITER {[}integer{]}     Sets the iteration number to be used for the first iteration of the current computation. At present this keyword applies to optimization procedures where its use can effect convergence criteria, timing of restarts, and so forth. The default in the absence of this keyword is to take the initial iteration as iteration 1.

NOSE\sphinxhyphen{}MASS {[}real{]}     This keyword sets the mass of particles making up the Nose\sphinxhyphen{}Hoover chain in that thermostating method. The default in the absence of the NOSE\sphinxhyphen{}MASS keyword is to use a mass of 0.1.

NOVERSION     Turns off the use of version numbers appended to the end of filenames as the method for generating filenames for updated copies of an existing file. The presence of this keyword results in direct use of input file names without a search for the highest available version, and requires the entry of specific output file names in many additional cases. By default, in the absence of this keyword, Tinker generates and attaches version numbers in a manner similar to the Digital OpenVMS operating system. For example, subsequent new versions of the file molecule.xyz would be written first to the file molecule.xyz\_2, then to molecule.xyz\_3, etc.

OCTAHEDRON     Specifies that the periodic “box” is a truncated octahedron with maximal distance across the truncated octahedron as given by the A\sphinxhyphen{}AXIS keyword. All other unit cell and periodic box size\sphinxhyphen{}defining keywords are ignored if the OCTAHEDRON keyword is present.

OPBEND {[}2 integers \& 1 real{]}     This keyword provides the values for a single Allinger MM\sphinxhyphen{}style out\sphinxhyphen{}of\sphinxhyphen{}plane angle bending potential parameter. The first integer modifier is the atom class of the central trigonal atom and the second integer is the atom class of the out\sphinxhyphen{}of\sphinxhyphen{}plane atom. The real number modifier gives the force constant value for the out\sphinxhyphen{}of\sphinxhyphen{}plane angle. The default units for the force constant are kcal/mole/radian\textasciicircum{}2, but this can be controlled via the OPBENDUNIT keyword.

OPBENDTERM {[}NONE/ONLY{]}     This keyword controls use of the Allinger MM\sphinxhyphen{}style out\sphinxhyphen{}of\sphinxhyphen{}plane bending potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

OPBENDUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the Allinger MM\sphinxhyphen{}style out\sphinxhyphen{}of\sphinxhyphen{}plane bending potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default of (Pi/180)\textasciicircum{}2 = 0.0003046 is used, if the OPBENDUNIT keyword is not given in the force field parameter file or the keyfile.

OPDIST {[}4 integers \& 1 real{]}     This keyword provides the values for a single out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential parameter. The first integer modifier is the atom class of the central trigonal atom and the three following integer modifiers are the atom classes of the three attached atoms. The real number modifier is the force constant for the harmonic function of the out\sphinxhyphen{}of\sphinxhyphen{}plane distance of the central atom. The default units for the force constant are kcal/mole/Ang\textasciicircum{}2, but this can be controlled via the OPDISTUNIT keyword.

OPDISTTERM {[}NONE/ONLY{]}     This keyword controls use of the out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

OPDISTUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the OPDISTUNIT keyword is not given in the force field parameter file or the keyfile.

OVERWRITE     Causes Tinker programs, such as minimizations, that output intermediate coordinate sets to create a single disk file for the intermediate results which is successively overwritten with the new intermediate coordinates as they become available. This keyword is essentially the opposite of the SAVECYCLE keyword.

PARAMETERS {[}file name{]}     Provides the name of the force field parameter file to be used for the current Tinker calculation. The standard file name extension for parameter files, .prm, is an optional part of the file name modifier. The default in the absence of the PARAMETERS keyword is to look for a parameter file with the same base name as the molecular system and ending in the .prm extension. If a valid parameter file is not found, the user will asked to provide a file name interactively.

PIATOM {[}1 integer \& 3 reals{]}     This keyword provides the values for the pisystem MO potential parameters for a single atom class belonging to a pisystem.

PIBOND {[}2 integers \& 2 reals{]}     This keyword provides the values for the pisystem MO potential parameters for a single type of pisystem bond.

PIBOND4 {[}2 integers \& 2 reals{]}     This keyword provides the values for the pisystem MO potential parameters for a single type of pisystem bond contained in a 4\sphinxhyphen{}membered ring.

PIBOND5 {[}2 integers \& 2 reals{]}     This keyword provides the values for the pisystem MO potential parameters for a single type of pisystem bond contained in a 5\sphinxhyphen{}membered ring.

PISYSTEM {[}integer list{]}     This keyword sets the atoms within a molecule that are part of a conjugated pi\sphinxhyphen{}orbital system. The keyword is followed on the same line by a list of atom numbers and/or atom ranges that constitute the pi\sphinxhyphen{}system. The Allinger MM force fields use this information to set up an MO calculation used to scale bond and torsion parameters involving pi\sphinxhyphen{}system atoms.

PITORS {[}2 integers \& 1 real{]}     This keyword provides the values for a single pi\sphinxhyphen{}orbital torsional angle potential parameter. The two integer modifiers give the atom class numbers for the atoms involved in the central bond of the torsional angle to be parameterized. The real modifier gives the value of the 2\sphinxhyphen{}fold Fourier amplitude for the torsional angle between p\sphinxhyphen{}orbitals centered on the defined bond atom classes. The default units for the stretch\sphinxhyphen{}torsion force constant can be controlled via the PITORSUNIT keyword.

PITORSTERM {[}NONE/ONLY{]}     This keyword controls use of the pi\sphinxhyphen{}orbital torsional angle potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

PITORSUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the pi\sphinxhyphen{}orbital torsional angle potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the PITORSUNIT keyword is not given in the force field parameter file or the keyfile.

PME\sphinxhyphen{}GRID {[}3 integers{]}     This keyword sets the dimensions of the charge grid used during particle mesh Ewald summation. The three modifiers give the size along the X\sphinxhyphen{}, Y\sphinxhyphen{} and Z\sphinxhyphen{}axes, respectively. If either the Y\sphinxhyphen{} or Z\sphinxhyphen{}axis dimensions are omitted, then they are set equal to the X\sphinxhyphen{}axis dimension. The default in the absence of the PME\sphinxhyphen{}GRID keyword is to set the grid size along each axis to the smallest power of 2, 3 and/or 5 which is at least as large as 1.5 times the axis length in Angstoms. Note that the FFT used by PME is not restricted to, but is most efficient for, grid sizes which are powers of 2, 3 and/or 5.

PME\sphinxhyphen{}ORDER {[}integer{]}     This keyword sets the order of the B\sphinxhyphen{}spline interpolation used during particle mesh Ewald summation. A default value of 8 is used in the absence of the PME\sphinxhyphen{}ORDER keyword.

POLAR\sphinxhyphen{}12\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to polarization interactions between 1\sphinxhyphen{}2 polarization groups, i.e., pairs of atoms that are in directly connected polarization groups. The default value of 0.0 is used, if the POLAR\sphinxhyphen{}12\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

POLAR\sphinxhyphen{}13\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to polarization interactions between 1\sphinxhyphen{}3 polarization groups, i.e., pairs of atoms that are in polarization groups separated by one other group. The default value of 0.0 is used, if the POLAR\sphinxhyphen{}13\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

POLAR\sphinxhyphen{}14\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to polarization interactions between 1\sphinxhyphen{}4 polarization groups, i.e., pairs of atoms that are in polarization groups separated by two other groups. The default value of 1.0 is used, if the POLAR\sphinxhyphen{}14\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

POLAR\sphinxhyphen{}15\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to polarization interactions between 1\sphinxhyphen{}5 polarization groups, i.e., pairs of atoms that are in polarization groups separated by three other groups. The default value of 1.0 is used, if the POLAR\sphinxhyphen{}15\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

POLAR\sphinxhyphen{}DAMP {[}2 reals{]}     Controls the strength of the damping function applied to induced dipoles and dipole polarization interaction energies. The first modifier sets the radius in Angstoms of a hypothetical atom with unit polarizability, while the second modifier sets the scale factor for the exponent of the  damping function. The default values for the radius and the scale factor are 1.662 and 1.0, respectively. Damping is eliminated entirely by using this keyword to set the radius value to zero.

POLAR\sphinxhyphen{}EPS {[}real{]}     This keyword sets the convergence criterion applied during computation of self\sphinxhyphen{}consistent induced dipoles. The calculation is deemed to have converged when the rms change in Debyes in the induced dipole at all polarizable sites is less than the value specified with this keyword. The default value in the absence of the keyword is 0.000001 Debyes.

POLAR\sphinxhyphen{}SOR {[}real{]}     Sets a successive overrelaxation (SOR) factor for use in computation of induced atomic dipoles. Optimal values for this keyword will speed the induced dipole calculation, and poor values can result in convergence failure. The default value in the absence of the POLAR\sphinxhyphen{}SOR keyword is 0.7 which often a reasonable value when short\sphinxhyphen{}range intramolecular polarization is present. For models lacking intramolecular polarization, keyword values closer to 1.0 may be optimal.

POLARIZATION {[}DIRECT/MUTUAL{]}     Selects between the use of direct and mutual dipole polarization for force fields that incorporate the polarization term. The DIRECT modifier avoids an iterative calculation by using only the permanent electric field in computation of induced dipoles. The MUTUAL option, which is the default in the absence of the POLARIZATION keyword, iterates the induced dipoles to self\sphinxhyphen{}consistency.

POLARIZE {[}1 integer, 1 real \& up to 4 integers{]}     This keyword provides the values for a single atomic dipole polarizability parameter. The integer modifier, if positive, gives the atom type number for which a polarizability parameter is to be defined. If the first integer modifier is negative, then the parameter value to follow applies only to the individual atom whose atom number is the negative of the modifier. The real number modifier gives the value of the dipole polarizability in Ang\textasciicircum{}3. The final integer modifiers list the atom type numbers of atoms directly bonded to the current atom and which will be considered to be part of the current atom’s polarization group.

POLARIZETERM {[}NONE/ONLY{]}     This keyword controls use of the atomic dipole polarization potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

POLYMER\sphinxhyphen{}CUTOFF {[}real{]}     Sets the value of an additional cutoff parameter needed for infinite polymer systems. This value must be set to less than half the minimal periodic box dimension and should be greater than the largest possible interatomic distance that can be subject to scaling or exclusion as a local electrostatic or van der Waals interaction. The default in the absence of the POLYMER\sphinxhyphen{}CUTOFF keyword is 5.5 Angstroms.

PRINTOUT {[}integer{]}     A general parameter for iterative procedures such as minimizations that sets the number of iterations between writes of status information to the standard output. The default value in the absence of the keyword is 1, i.e., the calculation status is given every iteration.

RADIUSRULE {[}ARITHMETIC/GEOMETRIC/CUBIC\sphinxhyphen{}MEAN{]}     Sets the functional form of the radius combining rule for heteroatomic van der Waals potential energy interactions. The default in the absence of the RADIUSRULE keyword is to use the arithmetic mean combining rule to get radii for heteroatomic interactions.

RADIUSSIZE {[}RADIUS/DIAMETER{]}     Determines whether the atom size values given in van der Waals parameters read from VDW keyword statements are interpreted as atomic radius or diameter values. The default in the absence of the RADIUSSIZE keyword is to assume that vdw size parameters are given as radius values.

RADIUSTYPE {[}R\sphinxhyphen{}MIN/SIGMA{]}     Determines whether atom size values given in van der Waals parameters read from VDW keyword statements are interpreted as potential minimum (Rmin) or LJ\sphinxhyphen{}style sigma values. The default in the absence of the RADIUSTYPE keyword is to assume that vdw size parameters are given as Rmin values.

RANDOMSEED {[}integer{]}     Followed by an integer value, this keyword sets the initial seed value for the random number generator used by Tinker. Setting RANDOMSEED to the same value as an earlier run will allow exact reproduction of the earlier calculation. (Note that this will not hold across different machine types.) RANDOMSEED should be set to a positive integer less than about 2 billion. In the absence of the RANDOMSEED keyword the seed is chosen “randomly” based upon the number of seconds that have elapsed in the current decade.

RATTLE {[}BONDS/ANGLES/DIATOMIC/TRIATOMIC/WATER{]}     Invokes the rattle algorithm, a velocity version of shake, on portions of a molecular system during a molecular dynamic calculation. The RATTLE keyword can be followed by any of the modifiers shown, in which case all occurrences of the modifier species are constrained at ideal values taken from the bond and angle parameters of the force field in use. In the absence of any modifier, RATTLE constrains all bonds to hydrogen atoms at ideal bond lengths.

RATTLE\sphinxhyphen{}DISTANCE {[}2 integers{]}     This keyword allows the use of a holonomic constraint between the two atoms whose numbers are specified on the keyword line. If the two atoms are involved in a covalent bond, then their distance is constrained to the ideal bond length from the force field. For nonbonded atoms, the rattle constraint is fixed at their distance in the input coordinate file.

RATTLE\sphinxhyphen{}LINE {[}integer{]}     This keyword

RATTLE\sphinxhyphen{}ORIGIN {[}integer{]}     This keyword

RATTLE\sphinxhyphen{}PLANE {[}integer{]}     This keyword

REACTIONFIELD {[}2 reals \& 1 integer{]}     This keyword provides parameters needed for the reaction field potential energy calculation. The two real modifiers give the radius of the dielectric cavity and the ratio of the bulk dielectric outside the cavity to that inside the cavity. The integer modifier gives the number of terms in the reaction field summation to be used. In the absence of the REACTIONFIELD keyword, the default values are a cavity of radius 1000000 Ang, a dielectric ratio of 80 and use of only the first term of the reaction field summation.

REDUCE {[}real{]}     Specifies the fraction between zero and one by which the path between starting and final conformational state will be shortened at each major cycle of the transition state location algorithm implemented by the SADDLE program. This causes the path endpoints to move up and out of the terminal structures toward the transition state region. In favorable cases, a nonzero value of the REDUCE modifier can speed convergence to the transition state. The default value in the absence of the REDUCE keyword is zero.

RESTRAIN\sphinxhyphen{}ANGLE {[}3 integers \& 3 reals{]}     This keyword implements a flat\sphinxhyphen{}welled harmonic potential that can be used to restrain the angle between three atoms to lie within a specified angle range. The integer modifiers contain the atom numbers of the three atoms whose angle is to be restrained.  The first real modifier is the force constant in kcal/degree\textasciicircum{}2 for the restraint. The last two real modifiers give the lower and upper bounds in degrees on the allowed angle values. If the angle lies between the lower and upper bounds, the restraint potential is zero. Outside the bounds, the harmonic restraint is applied. If the angle range modifiers are omitted, then the atoms are restrained to the angle found in the input structure. If the force constant is also omitted, a default value of 10.0 is used.

RESTRAIN\sphinxhyphen{}DISTANCE {[}2 integers \& 3 reals{]}     This keyword implements a flat\sphinxhyphen{}welled harmonic potential that can be used to restrain two atoms to lie within a specified distance range. The integer modifiers contain the atom numbers of the two atoms to be restrained. The first real modifier specifies the force constant in kcal/Ang\textasciicircum{}2 for the restraint. The next two real modifiers give the lower and upper bounds in Angstroms on the allowed distance range. If the interatomic distance lies between these lower and upper bounds, the restraint potential is zero. Outside the bounds, the harmonic restraint is applied. If the distance range modifiers are omitted, then the atoms are restrained to the interatomic distance found in the input structure. If the force constant is also omitted, a default value of 100.0 is used.

RESTRAIN\sphinxhyphen{}GROUPS {[}2 integers \& 3 reals{]}     This keyword implements a flat\sphinxhyphen{}welled harmonic distance restraint between the centers\sphinxhyphen{}of\sphinxhyphen{}mass of two groups of atoms. The integer modifiers are the numbers of the two groups which must be defined separately via the GROUP keyword. The first real modifier is the force constant in kcal/Ang\textasciicircum{}2 for the restraint. The last two real modifiers give the lower and upper bounds in Angstroms on the allowed intergroup center\sphinxhyphen{}of\sphinxhyphen{}mass distance values. If the distance range modifiers are omitted, then the groups are restrained to the distance found in the input structure. If the force constant is also omitted, a default value of 100.0 is used.

RESTRAIN\sphinxhyphen{}POSITION {[}1 integer \& 5 reals{]}     This keyword provides the ability to restrain an individual atom to a specified coordinate position. The initial integer modifier contains the atom number of the atom to be restrained. The first real modifier sets the force constant in kcal/Ang\textasciicircum{}2 for the harmonic restraint potential. The next three real number modifiers give the X\sphinxhyphen{}, Y\sphinxhyphen{} and Z\sphinxhyphen{}coordinates to which the atom is tethered. The final real modifier defines a sphere around the specified coordinates within which the restraint value is zero. If the exclusion sphere radius is omitted, it is taken to be zero. If  the coordinates are omitted, then the atom is restrained to the origin. If the force constant is also omitted, a default value of 100.0 is used.

RESTRAIN\sphinxhyphen{}TORSION {[}4 integers \& 3 reals{]}     This keyword implements a flat\sphinxhyphen{}welled harmonic potential that can be used to restrain the torsional angle between four atoms to lie within a specified angle range. The initial integer modifiers contains the atom numbers of the four atoms whose torsional angle, computed in the atom order listed, is to be restrained. The first real modifier gives a force constant in kcal/degree\textasciicircum{}2 for the restraint. The last two real modifiers give the lower and upper bounds in degrees on the allowed torsional angle values. The angle values given can wrap around across \sphinxhyphen{}180 and +180 degrees. Outside the allowed angle range, the harmonic restraint is applied. If the angle range modifiers are omitted, then the atoms are restrained to the torsional angle found in the input structure. If the force constant is also omitted, a default value of 1.0 is used.

RESTRAINTERM {[}NONE/ONLY{]}     This keyword controls use of the restraint potential energy terms. In the absence of a modifying option, this keyword turns on use of these potentials. The NONE option turns off use of these potential energy terms. The ONLY option turns off all potential energy terms except for these terms.

RXNFIELDTERM {[}NONE/ONLY{]}     This keyword controls use of the reaction field continuum solvation potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

SADDLEPOINT     The presence of this keyword allows Newton\sphinxhyphen{}style second derivative\sphinxhyphen{}based optimization routine used by NEWTON, NEWTROT and other programs to converge to saddlepoints as well as minima on the potential surface. By default, in the absence of the SADDLEPOINT keyword, checks are applied that prevent convergence to stationary points having directions of negative curvature.

SAVE\sphinxhyphen{}CYCLE     This keyword causes Tinker programs, such as minimizations, that output intermediate coordinate sets to save each successive set to the next consecutively numbered cycle file. The SAVE\sphinxhyphen{}CYCLE keyword is the opposite of the OVERWRITE keyword.

SAVE\sphinxhyphen{}FORCE     This keyword causes Tinker molecular dynamics calculations to save the values of the force components on each atom to a separate cycle file. These files are written whenever the atomic coordinate snapshots are written during the dynamics run. Each atomic force file name contains as a suffix the cycle number followed by the letter f.

SAVE\sphinxhyphen{}INDUCED     This keyword causes Tinker molecular dynamics calculations that involve polarizable atomic multipoles to save the values of the induced dipole components on each polarizable atom to a separate cycle file. These files are written whenever the atomic coordinate snapshots are written during the dynamics run. Each induced dipole file name contains as a suffix the cycle number followed by the letter u.

SAVE\sphinxhyphen{}VELOCITY     This keyword causes Tinker molecular dynamics calculations to save the values of the velocity components on each atom to a separate cycle file. These files are written whenever the atomic coordinate snapshots are written during the dynamics run. Each velocity file name contains as a suffix the cycle number followed by the letter v.

SLOPEMAX {[}real{]}     This keyword and its modifying value set the maximum allowed size of the ratio between the current and initial projected gradients during the line search phase of conjugate gradient or truncated Newton optimizations. If this ratio exceeds SLOPEMAX, then the initial step size is reduced by a factor of 10. The default value is usually set to 10000.0 when not specified via the SLOPEMAX keyword.

SMOOTHING {[}DEM/GDA/TOPHAT/STOPHAT{]}     This keyword activates the potential energy smoothing methods. Several variations are available depending on the value of the modifier used: DEM= Diffusion Equation Method with a standard Gaussian kernel; GDA= Gaussian Density Annealing as proposed by the Straub group; TOPHAT= a local DEM\sphinxhyphen{}like method using a finite range “tophat” kernel; STOPHAT= shifted tophat smoothing.

SOLVATE {[}ASP/SASA/ONION/STILL/HCT/ACE/GBSA{]}     Use of this keyword during energy calculations with any of the standard force fields turns on a continuum solvation free energy term. Several algorithms are available based on the modifier used: ASP= Eisenberg\sphinxhyphen{}McLachlan ASP method using the Wesson\sphinxhyphen{}Eisenberg vacuum\sphinxhyphen{}to\sphinxhyphen{}water parameters; SASA= the Ooi\sphinxhyphen{}Scheraga SASA method; ONION= the original 1990 Still “Onion\sphinxhyphen{}shell” GB/SA method; STILL= the 1997 analytical GB/SA method from Still’s group; HCT= the pairwise descreening method of Hawkins, Cramer and Truhlar; ACE= the Analytical Continuum Electrostatics solvation method from the Karplus group; GBSA= equivalent to the STILL modifier. At present, GB/SA\sphinxhyphen{}style methods are only valid for force fields that use simple partial charge electrostatics.

SOLVATETERM {[}NONE/ONLY{]}     This keyword controls use of the macroscopic solvation potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

SPACEGROUP {[}name{]}     This keyword selects the space group to be used in manipulation of crystal unit cells and asymmetric units. The name option must be chosen from one of the following currently implemented space groups: P1, P1(\sphinxhyphen{}), P21, Cc, P21/a, P21/n, P21/c, C2/c, P212121, Pna21, Pn21a, Cmc21, Pccn, Pbcn, Pbca, P41, I41/a, P4(\sphinxhyphen{})21c, P4(\sphinxhyphen{})m2, R3c, P6(3)/mcm, Fm3(\sphinxhyphen{})m, Im3(\sphinxhyphen{})m.

SPHERE {[}4 reals, or 1 integer \& 1 real{]}     This keyword provides an alternative to the ACTIVE and INACTIVE keywords for specification of subsets of active atoms. If four real number modifiers are provided, the first three are taken as X\sphinxhyphen{}, Y\sphinxhyphen{} and Z\sphinxhyphen{}coordinates and the fourth is the radius of a sphere centered at these coordinates. In this case, all atoms within the sphere at the start of the calculation are active throughout the calculation, while all other atoms are inactive. Similarly if one integer and real number are given, an “active” sphere with radius set by the real is centered on the system atom with atom number given by the integer modifier. Multiple SPHERE keyword lines can be present in a single keyfile, and the list of active atoms specified by the spheres is cumulative.

STEEPEST\sphinxhyphen{}DESCENT     This keyword forces the L\sphinxhyphen{}BFGS optimization routine used by the MINIMIZE program and other programs to perform steepest descent minimization. This option can be useful in conjunction with small step sizes for following minimum energy paths, but is generally inferior to the L\sphinxhyphen{}BFGS default for most optimization purposes.

STEPMAX {[}real{]}     This keyword and its modifying value set the maximum size of an individual step during the line search phase of conjugate gradient or truncated Newton optimizations. The step size is computed as the norm of the vector of changes in parameters being optimized. The default value depends on the particular Tinker program, but is usually in the range from 1.0 to 5.0 when not specified via the STEPMAX keyword.

STEPMIN {[}real{]}     This keyword and its modifying value set the minimum size of an individual step during the line search phase of conjugate gradient or truncated Newton optimizations. The step size is computed as the norm of the vector of changes in parameters being optimized. The default value is usually set to about 10\sphinxhyphen{}16 when not specified via the STEPMIN keyword.

STRBND {[}1 integer \& 3 reals{]}     This keyword provides the values for a single stretch\sphinxhyphen{}bend cross term potential parameter. The integer modifier gives the atom class number for the central atom of the bond angle involved in stretch\sphinxhyphen{}bend interactions. The real number modifiers give the force constant values to be used when the central atom of the angle is attached to 0, 1 or 2 additional hydrogen atoms, respectively. The default units for the stretch\sphinxhyphen{}bend force constant are kcal/mole/Ang\sphinxhyphen{}degree, but this can be controlled via the STRBNDUNIT keyword.

STRBNDTERM {[}NONE/ONLY{]}     This keyword controls use of the bond stretching\sphinxhyphen{}angle bending cross term potential energy. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

STRBNDUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the bond stretching\sphinxhyphen{}angle bending cross term potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the STRBNDUNIT keyword is not given in the force field parameter file or the keyfile.

STRTORS {[}2 integers \& 1 real{]}     This keyword provides the values for a single stretch\sphinxhyphen{}torsion cross term potential parameter. The two integer modifiers give the atom class numbers for the atoms involved in the central bond of the torsional angles to be parameterized. The real modifier gives the value of the stretch\sphinxhyphen{}torsion force constant for all torsional angles with the defined central bond atom classes. The default units for the stretch\sphinxhyphen{}torsion force constant can be controlled via the STRTORUNIT keyword.

STRTORTERM {[}NONE/ONLY{]}     This keyword controls use of the bond stretching\sphinxhyphen{}torsional angle cross term potential energy. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

STRTORUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the bond stretching\sphinxhyphen{}torsional angle cross term potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the STRTORUNIT keyword is not given in the force field parameter file or the keyfile.

TAPER {[}real{]}     This keyword allows modification of the cutoff windows for nonbonded potential energy interactions. The nonbonded terms are smoothly reduced from their standard value at the beginning of the cutoff window to zero at the far end of the window. The far end of the window is specified via the CUTOFF keyword or its potential function specific variants. The modifier value supplied with the TAPER keyword sets the beginning of the cutoff window. The modifier can be given either as an absolute distance value in Angstroms, or as a fraction between zero and one of the CUTOFF distance. The default value in the absence of the TAPER keyword ranges from 0.65 to 0.9 of the CUTOFF distance depending on the type of potential function. The windows are implemented via polynomial\sphinxhyphen{}based switching functions, in some cases combined with energy shifting.

TAU\sphinxhyphen{}PRESSURE {[}real{]}     Sets the coupling time in picoseconds for the Groningen\sphinxhyphen{}style pressure bath coupling used to control the system pressure during molecular dynamics calculations. A default value of 2.0 is used for TAU\sphinxhyphen{}PRESSURE in the absence of the keyword.

TAU\sphinxhyphen{}TEMPERATURE {[}real{]}     Sets the coupling time in picoseconds for the Groningen\sphinxhyphen{}style temperature bath coupling used to control the system temperature during molecular dynamics calculations. A default value of 0.1 is used for TAU\sphinxhyphen{}TEMPERATURE in the absence of the keyword.

THERMOSTAT {[}BERENDSEN/ANDERSEN{]}     This keyword selects a thermostat algorithm for use during molecular dynamics. Two modifiers are available, a Berendsen bath coupling method, and an Andersen stochastic collision method. The default in the absence of the THERMOSTAT keyword is to use the BERENDSEN algorithm.

TORSION {[}4 integers \& up to 6 real/real/integer triples{]}     This keyword provides the values for a single torsional angle parameter. The first four integer modifiers give the atom class numbers for the atoms involved in the torsional angle to be defined. Each of the remaining triples of real/real/integer modifiers give the amplitude, phase offset in degrees and periodicity of a particular torsional function term, respectively. Periodicities through 6\sphinxhyphen{}fold are allowed for torsional parameters.

TORSION4 {[}4 integers \& up to 6 real/real/integer triples{]}     This keyword provides the values for a single torsional angle parameter specific to atoms in 4\sphinxhyphen{}membered rings. The first four integer modifiers give the atom class numbers for the atoms involved in the torsional angle to be defined. The remaining triples of real number and integer modifiers operate as described above for the TORSION keyword.

TORSION5 {[}4 integers \& up to 6 real/real/integer triples{]}     This keyword provides the values for a single torsional angle parameter specific to atoms in 5\sphinxhyphen{}membered rings. The first four integer modifiers give the atom class numbers for the atoms involved in the torsional angle to be defined. The remaining triples of real number and integer modifiers operate as described above for the TORSION keyword.

TORSIONTERM {[}NONE/ONLY{]}     This keyword controls use of the torsional angle potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

TORSIONUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the torsional angle potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the TORSIONUNIT keyword is not given in the force field parameter file or the keyfile.

TORTOR {[}7 integers, then multiple lines of 2 integers and 1 real{]}     This keyword is used to provide the values for a single torsion\sphinxhyphen{}torsion parameter. The first five integer modifiers give the atom class numbers for the atoms involved in the two adjacent torsional angles to be defined. The last two integer modifiers contain the number of data grid points that lie along each axis of the torsion\sphinxhyphen{}torsion map. For example, this value will be 13 for a 30 degree torsional angle spacing, i.e., 360/30 = 12, but 13 values are required since data values for \sphinxhyphen{}180 and +180 degrees must both be supplied. The subsequent lines contain the torsion\sphinxhyphen{}torsion map data as the integer values in degrees of each torsional angle and the target energy value in kcal/mole.

TORTORTERM {[}NONE/ONLY{]}     This keyword controls use of the torsion\sphinxhyphen{}torsion potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

TORTORUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the torsion\sphinxhyphen{}torsion potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the TORTORUNIT keyword is not given in the force field parameter file or the keyfile.

TRIAL\sphinxhyphen{}DISTANCE              {[}CLASSIC/RANDOM/TRICOR/HAVEL integer/PAIRWISE integer{]}     Sets the method for selection of a trial distance matrix during distance geometry computations. The keyword takes a modifier that selects the method to be used. The HAVEL and PAIRWISE modifiers also require an additional integer value that specifies the number of atoms used in metrization and the percentage of metrization, respectively. The default in the absence of this keyword is to use the PAIRWISE method with 100 percent metrization. Further information on the various methods is given with the description of the Tinker distance geometry program.

TRIAL\sphinxhyphen{}DISTRIBUTION {[}real{]}     Sets the initial value for the mean of the Gaussian distribution used to select trial distances between the lower and upper bounds during distance geometry computations. The value given must be between 0 and 1 which represent the lower and upper bounds respectively. This keyword is rarely needed since Tinker will usually be able to choose a reasonable value by default.

TRUNCATE     Causes all distance\sphinxhyphen{}based nonbond energy cutoffs to be sharply truncated to an energy of zero at distances greater than the value set by the cutoff keyword(s) without use of any shifting, switching or smoothing schemes. At all distances within the cutoff sphere, the full interaction energy is computed.

UREY\sphinxhyphen{}CUBIC {[}real{]}     Sets the value of the cubic term in the Taylor series expansion form of the Urey\sphinxhyphen{}Bradley potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. The default value in the absence of the UREY\sphinxhyphen{}CUBIC keyword is zero; i.e., the cubic Urey\sphinxhyphen{}Bradley term is omitted.

UREY\sphinxhyphen{}QUARTIC {[}real{]}     Sets the value of the quartic term in the Taylor series expansion form of the Urey\sphinxhyphen{}Bradley potential energy. The real number modifier gives the value of the coefficient as a multiple of the quadratic coefficient. The default value in the absence of the UREY\sphinxhyphen{}QUARTIC keyword is zero; i.e., the quartic Urey\sphinxhyphen{}Bradley term is omitted.

UREYBRAD {[}3 integers \& 2 reals{]}     This keyword provides the values for a single Urey\sphinxhyphen{}Bradley cross term potential parameter. The integer modifiers give the atom class numbers for the three kinds of atoms involved in the angle for which a Urey\sphinxhyphen{}Bradley term is to be defined. The real number modifiers give the force constant value for the term and the target value for the 1\sphinxhyphen{}3 distance in Angstroms. The default units for the force constant are kcal/mole/Ang\textasciicircum{}2, but this can be controlled via the UREYUNIT keyword.

UREYTERM {[}NONE/ONLY{]}     This keyword controls use of the Urey\sphinxhyphen{}Bradley potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

UREYUNIT {[}real{]}     Sets the scale factor needed to convert the energy value computed by the Urey\sphinxhyphen{}Bradley potential into units of kcal/mole. The correct value is force field dependent and typically provided in the header of the master force field parameter file. The default value of 1.0 is used, if the UREYUNIT keyword is not given in the force field parameter file or the keyfile.

VDW {[}1 integer \& 3 reals{]}     This keyword provides values for a single van der Waals parameter. The integer modifier, if positive, gives the atom class number for which vdw parameters are to be defined. Note that vdw parameters are given for atom classes, not atom types. The three real number modifiers give the values of the atom size in Angstroms, homoatomic well depth in kcal/mole, and an optional reduction factor for univalent atoms.

VDW\sphinxhyphen{}12\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to van der Waals potential interactions between 1\sphinxhyphen{}2 connected atoms, i.e., atoms that are directly bonded. The default value of 0.0 is used, if the VDW\sphinxhyphen{}12\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

VDW\sphinxhyphen{}13\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to van der Waals potential interactions between 1\sphinxhyphen{}3 connected atoms, i.e., atoms separated by two covalent bonds. The default value of 0.0 is used, if the VDW\sphinxhyphen{}13\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

VDW\sphinxhyphen{}14\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to van der Waals potential interactions between 1\sphinxhyphen{}4 connected atoms, i.e., atoms separated by three covalent bonds. The default value of 1.0 is used, if the VDW\sphinxhyphen{}14\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

VDW\sphinxhyphen{}15\sphinxhyphen{}SCALE {[}real{]}     This keyword provides a multiplicative scale factor that is applied to van der Waals potential interactions between 1\sphinxhyphen{}5 connected atoms, i.e., atoms separated by four covalent bonds. The default value of 1.0 is used, if the VDW\sphinxhyphen{}15\sphinxhyphen{}SCALE keyword is not given in either the parameter file or the keyfile.

VDW\sphinxhyphen{}CUTOFF {[}real{]}     Sets the cutoff distance value in Angstroms for van der Waals potential energy interactions. The energy for any pair of van der Waals sites beyond the cutoff distance will be set to zero. Other keywords can be used to select a smoothing scheme near the cutoff distance. The default cutoff distance in the absence of the VDW\sphinxhyphen{}CUTOFF keyword is infinite for nonperiodic systems and 9.0 for periodic systems.

VDW\sphinxhyphen{}TAPER {[}real{]}     This keyword allows modification of the cutoff windows for van der Waals potential energy interactions. It is similar in form and action to the TAPER keyword, except that its value applies only to the vdw potential. The default value in the absence of the VDW\sphinxhyphen{}TAPER keyword is to begin the cutoff window at 0.9 of the vdw cutoff distance.

VDW14 {[}1 integer \& 2 reals{]}     This keyword provides values for a single van der Waals parameter for use in 1\sphinxhyphen{}4 nonbonded interactions. The integer modifier, if positive, gives the atom class number for which vdw parameters are to be defined. Note that vdw parameters are given for atom classes, not atom types. The two real number modifiers give the values of the atom size in Angstroms and the homoatomic well depth in kcal/mole. Reduction factors, if used, are carried over from the VDW keyword for the same atom class.

VDWPR {[}2 integers \& 2 reals{]}     This keyword provides the values for the vdw parameters for a single special heteroatomic pair of atoms. The integer modifiers give the pair of atom class numbers for which special vdw parameters are to be defined. The two real number modifiers give the values of the minimum energy contact distance in Angstroms and the well depth at the minimum distance in kcal/mole.

VDWTERM {[}NONE/ONLY{]}     This keyword controls use of the van der Waals repulsion\sphinxhyphen{}dispersion potential energy term. In the absence of a modifying option, this keyword turns on use of the potential. The NONE option turns off use of this potential energy term. The ONLY option turns off all potential energy terms except for this one.

VDWTYPE {[}LENNARD\sphinxhyphen{}JONES / BUCKINGHAM / BUFFERED\sphinxhyphen{}14\sphinxhyphen{}7 / MM3\sphinxhyphen{}HBOND / GAUSSIAN{]}     Sets the functional form for the van der Waals potential energy term. The text modifier gives the name of the functional form to be used. The GAUSSIAN modifier value implements a two or four Gaussian fit to the corresponding Lennard\sphinxhyphen{}Jones function for use with potential energy smoothing schemes. The default in the absence of the VDWTYPE keyword is to use the standard two parameter Lennard\sphinxhyphen{}Jones function.

VERBOSE     Turns on printing of secondary and informational output during a variety of Tinker computations; a subset of the more extensive output provided by the DEBUG keyword.

WALL {[}real{]}     Sets the radius of a spherical boundary used to maintain droplet boundary conditions. The real modifier specifies the desired approximate radius of the droplet. In practice, an artificial van der Waals wall is constructed at a fixed buffer distance of 2.5 Angstroms outside the specified radius. The effect is that atoms which attempt to move outside the region defined by the droplet radius will be forced toward the center.

WRITEOUT {[}integer{]}     A general parameter for iterative procedures such as minimizations that sets the number of iterations between writes of intermediate results (such as the current coordinates) to disk file(s). The default value in the absence of the keyword is 1, i.e., the intermediate results are written to file on every iteration. Whether successive intermediate results are saved to new files or replace previously written intermediate results is controlled by the OVERWRITE and SAVE\sphinxhyphen{}CYCLE keywords.


\chapter{Descriptions of Tinker Routines}
\label{\detokenize{text/routines:descriptions-of-tinker-routines}}\label{\detokenize{text/routines::doc}}
The distribution version of the Tinker package contains over 700 separate programs, subroutines and functions. This section contains a brief description of the purpose of most of these code units. Further information can be found in the comments located at the top of each source code file.

\sphinxstylestrong{ACTIVE Subroutine}

“active” sets the list of atoms that are used during each potential energy Function** calculation

\sphinxstylestrong{ADDBASE Subroutine}

“addbase” builds the Cartesian coordinates for a single nucleic acid base; coordinates are read from the Protein Data Bank file or found from internal coordinates, then atom types are assigned and connectivity data generated

\sphinxstylestrong{ADDBOND Subroutine}

“addbond” adds entries to the attached atoms list in order to generate a direct connection between two atoms

\sphinxstylestrong{ADDSIDE Subroutine}

“addside” builds the Cartesian coordinates for a single amino acid side chain; coordinates are read from the Protein Data Bank file or found from internal coordinates, then atom types are assigned and connectivity data generated

\sphinxstylestrong{ADJACENT Function}

“adjacent” finds an atom connected to atom “i1” other than atom “i2”; if no such atom exists, then the closest atom in space is returned

\sphinxstylestrong{ALCHEMY Program}

“alchemy” computes the free energy difference corresponding to a small perturbation by Boltzmann weighting the potential energy difference over a number of sample states; current version (incorrectly) considers the charge energy to be intermolecular in finding the perturbation energies

\sphinxstylestrong{ANALYSIS Subroutine}

“analysis” calls the series of routines needed to calculate the potential energy and perform energy partitioning analysis in terms of type of interaction or atom number

\sphinxstylestrong{ANALYZ4 Subroutine}

“analyz4” prints the energy to 4 decimal places and number of interactions for each component of the potential energy

\sphinxstylestrong{ANALYZ6 Subroutine}

“analyz6” prints the energy to 6 decimal places and number of interactions for each component of the potential energy

\sphinxstylestrong{ANALYZ8 Subroutine}

“analyz8” prints the energy to 8 decimal places and number of interactions for each component of the potential energy

\sphinxstylestrong{ANALYZE Program}

“analyze” computes and displays the total potential; options are provided to partition the energy by atom or by potential Function** type; parameters used in computing interactions can also be displayed by atom; output of large energy interactions and of electrostatic and inertial properties is available

\sphinxstylestrong{ANGLES Subroutine}

“angles” finds the total number of bond angles and stores the atom numbers of the atoms defining each angle; for each angle to a trivalent central atom, the third bonded atom is stored for use in out\sphinxhyphen{}of\sphinxhyphen{}plane bending

\sphinxstylestrong{ANNEAL Program}

“anneal” performs a simulated annealing protocol by means of variable temperature molecular dynamics using either linear, exponential or sigmoidal cooling schedules

\sphinxstylestrong{ANORM Function}

“anorm” finds the norm (length) of a vector; used as a service routine by the Connolly surface area and volume computation

\sphinxstylestrong{ARCHIVE Program}

“archive” is a utility Program** for coordinate files which concatenates multiple coordinate sets into a single archive file, or extracts individual coordinate sets from an archive

\sphinxstylestrong{ASET Subroutine}

“aset” computes by recursion the A Function**s used in the evaluation of Slater\sphinxhyphen{}type (STO) overlap integrals

\sphinxstylestrong{ATOMYZE Subroutine}

“atomyze” prints the potential energy components broken down by atom and to a choice of precision

\sphinxstylestrong{ATTACH Subroutine}

“attach” generates lists of 1\sphinxhyphen{}3, 1\sphinxhyphen{}4 and 1\sphinxhyphen{}5 connectivities starting from the previously determined list of attached atoms (ie, 1\sphinxhyphen{}2 connectivity)

\sphinxstylestrong{BASEFILE Subroutine}

“basefile” extracts from an input filename the portion consisting of any directory name and the base filename

\sphinxstylestrong{BCUCOF Subroutine}

“bcucof” determines the coefficient matrix needed for bicubic interpolation of a Function**, gradients and cross derivatives

\sphinxstylestrong{BCUINT Subroutine}

“bcuint” performs a bicubic interpolation of the Function** value on a 2D spline grid

\sphinxstylestrong{BCUINT1 Subroutine}

“bcuint1” performs a bicubic interpolation of the Function** value and gradient along the directions of a 2D spline grid

\sphinxstylestrong{BCUINT2 Subroutine}

“bcuint2” performs a bicubic interpolation of the Function** value, gradient and Hessain along the directions of a 2D spline grid

\sphinxstylestrong{BEEMAN Subroutine}

“beeman” performs a single molecular dynamics time step by means of a Beeman multistep recursion formula; the actual coefficients are Brooks’ “Better Beeman” values

\sphinxstylestrong{BETACF Function}

“betacf” computes a rapidly convergent continued fraction needed by routine “betai” to evaluate the cumulative Beta distribution

\sphinxstylestrong{BETAI Function}

“betai” evaluates the cumulative Beta distribution Function** as the probability that a random variable from a distribution with Beta parameters “a” and “b” will be less than “x”

\sphinxstylestrong{BIGBLOCK Subroutine}

“bigblock” replicates the coordinates of a single unit cell to give a larger block of repeated units

\sphinxstylestrong{BITORS Subroutine}

“bitors” finds the total number of bitorsions, pairs of overlapping dihedral angles, and the numbers of the five atoms defining each bitorsion

\sphinxstylestrong{BMAX Function}

“bmax” computes the maximum order of the B Function**s needed for evaluation of Slater\sphinxhyphen{}type (STO) overlap integrals

\sphinxstylestrong{BNDERR Function}

“bnderr” is the distance bound error Function** and derivatives; this version implements the original and Havel’s normalized lower bound penalty, the normalized version is preferred when lower bounds are small (as with NMR NOE restraints), the original penalty is needed if large lower bounds are present

\sphinxstylestrong{BONDS Subroutine}

“bonds” finds the total number of covalent bonds and stores the atom numbers of the atoms defining each bond

\sphinxstylestrong{BORN Subroutine}

“born” computes the Born radius of each atom for use with the various GB/SA solvation models

\sphinxstylestrong{BORN1 Subroutine}

“born1” computes derivatives of the Born radii with respect to atomic coordinates and increments total energy derivatives and virial components for potentials involving Born radii

\sphinxstylestrong{BOUNDS Subroutine}

“bounds” finds the center of mass of each molecule and translates any stray molecules back into the periodic box

\sphinxstylestrong{BSET Subroutine}

“bset” computes by downward recursion the B Function**s used in the evaluation of Slater\sphinxhyphen{}type (STO) overlap integrals

\sphinxstylestrong{BSPLINE Subroutine}

“bspline” calculates the coefficients for an n\sphinxhyphen{}th order B\sphinxhyphen{}spline approximation

\sphinxstylestrong{BSPLINE1 Subroutine}

“bspline1” calculates the coefficients and derivative coefficients for an n\sphinxhyphen{}th order B\sphinxhyphen{}spline approximation

\sphinxstylestrong{BSSTEP Subroutine}

“bsstep” takes a single Bulirsch\sphinxhyphen{}Stoer step with monitoring of local truncation error to ensure accuracy

\sphinxstylestrong{CALENDAR Subroutine}

“calendar” returns the current time as a set of integer values representing the year, month, day, hour, minute and second

\sphinxstylestrong{CELLATOM Subroutine}

“cellatom” completes the addition of a symmetry related atom to a unit cell by updating the atom type and attachment arrays

\sphinxstylestrong{CENTER Subroutine}

“center” moves the weighted centroid of each coordinate set to the origin during least squares superposition

\sphinxstylestrong{CERROR Subroutine}

“cerror” is the error handling routine for the Connolly surface area and volume computation

\sphinxstylestrong{CFFTB Subroutine}

“cfftb” computes the backward complex discrete Fourier transform, the Fourier synthesis

\sphinxstylestrong{CFFTB1 Subroutine}

\sphinxstylestrong{CFFTF Subroutine}

“cfftf” computes the forward complex discrete Fourier transform, the Fourier analysis

\sphinxstylestrong{CFFTF1 Subroutine}

\sphinxstylestrong{CFFTI Subroutine}

“cffti” initializes the array “wsave” which is used in both forward and backward transforms; the prime factorization of “n” together with a tabulation of the trigonometric Function**s are computed and stored in “wsave”

\sphinxstylestrong{CFFTI1 Subroutine}

\sphinxstylestrong{CHIRER Function}

“chirer” computes the chirality error and its derivatives with respect to atomic Cartesian coordinates as a sum the squares of deviations of chiral volumes from target values

\sphinxstylestrong{CHKCLASH Subroutine}

“chkclash” determines if there are any atom clashes which might cause trouble on subsequent energy evaluation

\sphinxstylestrong{CHKPOLE Subroutine}

“chkpole” inverts atomic multipole moments as necessary at sites with chiral local reference frame definitions

\sphinxstylestrong{CHKRING Subroutine}

“chkring” tests angles to be constrained for their presence in small rings and removes constraints that are redundant

\sphinxstylestrong{CHKSIZE Subroutine}

“chksize” computes a measure of overall global structural expansion or compaction from the number of excess upper or lower bounds matrix violations

\sphinxstylestrong{CHKTREE Subroutine}

“chktree” tests a minimum energy structure to see if it belongs to the correct progenitor in the existing map

\sphinxstylestrong{CHKXYZ Subroutine}

“chkxyz” finds any pairs of atoms with identical Cartesian coordinates, and prints a warning message

\sphinxstylestrong{CHOLESKY Subroutine}

“cholesky” uses a modified Cholesky method to solve the linear system Ax = b, returning “x” in “b”; “A” is assumed to be a real symmetric positive definite matrix with its diagonal and upper triangle stored by rows

\sphinxstylestrong{CIRPLN Subroutine}

\sphinxstylestrong{CJKM Function}

“cjkm” computes the coefficients of spherical harmonics expressed in prolate spheroidal coordinates

\sphinxstylestrong{CLIMBER Subroutine}

\sphinxstylestrong{CLIMBRGD Subroutine}

\sphinxstylestrong{CLIMBROT Subroutine}

\sphinxstylestrong{CLIMBTOR Subroutine}

\sphinxstylestrong{CLIMBXYZ Subroutine}

\sphinxstylestrong{CLOCK Subroutine}

“clock” determines elapsed CPU time in seconds since the start of the job

\sphinxstylestrong{CLUSTER Subroutine}

“cluster” gets the partitioning of the system into groups and stores a list of the group to which each atom belongs

\sphinxstylestrong{COLUMN Subroutine}

“column” takes the off\sphinxhyphen{}diagonal Hessian elements stored as sparse rows and sets up indices to allow column access

\sphinxstylestrong{COMMAND Subroutine}

“command” uses the standard Unix\sphinxhyphen{}like iargc/getarg routines to get the number and values of arguments specified on the command line at Program** runtime

\sphinxstylestrong{COMPRESS Subroutine}

“compress” transfers only the non\sphinxhyphen{}buried tori from the temporary tori arrays to the final tori arrays

\sphinxstylestrong{CONNECT Subroutine}

“connect” sets up the attached atom arrays starting from a set of internal coordinates

\sphinxstylestrong{CONNOLLY Subroutine}

“connolly” uses the algorithms from the AMS/VAM Program**s of Michael Connolly to compute the analytical molecular surface area and volume of a collection of spherical atoms; thus it implements Fred Richards’ molecular surface definition as a set of analytically defined spherical and toroidal polygons

\sphinxstylestrong{CONTACT Subroutine}

“contact” constructs the contact surface, cycles and convex faces

\sphinxstylestrong{CONTROL Subroutine}

“control” gets initial values for parameters that determine the output style and information level provided by Tinker

\sphinxstylestrong{COORDS Subroutine}

“coords” converts the three principal eigenvalues/vectors from the metric matrix into atomic coordinates, and calls a routine to compute the rms deviation from the bounds

\sphinxstylestrong{CORRELATE Program}

“correlate” computes the time correlation Function** of some user\sphinxhyphen{}supplied property from individual snapshot frames taken from a molecular dynamics or other trajectory

\sphinxstylestrong{CREATEJVM Subroutine}

\sphinxstylestrong{CREATESERVER Subroutine}

\sphinxstylestrong{CREATESYSTEM Subroutine}

\sphinxstylestrong{CREATEUPDATE Subroutine}

\sphinxstylestrong{CRYSTAL Program}

“crystal” is a utility Program** which converts between fractional and Cartesian coordinates, and can generate full unit cells from asymmetric units

\sphinxstylestrong{CUTOFFS Subroutine}

“cutoffs” initializes and stores spherical energy cutoff distance windows, Hessian element and Ewald sum cutoffs, and the pairwise neighbor generation method

\sphinxstylestrong{CYTSY Subroutine}

“cytsy” solves a system of linear equations for a cyclically tridiagonal, symmetric, positive definite matrix

\sphinxstylestrong{CYTSYP Subroutine}

“cytsyp” finds the Cholesky factors of a cyclically tridiagonal symmetric, positive definite matrix given by two vectors

\sphinxstylestrong{CYTSYS Subroutine}

“cytsys” solves a cyclically tridiagonal linear system given the Cholesky factors

\sphinxstylestrong{D1D2 Function}

“d1d2” is a utility Function** used in computation of the reaction field recursive summation elements

\sphinxstylestrong{DELETE Subroutine}

“delete” removes a specified atom from the Cartesian coordinates list and shifts the remaining atoms

\sphinxstylestrong{DEPTH Function}

\sphinxstylestrong{DESTROYJVM Subroutine}

\sphinxstylestrong{DESTROYSERVER Subroutine}

\sphinxstylestrong{DFTMOD Subroutine}

“dftmod” computes the modulus of the discrete Fourier transform of “bsarray”, storing it into “bsmod”

\sphinxstylestrong{DIAGQ Subroutine}

“diagq” is a matrix diagonalization routine which is derived from the classical given, housec, and eigen algorithms with several modifications to increase the efficiency and accuracy

\sphinxstylestrong{DIFFEQ Subroutine}

“diffeq” performs the numerical integration of an ordinary differential equation using an adaptive stepsize method to solve the corresponding coupled first\sphinxhyphen{}order equations of the general form dyi/dx = f(x,y1,…,yn) for yi = y1,…,yn

\sphinxstylestrong{DIFFUSE Program}

“diffuse” finds the self\sphinxhyphen{}diffusion constant for a homogeneous liquid via the Einstein relation from a set of stored molecular dynamics frames; molecular centers of mass are unfolded and mean squared displacements are computed versus time separation

\sphinxstylestrong{DIST2 Function}

“dist2” finds the distance squared between two points; used as a service routine by the Connolly surface area and volume computation

\sphinxstylestrong{DISTGEOM Program}

“distgeom” uses a metric matrix distance geometry procedure to generate structures with interpoint distances that lie within specified bounds, with chiral centers that maintain chirality, and with torsional angles restrained to desired values; the user also has the ability to interactively inspect and alter the triangle smoothed bounds matrix prior to embedding

\sphinxstylestrong{DMDUMP Subroutine}

“dmdump” puts the distance matrix of the final structure into the upper half of a matrix, the distance of each atom to the centroid on the diagonal, and the individual terms of the bounds errors into the lower half of the matrix

\sphinxstylestrong{DOCUMENT Program}

“document” generates a formatted description of all the code modules or common blocks, an index of routines called by each source code module, a listing of all valid keywords, a list of include file dependencies as needed by a Unix\sphinxhyphen{}style Makefile, or a formatted force field parameter set summary

\sphinxstylestrong{DOT Function}

“dot” finds the dot product of two vectors

\sphinxstylestrong{DSTMAT Subroutine}

“dstmat” selects a distance matrix containing values between the previously smoothed upper and lower bounds; the distance values are chosen from uniform distributions, in a triangle correlated fashion, or using random partial metrization

\sphinxstylestrong{DYNAMIC Program}

“dynamic” computes a molecular dynamics trajectory in any of several statistical mechanical ensembles with optional periodic boundaries and optional coupling to temperature and pressure baths alternatively a stochastic dynamics trajectory can be generated

\sphinxstylestrong{EANGANG Subroutine}

“eangang” calculates the angle\sphinxhyphen{}angle potential energy

\sphinxstylestrong{EANGANG1 Subroutine}

“eangang1” calculates the angle\sphinxhyphen{}angle potential energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EANGANG2 Subroutine}

“eangang2” calculates the angle\sphinxhyphen{}angle potential energy second derivatives with respect to Cartesian coordinates using finite difference methods

\sphinxstylestrong{EANGANG2A Subroutine}

“eangang2a” calculates the angle\sphinxhyphen{}angle first derivatives for a single interaction with respect to Cartesian coordinates; used in computation of finite difference second derivatives

\sphinxstylestrong{EANGANG3 Subroutine}

“eangang3” calculates the angle\sphinxhyphen{}angle potential energy; also partitions the energy among the atoms

\sphinxstylestrong{EANGLE Subroutine}

“eangle” calculates the angle bending potential energy; projected in\sphinxhyphen{}plane angles at trigonal centers or Fourier angle bending terms are optionally used

\sphinxstylestrong{EANGLE1 Subroutine}

“eangle1” calculates the angle bending potential energy and the first derivatives with respect to Cartesian coordinates; projected in\sphinxhyphen{}plane angles at trigonal centers or Fourier angle bending terms are optionally used

\sphinxstylestrong{EANGLE2 Subroutine}

“eangle2” calculates second derivatives of the angle bending energy for a single atom using a mixture of analytical and finite difference methods; projected in\sphinxhyphen{}plane angles at trigonal centers or Fourier angle bending terms are optionally used

\sphinxstylestrong{EANGLE2A Subroutine}

“eangle2a” calculates bond angle bending potential energy second derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EANGLE2B Subroutine}

“eangle2b” computes projected in\sphinxhyphen{}plane bending first derivatives for a single angle with respect to Cartesian coordinates; used in computation of finite difference second derivatives

\sphinxstylestrong{EANGLE3 Subroutine}

“eangle3” calculates the angle bending potential energy, also partitions the energy among the atoms; projected in\sphinxhyphen{}plane angles at trigonal centers or Fourier angle bending terms are optionally used

\sphinxstylestrong{EBOND Subroutine}

“ebond” calculates the bond stretching energy

\sphinxstylestrong{EBOND1 Subroutine}

“ebond1” calculates the bond stretching energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EBOND2 Subroutine}

“ebond2” calculates second derivatives of the bond stretching energy for a single atom at a time

\sphinxstylestrong{EBOND3 Subroutine}

“ebond3” calculates the bond stretching energy; also partitions the energy among the atoms

\sphinxstylestrong{EBUCK Subroutine}

“ebuck” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy

\sphinxstylestrong{EBUCK0A Subroutine}

“ebuck0a” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy using a pairwise double loop

\sphinxstylestrong{EBUCK0B Subroutine}

“ebuck0b” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy using the method of lights to locate neighboring atoms

\sphinxstylestrong{EBUCK0C Subroutine}

“ebuck0c” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy via a Gaussian approximation for potential energy smoothing

\sphinxstylestrong{EBUCK1 Subroutine}

“ebuck1” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EBUCK1A Subroutine}

“ebuck1a” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and its first derivatives using a pairwise double loop

\sphinxstylestrong{EBUCK1B Subroutine}

“ebuck1b” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and its first derivatives using the method of lights to locate neighboring atoms

\sphinxstylestrong{EBUCK1C Subroutine}

“ebuck1c” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and its first derivatives via a Gaussian approximation for potential energy smoothing

\sphinxstylestrong{EBUCK2 Subroutine}

“ebuck2” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals second derivatives for a single atom at a time

\sphinxstylestrong{EBUCK2A Subroutine}

“ebuck2a” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals second derivatives using a double loop over relevant atom pairs

\sphinxstylestrong{EBUCK2B Subroutine}

“ebuck2b” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals second derivatives via a Gaussian approximation for use with potential energy smoothing

\sphinxstylestrong{EBUCK3 Subroutine}

“ebuck3” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and partitions the energy among the atoms

\sphinxstylestrong{EBUCK3A Subroutine}

“ebuck3a” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and partitions the energy among the atoms using a pairwise double loop

\sphinxstylestrong{EBUCK3B Subroutine}

“ebuck3b” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy and also partitions the energy among the atoms using the method of lights to locate neighboring atoms

\sphinxstylestrong{EBUCK3C Subroutine}

“ebuck3c” calculates the Buckingham exp\sphinxhyphen{}6 van der Waals energy via a Gaussian approximation for potential energy smoothing

\sphinxstylestrong{ECHARGE Subroutine}

“echarge” calculates the charge\sphinxhyphen{}charge interaction energy

\sphinxstylestrong{ECHARGE0A Subroutine}

“echarge0a” calculates the charge\sphinxhyphen{}charge interaction energy using a pairwise double loop

\sphinxstylestrong{ECHARGE0B Subroutine}

“echarge0b” calculates the charge\sphinxhyphen{}charge interaction energy using the method of lights to locate neighboring atoms

\sphinxstylestrong{ECHARGE0C Subroutine}

“echarge0c” calculates the charge\sphinxhyphen{}charge interaction energy for use with potential smoothing methods

\sphinxstylestrong{ECHARGE0D Subroutine}

“echarge0d” calculates the charge\sphinxhyphen{}charge interaction energy using a particle mesh Ewald summation

\sphinxstylestrong{ECHARGE0E Subroutine}

“echarge0e” calculates the charge\sphinxhyphen{}charge interaction energy using a particle mesh Ewald summation and the method of lights to locate neighboring atoms

\sphinxstylestrong{ECHARGE1 Subroutine}

“echarge1” calculates the charge\sphinxhyphen{}charge interaction energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ECHARGE1A Subroutine}

“echarge1a” calculates the charge\sphinxhyphen{}charge interaction energy and first derivatives with respect to Cartesian coordinates using a pairwise double loop

\sphinxstylestrong{ECHARGE1B Subroutine}

“echarge1b” calculates the charge\sphinxhyphen{}charge interaction energy and first derivatives with respect to Cartesian coordinates using the method of lights to locate neighboring atoms

\sphinxstylestrong{ECHARGE1C Subroutine}

“echarge1c” calculates the charge\sphinxhyphen{}charge interaction energy and first derivatives with respect to Cartesian coordinates for use with potential smoothing methods

\sphinxstylestrong{ECHARGE1D Subroutine}

“echarge1d” calculates the charge\sphinxhyphen{}charge interaction energy and first derivatives with respect to Cartesian coordinates using a particle mesh Ewald summation

\sphinxstylestrong{ECHARGE2 Subroutine}

“echarge2” calculates second derivatives of the charge\sphinxhyphen{}charge interaction energy for a single atom

\sphinxstylestrong{ECHARGE2A Subroutine}

“echarge2a” calculates second derivatives of the charge\sphinxhyphen{}charge interaction energy for a single atom using a pairwise double loop

\sphinxstylestrong{ECHARGE2B Subroutine}

“echarge2b” calculates second derivatives of the charge\sphinxhyphen{}charge interaction energy for a single atom for use with potential smoothing methods

\sphinxstylestrong{ECHARGE2C Subroutine}

“echarge2c” calculates second derivatives of the charge\sphinxhyphen{}charge interaction energy for a single atom using a particle mesh Ewald summation

\sphinxstylestrong{ECHARGE3 Subroutine}

“echarge3” calculates the charge\sphinxhyphen{}charge interaction energy and partitions the energy among the atoms

\sphinxstylestrong{ECHARGE3A Subroutine}

“echarge3a” calculates the charge\sphinxhyphen{}charge interaction energy and partitions the energy among the atoms using a pairwise double loop

\sphinxstylestrong{ECHARGE3B Subroutine}

“echarge3b” calculates the charge\sphinxhyphen{}charge interaction energy and partitions the energy among the atoms using the method of lights to locate neighboring atoms

\sphinxstylestrong{ECHARGE3C Subroutine}

“echarge3c” calculates the charge\sphinxhyphen{}charge interaction energy and partitions the energy among the atoms for use with potential smoothing methods

\sphinxstylestrong{ECHARGE3D Subroutine}

“echarge3d” calculates the charge\sphinxhyphen{}charge interaction energy and partitions the energy among the atoms using a particle mesh Ewald summation

\sphinxstylestrong{ECHARGE3E Subroutine}

“echarge3e” calculates the charge\sphinxhyphen{}charge interaction energy and partitions the energy among the atoms using a particle mesh Ewald summation and the method of lights to locate neighboring atoms

\sphinxstylestrong{ECHGDPL Subroutine}

“echgdpl” calculates the charge\sphinxhyphen{}dipole interaction energy

\sphinxstylestrong{ECHGDPL1 Subroutine}

“echgdpl1” calculates the charge\sphinxhyphen{}dipole interaction energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ECHGDPL2 Subroutine}

“echgdpl2” calculates second derivatives of the charge\sphinxhyphen{}dipole interaction energy for a single atom

\sphinxstylestrong{ECHGDPL3 Subroutine}

“echgdpl3” calculates the charge\sphinxhyphen{}dipole interaction energy; also partitions the energy among the atoms

\sphinxstylestrong{EDIPOLE Subroutine}

“edipole” calculates the dipole\sphinxhyphen{}dipole interaction energy

\sphinxstylestrong{EDIPOLE1 Subroutine}

“edipole1” calculates the dipole\sphinxhyphen{}dipole interaction energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EDIPOLE2 Subroutine}

“edipole2” calculates second derivatives of the dipole\sphinxhyphen{}dipole interaction energy for a single atom

\sphinxstylestrong{EDIPOLE3 Subroutine}

“edipole3” calculates the dipole\sphinxhyphen{}dipole interaction energy; also partitions the energy among the atoms

\sphinxstylestrong{EGAUSS Subroutine}

“egauss” calculates the Gaussian expansion van der Waals interaction energy

\sphinxstylestrong{EGAUSS0A Subroutine}

“egauss0a” calculates the Gaussian expansion van der Waals interaction energy using a pairwise double loop

\sphinxstylestrong{EGAUSS0B Subroutine}

“egauss0b” calculates the Gaussian expansion van der Waals interaction energy for use with potential energy smoothing

\sphinxstylestrong{EGAUSS1 Subroutine}

“egauss1” calculates the Gaussian expansion van der Waals interaction energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EGAUSS1A Subroutine}

“egauss1a” calculates the Gaussian expansion van der Waals interaction energy and its first derivatives using a pairwise double loop

\sphinxstylestrong{EGAUSS1B Subroutine}

“egauss1b” calculates the Gaussian expansion van der Waals interaction energy and its first derivatives for use with stophat potential energy smoothing

\sphinxstylestrong{EGAUSS2 Subroutine}

“egauss2” calculates the Gaussian expansion van der Waals second derivatives for a single atom at a time

\sphinxstylestrong{EGAUSS2A Subroutine}

“egauss2a” calculates the Gaussian expansion van der Waals second derivatives using a pairwise double loop

\sphinxstylestrong{EGAUSS2B Subroutine}

“egauss2b” calculates the Gaussian expansion van der Waals second derivatives for stophat potential energy smoothing

\sphinxstylestrong{EGAUSS3 Subroutine}

“egauss3” calculates the Gaussian expansion van der Waals interaction energy and partitions the energy among the atoms

\sphinxstylestrong{EGAUSS3A Subroutine}

“egauss3a” calculates the Gaussian expansion van der Waals interaction energy and partitions the energy among the atoms using a pairwise double loop

\sphinxstylestrong{EGAUSS3B Subroutine}

“egauss3b” calculates the Gaussian expansion van der Waals interaction energy and partitions the energy among the atoms using a pairwise double loop

\sphinxstylestrong{EGBSA0A Subroutine}

“egbsa0a” calculates the generalized Born polarization energy for the GB/SA solvation models

\sphinxstylestrong{EGBSA0B Subroutine}

“egbsa0b” calculates the generalized Born polarization energy for the GB/SA solvation models for use with potential smoothing methods via analogy to the smoothing of Coulomb’s law

\sphinxstylestrong{EGBSA1A Subroutine}

“egbsa1a” calculates the generalized Born energy and first derivatives of the GB/SA solvation models

\sphinxstylestrong{EGBSA1B Subroutine}

“egbsa1b” calculates the generalized Born energy and first derivatives of the GB/SA solvation models for use with potential smoothing methods

\sphinxstylestrong{EGBSA2A Subroutine}

“egbsa2a” calculates second derivatives of the generalized Born energy term for the GB/SA solvation models

\sphinxstylestrong{EGBSA2B Subroutine}

“egbsa2b” calculates second derivatives of the generalized Born energy term for the GB/SA solvation models for use with potential smoothing methods

\sphinxstylestrong{EGBSA3A Subroutine}

“egbsa3a” calculates the generalized Born energy term for the GB/SA solvation models; also partitions the energy among the atoms

\sphinxstylestrong{EGBSA3B Subroutine}

“egbsa3b” calculates the generalized Born polarization energy for the GB/SA solvation models for use with potential smoothing methods via analogy to the smoothing of Coulomb’s law; also partitions the energy among the atoms

\sphinxstylestrong{EGEOM Subroutine}

“egeom” calculates the energy due to restraints on positions, distances, angles and torsions as well as Gaussian basin and spherical droplet restraints

\sphinxstylestrong{EGEOM1 Subroutine}

“egeom1” calculates the energy and first derivatives with respect to Cartesian coordinates due to restraints on positions, distances, angles and torsions as well as Gaussian basin and spherical droplet restraints

\sphinxstylestrong{EGEOM2 Subroutine}

“egeom2” calculates second derivatives of restraints on positions, distances, angles and torsions as well as Gaussian basin and spherical droplet restraints

\sphinxstylestrong{EGEOM3 Subroutine}

“egeom3” calculates the energy due to restraints on positions, distances, angles and torsions as well as Gaussian basin and droplet restraints; also partitions energy among the atoms

\sphinxstylestrong{EHAL Subroutine}

“ehal” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy

\sphinxstylestrong{EHAL0A Subroutine}

“ehal0a” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy using a pairwise double loop

\sphinxstylestrong{EHAL0B Subroutine}

“ehal0a” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy using the method of lights to locate neighboring atoms

\sphinxstylestrong{EHAL1 Subroutine}

“ehal1” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EHAL1A Subroutine}

“ehal1a” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy and its first derivatives with respect to Cartesian coordinates using a pairwise double loop

\sphinxstylestrong{EHAL1B Subroutine}

“ehal1b” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy and its first derivatives with respect to Cartesian coordinates using the method of lights to locate neighboring atoms

\sphinxstylestrong{EHAL2 Subroutine}

“ehal2” calculates the buffered 14\sphinxhyphen{}7 van der Waals second derivatives for a single atom at a time

\sphinxstylestrong{EHAL3 Subroutine}

“ehal3” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy and partitions the energy among the atoms

\sphinxstylestrong{EHAL3A Subroutine}

“ehal3a” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy and partitions the energy among the atoms using a pairwise double loop

\sphinxstylestrong{EHAL3B Subroutine}

“ehal3b” calculates the buffered 14\sphinxhyphen{}7 van der Waals energy and also partitions the energy among the atoms using the method of lights to locate neighboring atoms

\sphinxstylestrong{EIGEN Subroutine}

“eigen” uses the power method to compute the largest eigenvalues and eigenvectors of the metric matrix, “valid” is set true if the first three eigenvalues are positive

\sphinxstylestrong{EIGENRGD Subroutine}

\sphinxstylestrong{EIGENROT Subroutine}

\sphinxstylestrong{EIGENROT Subroutine}

\sphinxstylestrong{EIGENTOR Subroutine}

\sphinxstylestrong{EIGENXYZ Subroutine}

\sphinxstylestrong{EIMPROP Subroutine}

“eimprop” calculates the improper dihedral potential energy

\sphinxstylestrong{EIMPROP1 Subroutine}

“eimprop1” calculates improper dihedral energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EIMPROP2 Subroutine}

“eimprop2” calculates second derivatives of the improper dihedral angle energy for a single atom

\sphinxstylestrong{EIMPROP3 Subroutine}

“eimprop3” calculates the improper dihedral potential energy; also partitions the energy terms among the atoms

\sphinxstylestrong{EIMPTOR Subroutine}

“eimptor” calculates the improper torsion potential energy

\sphinxstylestrong{EIMPTOR1 Subroutine}

“eimptor1” calculates improper torsion energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EIMPTOR2 Subroutine}

“eimptor2” calculates second derivatives of the improper torsion energy for a single atom

\sphinxstylestrong{EIMPTOR3 Subroutine}

“eimptor3” calculates the improper torsion potential energy; also partitions the energy terms among the atoms

\sphinxstylestrong{ELJ Subroutine}

“elj” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy

\sphinxstylestrong{ELJ0A Subroutine}

“elj0a” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy using a pairwise double loop

\sphinxstylestrong{ELJ0B Subroutine}

“elj0b” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy using the method of lights to locate neighboring atoms

\sphinxstylestrong{ELJ0C Subroutine}

“elj0c” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy via a Gaussian approximation for potential energy smoothing

\sphinxstylestrong{ELJ0D Subroutine}

“elj0d” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy for use with stophat potential energy smoothing

\sphinxstylestrong{ELJ1 Subroutine}

“elj1” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ELJ1A Subroutine}

“elj1a” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and its first derivatives using a pairwise double loop

\sphinxstylestrong{ELJ1B Subroutine}

“elj1b” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and its first derivatives using the method of lights to locate neighboring atoms

\sphinxstylestrong{ELJ1C Subroutine}

“elj1c” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy  and its first derivatives via a Gaussian approximation for  potential energy smoothing

\sphinxstylestrong{ELJ1D Subroutine}

“elj1d” calculates the van der Waals interaction energy and its first derivatives for use with stophat potential energy smoothing

\sphinxstylestrong{ELJ2 Subroutine}

“elj2” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals second derivatives for a single atom at a time

\sphinxstylestrong{ELJ2A Subroutine}

“elj2a” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals second derivatives using a double loop over relevant atom pairs

\sphinxstylestrong{ELJ2B Subroutine}

“elj2b” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals second derivatives via a Gaussian approximation for use with potential energy smoothing

\sphinxstylestrong{ELJ2C Subroutine}

“elj2c” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals second derivatives for use with stophat potential energy smoothing

\sphinxstylestrong{ELJ3 Subroutine}

“elj3” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and also partitions the energy among the atoms

\sphinxstylestrong{ELJ3A Subroutine}

“elj3a” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and also partitions the energy among the atoms using a pairwise double loop

\sphinxstylestrong{ELJ3B Subroutine}

“elj3b” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and also partitions the energy among the atoms using the method of lights to locate neighboring atoms

\sphinxstylestrong{ELJ3C Subroutine}

“elj3c” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and also partitions the energy among the atoms via a Gaussian approximation for potential energy smoothing

\sphinxstylestrong{ELJ3D Subroutine}

“elj3d” calculates the Lennard\sphinxhyphen{}Jones 6\sphinxhyphen{}12 van der Waals energy and also partitions the energy among the atoms for use with stophat potential energy smoothing

\sphinxstylestrong{EMBED Subroutine}

“embed” is a distance geometry routine patterned after the ideas of Gordon Crippen, Irwin Kuntz and Tim Havel; it takes as input a set of upper and lower bounds on the interpoint distances, chirality restraints and torsional restraints, and attempts to generate a set of coordinates that satisfy the input bounds and restraints

\sphinxstylestrong{EMETAL Subroutine}

“emetal” calculates the transition metal ligand field energy

\sphinxstylestrong{EMETAL1 Subroutine}

“emetal1” calculates the transition metal ligand field energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EMETAL2 Subroutine}

“emetal2” calculates the transition metal ligand field second derivatives for a single atom at a time

\sphinxstylestrong{EMETAL3 Subroutine}

“emetal3” calculates the transition metal ligand field energy and also partitions the energy among the atoms

\sphinxstylestrong{EMM3HB Subroutine}

“emm3hb” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy

\sphinxstylestrong{EMM3HB0A Subroutine}

“emm3hb0a” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy using a pairwise double loop

\sphinxstylestrong{EMM3HB0B Subroutine}

“emm3hb0b” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy using the method of lights to locate neighboring atoms

\sphinxstylestrong{EMM3HB1 Subroutine}

“emm3hb1” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy with respect to Cartesian coordinates

\sphinxstylestrong{EMM3HB1A Subroutine}

“emm3hb1a” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy with respect to Cartesian coordinates using a pairwise double loop

\sphinxstylestrong{EMM3HB1B Subroutine}

“emm3hb1b” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy with respect to Cartesian coordinates using the method of lights to locate neighboring atoms

\sphinxstylestrong{EMM3HB2 Subroutine}

“emm3hb2” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding second derivatives for a single atom at a time

\sphinxstylestrong{EMM3HB3 Subroutine}

“emm3hb3” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy, and partitions the energy among the atoms

\sphinxstylestrong{EMM3HB3A Subroutine}

“emm3hb3” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy, and partitions the energy among the atoms

\sphinxstylestrong{EMM3HB3B Subroutine}

“emm3hb3b” calculates the MM3 exp\sphinxhyphen{}6 van der Waals and directional charge transfer hydrogen bonding energy using the method of lights to locate neighboring atoms

\sphinxstylestrong{EMPOLE Subroutine}

“empole” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability

\sphinxstylestrong{EMPOLE0A Subroutine}

“empole0a” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability using a pairwise double loop

\sphinxstylestrong{EMPOLE0B Subroutine}

“empole0b” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability using a regular Ewald summation

\sphinxstylestrong{EMPOLE1 Subroutine}

“empole1” calculates the multipole and dipole polarization energy and derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EMPOLE1A Subroutine}

“empole1a” calculates the multipole and dipole polarization energy and derivatives with respect to Cartesian coordinates using a pairwise double loop

\sphinxstylestrong{EMPOLE1B Subroutine}

“empole1b” calculates the multipole and dipole polarization energy and derivatives with respect to Cartesian coordinates using a regular Ewald summation

\sphinxstylestrong{EMPOLE2 Subroutine}

“empole2” calculates second derivatives of the multipole and dipole polarization energy for a single atom at a time

\sphinxstylestrong{EMPOLE2A Subroutine}

“empole2a” computes multipole and dipole polarization first derivatives for a single atom with respect to Cartesian coordinates; used to get finite difference second derivatives

\sphinxstylestrong{EMPOLE3 Subroutine}

“empole3” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability, and partitions the energy among the atoms

\sphinxstylestrong{EMPOLE3A Subroutine}

“empole3a” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability, and partitions the energy among the atoms using a double loop

\sphinxstylestrong{EMPOLE3B Subroutine}

“empole3b” calculates the electrostatic energy due to atomic multipole interactions and dipole polarizability, and partitions the energy among the atoms using a regular Ewald summation

\sphinxstylestrong{ENERGY Function}

“energy” calls the Subroutine**s to calculate the potential energy terms and sums up to form the total energy

\sphinxstylestrong{ENRGYZE Subroutine}

“energyze” is an auxiliary routine for the analyze Program** that performs the energy analysis and prints the total and intermolecular energies

\sphinxstylestrong{EOPBEND Subroutine}

“eopbend” computes the out\sphinxhyphen{}of\sphinxhyphen{}plane bend potential energy at trigonal centers via a Wilson\sphinxhyphen{}Decius\sphinxhyphen{}Cross angle bend

\sphinxstylestrong{EOPBEND1 Subroutine}

“eopbend1” computes the out\sphinxhyphen{}of\sphinxhyphen{}plane bend potential energy and first derivatives at trigonal centers via a Wilson\sphinxhyphen{}Decius\sphinxhyphen{}Cross angle bend

\sphinxstylestrong{EOPBEND2 Subroutine}

“eopbend2” calculates second derivatives of the out\sphinxhyphen{}of\sphinxhyphen{}plane bend energy via a Wilson\sphinxhyphen{}Decius\sphinxhyphen{}Cross angle bend for a single atom using finite difference methods

\sphinxstylestrong{EOPBEND2A Subroutine}

“eopbend2a” calculates out\sphinxhyphen{}of\sphinxhyphen{}plane bending first derivatives at a trigonal center via a Wilson\sphinxhyphen{}Decius\sphinxhyphen{}Cross angle bend; used in computation of finite difference second derivatives

\sphinxstylestrong{EOPBEND3 Subroutine}

“eopbend3” computes the out\sphinxhyphen{}of\sphinxhyphen{}plane bend potential energy at trigonal centers via a Wilson\sphinxhyphen{}Decius\sphinxhyphen{}Cross angle bend; also partitions the energy among the atoms

\sphinxstylestrong{EOPDIST Subroutine}

“eopdist” computes the out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential energy at trigonal centers via the central atom height

\sphinxstylestrong{EOPDIST1 Subroutine}

“eopdist1” computes the out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential energy and first derivatives at trigonal centers via the central atom height

\sphinxstylestrong{EOPDIST2 Subroutine}

“eopdist2” calculates second derivatives of the out\sphinxhyphen{}of\sphinxhyphen{}plane distance energy for a single atom via the central atom height

\sphinxstylestrong{EOPDIST3 Subroutine}

“eopdist3” computes the out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential energy at trigonal centers via the central atom height; also partitions the energy among the atoms

\sphinxstylestrong{EPITORS Subroutine}

“epitors” calculates the pi\sphinxhyphen{}orbital torsion potential energy

\sphinxstylestrong{EPITORS1 Subroutine}

“epitors1” calculates the pi\sphinxhyphen{}orbital torsion potential energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EPITORS2 Subroutine}

“epitors2” calculates the second derivatives of the pi\sphinxhyphen{}orbital torsion energy for a single atom using finite difference methods

\sphinxstylestrong{EPITORS2A Subroutine}

“epitors2a” calculates the pi\sphinxhyphen{}orbital torsion first derivatives; used in computation of finite difference second derivatives

\sphinxstylestrong{EPITORS3 Subroutine}

“epitors3” calculates the pi\sphinxhyphen{}orbital torsion potential energy; also partitions the energy terms among the atoms

\sphinxstylestrong{EPME Subroutine}

“epme” computes the reciprocal space energy for a particle mesh Ewald summation over partial charges

\sphinxstylestrong{EPME1 Subroutine}

“epme1” computes the reciprocal space energy and first derivatives for a particle mesh Ewald summation

\sphinxstylestrong{EPME3 Subroutine}

“epme3” computes the reciprocal space energy for a particle mesh Ewald summation over partial charges and prints information about the energy over the charge grid points

\sphinxstylestrong{EPUCLC Subroutine}

\sphinxstylestrong{EREAL Subroutine}

“ereal” evaluates the real space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability

\sphinxstylestrong{EREAL1 Subroutine}

“ereal1” evaluates the real space portion of the regular Ewald summation energy and gradient due to atomic multipole interactions and dipole polarizability

\sphinxstylestrong{EREAL3 Subroutine}

“ereal3” evaluates the real space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability and partitions the energy among the atoms

\sphinxstylestrong{ERECIP Subroutine}

“erecip” evaluates the reciprocal space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability

\sphinxstylestrong{ERECIP1 Subroutine}

“erecip1” evaluates the reciprocal space portion of the regular Ewald summation energy and gradient due to atomic multipole interactions and dipole polarizability

\sphinxstylestrong{ERECIP3 Subroutine}

“erecip3” evaluates the reciprocal space portion of the regular Ewald summation energy due to atomic multipole interactions and dipole polarizability, and prints information about the energy over the reciprocal lattice vectors

\sphinxstylestrong{ERF Function}

“erf” computes a numerical approximation to the value of the error Function** via a Chebyshev approximation

\sphinxstylestrong{ERFC Function}

“erfc” computes a numerical approximation to the value of the complementary error Function** via a Chebyshev approximation

\sphinxstylestrong{ERFCORE Subroutine}

“erfcore” evaluates erf(x) or erfc(x) for a real argument x; when called with mode set to 0 it returns erf, a mode of 1 returns erfc; uses rational Function**s that approximate erf(x) and erfc(x) to at least 18 significant decimal digits

\sphinxstylestrong{ERFIK Subroutine}

“erfik” compute the reaction field energy due to a single pair of atomic multipoles

\sphinxstylestrong{ERFINV Function}

“erfinv” evaluates the inverse of the error Function** erf for a real argument in the range (\sphinxhyphen{}1,1) using a rational Function** approximation followed by cycles of Newton\sphinxhyphen{}Raphson correction

\sphinxstylestrong{ERXNFLD Subroutine}

“erxnfld” calculates the macroscopic reaction field energy arising from a set of atomic multipoles

\sphinxstylestrong{ERXNFLD1 Subroutine}

“erxnfld1” calculates the macroscopic reaction field energy and derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ERXNFLD2 Subroutine}

“erxnfld2” calculates second derivatives of the macroscopic reaction field energy for a single atom at a time

\sphinxstylestrong{ERXNFLD3 Subroutine}

“erxnfld3” calculates the macroscopic reaction field energy, and also partitions the energy among the atoms

\sphinxstylestrong{ESOLV Subroutine}

“esolv” calculates the continuum solvation energy via either the Eisenberg\sphinxhyphen{}McLachlan ASP model, Ooi\sphinxhyphen{}Scheraga SASA model, various GB/SA methods or the ACE model

\sphinxstylestrong{ESOLV1 Subroutine}

“esolv1” calculates the continuum solvation energy and first derivatives with respect to Cartesian coordinates using either the Eisenberg\sphinxhyphen{}McLachlan ASP, Ooi\sphinxhyphen{}Scheraga SASA or various GB/SA solvation models

\sphinxstylestrong{ESOLV2 Subroutine}

“esolv2” calculates second derivatives of the continuum solvation energy using either the Eisenberg\sphinxhyphen{}McLachlan ASP, Ooi\sphinxhyphen{}Scheraga SASA or various GB/SA solvation models

\sphinxstylestrong{ESOLV3 Subroutine}

“esolv3” calculates the continuum solvation energy using either the Eisenberg\sphinxhyphen{}McLachlan ASP model, Ooi\sphinxhyphen{}Scheraga SASA model, various GB/SA methods or the ACE model; also partitions the energy among the atoms

\sphinxstylestrong{ESTRBND Subroutine}

“estrbnd” calculates the stretch\sphinxhyphen{}bend potential energy

\sphinxstylestrong{ESTRBND1 Subroutine}

“estrbnd1” calculates the stretch\sphinxhyphen{}bend potential energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ESTRBND2 Subroutine}

“estrbnd2” calculates the stretch\sphinxhyphen{}bend potential energy second derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ESTRBND3 Subroutine}

“estrbnd3” calculates the stretch\sphinxhyphen{}bend potential energy; also partitions the energy among the atoms

\sphinxstylestrong{ESTRTOR Subroutine}

“estrtor” calculates the stretch\sphinxhyphen{}torsion potential energy

\sphinxstylestrong{ESTRTOR1 Subroutine}

“estrtor1” calculates the stretch\sphinxhyphen{}torsion energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ESTRTOR2 Subroutine}

“estrtor2” calculates the stretch\sphinxhyphen{}torsion potential energy second derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ESTRTOR3 Subroutine}

“estrtor3” calculates the stretch\sphinxhyphen{}torsion potential energy; also partitions the energy terms among the atoms

\sphinxstylestrong{ETORS Subroutine}

“etors” calculates the torsional potential energy

\sphinxstylestrong{ETORS0A Subroutine}

“etors0a” calculates the torsional potential energy using a standard sum of Fourier terms

\sphinxstylestrong{ETORS0B Subroutine}

“etors0b” calculates the torsional potential energy for use with potential energy smoothing methods

\sphinxstylestrong{ETORS1 Subroutine}

“etors1” calculates the torsional potential energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ETORS1A Subroutine}

“etors1a” calculates the torsional potential energy and first derivatives with respect to Cartesian coordinates using a standard sum of Fourier terms

\sphinxstylestrong{ETORS1B Subroutine}

“etors1b” calculates the torsional potential energy and first derivatives with respect to Cartesian coordinates for use with potential energy smoothing methods

\sphinxstylestrong{ETORS2 Subroutine}

“etors2” calculates the second derivatives of the torsional energy for a single atom

\sphinxstylestrong{ETORS2A Subroutine}

“etors2a” calculates the second derivatives of the torsional energy for a single atom using a standard sum of Fourier terms

\sphinxstylestrong{ETORS2B Subroutine}

“etors2b” calculates the second derivatives of the torsional energy for a single atom for use with potential energy smoothing methods

\sphinxstylestrong{ETORS3 Subroutine}

“etors3” calculates the torsional potential energy; also partitions the energy among the atoms

\sphinxstylestrong{ETORS3A Subroutine}

“etors3a” calculates the torsional potential energy using a standard sum of Fourier terms and partitions the energy among the atoms

\sphinxstylestrong{ETORS3B Subroutine}

“etors3b” calculates the torsional potential energy for use with potential energy smoothing methods and partitions the energy among the atoms

\sphinxstylestrong{ETORTOR Subroutine}

“etortor” calculates the torsion\sphinxhyphen{}torsion potential energy

\sphinxstylestrong{ETORTOR1 Subroutine}

“etortor1” calculates the torsion\sphinxhyphen{}torsion energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ETORTOR2 Subroutine}

“etortor2” calculates the torsion\sphinxhyphen{}torsion potential energy second derivatives with respect to Cartesian coordinates

\sphinxstylestrong{ETORTOR3 Subroutine}

“etortor3” calculates the torsion\sphinxhyphen{}torsion potential energy; also partitions the energy terms among the atoms

\sphinxstylestrong{EUREY Subroutine}

“eurey” calculates the Urey\sphinxhyphen{}Bradley 1\sphinxhyphen{}3 interaction energy

\sphinxstylestrong{EUREY1 Subroutine}

“eurey1” calculates the Urey\sphinxhyphen{}Bradley interaction energy and its first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{EUREY2 Subroutine}

“eurey2” calculates second derivatives of the Urey\sphinxhyphen{}Bradley interaction energy for a single atom at a time

\sphinxstylestrong{EUREY3 Subroutine}

“eurey3” calculates the Urey\sphinxhyphen{}Bradley energy; also partitions the energy among the atoms

\sphinxstylestrong{EWALDCOF Subroutine}

“ewaldcof” finds a value of the Ewald coefficient such that all terms beyond the specified cutoff distance will have an value less than a specified tolerance

\sphinxstylestrong{EXPLORE Subroutine}

“explore” uses simulated annealing on an initial crude embedded distance geoemtry structure to refine versus the bound, chirality, planarity and torsional error Function**s

\sphinxstylestrong{EXTRA Subroutine}

“extra” calculates any additional user defined potential energy contribution

\sphinxstylestrong{EXTRA1 Subroutine}

“extra1” calculates any additional user defined potential energy contribution and its first derivatives

\sphinxstylestrong{EXTRA2 Subroutine}

“extra2” calculates second derivatives of any additional user defined potential energy contribution for a single atom at a time

\sphinxstylestrong{EXTRA3 Subroutine}

“extra3” calculates any additional user defined potential contribution and also partitions the energy among the atoms

\sphinxstylestrong{FATAL Subroutine}

“fatal” terminates execution due to a user request, a severe error or some other nonstandard condition

\sphinxstylestrong{FFTBACK Subroutine}

\sphinxstylestrong{FFTFRONT Subroutine}

\sphinxstylestrong{FFTSETUP Subroutine}

\sphinxstylestrong{FIELD Subroutine}

“field” sets the force field potential energy Function**s from a parameter file and modifications specified in a keyfile

\sphinxstylestrong{FINAL Subroutine}

“final” performs any final Program** actions, prints a status message, and then pauses if necessary to avoid closing the execution window

\sphinxstylestrong{FINDATM Subroutine}

“findatm” locates a specific PDB atom name type within a range of atoms from the PDB file, returns zero if the name type was not found

\sphinxstylestrong{FIXPDB Subroutine}

“fixpdb” corrects problems with PDB files by converting residue and atom names to the forms used by Tinker

\sphinxstylestrong{FRACDIST Subroutine}

“fracdist” computes a normalized distribution of the pairwise fractional distances between the smoothed upper and lower bounds

\sphinxstylestrong{FREEUNIT Function}

“freeunit” finds an unopened Fortran I/O unit and returns its numerical value from 1 to 99; the units already assigned to “input” and “iout” (usually 5 and 6) are skipped since they have special meaning as the default I/O units

\sphinxstylestrong{GAMMLN Function}

“gammln” uses a series expansion due to Lanczos to compute the natural logarithm of the Gamma Function** at “x” in {[}0,1{]}

\sphinxstylestrong{GDA Program}

“gda” implements Gaussian Density Annealing (GDA) algorithm for global optimization via simulated annealing

\sphinxstylestrong{GDA1 Subroutine}

\sphinxstylestrong{GDA2 Function}

\sphinxstylestrong{GDA3 Subroutine}

\sphinxstylestrong{GDASTAT Subroutine}

\sphinxstylestrong{GENDOT Subroutine}

“gendot” finds the coordinates of a specified number of surface points for a sphere with the input radius and coordinate center

\sphinxstylestrong{GEODESIC Subroutine}

“geodesic” smooths the upper and lower distance bounds via the triangle inequality using a sparse matrix version of a shortest path algorithm

\sphinxstylestrong{GEOMETRY Function}

“geometry” finds the value of the interatomic distance, angle or dihedral angle defined by two to four input atoms

\sphinxstylestrong{GETBASE Subroutine}

“getbase” finds the base heavy atoms for a single nucleotide residue and copies the names and coordinates to the Protein Data Bank file

\sphinxstylestrong{GETIME Subroutine}

“getime” gets elapsed CPU time in seconds for an interval

\sphinxstylestrong{GETINT Subroutine}

“getint” asks for an internal coordinate file name, then reads the internal coordinates and computes Cartesian coordinates

\sphinxstylestrong{GETKEY Subroutine}

“getkey” finds a valid keyfile and stores its contents as line images for subsequent keyword parameter searching

\sphinxstylestrong{GETMOL2 Subroutine}

“getmol2” asks for a Sybyl MOL2 molecule file name, then reads the coordinates from the file

\sphinxstylestrong{GETMONITOR Subroutine}

\sphinxstylestrong{GETNUCH Subroutine}

“getnuch” finds the nucleotide hydrogen atoms for a single residue and copies the names and coordinates to the Protein Data Bank file

\sphinxstylestrong{GETNUMB Subroutine}

“getnumb” searchs an input string from left to right for an integer and puts the numeric value in “number”; returns zero with “next” unchanged if no integer value is found

\sphinxstylestrong{GETPDB Subroutine}

“getpdb” asks for a Protein Data Bank file name, then reads in the coordinates file

\sphinxstylestrong{GETPRB Subroutine}

“getprb” tests for a possible probe position at the interface between three neighboring atoms

\sphinxstylestrong{GETPRM Subroutine}

“getprm” finds the potential energy parameter file and then opens and reads the parameters

\sphinxstylestrong{GETPROH Subroutine}

“getproh” finds the hydrogen atoms for a single amino acid residue and copies the names and coordinates to the Protein Data Bank file

\sphinxstylestrong{GETREF Subroutine}

“getref” copies structure information from the reference area into the standard variables for the current system structure

\sphinxstylestrong{GETSEQ Subroutine}

“getseq” asks the user for the amino acid sequence and torsional angle values needed to define a peptide

\sphinxstylestrong{GETSEQN Subroutine}

“getseqn” asks the user for the nucleotide sequence and torsional angle values needed to define a nucleic acid

\sphinxstylestrong{GETSIDE Subroutine}

“getside” finds the side chain heavy atoms for a single amino acid residue and copies the names and coordinates to the Protein Data Bank file

\sphinxstylestrong{GETSTRING Subroutine}

“getstring” searchs for a quoted text string within an input character string; the region between the first and second quotes is returned as the “text”; if the actual text is too long, only the first part is returned

\sphinxstylestrong{GETTEXT Subroutine}

“gettext” searchs an input string for the first string of non\sphinxhyphen{}blank characters; the region from a non\sphinxhyphen{}blank character to the first blank space is returned as “text”; if the actual text is too long, only the first part is returned

\sphinxstylestrong{GETTOR Subroutine}

“gettor” tests for a possible torus position at the interface between two atoms, and finds the torus radius, center and axis

\sphinxstylestrong{GETWORD Subroutine}

“getword” searchs an input string for the first alphabetic character (A\sphinxhyphen{}Z or a\sphinxhyphen{}z); the region from this first character to the first blank space or comma is returned as a “word”; if the actual word is too long, only the first part is returned

\sphinxstylestrong{GETXYZ Subroutine}

“getxyz” asks for a Cartesian coordinate file name, then reads in the coordinates file

\sphinxstylestrong{GRADIENT Subroutine}

“gradient” calls Subroutine**s to calculate the potential energy and first derivatives with respect to Cartesian coordinates

\sphinxstylestrong{GRADRGD Subroutine}

“gradrgd” calls Subroutine**s to calculate the potential energy and first derivatives with respect to rigid body coordinates

\sphinxstylestrong{GRADROT Subroutine}

“gradrot” calls Subroutine**s to calculate the potential energy and its torsional first derivatives

\sphinxstylestrong{GRAFIC Subroutine}

“grafic” outputs the upper \& lower triangles and diagonal of a square matrix in a schematic form for visual inspection

\sphinxstylestrong{GROUPS Subroutine}

“groups” tests a set of atoms to see if all are members of a single atom group or a pair of atom groups; if so, then the correct intra\sphinxhyphen{} or intergroup weight is assigned

\sphinxstylestrong{GRPLINE Subroutine}

“grpline” tests each atom group for linearity of the sites contained in the group

\sphinxstylestrong{GYRATE Subroutine}

“gyrate” computes the radius of gyration of a molecular system from its atomic coordinates

\sphinxstylestrong{HANGLE Subroutine}

“hangle” constructs hybrid angle bending parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HATOM Subroutine}

“hatom” assigns a new atom type to each hybrid site

\sphinxstylestrong{HBOND Subroutine}

“hbond” constructs hybrid bond stretch parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HCHARGE Subroutine}

“hcharge” constructs hybrid charge interaction parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HDIPOLE Subroutine}

“hdipole” constructs hybrid dipole interaction parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HESSIAN Subroutine}

“hessian” calls Subroutine**s to calculate the Hessian elements for each atom in turn with respect to Cartesian coordinates

\sphinxstylestrong{HESSRGD Subroutine}

“hessrgd” computes the numerical Hessian elements with respect to rigid body coordinates via 6*ngroup+1 gradient evaluations

\sphinxstylestrong{HESSROT Subroutine}

“hessrot” computes the numerical Hessian elements with respect to torsional angles; either the full matrix or just the diagonal can be calculated; the full matrix needs nomega+1 gradient evaluations while the diagonal requires just two gradient calls

\sphinxstylestrong{HIMPTOR Subroutine}

“himptor” constructs hybrid improper torsional parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HSTRBND Subroutine}

“hstrbnd” constructs hybrid stretch\sphinxhyphen{}bend parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HSTRTOR Subroutine}

“hstrtor” constructs hybrid stretch\sphinxhyphen{}torsion parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HTORS Subroutine}

“htors” constructs hybrid torsional parameters for a given initial state, final state and “lambda” value

\sphinxstylestrong{HVDW Subroutine}

“hvdw” constructs hybrid van der Waals  parameters given an initial state, final state and “lambda” value

\sphinxstylestrong{HYBRID Subroutine}

“hybrid” constructs the hybrid hamiltonian for a specified initial state, final state and mutation parameter “lambda”

\sphinxstylestrong{IJKPTS Subroutine}

“ijkpts” stores a set of indices used during calculation of macroscopic reaction field energetics

\sphinxstylestrong{IMAGE Subroutine}

“image” takes the components of pairwise distance between two points in the same or neighboring periodic boxes and converts to the components of the minimum image distance

\sphinxstylestrong{IMPOSE Subroutine}

“impose” performs the least squares best superposition of two atomic coordinate sets via a quaternion method; upon return, the first coordinate set is unchanged while the second set is translated and rotated to give best fit; the final root mean square fit is returned in “rmsvalue”

\sphinxstylestrong{INDUCE Subroutine}

“induce” computes the induced dipole moment at each polarizable site due to direct or mutual polarization; assumes that multipole components have already been rotated into the global coordinate frame

\sphinxstylestrong{INDUCE0A Subroutine}

“induce0a” computes the induced dipole moment at each polarizable site using a pairwise double loop

\sphinxstylestrong{INDUCE0B Subroutine}

“induce0b” computes the induced dipole moment at each polarizable site using a regular Ewald summation

\sphinxstylestrong{INEDGE Subroutine}

“inedge” inserts a concave edge into the linked list for its temporary torus

\sphinxstylestrong{INERTIA Subroutine}

“inertia” computes the principal moments of inertia for the system, and optionally translates the center of mass to the origin and rotates the principal axes onto the global axes

\sphinxstylestrong{INITERR Function}

“initerr” is the initial error Function** and derivatives for a distance geometry embedding; it includes components from the local geometry and torsional restraint errors

\sphinxstylestrong{INITIAL Subroutine}

“initial” sets up original values for some parameters and variables that might not otherwise get initialized

\sphinxstylestrong{INITPRM Subroutine}

“initprm” completely initializes a force field by setting all parameters to zero and using defaults for control values

\sphinxstylestrong{INITRES Subroutine}

“initres” sets names for biopolymer residue types used in PDB file conversion and automated generation of structures

\sphinxstylestrong{INITROT Subroutine}

“initrot” sets the torsional angles which are to be rotated in subsequent computation, by default automatically selects all rotatable single bonds; assumes internal coordinates have already been setup

\sphinxstylestrong{INSERT Subroutine}

“insert” adds the specified atom to the Cartesian coordinates list and shifts the remaining atoms

\sphinxstylestrong{INTEDIT Program}

“intedit” allows the user to extract information from or alter the values within an internal coordinates file

\sphinxstylestrong{INTXYZ Program}

“intxyz” takes as input an internal coordinates file, converts to and then writes out Cartesian coordinates

\sphinxstylestrong{INVBETA Function}

“invbeta” computes the inverse Beta distribution Function** via a combination of Newton iteration and bisection search

\sphinxstylestrong{INVERT Subroutine}

“invert” inverts a matrix using the Gauss\sphinxhyphen{}Jordan method

\sphinxstylestrong{IPEDGE Subroutine}

“ipedge” inserts convex edge into linked list for atom

\sphinxstylestrong{ISPLPE Subroutine}

“isplpe” computes the coefficients for a cubic periodic interpolating spline

\sphinxstylestrong{JACOBI Subroutine}

“jacobi” performs a matrix diagonalization of a real symmetric matrix by the method of Jacobi rotations

\sphinxstylestrong{KANGANG Subroutine}

“kangang” assigns the parameters for angle\sphinxhyphen{}angle cross term interactions and processes new or changed parameter values

\sphinxstylestrong{KANGLE Subroutine}

“kangle” assigns the force constants and ideal angles for the bond angles; also processes new or changed parameters

\sphinxstylestrong{KATOM Subroutine}

“katom” assigns an atom type definitions to each atom in the structure and processes any new or changed values

\sphinxstylestrong{KBOND Subroutine}

“kbond” assigns a force constant and ideal bond length to each bond in the structure and processes any new or changed parameter values

\sphinxstylestrong{KCHARGE Subroutine}

“kcharge” assigns partial charges to the atoms within the structure and processes any new or changed values

\sphinxstylestrong{KCHIRAL Subroutine}

“kchiral” determines the target value for each chirality and planarity restraint as the signed volume of the parallelpiped spanned by vectors from a common atom to each of three other atoms

\sphinxstylestrong{KDIPOLE Subroutine}

“kdipole” assigns bond dipoles to the bonds within the structure and processes any new or changed values

\sphinxstylestrong{KENEG Subroutine}

“keneg” applies primary and secondary electronegativity bond length corrections to applicable bond parameters

\sphinxstylestrong{KEWALD Subroutine}

“kewald” assigns both regular Ewald summation and particle mesh Ewald parameters for a periodic box

\sphinxstylestrong{KGEOM Subroutine}

“kgeom” asisgns parameters for geometric restraint terms to be included in the potential energy calculation

\sphinxstylestrong{KIMPROP Subroutine}

“kimprop” assigns potential parameters to each improper dihedral in the structure and processes any changed values

\sphinxstylestrong{KIMPTOR Subroutine}

“kimptor” assigns torsional parameters to each improper torsion in the structure and processes any changed values

\sphinxstylestrong{KINETIC Subroutine}

“kinetic” computes the total kinetic energy and kinetic energy contributions to the pressure tensor by summing over velocities

\sphinxstylestrong{KMETAL Subroutine}

“kmetal” assigns ligand field parameters to transition metal atoms and processes any new or changed parameter values

\sphinxstylestrong{KMPOLE Subroutine}

“kmpole” assigns atomic multipole moments to the atoms of the structure and processes any new or changed values

\sphinxstylestrong{KOPBEND Subroutine}

“kopbend” assigns the force constants for out\sphinxhyphen{}of\sphinxhyphen{}plane bending at trigonal centers via Wilson\sphinxhyphen{}Decius\sphinxhyphen{}Cross angle bends; also processes any new or changed parameter values

\sphinxstylestrong{KOPDIST Subroutine}

“kopdist” assigns the force constants for out\sphinxhyphen{}of\sphinxhyphen{}plane distance at trigonal centers via the central atom height; also processes any new or changed parameter values

\sphinxstylestrong{KORBIT Subroutine}

“korbit” assigns pi\sphinxhyphen{}orbital parameters to conjugated systems and processes any new or changed parameters

\sphinxstylestrong{KPITORS Subroutine}

“kpitors” assigns pi\sphinxhyphen{}orbital torsion parameters to torsions needing them, and processes any new or changed values

\sphinxstylestrong{KPOLAR Subroutine}

“kpolar” assigns atomic dipole polarizabilities to the atoms within the structure and processes any new or changed values

\sphinxstylestrong{KSOLV Subroutine}

“ksolv” assigns continuum solvation energy parameters for the Eisenberg\sphinxhyphen{}McLachlan ASP, Ooi\sphinxhyphen{}Scheraga SASA or various GB/SA solvation models

\sphinxstylestrong{KSTRBND Subroutine}

“kstrbnd” assigns the parameters for the stretch\sphinxhyphen{}bend interactions and processes new or changed parameter values

\sphinxstylestrong{KSTRTOR Subroutine}

“kstrtor” assigns stretch\sphinxhyphen{}torsion parameters to torsions needing them, and processes any new or changed values

\sphinxstylestrong{KTORS Subroutine}

“ktors” assigns torsional parameters to each torsion in the structure and processes any new or changed values

\sphinxstylestrong{KTORTOR Subroutine}

“ktortor” assigns torsion\sphinxhyphen{}torsion parameters to adjacent torsion pairs and processes any new or changed values

\sphinxstylestrong{KUREY Subroutine}

“kurey” assigns the force constants and ideal distances for the Urey\sphinxhyphen{}Bradley 1\sphinxhyphen{}3 interactions; also processes any new or changed parameter values

\sphinxstylestrong{KVDW Subroutine}

“kvdw” assigns the parameters to be used in computing the van der Waals interactions and processes any new or changed values for these parameters

\sphinxstylestrong{LATTICE Subroutine}

“lattice” stores the periodic box dimensions and sets angle values to be used in computing fractional coordinates

\sphinxstylestrong{LBFGS Subroutine}

“lbfgs” is a limited memory BFGS quasi\sphinxhyphen{}newton nonlinear optimization routine

\sphinxstylestrong{LIGASE Subroutine}

“ligase” translates a nucleic acid structure in Protein Data Bank format to a Cartesian coordinate file and sequence file

\sphinxstylestrong{LIGHTS Subroutine}

“lights” computes the set of nearest neighbor interactions using the method of lights algorithm

\sphinxstylestrong{LINBODY Subroutine}

“linbody” finds the angular velocity of a linear rigid body given the inertia tensor and angular momentum

\sphinxstylestrong{LMSTEP Subroutine}

“lmstep” computes the Levenberg\sphinxhyphen{}Marquardt step during a nonlinear least squares calculation; this version is based upon ideas from the Minpack routine LMPAR together with with the internal doubling strategy of Dennis and Schnabel

\sphinxstylestrong{LOCALMIN Subroutine}

“localmin” is used during normal mode local search to perform a Cartesian coordinate energy minimization

\sphinxstylestrong{LOCALRGD Subroutine}

“localrgd” is used during the PSS local search procedure to perform a rigid body energy minimization

\sphinxstylestrong{LOCALROT Subroutine}

“localrot” is used during the PSS local search procedure to perform a torsional space energy minimization

\sphinxstylestrong{LOCALXYZ Subroutine}

“localxyz” is used during the potential smoothing and search procedure to perform a local optimization at the current smoothing level

\sphinxstylestrong{LOCERR Function}

“locerr” is the local geometry error Function** and derivatives including the 1\sphinxhyphen{}2, 1\sphinxhyphen{}3 and 1\sphinxhyphen{}4 distance bound restraints

\sphinxstylestrong{LOWCASE Subroutine}

“lowcase” converts a text string to all lower case letters

\sphinxstylestrong{MAJORIZE Subroutine}

“majorize” refines the projected coordinates by attempting to minimize the least square residual between the trial distance matrix and the distances computed from the coordinates

\sphinxstylestrong{MAKEINT Subroutine}

“makeint” converts Cartesian to internal coordinates where selection of internal coordinates is controlled by “mode”

\sphinxstylestrong{MAKEPDB Subroutine}

“makexyz” converts a set of Cartesian coordinates to Protein Data Bank format with special handling for systems consisting of polypeptide chains, ligands and water molecules

\sphinxstylestrong{MAKEREF Subroutine}

“makeref” copies the information contained in the “xyz” file of the current structure into corresponding reference areas

\sphinxstylestrong{MAKEXYZ Subroutine}

“makexyz” generates a complete set of Cartesian coordinates for a full structure from the internal coordinate values

\sphinxstylestrong{MAPCHECK Subroutine}

“mapcheck” checks the current minimum energy structure for possible addition to the master list of local minima

\sphinxstylestrong{MAXWELL Function}

“maxwell” returns a speed in Angstroms/picosecond randomly selected from a 3\sphinxhyphen{}D Maxwell\sphinxhyphen{}Boltzmann distribution for the specified particle mass and system temperature

\sphinxstylestrong{MCM1 Function}

“mcm1” is a service routine that computes the energy and gradient for truncated Newton optimization in Cartesian coordinate space

\sphinxstylestrong{MCM2 Subroutine}

“mcm2” is a service routine that computes the sparse matrix Hessian elements for truncated Newton optimization in Cartesian coordinate space

\sphinxstylestrong{MCMSTEP Function}

“mcmstep” implements the minimization phase of an MCM step via Cartesian minimization following a Monte Carlo step

\sphinxstylestrong{MDINIT Subroutine}

“mdinit” initializes the velocities and accelerations for a molecular dynamics trajectory, including restarts

\sphinxstylestrong{MDREST Subroutine}

“mdrest” finds and removes any translational or rotational kinetic energy of the overall system center of mass

\sphinxstylestrong{MDSAVE Subroutine}

“mdsave” writes molecular dynamics trajectory snapshots and auxiliary files with velocity and induced dipole information; also checks for user requested termination of a simulation

\sphinxstylestrong{MDSTAT Subroutine}

“mdstat” is called at each molecular dynamics time step to form statistics on various average values and fluctuations, and to periodically save the state of the trajectory

\sphinxstylestrong{MEASFN Subroutine}

\sphinxstylestrong{MEASFP Subroutine}

\sphinxstylestrong{MEASFS Subroutine}

\sphinxstylestrong{MEASPM Subroutine}

“measpm” computes the volume of a single prism section of the full interior polyhedron

\sphinxstylestrong{MECHANIC Subroutine}

“mechanic” sets up needed parameters for the potential energy calculation and reads in many of the user selectable options

\sphinxstylestrong{MERGE Subroutine}

“merge” combines the reference and current structures into a single new “current” structure containing the reference atoms followed by the atoms of the current structure

\sphinxstylestrong{METRIC Subroutine}

“metric” takes as input the trial distance matrix and computes the metric matrix of all possible dot products between the atomic vectors and the center of mass using the law of cosines and the following formula for the distances to the center of mass:

\sphinxstylestrong{MIDERR Function}

“miderr” is the secondary error Function** and derivatives for a distance geometry embedding; it includes components from the distance bounds, local geometry, chirality and torsional restraint errors

\sphinxstylestrong{MINIMIZ1 Function}

“minimiz1” is a service routine that computes the energy and gradient for a low storage BFGS optimization in Cartesian coordinate space

\sphinxstylestrong{MINIMIZE Program}

“minimize” performs energy minimization in Cartesian coordinate space using a low storage BFGS nonlinear optimization

\sphinxstylestrong{MINIROT Program}

“minirot” performs an energy minimization in torsional angle space using a low storage BFGS nonlinear optimization

\sphinxstylestrong{MINIROT1 Function}

“minirot1” is a service routine that computes the energy and gradient for a low storage BFGS nonlinear optimization in torsional angle space

\sphinxstylestrong{MINPATH Subroutine}

“minpath” is a routine for finding the triangle smoothed upper and lower bounds of each atom to a specified root atom using a sparse variant of the Bellman\sphinxhyphen{}Ford shortest path algorithm

\sphinxstylestrong{MINRIGID Program}

“minrigid” performs an energy minimization of rigid body atom groups using a low storage BFGS nonlinear optimization

\sphinxstylestrong{MINRIGID1 Function}

“minrigid1” is a service routine that computes the energy and gradient for a low storage BFGS nonlinear optimization of rigid bodies

\sphinxstylestrong{MMID Subroutine}

“mmid” implements a modified midpoint method to advance the integration of a set of first order differential equations

\sphinxstylestrong{MODECART Subroutine}

\sphinxstylestrong{MODEROT Subroutine}

\sphinxstylestrong{MODESRCH Subroutine}

\sphinxstylestrong{MODETORS Subroutine}

\sphinxstylestrong{MODULI Subroutine}

“moduli” sets the moduli of the inverse discrete Fourier transform of the B\sphinxhyphen{}splines; bsmod{[}1\sphinxhyphen{}3{]} hold these values, nfft{[}1\sphinxhyphen{}3{]} are the grid dimensions, bsorder is the order of B\sphinxhyphen{}spline approximation

\sphinxstylestrong{MOLECULE Subroutine}

“molecule” counts the molecules, assigns each atom to its molecule and computes the mass of each molecule

\sphinxstylestrong{MOLUIND Subroutine}

“moluind” computes the molecular induced dipole components in the presence of an external electric field

\sphinxstylestrong{MOMENTS Subroutine}

“moments” computes the total electric charge, dipole and quadrupole moments for the entire system as a sum over the partial charges, bond dipoles and atomic multipole moments

\sphinxstylestrong{MONTE Program}

“monte” performs a Monte Carlo/MCM conformational search using either Cartesian single atom or torsional move sets

\sphinxstylestrong{MUTATE Subroutine}

“mutate” constructs the hybrid hamiltonian for a specified initial state, final state and mutation parameter “lambda”

\sphinxstylestrong{NEEDUPDATE Subroutine}

\sphinxstylestrong{NEIGHBOR Subroutine}

“neighbor” finds all of the neighbors of each atom

\sphinxstylestrong{NEWATM Subroutine}

“newatm” creates and defines an atom needed for the Cartesian coordinates file, but which may not present in the original Protein Data Bank file

\sphinxstylestrong{NEWTON Program}

“newton” performs an energy minimization in Cartesian coordinate space using a truncated Newton method

\sphinxstylestrong{NEWTON1 Function}

“newton1” is a service routine that computes the energy and gradient for truncated Newton optimization in Cartesian coordinate space

\sphinxstylestrong{NEWTON2 Subroutine}

“newton2” is a service routine that computes the sparse matrix Hessian elements for truncated Newton optimization in Cartesian coordinate space

\sphinxstylestrong{NEWTROT Program}

“newtrot” performs an energy minimization in torsional angle space using a truncated Newton conjugate gradient method

\sphinxstylestrong{NEWTROT1 Function}

“newtrot1” is a service routine that computes the energy and gradient for truncated Newton conjugate gradient optimization in torsional angle space

\sphinxstylestrong{NEWTROT2 Subroutine}

“newtrot2” is a service routine that computes the sparse matrix Hessian elements for truncated Newton optimization in torsional angle space

\sphinxstylestrong{NEXTARG Subroutine}

“nextarg” finds the next unused command line argument and returns it in the input character string

\sphinxstylestrong{NEXTTEXT Function}

“nexttext” finds and returns the location of the first non\sphinxhyphen{}blank character within an input text string; zero is returned if no such character is found

\sphinxstylestrong{NORMAL Function}

“normal” generates a random number from a normal Gaussian distribution with a mean of zero and a variance of one

\sphinxstylestrong{NUCBASE Subroutine}

“nucbase” builds the side chain for a single nucleotide base in terms of internal coordinates

\sphinxstylestrong{NUCCHAIN Subroutine}

“nucchain” builds up the internal coordinates for a nucleic acid sequence from the sugar type, backbone and glycosidic torsional values

\sphinxstylestrong{NUCLEIC Program}

“nucleic” builds the internal and Cartesian coordinates of a polynucleotide from nucleic acid sequence and torsional angle values for the nucleic acid backbone and side chains

\sphinxstylestrong{NUMBER Function}

“number” converts a text numeral into an integer value; the input string must contain only numeric characters

\sphinxstylestrong{NUMERAL Subroutine}

“numeral” converts an input integer number into the corresponding right\sphinxhyphen{} or left\sphinxhyphen{}justified text numeral

\sphinxstylestrong{NUMGRAD Subroutine}

“numgrad” computes the gradient of the objective Function** “fvalue” with respect to Cartesian coordinates of the atoms via a two\sphinxhyphen{}sided numerical differentiation

\sphinxstylestrong{OCVM Subroutine}

“ocvm” is an optimally conditioned variable metric nonlinear optimization routine without line searches

\sphinxstylestrong{OLDATM Subroutine}

“oldatm” get the Cartesian coordinates for an atom from the Protein Data Bank file, then assigns the atom type and atomic connectivities

\sphinxstylestrong{OPENEND Subroutine}

“openend” opens a file on a Fortran unit such that the position is set to the bottom for appending to the end of the file

\sphinxstylestrong{OPTIMIZ1 Function}

“optimiz1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization in Cartesian coordinate space

\sphinxstylestrong{OPTIMIZE Program}

“optimize” performs energy minimization in Cartesian coordinate space using an optimally conditioned variable metric method

\sphinxstylestrong{OPTIROT Program}

“optirot” performs an energy minimization in torsional angle space using an optimally conditioned variable metric method

\sphinxstylestrong{OPTIROT1 Function}

“optirot1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization in torsional angle space

\sphinxstylestrong{OPTRIGID Program}

“optrigid” performs an energy minimization of rigid body atom groups using an optimally conditioned variable metric method

\sphinxstylestrong{OPTRIGID1 Function}

“optrigid1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization of rigid bodies

\sphinxstylestrong{OPTSAVE Subroutine}

“optsave” is used by the optimizers to write imtermediate coordinates and other relevant information; also checks for user requested termination of an optimization

\sphinxstylestrong{ORBITAL Subroutine}

“orbital” finds and organizes lists of atoms in a pisystem, bonds connecting pisystem atoms and torsions whose two central atoms are both pisystem atoms

\sphinxstylestrong{ORIENT Subroutine}

“orient” computes a set of reference Cartesian coordinates in standard orientation for each rigid body atom group

\sphinxstylestrong{ORTHOG Subroutine}

“orthog” performs an orthogonalization of an input matrix via the modified Gram\sphinxhyphen{}Schmidt algorithm

\sphinxstylestrong{OVERLAP Subroutine}

“overlap” computes the overlap for two parallel p\sphinxhyphen{}orbitals given the atomic numbers and distance of separation

\sphinxstylestrong{PARAMYZE Subroutine}

“paramyze” prints the force field parameters used in the computation of each of the potential energy terms

\sphinxstylestrong{PASSB Subroutine}

\sphinxstylestrong{PASSB2 Subroutine}

\sphinxstylestrong{PASSB3 Subroutine}

\sphinxstylestrong{PASSB4 Subroutine}

\sphinxstylestrong{PASSB5 Subroutine}

\sphinxstylestrong{PASSF Subroutine}

\sphinxstylestrong{PASSF2 Subroutine}

\sphinxstylestrong{PASSF3 Subroutine}

\sphinxstylestrong{PASSF4 Subroutine}

\sphinxstylestrong{PASSF5 Subroutine}

\sphinxstylestrong{PATH Program}

“path” locates a series of structures equally spaced along a conformational pathway connecting the input reactant and product structures; a series of constrained optimizations orthogonal to the path is done via Lagrangian multipliers

\sphinxstylestrong{PATH1 Function}

\sphinxstylestrong{PATHPNT Subroutine}

“pathpnt” finds a structure on the synchronous transit path with the specified path value “t”

\sphinxstylestrong{PATHSCAN Subroutine}

“pathscan” makes a scan of a synchronous transit pathway by computing structures and energies for specific path values

\sphinxstylestrong{PATHVAL Subroutine}

“pathval” computes the synchronous transit path value for the specified structure

\sphinxstylestrong{PDBATM Subroutine}

“pdbatm” adds an atom to the Protein Data Bank file

\sphinxstylestrong{PDBXYZ Program}

“pdbxyz” takes as input a Protein Data Bank file and then converts to and writes out a Cartesian coordinates file and, for biopolymers, a sequence file

\sphinxstylestrong{PIALTER Subroutine}

“pialter” first modifies bond lengths and force constants according to the standard bond slope parameters and the bond order values stored in “pnpl”; also alters some 2\sphinxhyphen{}fold torsional parameters based on the bond\sphinxhyphen{}order * beta matrix

\sphinxstylestrong{PIMOVE Subroutine}

“pimove” rotates the vector between atoms “list(1)” and “list(2)” so that atom 1 is at the origin and atom 2 along the x\sphinxhyphen{}axis; the atoms defining the respective planes are also moved and their bond lengths normalized

\sphinxstylestrong{PIPLANE Subroutine}

“piplane” selects the three atoms which specify the plane perpendicular to each p\sphinxhyphen{}orbital; the current version will fail in certain situations, including ketenes, allenes, and isolated or adjacent triple bonds

\sphinxstylestrong{PISCF Subroutine}

“piscf” performs an scf molecular orbital calculation for the pisystem using a modified Pariser\sphinxhyphen{}Parr\sphinxhyphen{}Pople method

\sphinxstylestrong{PITILT Subroutine}

“pitilt” calculates for each pibond the ratio of the actual p\sphinxhyphen{}orbital overlap integral to the ideal overlap if the same orbitals were perfectly parallel

\sphinxstylestrong{PLACE Subroutine}

“place” finds the probe sites by putting the probe sphere tangent to each triple of neighboring atoms

\sphinxstylestrong{POLARGRP Subroutine}

“polargrp” generates members of the polarization group of each atom and separate lists of the 1\sphinxhyphen{}2, 1\sphinxhyphen{}3 and 1\sphinxhyphen{}4 group connectivities

\sphinxstylestrong{POLARIZE Program}

“polarize” computes the molecular polarizability by applying an external field along each axis followed by diagonalization of the resulting polarizability tensor

\sphinxstylestrong{POLYMER Subroutine}

“polymer” tests for the presence of an infinite polymer extending across periodic boundaries

\sphinxstylestrong{POLYP Subroutine}

“polyp” is a polynomial product routine that multiplies two algebraic forms

\sphinxstylestrong{POTNRG Function}

\sphinxstylestrong{POTOFF Subroutine}

“potoff” clears the forcefield definition by turning off the use of each of the potential energy Function**s

\sphinxstylestrong{POWER Subroutine}

“power” uses the power method with deflation to compute the few largest eigenvalues and eigenvectors of a symmetric matrix

\sphinxstylestrong{PRECISE Function}

“precise” finds a machine precision value as selected by the input argument: (1) the smallest positive floating point value, (2) the smallest relative floating point spacing, (3) the largest relative floating point spacing

\sphinxstylestrong{PRECOND Subroutine}

“precond” solves a simplified version of the Newton equations Ms = r, and uses the result to precondition linear conjugate gradient iterations on the full Newton equations in “tnsolve”

\sphinxstylestrong{PRESSURE Subroutine}

“pressure” uses the internal virial to find the pressure in a periodic box and maintains a constant desired pressure by scaling the coordinates via coupling to an external constant pressure bath

\sphinxstylestrong{PRMKEY Subroutine}

“field” parses a text string to extract keywords related to force field potential energy Function**al forms and constants

\sphinxstylestrong{PROCHAIN Subroutine}

“prochain” builds up the internal coordinates for an amino acid sequence from the phi, psi, omega and chi values

\sphinxstylestrong{PROJCT Subroutine}

\sphinxstylestrong{PROMO Subroutine}

“promo” writes a short message containing information about the Tinker version number and the copyright notice

\sphinxstylestrong{PROPERTY Function}

“property” takes two input snapshot frames and computes the value of the property for which the correlation Function** is being accumulated

\sphinxstylestrong{PROPYZE Subroutine}

“propyze” finds and prints the total charge, dipole moment components, radius of gyration and moments of inertia

\sphinxstylestrong{PROSIDE Subroutine}

“proside” builds the side chain for a single amino acid residue in terms of internal coordinates

\sphinxstylestrong{PROTEIN Program}

“protein” builds the internal and Cartesian coordinates of a polypeptide from amino acid sequence and torsional angle values for the peptide backbone and side chains

\sphinxstylestrong{PRTARC Subroutine}

“prtarc” writes out a set of Cartesian coordinates for all active atoms in the Tinker XYZ archive format

\sphinxstylestrong{PRTCAR Subroutine}

“prtcar” writes out a set of Cartesian coordinates for all active atoms in the Accelerys InsightII .car format

\sphinxstylestrong{PRTDYN Subroutine}

“prtdyn” writes out the information needed to restart a molecular dynamics trajectory to an external disk file

\sphinxstylestrong{PRTERR Subroutine}

“prterr” writes out a set of coordinates to a disk file prior to aborting on a serious error

\sphinxstylestrong{PRTINT Subroutine}

“prtint” writes out a set of Z\sphinxhyphen{}matrix internal coordinates to an external disk file

\sphinxstylestrong{PRTMOL2 Program}

“prtmol2” writes out a set of coordinates in Sybyl MOL2 format to an external disk file

\sphinxstylestrong{PRTPDB Subroutine}

“prtpdb” writes out a set of Protein Data Bank coordinates to an external disk file

\sphinxstylestrong{PRTPRM Subroutine}

“prtprm” writes out a formatted listing of the default set of potential energy parameters for a force field

\sphinxstylestrong{PRTSEQ Subroutine}

“prtseq” writes out a biopolymer sequence to an external disk file with 15 residues per line and distinct chains separated by blank lines

\sphinxstylestrong{PRTXMOL Subroutine}

“prtxmol” writes out a set of Cartesian coordinates for all active atoms in a simple, generic XYZ format originally used by the XMOL Program**

\sphinxstylestrong{PRTXYZ Subroutine}

“prtxyz” writes out a set of Cartesian coordinates to an external disk file

\sphinxstylestrong{PSS Program}

“pss” implements the potential smoothing plus search method for global optimization in Cartesian coordinate space with local searches performed in Cartesian or torsional space

\sphinxstylestrong{PSS1 Function}

“pss1” is a service routine that computes the energy and gradient during PSS global optimization in Cartesian coordinate space

\sphinxstylestrong{PSS2 Subroutine}

“pss2” is a service routine that computes the sparse matrix Hessian elements during PSS global optimization in Cartesian coordinate space

\sphinxstylestrong{PSSRGD1 Function}

“pssrgd1” is a service routine that computes the energy and gradient during PSS global optimization over rigid bodies

\sphinxstylestrong{PSSRIGID Program}

“pssrigid” implements the potential smoothing plus search method for global optimization for a set of rigid bodies

\sphinxstylestrong{PSSROT Program}

“pssrot” implements the potential smoothing plus search method for global optimization in torsional space

\sphinxstylestrong{PSSROT1 Function}

“pssrot1” is a service routine that computes the energy and gradient during PSS global optimization in torsional space

\sphinxstylestrong{PSSWRITE Subroutine}

\sphinxstylestrong{PTINCY Function}

\sphinxstylestrong{PZEXTR Subroutine}

“pzextr” is a polynomial extrapolation routine used during Bulirsch\sphinxhyphen{}Stoer integration of ordinary differential equations

\sphinxstylestrong{QRFACT Subroutine}

“qrfact” performs Householder transformations with column pivoting (optional) to compute a QR factorization of the m by n matrix a; the routine determines an orthogonal matrix q, a permutation matrix p, and an upper trapezoidal matrix r with diagonal elements of nonincreasing magnitude, such that a*p = q*r; the Householder transformation for column k, k = 1,2,…,min(m,n), is of the form

\sphinxstylestrong{QRSOLVE Subroutine}

“qrsolve” solves a*x=b and d*x=0 in the least squares sense; normally used in combination with routine “qrfact” to solve least squares problems

\sphinxstylestrong{QUATFIT Subroutine}

“quatfit” uses a quaternion\sphinxhyphen{}based method to achieve the best fit superposition of two sets of coordinates

\sphinxstylestrong{RADIAL Program}

“radial” finds the radial distribution Function** for a specified pair of atom types via analysis of a set of coordinate frames

\sphinxstylestrong{RANDOM Function}

“random” generates a random number on {[}0,1{]} via a long period generator due to L’Ecuyer with Bays\sphinxhyphen{}Durham shuffle

\sphinxstylestrong{RANVEC Subroutine}

“ranvec” generates a unit vector in 3\sphinxhyphen{}dimensional space with uniformly distributed random orientation

\sphinxstylestrong{RATTLE Subroutine}

“rattle” implements the first portion of the rattle algorithm by correcting atomic positions and half\sphinxhyphen{}step velocities to maintain interatomic distance and absolute spatial constraints

\sphinxstylestrong{RATTLE2 Subroutine}

“rattle2” implements the second portion of the rattle algorithm by correcting the full\sphinxhyphen{}step velocities in order to maintain interatomic distance constraints

\sphinxstylestrong{READBLK Subroutine}

“readblk” reads in a set of snapshot frames and transfers the values to internal arrays for use in the computation of time correlation Function**s

\sphinxstylestrong{READDYN Subroutine}

“readdyn” get the positions, velocities and accelerations for a molecular dynamics restart from an external disk file

\sphinxstylestrong{READINT Subroutine}

“readint” gets a set of Z\sphinxhyphen{}matrix internal coordinates from an external file

\sphinxstylestrong{READMOL2 Subroutine}

“readmol2” gets a set of Sybyl MOL2 coordinates from an external disk file

\sphinxstylestrong{READPDB Subroutine}

“readpdb” gets a set of Protein Data Bank coordinates from an external disk file

\sphinxstylestrong{READPRM Subroutine}

“readprm” processes the potential energy parameter file in order to define the default force field parameters

\sphinxstylestrong{READSEQ Subroutine}

“readseq” gets a biopolymer sequence containing one or more separate chains from an external file; all lines containing sequence must begin with the starting sequence number, the actual sequence is read from subsequent nonblank characters

\sphinxstylestrong{READXYZ Subroutine}

“readxyz” gets a set of Cartesian coordinates from an external disk file

\sphinxstylestrong{REFINE Subroutine}

“refine” performs minimization of the atomic coordinates of an initial crude embedded distance geometry structure versus the bound, chirality, planarity and torsional error Function**s

\sphinxstylestrong{RELEASEMONITOR Subroutine}

\sphinxstylestrong{REPLICA Subroutine}

“replica” decides between images and replicates for generation of periodic boundary conditions, and sets the cell replicate list if the replicates method is to be used

\sphinxstylestrong{RFINDEX Subroutine}

“rfindex” finds indices for each multipole site for use in computing reaction field energetics

\sphinxstylestrong{RGDSRCH Subroutine}

\sphinxstylestrong{RGDSTEP Subroutine}

“rgdstep” performs a single molecular dynamics time step for a rigid body calculation

\sphinxstylestrong{RIBOSOME Subroutine}

“ribosome” translates a polypeptide structure in Protein Data Bank format to a Cartesian coordinate file and sequence file

\sphinxstylestrong{RIGIDXYZ Subroutine}

“rigidxyz” computes Cartesian coordinates for a rigid body group via rotation and translation of reference coordinates

\sphinxstylestrong{RINGS Subroutine}

“rings” searches the structure for small rings and stores their constituent atoms

\sphinxstylestrong{RMSERROR Subroutine}

“rmserror” computes the maximum absolute deviation and the rms deviation from the distance bounds, and the number and rms value of the distance restraint violations

\sphinxstylestrong{RMSFIT Function}

“rmsfit” computes the rms fit of two coordinate sets

\sphinxstylestrong{ROTANG Function}

\sphinxstylestrong{ROTCHECK Function}

“rotcheck” tests a specified candidate rotatable bond for the disallowed case where inactive atoms are found on both sides of the candidate bond

\sphinxstylestrong{ROTEULER Subroutine}

“roteuler” computes a set of Euler angle values consistent with an input rotation matrix

\sphinxstylestrong{ROTLIST Subroutine}

“rotlist” generates the minimum list of all the atoms lying to one side of a pair of directly bonded atoms; optionally finds the minimal list by choosing the side with fewer atoms

\sphinxstylestrong{ROTMAT Subroutine}

“rotmat” finds the rotation matrix that converts from the local coordinate system to the global frame at a multipole site

\sphinxstylestrong{ROTPOLE Subroutine}

“rotpole” constructs the set of atomic multipoles in the global frame by applying the correct rotation matrix for each site

\sphinxstylestrong{ROTRGD Subroutine}

“rotrgd” finds the rotation matrix for a rigid body due to a single step of dynamics

\sphinxstylestrong{ROTSITE Subroutine}

“rotsite” computes the atomic multipoles at a specified site in the global coordinate frame by applying a rotation matrix

\sphinxstylestrong{SADDLE Program}

“saddle” finds a transition state between two conformational minima using a combination of ideas from the synchronous transit (Halgren\sphinxhyphen{}Lipscomb) and quadratic path (Bell\sphinxhyphen{}Crighton) methods

\sphinxstylestrong{SADDLE1 Function}

“saddle1” is a service routine that computes the energy and gradient for transition state optimization

\sphinxstylestrong{SADDLES Subroutine}

“saddles” constructs circles, convex edges and saddle faces

\sphinxstylestrong{SCAN Program}

“scan” attempts to find all the local minima on a potential energy surface via an iterative series of local searches

\sphinxstylestrong{SCAN1 Function}

“scan1” is a service routine that computes the energy and gradient during exploration of a potential energy surface via iterative local search

\sphinxstylestrong{SCAN2 Subroutine}

“scan2” is a service routine that computes the sparse matrix Hessian elements during exploration of a potential energy surface via iterative local search

\sphinxstylestrong{SDAREA Subroutine}

“sdarea” optionally scales the atomic friction coefficient of each atom based on its accessible surface area

\sphinxstylestrong{SDSTEP Subroutine}

“sdstep” performs a single stochastic dynamics time step via a velocity Verlet integration algorithm

\sphinxstylestrong{SDTERM Subroutine}

“sdterm” gets frictional and random force terms needed to update positions and velocities via stochastic dynamics

\sphinxstylestrong{SEARCH Subroutine}

“search” is a unidimensional line search based upon parabolic extrapolation and cubic interpolation using both Function** and gradient values; if forced to search in an uphill direction, return is after the initial step

\sphinxstylestrong{SETACCELERATION Subroutine}

\sphinxstylestrong{SETATOMIC Subroutine}

\sphinxstylestrong{SETATOMTYPES Subroutine}

\sphinxstylestrong{SETCHARGE Subroutine}

\sphinxstylestrong{SETCONNECTIVITY Subroutine}

\sphinxstylestrong{SETCOORDINATES Subroutine}

\sphinxstylestrong{SETENERGY Subroutine}

\sphinxstylestrong{SETFILE Subroutine}

\sphinxstylestrong{SETFORCEFIELD Subroutine}

\sphinxstylestrong{SETGRADIENTS Subroutine}

\sphinxstylestrong{SETIME Subroutine}

“setime” initializes the elapsed interval CPU timer

\sphinxstylestrong{SETINDUCED Subroutine}

\sphinxstylestrong{SETKEYWORD Subroutine}

\sphinxstylestrong{SETMASS Subroutine}

\sphinxstylestrong{SETNAME Subroutine}

\sphinxstylestrong{SETSTEP Subroutine}

\sphinxstylestrong{SETSTORY Subroutine}

\sphinxstylestrong{SETTIME Subroutine}

\sphinxstylestrong{SETUPDATED Subroutine}

\sphinxstylestrong{SETVELOCITY Subroutine}

\sphinxstylestrong{SHAKEUP Subroutine}

“shakeup” initializes any holonomic constraints for use with the rattle algorithm during molecular dynamics

\sphinxstylestrong{SIGMOID Function}

“sigmoid” implements a normalized sigmoidal Function** on the interval {[}0,1{]}; the curves connect (0,0) to (1,1) and have a cooperativity controlled by beta, they approach a straight line as beta \sphinxhyphen{}\textgreater{} 0 and get more nonlinear as beta increases

\sphinxstylestrong{SKTDYN Subroutine}

“sktdyn” sends the current dynamics info via a socket

\sphinxstylestrong{SKTINIT Subroutine}

“sktinit” sets up socket communication with the graphical user interface by starting a Java virtual machine, initiating a server, and loading an object with system information

\sphinxstylestrong{SKTKILL Subroutine}

“sktkill” closes the server and Java virtual machine

\sphinxstylestrong{SKTOPT Subroutine}

“sktopt” sends the current optimization info via a socket

\sphinxstylestrong{SLATER Subroutine}

“slater” is a general routine for computing the overlap integrals between two Slater\sphinxhyphen{}type orbitals

\sphinxstylestrong{SMOOTH Subroutine}

“smooth” sets the type of smoothing method and the extent of surface deformation for use with potential energy smoothing

\sphinxstylestrong{SNIFFER Program}

“sniffer” performs a global energy minimization using a discrete version of Griewank’s global search trajectory

\sphinxstylestrong{SNIFFER1 Function}

“sniffer1” is a service routine that computes the energy and gradient for the Sniffer global optimization method

\sphinxstylestrong{SOAK Subroutine}

“soak” takes a currently defined solute system and places it into a solvent box, with removal of any solvent molecules that overlap the solute

\sphinxstylestrong{SORT Subroutine}

“sort” takes an input list of integers and sorts it into ascending order using the Heapsort algorithm

\sphinxstylestrong{SORT10 Subroutine}

“sort10” takes an input list of character strings and sorts it into alphabetical order using the Heapsort algorithm, duplicate values are removed from the final sorted list

\sphinxstylestrong{SORT2 Subroutine}

“sort2” takes an input list of reals and sorts it into ascending order using the Heapsort algorithm; it also returns a key into the original ordering

\sphinxstylestrong{SORT3 Subroutine}

“sort3” takes an input list of integers and sorts it into ascending order using the Heapsort algorithm; it also returns a key into the original ordering

\sphinxstylestrong{SORT4 Subroutine}

“sort4” takes an input list of integers and sorts it into ascending absolute value using the Heapsort algorithm

\sphinxstylestrong{SORT5 Subroutine}

“sort5” takes an input list of integers and sorts it into ascending order based on each value modulo “m”

\sphinxstylestrong{SORT6 Subroutine}

“sort6” takes an input list of character strings and sorts it into alphabetical order using the Heapsort algorithm

\sphinxstylestrong{SORT7 Subroutine}

“sort7” takes an input list of character strings and sorts it into alphabetical order using the Heapsort algorithm; it also returns a key into the original ordering

\sphinxstylestrong{SORT8 Subroutine}

“sort8” takes an input list of integers and sorts it into ascending order using the Heapsort algorithm, duplicate values are removed from the final sorted list

\sphinxstylestrong{SORT9 Subroutine}

“sort9” takes an input list of reals and sorts it into ascending order using the Heapsort algorithm, duplicate values are removed from the final sorted list

\sphinxstylestrong{SPACEFILL Program}

“spacefill” computes the surface area and volume of a structure; the van der Waals, accessible\sphinxhyphen{}excluded, and contact\sphinxhyphen{}reentrant definitions are available

\sphinxstylestrong{SPECTRUM Program}

“spectrum” computes a power spectrum over a wavelength range from the velocity autocorrelation as a Function** of time

\sphinxstylestrong{SQUARE Subroutine}

“square” is a nonlinear least squares routine derived from the IMSL routine BCLSF and More’s Minpack routine LMDER; the Jacobian is estimated by finite differences and bounds can be specified for the variables to be refined

\sphinxstylestrong{SUFFIX Subroutine}

“suffix” checks a filename for the presence of an extension, and appends an extension if none is found

\sphinxstylestrong{SUPERPOSE Program}

“superpose” takes pairs of structures and superimposes them in the optimal least squares sense; it will attempt to match all atom pairs or only those specified by the user

\sphinxstylestrong{SURFACE Subroutine}

“surface” performs an analytical computation of the weighted solvent accessible surface area of each atom and the first derivatives of the area with respect to Cartesian coordinates

\sphinxstylestrong{SURFATOM Subroutine}

“surfatom” performs an analytical computation of the surface area of a specified atom; a simplified version of “surface”

\sphinxstylestrong{SWITCH Subroutine}

“switch” sets the coeffcients used by the fifth and seventh order polynomial switching Function**s for spherical cutoffs

\sphinxstylestrong{SYBYLXYZ Program}

“sybylxyz” takes as input a Sybyl MOL2 coordinates file, converts to and then writes out Cartesian coordinates

\sphinxstylestrong{SYMMETRY Subroutine}

“symmetry” applies symmetry operators to the fractional coordinates of the asymmetric unit in order to generate the symmetry related atoms of the full unit cell

\sphinxstylestrong{TANGENT Subroutine}

“tangent” finds the projected gradient on the synchronous transit path for a point along the transit pathway

\sphinxstylestrong{TEMPER Subroutine}

“temper” applies a velocity correction at the half time step as needed for the Nose\sphinxhyphen{}Hoover extended system thermostat

\sphinxstylestrong{TEMPER2 Subroutine}

“temper2” computes the instantaneous temperature and applies a thermostat via Berendsen velocity scaling, Andersen stochastic collisions, Langevin piston or Nose\sphinxhyphen{}Hoover extended systems

\sphinxstylestrong{TESTGRAD Program}

“testgrad” computes and compares the analytical and numerical gradient vectors of the potential energy Function** with respect to Cartesian coordinates

\sphinxstylestrong{TESTHESS Program}

“testhess” computes and compares the analytical and numerical Hessian matrices of the potential energy Function** with respect to Cartesian coordinates

\sphinxstylestrong{TESTLIGHT Program}

“testlight” performs a set of timing tests to compare the evaluation of potential energy and energy/gradient using the method of lights with a double loop over all atom pairs

\sphinxstylestrong{TESTROT Program}

“testrot” computes and compares the analytical and numerical gradient vectors of the potential energy Function** with respect to rotatable torsional angles

\sphinxstylestrong{TIMER Program}

“timer” measures the CPU time required for file reading and parameter assignment, potential energy computation, energy and gradient computation, and Hessian matrix evaluation

\sphinxstylestrong{TIMEROT Program}

“timerot” measures the CPU time required for file reading and parameter assignment, potential energy computation, energy and gradient over torsions, and torsional angle Hessian matrix evaluation

\sphinxstylestrong{TNCG Subroutine}

“tncg” implements a truncated Newton optimization algorithm in which a preconditioned linear conjugate gradient method is used to approximately solve Newton’s equations; special features include use of an explicit sparse Hessian or finite\sphinxhyphen{}difference gradient\sphinxhyphen{}Hessian products within the PCG iteration; the exact Newton search directions can be used optionally; by default the algorithm checks for negative curvature to prevent convergence to a stationary point having negative eigenvalues; if a saddle point is desired this test can be removed by disabling “negtest”

\sphinxstylestrong{TNSOLVE Subroutine}

“tnsolve” uses a linear conjugate gradient method to find an approximate solution to the set of linear equations represented in matrix form by Hp = \sphinxhyphen{}g (Newton’s equations)

\sphinxstylestrong{TORPHASE Subroutine}

“torphase” sets the n\sphinxhyphen{}fold amplitude and phase values for each torsion via sorting of the input parameters

\sphinxstylestrong{TORQUE Subroutine}

“torque” takes the torque values on sites defined by local coordinate frames and distributes thme to convert to forces on the original sites and sites specifying the local frames

\sphinxstylestrong{TORQUE1 Subroutine}

“torque1” takes the torque value on a site defined by a local coordinate frame and distributes it to convert to forces on the original site and sites specifying the local frame

\sphinxstylestrong{TORSER Function}

“torser” computes the torsional error Function** and its first derivatives with respect to the atomic Cartesian coordinates based on the deviation of specified torsional angles from desired values, the contained bond angles are also restrained to avoid a numerical instability

\sphinxstylestrong{TORSIONS Subroutine}

“torsions” finds the total number of dihedral angles and the numbers of the four atoms defining each dihedral angle

\sphinxstylestrong{TORUS Subroutine}

“torus” sets a list of all of the temporary torus positions by testing for a torus between each atom and its neighbors

\sphinxstylestrong{TOTERR Function}

“toterr” is the error Function** and derivatives for a distance geometry embedding; it includes components from the distance bounds, hard sphere contacts, local geometry, chirality and torsional restraint errors

\sphinxstylestrong{TRANSIT Function}

“transit” evaluates the synchronous transit Function** and gradient; linear and quadratic transit paths are available

\sphinxstylestrong{TRIANGLE Subroutine}

“triangle” smooths the upper and lower distance bounds via the triangle inequality using a full\sphinxhyphen{}matrix variant of the Floyd\sphinxhyphen{}Warshall shortest path algorithm; this routine is usually much slower than the sparse matrix shortest path methods in “geodesic” and “trifix”, and should be used only for comparison with answers generated by those routines

\sphinxstylestrong{TRIFIX Subroutine}

“trifix” rebuilds both the upper and lower distance bound matrices following tightening of one or both of the bounds between a specified pair of atoms, “p” and “q”, using a modification of Murchland’s shortest path update algorithm

\sphinxstylestrong{TRIMTEXT Function}

“trimtext” finds and returns the location of the last non\sphinxhyphen{}blank character before the first null character in an input text string; the Function** returns zero if no such character is found

\sphinxstylestrong{TRIPLE Function}

“triple” finds the triple product of three vectors; used as a service routine by the Connolly surface area and volume computation

\sphinxstylestrong{TRUST Subroutine}

“trust” updates the model trust region for a nonlinear least squares calculation; this version is based on the ideas found in NL2SOL and in Dennis and Schnabel’s book

\sphinxstylestrong{UDIRECT1 Subroutine}

“udirect1” computes the reciprocal space contribution of the permanent atomic multipole moments to the electrostatic field for use in finding the direct induced dipole moments via a regular Ewald summation

\sphinxstylestrong{UDIRECT2 Subroutine}

“udirect2” computes the real space contribution of the permanent atomic multipole moments to the electrostatic field for use in finding the direct induced dipole moments via a regular Ewald summation

\sphinxstylestrong{UFIELD Subroutine}

“ufield” finds the field at each polarizable site due to the induced dipoles at the other sites using Thole’s method to damp the field at close range

\sphinxstylestrong{UMUTUAL1 Subroutine}

“umutual1” computes the reciprocal space contribution of the induced atomic dipole moments to the electrostatic field for use in iterative calculation of induced dipole moments via a regular Ewald summation

\sphinxstylestrong{UMUTUAL2 Subroutine}

“umutual2” computes the real space contribution of the induced atomic dipole moments to the electrostatic field for use in iterative calculation of induced dipole moments via a regular Ewald summation

\sphinxstylestrong{UNITCELL Subroutine}

“unitcell” gets the periodic boundary box size and related values from an external keyword file

\sphinxstylestrong{UPCASE Subroutine}

“upcase” converts a text string to all upper case letters

\sphinxstylestrong{VAM Subroutine}

“vam” takes the analytical molecular surface defined as a collection of spherical and toroidal polygons and uses it to compute the volume and surface area

\sphinxstylestrong{VCROSS Subroutine}

“vcross” finds the cross product of two vectors

\sphinxstylestrong{VDWERR Function}

“vdwerr” is the hard sphere van der Waals bound error Function** and derivatives that penalizes close nonbonded contacts, pairwise neighbors are generated via the method of lights

\sphinxstylestrong{VECANG Function}

“vecang” finds the angle between two vectors handed with respect to a coordinate axis; returns an angle in the range {[}0,2*pi{]}

\sphinxstylestrong{VERLET Subroutine}

“verlet” performs a single molecular dynamics time step by means of the velocity Verlet multistep recursion formula

\sphinxstylestrong{VERSION Subroutine}

“version” checks the name of a file about to be opened; if if “old” status is passed, the name of the highest current version is returned; if “new” status is passed the filename of the next available unused version is generated

\sphinxstylestrong{VIBRATE Program}

“vibrate” performs a vibrational normal mode analysis; the Hessian matrix of second derivatives is determined and then diagonalized both directly and after mass weighting; output consists of the eigenvalues of the force constant matrix as well as the vibrational frequencies and displacements

\sphinxstylestrong{VIBRIGID Program}

“vibrigid” computes the eigenvalues and eigenvectors of the Hessian matrix over rigid body degrees of freedom

\sphinxstylestrong{VIBROT Program}

“vibrot” computes the eigenvalues and eigenvectors of the torsional Hessian matrix

\sphinxstylestrong{VNORM Subroutine}

“vnorm” normalizes a vector to unit length; used as a service routine by the Connolly surface area and volume computation

\sphinxstylestrong{VOLUME Subroutine}

“volume” calculates the excluded volume via the Connolly analytical volume and surface area algorithm

\sphinxstylestrong{VOLUME1 Subroutine}

“volume1” calculates first derivatives of the total excluded volume with respect to the Cartesian coordinates of each atom

\sphinxstylestrong{VOLUME2 Subroutine}

“volume2” calculates second derivatives of the total excluded volume with respect to the Cartesian coordinates of the atoms

\sphinxstylestrong{WATSON Subroutine}

“watson” uses a rigid body optimization to approximately align the paired strands of a nucleic acid double helix

\sphinxstylestrong{WATSON1 Function}

“watson1” is a service routine that computes the energy and gradient for optimally conditioned variable metric optimization of rigid bodies

\sphinxstylestrong{XTALERR Subroutine}

“xtalerr” computes an error Function** value derived from derivatives with respect to lattice parameters, lattice energy and monomer dipole moments

\sphinxstylestrong{XTALFIT Program}

“xtalfit” computes an optimized set of potential energy parameters for user specified van der Waals and electrostatic interactions by fitting to crystal structure, lattice energy and monomer dipole moment data

\sphinxstylestrong{XTALLAT1 Function}

“xtalmol1” is a service routine that computes the energy and numerical gradient with respect to the six lattice lengths and angles for a crystal energy minimization

\sphinxstylestrong{XTALMIN Program}

“xtalmin” performs a full crystal energy minimization by alternating cycles of truncated Newton optimization over atomic coordinates with variable metric optimization over the six lattice dimensions and angles

\sphinxstylestrong{XTALMOL1 Function}

“xtalmol1” is a service routine that computes the energy and gradient with respect to the atomic Cartesian coordinates for a crystal energy minimization

\sphinxstylestrong{XTALMOL2 Subroutine}

“xtalmol2” is a service routine that computes the sparse matrix Hessian elements with respect to the atomic Cartesian coordinates for a crystal energy minimization

\sphinxstylestrong{XTALMOVE Subroutine}

“xtalmove” converts fractional to Cartesian coordinates for rigid molecules during fitting of force field parameters to crystal structure data

\sphinxstylestrong{XTALPRM Subroutine}

“xtalprm” stores or retrieves a crystal structure; used to make a previously stored structure the currently active structure, or to store a structure for later use; only provides for the intermolecular energy terms

\sphinxstylestrong{XTALWRT Subroutine}

“xtalwrt” is a utility that prints intermediate results during fitting of force field parameters to crystal data

\sphinxstylestrong{XYZATM Subroutine}

“xyzatm” computes the Cartesian coordinates of a single atom from its defining internal coordinate values

\sphinxstylestrong{XYZEDIT Program}

“xyzedit” provides for modification and manipulation of the contents of a Cartesian coordinates file

\sphinxstylestrong{XYZINT Program}

“xyzint” takes as input a Cartesian coordinates file, then converts to and writes out an internal coordinates file

\sphinxstylestrong{XYZPDB Program}

“xyzpdb” takes as input a Cartesian coordinates file, then converts to and writes out a Protein Data Bank file

\sphinxstylestrong{XYZRIGID Subroutine}

“xyzrigid” computes the center of mass and Euler angle rigid body coordinates for each atom group in the system

\sphinxstylestrong{XYZSYBYL Program}

“xyzsybyl” takes as input a Cartesian coordinates file, converts to and then writes out a Sybyl MOL2 file

\sphinxstylestrong{ZATOM Subroutine}

“zatom” adds an atom to the end of the current Z\sphinxhyphen{}matrix and then increments the atom counter; atom type, defining atoms and internal coordinates are passed as arguments

\sphinxstylestrong{ZHELP Subroutine}

“zhelp” prints the general information and instructions for the Z\sphinxhyphen{}matrix editing Program**

\sphinxstylestrong{ZVALUE Subroutine}

“zvalue” gets user supplied values for selected coordinates as needed by the internal coordinate editing Program**


\chapter{Description of Modules \& Global Variables}
\label{\detokenize{text/modules:description-of-modules-global-variables}}\label{\detokenize{text/modules::doc}}
The Fortran modules found in the Tinker package are listed below along with a brief description of the variables associated with each module. Each individual module contains a set of globally allocated variables available to any program unit upon inclusion of that module. A source listing containing each of the Tinker functions and subroutines and its included modules can be produced by running the “listing.make” script found in the distribution.

\begin{DUlineblock}{0em}
\item[] \sphinxstylestrong{ACTION Module \sphinxhyphen{} total number of each energy term computed}
\item[] neb   number of bond stretch energy terms computed
\item[] nea   number of angle bend energy terms computed
\item[] neba  number of stretch\sphinxhyphen{}bend energy terms computed
\item[] neub  number of Urey\sphinxhyphen{}Bradley energy terms computed
\item[] neaa  number of angle\sphinxhyphen{}angle energy terms computed
\item[] neopb number of out\sphinxhyphen{}of\sphinxhyphen{}plane bend energy terms computed
\item[] neopd number of out\sphinxhyphen{}of\sphinxhyphen{}plane distance energy terms computed
\item[] neid  number of improper dihedral energy terms computed
\item[] neit  number of improper torsion energy terms computed
\item[] net   number of torsional energy terms computed
\item[] nept  number of pi\sphinxhyphen{}orbital torsion energy terms computed
\item[] nebt  number of stretch\sphinxhyphen{}torsion energy terms computed
\item[] nett  number of torsion\sphinxhyphen{}torsion energy terms computed
\item[] nev   number of van der Waals energy terms computed
\item[] nec   number of charge\sphinxhyphen{}charge energy terms computed
\item[] necd  number of charge\sphinxhyphen{}dipole energy terms computed
\item[] ned   number of dipole\sphinxhyphen{}dipole energy terms computed
\item[] nem   number of multipole energy terms computed
\item[] nep   number of polarization energy terms computed
\item[] new   number of Ewald summation energy terms computed
\item[] ner   number of reaction field energy terms computed
\item[] nes   number of solvation energy terms computed
\item[] nelf  number of metal ligand field energy terms computed
\item[] neg   number of geometric restraint energy terms computed
\item[] nex   number of extra energy terms computed
\end{DUlineblock}

\begin{DUlineblock}{0em}
\item[] \sphinxstylestrong{ALIGN Modules \sphinxhyphen{} information for superposition of structures}
\item[] wfit  weights assigned to atom pairs during superposition
\item[] nfit  number of atoms to use in superimposing two structures
\item[] ifit  atom numbers of pairs of atoms to be superimposed
\end{DUlineblock}

ANALYZ  energy components partitioned over atoms

aesum   total potential energy partitioned over atoms
aeb     bond stretch energy partitioned over atoms
aea     angle bend energy partitioned over atoms
aeba    stretch\sphinxhyphen{}bend energy partitioned over atoms
aeub    Urey\sphinxhyphen{}Bradley energy partitioned over atoms
aeaa    angle\sphinxhyphen{}angle energy partitioned over atoms
aeopb   out\sphinxhyphen{}of\sphinxhyphen{}plane bend energy partitioned over atoms
aeopd   out\sphinxhyphen{}of\sphinxhyphen{}plane distance energy partitioned over atoms
aeid    improper dihedral energy partitioned over atoms
aeit    improper torsion energy partitioned over atoms
aet     torsional energy partitioned over atoms
aept    pi\sphinxhyphen{}orbital torsion energy partitioned over atoms
aebt    stretch\sphinxhyphen{}torsion energy partitioned over atoms
aett    torsion\sphinxhyphen{}torsion energy partitioned over atoms
aev     van der Waals energy partitioned over atoms
aec     charge\sphinxhyphen{}charge energy partitioned over atoms
aecd    charge\sphinxhyphen{}dipole energy partitioned over atoms
aed     dipole\sphinxhyphen{}dipole energy partitioned over atoms
aem     multipole energy partitioned over atoms
aep     polarization energy partitioned over atoms
aer     reaction field energy partitioned over atoms
aes     solvation energy partitioned over atoms
aelf    metal ligand field energy partitioned over atoms
aeg     geometric restraint energy partitioned over atoms
aex     extra energy term partitioned over atoms

ANGANG  angle\sphinxhyphen{}angle terms in current structure

kaa     force constant for angle\sphinxhyphen{}angle cross terms
nangang total number of angle\sphinxhyphen{}angle interactions
iaa     angle numbers used in each angle\sphinxhyphen{}angle term

ANGLE   bond angles within the current structure

ak      harmonic angle force constant (kcal/mole/rad**2)
anat    ideal bond angle or phase shift angle (degrees)
afld    periodicity for Fourier bond angle term
nangle  total number of bond angles in the system
iang    numbers of the atoms in each bond angle
angtyp  potential energy function type for each bond angle

ANGPOT  specifics of bond angle functional forms

cang    cubic coefficient in angle bending potential
qang    quartic coefficient in angle bending potential
pang    quintic coefficient in angle bending potential
sang    sextic coefficient in angle bending potential
angunit convert angle bending energy to kcal/mole
stbnunit        convert stretch\sphinxhyphen{}bend energy to kcal/mole
aaunit  convert angle\sphinxhyphen{}angle energy to kcal/mole
opbunit convert out\sphinxhyphen{}of\sphinxhyphen{}plane bend energy to kcal/mole
opdunit convert out\sphinxhyphen{}of\sphinxhyphen{}plane distance energy to kcal/mole
mm2stbn logical flag governing use of MM2\sphinxhyphen{}style stretch\sphinxhyphen{}bend

ARGUE   command line arguments at program startup

maxarg  maximum number of command line arguments
narg    number of command line arguments to the program
listarg flag to mark available command line arguments
arg     strings containing the command line arguments

ATMLST  local geometry terms involving each atom

bndlist list of the bond numbers involving each atom
anglist list of the angle numbers centered on each atom

ATMTYP  atomic properties for each current atom

mass    atomic weight for each atom in the system
tag     integer atom labels from input coordinates file
class   atom class number for each atom in the system
atomic  atomic number for each atom in the system
valence valence number for each atom in the system
name    atom name for each atom in the system
story   descriptive type for each atom in system

ATOMS   number, position and type of current atoms

x       current x\sphinxhyphen{}coordinate for each atom in the system
y       current y\sphinxhyphen{}coordinate for each atom in the system
z       current z\sphinxhyphen{}coordinate for each atom in the system
n       total number of atoms in the current system
type    atom type number for each atom in the system

BATH    temperature and pressure control parameters

maxnose maximum length of the Nose\sphinxhyphen{}Hoover chain
kelvin0 target value for the system temperature (K)
kelvin  variable target temperature for thermostat (K)
atmsph  target value for the system pressure (atm)
tautemp time constant for Berendsen thermostat (psec)
taupres time constant for Berendsen barostat (psec)
compress        isothermal compressibility of medium (atm\sphinxhyphen{}1)
collide collision frequency for Andersen thermostat
xnh     position of each chained Nose\sphinxhyphen{}Hoover thermostat
vnh     velocity of each chained Nose\sphinxhyphen{}Hoover thermostat
qnh     mass for each chained Nose\sphinxhyphen{}Hoover thermostat
gnh     coupling between chained Nose\sphinxhyphen{}Hoover thermostats
isothermal      logical flag governing use of temperature control
isobaric        logical flag governing use of pressure control
tempvary        logical flag to enable variable target thermostat
thermostat      choice of temperature control method to be used
barostat        choice of pressure control method to be used

BITOR   bitorsions within the current structure

nbitor  total number of bitorsions in the system
ibitor  numbers of the atoms in each bitorsion

BNDPOT  specifics of bond stretch functional forms

cbnd    cubic coefficient in bond stretch potential
qbnd    quartic coefficient in bond stretch potential
bndunit convert bond stretch energy to kcal/mole
bndtyp  type of bond stretch potential energy function

BOND    covalent bonds in the current structure

bk      bond stretch force constants (kcal/mole/Ang**2)
bl      ideal bond length values in Angstroms
nbond   total number of bond stretches in the system
ibnd    numbers of the atoms in each bond stretch

BORDER  bond orders for a conjugated pisystem

pbpl    pi\sphinxhyphen{}bond orders for bonds in “planar” pisystem
pnpl    pi\sphinxhyphen{}bond orders for bonds in “nonplanar” pisystem

BOUND   control of periodic boundary conditions

polycut cutoff distance for infinite polymer nonbonds
polycut2        square of infinite polymer nonbond cutoff
use\_bounds      flag to use periodic boundary conditions
use\_image       flag to use images for periodic system
use\_replica     flag to use replicates for periodic system
use\_polymer     flag to mark presence of infinite polymer

BOXES   parameters for periodic boundary conditions

xbox    length in Angs of a\sphinxhyphen{}axis of periodic box
ybox    length in Angs of b\sphinxhyphen{}axis of periodic box
zbox    length in Angs of c\sphinxhyphen{}axis of periodic box
alpha   angle in degrees between b\sphinxhyphen{} and c\sphinxhyphen{}axes of box
beta    angle in degrees between a\sphinxhyphen{} and c\sphinxhyphen{}axes of box
gamma   angle in degrees between a\sphinxhyphen{} and b\sphinxhyphen{}axes of box
xbox2   half of the a\sphinxhyphen{}axis length of periodic box
ybox2   half of the b\sphinxhyphen{}axis length of periodic box
zbox2   half of the c\sphinxhyphen{}axis length of periodic box
box34   three\sphinxhyphen{}fourths axis length of truncated octahedron
recip   reciprocal lattice vectors as matrix columns
volbox  volume in Ang**3 of the periodic box
beta\_sin        sine of the beta periodic box angle
beta\_cos        cosine of the beta periodic box angle
gamma\_sin       sine of the gamma periodic box angle
gamma\_cos       cosine of the gamma periodic box angle
beta\_term       term used in generating triclinic box
gamma\_term      term used in generating triclinic box
orthogonal      flag to mark periodic box as orthogonal
monoclinic      flag to mark periodic box as monoclinic
triclinic       flag to mark periodic box as triclinic
octahedron      flag to mark box as truncated octahedron
spacegrp        space group symbol for the unitcell type

CELL    periodic boundaries using replicated cells

xcell   length of the a\sphinxhyphen{}axis of the complete replicated cell
ycell   length of the b\sphinxhyphen{}axis of the complete replicated cell
zcell   length of the c\sphinxhyphen{}axis of the complete replicated cell
xcell2  half the length of the a\sphinxhyphen{}axis of the replicated cell
ycell2  half the length of the b\sphinxhyphen{}axis of the replicated cell
zcell2  half the length of the c\sphinxhyphen{}axis of the replicated cell
ncell   total number of cell replicates for periodic boundaries
icell   offset along axes for each replicate periodic cell

CHARGE  partial charges for the current structure

pchg    magnitude of the partial charges (e\sphinxhyphen{})
nion    total number of partial charges in system
iion    number of the atom site for each partial charge
jion    neighbor generation site for each partial charge
kion    cutoff switching site for each partial charge
chglist partial charge site for each atom (0=no charge)

CHGPOT  specifics of charge\sphinxhyphen{}charge functional form

dielec  dielectric constant for electrostatic interactions
c2scale factor by which 1\sphinxhyphen{}2 charge interactions are scaled
c3scale factor by which 1\sphinxhyphen{}3 charge interactions are scaled
c4scale factor by which 1\sphinxhyphen{}4 charge interactions are scaled
c5scale factor by which 1\sphinxhyphen{}5 charge interactions are scaled
neutnbr logical flag governing use of neutral group neighbors
neutcut logical flag governing use of neutral group cutoffs

CHRONO  timing statistics for the current program

cputim  elapsed cpu time in seconds since start of program

COUPLE  near\sphinxhyphen{}neighbor atom connectivity lists

maxn13  maximum number of atoms 1\sphinxhyphen{}3 connected to an atom
maxn14  maximum number of atoms 1\sphinxhyphen{}4 connected to an atom
maxn15  maximum number of atoms 1\sphinxhyphen{}5 connected to an atom
n12     number of atoms directly bonded to each atom
i12     atom numbers of atoms 1\sphinxhyphen{}2 connected to each atom
n13     number of atoms in a 1\sphinxhyphen{}3 relation to each atom
i13     atom numbers of atoms 1\sphinxhyphen{}3 connected to each atom
n14     number of atoms in a 1\sphinxhyphen{}4 relation to each atom
i14     atom numbers of atoms 1\sphinxhyphen{}4 connected to each atom
n15     number of atoms in a 1\sphinxhyphen{}5 relation to each atom
i15     atom numbers of atoms 1\sphinxhyphen{}5 connected to each atom

CUTOFF  cutoff distances for energy interactions

vdwcut  cutoff distance for van der Waals interactions
chgcut  cutoff distance for charge\sphinxhyphen{}charge interactions
dplcut  cutoff distance for dipole\sphinxhyphen{}dipole interactions
mpolecut        cutoff distance for atomic multipole interactions
vdwtaper        distance at which van der Waals switching begins
chgtaper        distance at which charge\sphinxhyphen{}charge switching begins
dpltaper        distance at which dipole\sphinxhyphen{}dipole switching begins
mpoletaper      distance at which atomic multipole switching begins
ewaldcut        cutoff distance for direct space Ewald summation
use\_ewald       logical flag governing use of Ewald summation term
use\_lights      logical flag to use method of lights neighbors

DERIV   Cartesian coordinate derivative components

desum   total energy Cartesian coordinate derivatives
deb     bond stretch Cartesian coordinate derivatives
dea     angle bend Cartesian coordinate derivatives
deba    stretch\sphinxhyphen{}bend Cartesian coordinate derivatives
deub    Urey\sphinxhyphen{}Bradley Cartesian coordinate derivatives
deaa    angle\sphinxhyphen{}angle Cartesian coordinate derivatives
deopb   out\sphinxhyphen{}of\sphinxhyphen{}plane bend Cartesian coordinate derivatives
deopd   out\sphinxhyphen{}of\sphinxhyphen{}plane distance Cartesian coordinate derivatives
deid    improper dihedral Cartesian coordinate derivatives
deit    improper torsion Cartesian coordinate derivatives
det     torsional Cartesian coordinate derivatives
dept    pi\sphinxhyphen{}orbital torsion Cartesian coordinate derivatives
debt    stretch\sphinxhyphen{}torsion Cartesian coordinate derivatives
dett    torsion\sphinxhyphen{}torsion Cartesian coordinate derivatives
dev     van der Waals Cartesian coordinate derivatives
dec     charge\sphinxhyphen{}charge Cartesian coordinate derivatives
decd    charge\sphinxhyphen{}dipole Cartesian coordinate derivatives
ded     dipole\sphinxhyphen{}dipole Cartesian coordinate derivatives
dem     multipole Cartesian coordinate derivatives
dep     polarization Cartesian coordinate derivatives
der     reaction field Cartesian coordinate derivatives
des     solvation Cartesian coordinate derivatives
delf    metal ligand field Cartesian coordinate derivatives
deg     geometric restraint Cartesian coordinate derivatives
dex     extra energy term Cartesian coordinate derivatives

DIPOLE  atom \& bond dipoles for current structure

bdpl    magnitude of each of the dipoles (Debyes)
sdpl    position of each dipole between defining atoms
ndipole total number of dipoles in the system
idpl    numbers of atoms that define each dipole

DISGEO  distance geometry bounds and parameters

bnd     distance geometry upper and lower bounds matrix
vdwrad  hard sphere radii for distance geometry atoms
vdwmax  maximum value of hard sphere sum for an atom pair
compact index of local distance compaction on embedding
pathmax maximum value of upper bound after smoothing
use\_invert      flag to use enantiomer closest to input structure
use\_anneal      flag to use simulated annealing refinement

DOMEGA  derivative components over torsions

tesum   total energy derivatives over torsions
teb     bond stretch derivatives over torsions
tea     angle bend derivatives over torsions
teba    stretch\sphinxhyphen{}bend derivatives over torsions
teub    Urey\sphinxhyphen{}Bradley derivatives over torsions
teaa    angle\sphinxhyphen{}angle derivatives over torsions
teopb   out\sphinxhyphen{}of\sphinxhyphen{}plane bend derivatives over torsions
teopd   out\sphinxhyphen{}of\sphinxhyphen{}plane distance derivatives over torsions
teid    improper dihedral derivatives over torsions
teit    improper torsion derivatives over torsions
tet     torsional derivatives over torsions
tept    pi\sphinxhyphen{}orbital torsion derivatives over torsions
tebt    stretch\sphinxhyphen{}torsion derivatives over torsions
tett    torsion\sphinxhyphen{}torsion derivatives over torsions
tev     van der Waals derivatives over torsions
tec     charge\sphinxhyphen{}charge derivatives over torsions
tecd    charge\sphinxhyphen{}dipole derivatives over torsions
ted     dipole\sphinxhyphen{}dipole derivatives over torsions
tem     atomic multipole derivatives over torsions
tep     polarization derivatives over torsions
ter     reaction field derivatives over torsions
tes     solvation derivatives over torsions
telf    metal ligand field derivatives over torsions
teg     geometric restraint derivatives over torsions
tex     extra energy term derivatives over torsions

ENERGI  individual potential energy components

esum    total potential energy of the system
eb      bond stretch potential energy of the system
ea      angle bend potential energy of the system
eba     stretch\sphinxhyphen{}bend potential energy of the system
eub     Urey\sphinxhyphen{}Bradley potential energy of the system
eaa     angle\sphinxhyphen{}angle potential energy of the system
eopb    out\sphinxhyphen{}of\sphinxhyphen{}plane bend potential energy of the system
eopd    out\sphinxhyphen{}of\sphinxhyphen{}plane distance potential energy of the system
eid     improper dihedral potential energy of the system
eit     improper torsion potential energy of the system
et      torsional potential energy of the system
ept     pi\sphinxhyphen{}orbital torsion potential energy of the system
ebt     stretch\sphinxhyphen{}torsion potential energy of the system
ett     torsion\sphinxhyphen{}torsion potential energy of the system
ev      van der Waals potential energy of the system
ec      charge\sphinxhyphen{}charge potential energy of the system
ecd     charge\sphinxhyphen{}dipole potential energy of the system
ed      dipole\sphinxhyphen{}dipole potential energy of the system
em      atomic multipole potential energy of the system
ep      polarization potential energy of the system
er      reaction field potential energy of the system
es      solvation potential energy of the system
elf     metal ligand field potential energy of the system
eg      geometric restraint potential energy of the system
ex      extra term potential energy of the system

EWALD   parameters for regular or PM Ewald summation

aewald  Ewald convergence coefficient value (Ang\sphinxhyphen{}1)
frecip  fractional cutoff value for reciprocal sphere
tinfoil flag governing use of tinfoil boundary conditions

EWREG   exponential factors for regular Ewald sum

maxvec  maximum number of k\sphinxhyphen{}vectors per reciprocal axis
ejc     exponental factors for cosine along the j\sphinxhyphen{}axis
ejs     exponental factors for sine along the j\sphinxhyphen{}axis
ekc     exponental factors for cosine along the k\sphinxhyphen{}axis
eks     exponental factors for sine along the k\sphinxhyphen{}axis
elc     exponental factors for cosine along the l\sphinxhyphen{}axis
els     exponental factors for sine along the l\sphinxhyphen{}axis

FACES   variables for Connolly area and volume

maxnbr  maximum number of neighboring atom pairs
maxtt   maximum number of temporary tori
maxt    maximum number of total tori
maxp    maximum number of probe positions
maxv    maximum number of vertices
maxen   maximum number of concave edges
maxfn   maximum number of concave faces
maxc    maximum number of circles
maxep   maximum number of convex edges
maxfs   maximum number of saddle faces
maxcy   maximum number of cycles
mxcyep  maximum number of cycle convex edges
maxfp   maximum number of convex faces
mxfpcy  maximum number of convex face cycles

FIELDS  molecular mechanics force field description

biotyp  force field atom type of each biopolymer type
forcefield      string used to describe the current forcefield

FILES   name and number of current structure files

nprior  number of previously existing cycle files
ldir    length in characters of the directory name
leng    length in characters of the base filename
filename        base filename used by default for all files
outfile output filename used for intermediate results

FRACS   atom distances to molecular center of mass

xfrac   fractional coordinate along a\sphinxhyphen{}axis of center of mass
yfrac   fractional coordinate along b\sphinxhyphen{}axis of center of mass
zfrac   fractional coordinate along c\sphinxhyphen{}axis of center of mass

GROUP   partitioning of system into atom groups

grpmass total mass of all the atoms in each group
wgrp    weight for each set of group\sphinxhyphen{}group interactions
ngrp    total number of atom groups in the system
kgrp    contiguous list of the atoms in each group
igrp    first and last atom of each group in the list
grplist number of the group to which each atom belongs
use\_group       flag to use partitioning of system into groups
use\_intra       flag to include only intragroup interactions
use\_inter       flag to include only intergroup interactions

HESCUT  cutoff value for Hessian matrix elements

hesscut magnitude of smallest allowed Hessian element

HESSN   Cartesian Hessian elements for a single atom

hessx   Hessian elements for x\sphinxhyphen{}component of current atom
hessy   Hessian elements for y\sphinxhyphen{}component of current atom
hessz   Hessian elements for z\sphinxhyphen{}component of current atom

IMPROP  improper dihedrals in the current structure

kprop   force constant values for improper dihedral angles
vprop   ideal improper dihedral angle value in degrees
niprop  total number of improper dihedral angles in the system
iiprop  numbers of the atoms in each improper dihedral angle

IMPTOR  improper torsions in the current structure

itors1  1\sphinxhyphen{}fold amplitude and phase for each improper torsion
itors2  2\sphinxhyphen{}fold amplitude and phase for each improper torsion
itors3  3\sphinxhyphen{}fold amplitude and phase for each improper torsion
nitors  total number of improper torsional angles in the system
iitors  numbers of the atoms in each improper torsional angle

INFORM  control values for I/O and program flow

digits  decimal places output for energy and coordinates
iprint  steps between status printing (0=no printing)
iwrite  steps between coordinate dumps (0=no dumps)
isend   steps between socket communication (0=no sockets)
verbose logical flag to turn on extra information
debug   logical flag to turn on full debug printing
holdup  logical flag to wait for carriage return on exit
abort   logical flag to stop execution at next chance

INTER   sum of intermolecular energy components

einter  total intermolecular potential energy

IOUNIT  Fortran input/output (I/O) unit numbers

iout    Fortran I/O unit for major output (default=6)
input   Fortran I/O unit for major input (default=5)

KANANG  forcefield parameters for angle\sphinxhyphen{}angle terms

anan    angle\sphinxhyphen{}angle cross term parameters for each atom class

KANGS   forcefield parameters for bond angle bending

maxna   maximum number of harmonic angle bend parameter entries
maxna5  maximum number of 5\sphinxhyphen{}membered ring angle bend entries
maxna4  maximum number of 4\sphinxhyphen{}membered ring angle bend entries
maxna3  maximum number of 3\sphinxhyphen{}membered ring angle bend entries
maxnaf  maximum number of Fourier angle bend parameter entries
acon    force constant parameters for harmonic angle bends
acon5   force constant parameters for 5\sphinxhyphen{}ring angle bends
acon4   force constant parameters for 4\sphinxhyphen{}ring angle bends
acon3   force constant parameters for 3\sphinxhyphen{}ring angle bends
aconf   force constant parameters for Fourier angle bends
ang     bond angle parameters for harmonic angle bends
ang5    bond angle parameters for 5\sphinxhyphen{}ring angle bends
ang4    bond angle parameters for 4\sphinxhyphen{}ring angle bends
ang3    bond angle parameters for 3\sphinxhyphen{}ring angle bends
angf    phase shift angle and periodicity for Fourier bends
ka      string of atom classes for harmonic angle bends
ka5     string of atom classes for 5\sphinxhyphen{}ring angle bends
ka4     string of atom classes for 4\sphinxhyphen{}ring angle bends
ka3     string of atom classes for 3\sphinxhyphen{}ring angle bends
kaf     string of atom classes for Fourier angle bends

KATOMS  forcefield parameters for the atom types

weight  average atomic mass of each atom type
atmcls  atom class number for each of the atom types
atmnum  atomic number for each of the atom types
ligand  number of atoms to be attached to each atom type
symbol  modified atomic symbol for each atom type
describe        string identifing each of the atom types

KBONDS  forcefield parameters for bond stretching

maxnb   maximum number of bond stretch parameter entries
maxnb5  maximum number of 5\sphinxhyphen{}membered ring bond stretch entries
maxnb4  maximum number of 4\sphinxhyphen{}membered ring bond stretch entries
maxnb3  maximum number of 3\sphinxhyphen{}membered ring bond stretch entries
maxnel  maximum number of electronegativity bond corrections
bcon    force constant parameters for harmonic bond stretch
bcon5   force constant parameters for 5\sphinxhyphen{}ring bond stretch
bcon4   force constant parameters for 4\sphinxhyphen{}ring bond stretch
bcon3   force constant parameters for 3\sphinxhyphen{}ring bond stretch
blen    bond length parameters for harmonic bond stretch
blen5   bond length parameters for 5\sphinxhyphen{}ring bond stretch
blen4   bond length parameters for 4\sphinxhyphen{}ring bond stretch
blen3   bond length parameters for 3\sphinxhyphen{}ring bond stretch
dlen    electronegativity bond length correction parameters
kb      string of atom classes for harmonic bond stretch
kb5     string of atom classes for 5\sphinxhyphen{}ring bond stretch
kb4     string of atom classes for 4\sphinxhyphen{}ring bond stretch
kb3     string of atom classes for 3\sphinxhyphen{}ring bond stretch
kel     string of atom classes for electronegativity corrections

KCHRGE  forcefield parameters for partial charges

chg     partial charge parameters for each atom type

KDIPOL  forcefield parameters for bond dipoles

maxnd   maximum number of bond dipole parameter entries
maxnd5  maximum number of 5\sphinxhyphen{}membered ring dipole entries
maxnd4  maximum number of 4\sphinxhyphen{}membered ring dipole entries
maxnd3  maximum number of 3\sphinxhyphen{}membered ring dipole entries
dpl     dipole moment parameters for bond dipoles
dpl5    dipole moment parameters for 5\sphinxhyphen{}ring dipoles
dpl4    dipole moment parameters for 4\sphinxhyphen{}ring dipoles
dpl3    dipole moment parameters for 3\sphinxhyphen{}ring dipoles
pos     dipole position parameters for bond dipoles
pos5    dipole position parameters for 5\sphinxhyphen{}ring dipoles
pos4    dipole position parameters for 4\sphinxhyphen{}ring dipoles
pos3    dipole position parameters for 3\sphinxhyphen{}ring dipoles
kd      string of atom classes for bond dipoles
kd5     string of atom classes for 5\sphinxhyphen{}ring dipoles
kd4     string of atom classes for 4\sphinxhyphen{}ring dipoles
kd3     string of atom classes for 3\sphinxhyphen{}ring dipoles

KEYS    contents of current keyword parameter file

nkey    number of nonblank lines in the keyword file
keyline contents of each individual keyword file line

KGEOMS  parameters for the geometrical restraints

xpfix   x\sphinxhyphen{}coordinate target for each restrained position
ypfix   y\sphinxhyphen{}coordinate target for each restrained position
zpfix   z\sphinxhyphen{}coordinate target for each restrained position
pfix    force constant and flat\sphinxhyphen{}well range for each position
dfix    force constant and target range for each distance
afix    force constant and target range for each angle
tfix    force constant and target range for each torsion
gfix    force constant and target range for each group distance
chir    force constant and target range for chiral centers
depth   depth of shallow Gaussian basin restraint
width   exponential width coefficient of Gaussian basin
rwall   radius of spherical droplet boundary restraint
npfix   number of position restraints to be applied
ipfix   atom number involved in each position restraint
kpfix   flags to use x\sphinxhyphen{}, y\sphinxhyphen{}, z\sphinxhyphen{}coordinate position restraints
ndfix   number of distance restraints to be applied
idfix   atom numbers defining each distance restraint
nafix   number of angle restraints to be applied
iafix   atom numbers defining each angle restraint
ntfix   number of torsional restraints to be applied
itfix   atom numbers defining each torsional restraint
ngfix   number of group distance restraints to be applied
igfix   group numbers defining each group distance restraint
nchir   number of chirality restraints to be applied
ichir   atom numbers defining each chirality restraint
use\_basin       logical flag governing use of Gaussian basin
use\_wall        logical flag governing use of droplet boundary

KHBOND  forcefield parameters for H\sphinxhyphen{}bonding terms

maxnhb  maximum number of hydrogen bonding pair entries
radhb   radius parameter for hydrogen bonding pairs
epshb   well depth parameter for hydrogen bonding pairs
khb     string of atom types for hydrogen bonding pairs

KIPROP  forcefield parameters for improper dihedral

maxndi  maximum number of improper dihedral parameter entries
dcon    force constant parameters for improper dihedrals
tdi     ideal dihedral angle values for improper dihedrals
kdi     string of atom classes for improper dihedral angles

KITORS  forcefield parameters for improper torsions

maxnti  maximum number of improper torsion parameter entries
ti1     torsional parameters for improper 1\sphinxhyphen{}fold rotation
ti2     torsional parameters for improper 2\sphinxhyphen{}fold rotation
ti3     torsional parameters for improper 3\sphinxhyphen{}fold rotation
kti     string of atom classes for improper torsional parameters

KMULTI  forcefield parameters for atomic multipoles

maxnmp  maximum number of atomic multipole parameter entries
multip  atomic monopole, dipole and quadrupole values
mpaxis  type of local axis definition for atomic multipoles
kmp     string of atom types for atomic multipoles

KOPBND  forcefield parameters for out\sphinxhyphen{}of\sphinxhyphen{}plane bend

maxnopb maximum number of out\sphinxhyphen{}of\sphinxhyphen{}plane bending entries
copb    force constant parameters for out\sphinxhyphen{}of\sphinxhyphen{}plane bending
kaopb   string of atom classes for out\sphinxhyphen{}of\sphinxhyphen{}plane bending

KOPDST  forcefield parameters for out\sphinxhyphen{}plane distance

maxnopb maximum number of out\sphinxhyphen{}of\sphinxhyphen{}plane distance entries
copb    force constant parameters for out\sphinxhyphen{}of\sphinxhyphen{}plane distance
kaopb   string of atom classes for out\sphinxhyphen{}of\sphinxhyphen{}plane distance

KORBS   forcefield parameters for pisystem orbitals

maxnpi  maximum number of pisystem bond parameter entries
electron        number of pi\sphinxhyphen{}electrons for each atom class
ionize  ionization potential for each atom class
repulse repulsion integral value for each atom class
sslope  slope for bond stretch vs. pi\sphinxhyphen{}bond order
tslope  slope for 2\sphinxhyphen{}fold torsion vs. pi\sphinxhyphen{}bond order
kpi     string of atom classes for pisystem bonds

KPITOR  forcefield parameters for pi\sphinxhyphen{}orbit torsions

maxnpt  maximum number of pi\sphinxhyphen{}orbital torsion parameter entries
ptcon   force constant parameters for pi\sphinxhyphen{}orbital torsions
kpt     string of atom classes for pi\sphinxhyphen{}orbital torsion terms

KPOLR   forcefield parameters for polarizability

polr    dipole polarizability parameters for each atom type
pgrp    connected types in polarization group of each atom type

KSTBND  forcefield parameters for stretch\sphinxhyphen{}bending

stbn    stretch\sphinxhyphen{}bending parameters for each atom class

KSTTOR  forcefield parameters for stretch\sphinxhyphen{}torsions

maxnbt  maximum number of stretch\sphinxhyphen{}torsion parameter entries
btcon   force constant parameters for stretch\sphinxhyphen{}torsion
kbt     string of atom classes for bonds in stretch\sphinxhyphen{}torsion

KTORSN  forcefield parameters for torsional angles

maxnt   maximum number of torsional angle parameter entries
maxnt5  maximum number of 5\sphinxhyphen{}membered ring torsion entries
maxnt4  maximum number of 4\sphinxhyphen{}membered ring torsion entries
t1      torsional parameters for standard 1\sphinxhyphen{}fold rotation
t2      torsional parameters for standard 2\sphinxhyphen{}fold rotation
t3      torsional parameters for standard 3\sphinxhyphen{}fold rotation
t4      torsional parameters for standard 4\sphinxhyphen{}fold rotation
t5      torsional parameters for standard 5\sphinxhyphen{}fold rotation
t6      torsional parameters for standard 6\sphinxhyphen{}fold rotation
t15     torsional parameters for 1\sphinxhyphen{}fold rotation in 5\sphinxhyphen{}ring
t25     torsional parameters for 2\sphinxhyphen{}fold rotation in 5\sphinxhyphen{}ring
t35     torsional parameters for 3\sphinxhyphen{}fold rotation in 5\sphinxhyphen{}ring
t45     torsional parameters for 4\sphinxhyphen{}fold rotation in 5\sphinxhyphen{}ring
t55     torsional parameters for 5\sphinxhyphen{}fold rotation in 5\sphinxhyphen{}ring
t65     torsional parameters for 6\sphinxhyphen{}fold rotation in 5\sphinxhyphen{}ring
t14     torsional parameters for 1\sphinxhyphen{}fold rotation in 4\sphinxhyphen{}ring
t24     torsional parameters for 2\sphinxhyphen{}fold rotation in 4\sphinxhyphen{}ring
t34     torsional parameters for 3\sphinxhyphen{}fold rotation in 4\sphinxhyphen{}ring
t44     torsional parameters for 4\sphinxhyphen{}fold rotation in 4\sphinxhyphen{}ring
t54     torsional parameters for 5\sphinxhyphen{}fold rotation in 4\sphinxhyphen{}ring
t64     torsional parameters for 6\sphinxhyphen{}fold rotation in 4\sphinxhyphen{}ring
kt      string of atom classes for torsional angles
kt5     string of atom classes for 5\sphinxhyphen{}ring torsions
kt4     string of atom classes for 4\sphinxhyphen{}ring torsions

KTRTOR  forcefield parameters for torsion\sphinxhyphen{}torsions

maxntt  maximum number of torsion\sphinxhyphen{}torsion parameter entries
maxtgrd maximum dimension of torsion\sphinxhyphen{}torsion spline grid
maxtgrd2        maximum number of torsion\sphinxhyphen{}torsion spline grid points
ttx     angle values for first torsion of spline grid
tty     angle values for second torsion of spline grid
tbf     function values at points on spline grid
tbx     gradient over first torsion of spline grid
tby     gradient over second torsion of spline grid
tbxy    Hessian cross components over spline grid
tnx     number of columns in torsion\sphinxhyphen{}torsion spline grid
tny     number of rows in torsion\sphinxhyphen{}torsion spline grid
ktt     string of torsion\sphinxhyphen{}torsion atom classes

KURYBR  forcefield parameters for Urey\sphinxhyphen{}Bradley terms

maxnu   maximum number of Urey\sphinxhyphen{}Bradley parameter entries
ucon    force constant parameters for Urey\sphinxhyphen{}Bradley terms
dst13   ideal 1\sphinxhyphen{}3 distance parameters for Urey\sphinxhyphen{}Bradley terms
ku      string of atom classes for Urey\sphinxhyphen{}Bradley terms

KVDWPR  forcefield parameters for special vdw terms

maxnvp  maximum number of special van der Waals pair entries
radpr   radius parameter for special van der Waals pairs
epspr   well depth parameter for special van der Waals pairs
kvpr    string of atom classes for special van der Waals pairs

KVDWS   forcefield parameters for van der Waals terms

rad     van der Waals radius parameter for each atom class
eps     van der Waals well depth parameter for each atom class
rad4    van der Waals radius parameter in 1\sphinxhyphen{}4 interactions
eps4    van der Waals well depth parameter in 1\sphinxhyphen{}4 interactions
reduct  van der Waals reduction factor for each atom class

LIGHT   indices for method of lights pair neighbors

nlight  total number of sites for method of lights calculation
kbx     low index of neighbors of each site in the x\sphinxhyphen{}sorted list
kby     low index of neighbors of each site in the y\sphinxhyphen{}sorted list
kbz     low index of neighbors of each site in the z\sphinxhyphen{}sorted list
kex     high index of neighbors of each site in the x\sphinxhyphen{}sorted list
key     high index of neighbors of each site in the y\sphinxhyphen{}sorted list
kez     high index of neighbors of each site in the z\sphinxhyphen{}sorted list
locx    pointer from x\sphinxhyphen{}sorted list into original interaction list
locy    pointer from y\sphinxhyphen{}sorted list into original interaction list
locz    pointer from z\sphinxhyphen{}sorted list into original interaction list
rgx     pointer from original interaction list into x\sphinxhyphen{}sorted list
rgy     pointer from original interaction list into y\sphinxhyphen{}sorted list
rgz     pointer from original interaction list into z\sphinxhyphen{}sorted list

LINMIN  parameters for line search minimization

stpmin  minimum step length in current line search direction
stpmax  maximum step length in current line search direction
cappa   stringency of line search (0=tight \textless{} cappa \textless{} 1=loose)
slpmax  projected gradient above which stepsize is reduced
angmax  maximum angle between search direction and \sphinxhyphen{}gradient
intmax  maximum number of interpolations during line search

MATH    mathematical and geometrical constants

radian  conversion factor from radians to degrees
pi      numerical value of the geometric constant
sqrtpi  numerical value of the square root of Pi
logten  numerical value of the natural log of ten
sqrttwo numerical value of the square root of two
twosix  numerical value of the sixth root of two

MDSTUF  control of molecular dynamics trajectory

nfree   total number of degrees of freedom for a system
velsave flag to save velocity vector components to a file
frcsave flag to save force vector components to a file
uindsave        flag to save induced atomic dipoles to a file
integrate       type of molecular dynamics integration algorithm

MINIMA  general parameters for minimizations

fctmin  value below which function is deemed optimized
hguess  initial value for the H\sphinxhyphen{}matrix diagonal elements
maxiter maximum number of iterations during optimization
nextiter        iteration number to use for the first iteration

MOLCUL  individual molecules within current system

molmass molecular weight for each molecule in the system
totmass total weight of all the molecules in the system
nmol    total number of separate molecules in the system
kmol    contiguous list of the atoms in each molecule
imol    first and last atom of each molecule in the list
molcule number of the molecule to which each atom belongs

MOLDYN  velocity and acceleration on MD trajectory

v       current velocity of each atom along the x,y,z\sphinxhyphen{}axes
a       current acceleration of each atom along x,y,z\sphinxhyphen{}axes
aold    previous acceleration of each atom along x,y,z\sphinxhyphen{}axes

MOMENT  components of electric multipole moments

netchg  net electric charge for the total system
netdpl  dipole moment magnitude for the total system
netqdp  diagonal quadrupole (Qxx, Qyy, Qzz) for system
xdpl    dipole vector x\sphinxhyphen{}component in the global frame
ydpl    dipole vector y\sphinxhyphen{}component in the global frame
zdpl    dipole vector z\sphinxhyphen{}component in the global frame
xxqdp   quadrupole tensor xx\sphinxhyphen{}component in global frame
xyqdp   quadrupole tensor xy\sphinxhyphen{}component in global frame
xzqdp   quadrupole tensor xz\sphinxhyphen{}component in global frame
yxqdp   quadrupole tensor yx\sphinxhyphen{}component in global frame
yyqdp   quadrupole tensor yy\sphinxhyphen{}component in global frame
yzqdp   quadrupole tensor yz\sphinxhyphen{}component in global frame
zxqdp   quadrupole tensor zx\sphinxhyphen{}component in global frame
zyqdp   quadrupole tensor zy\sphinxhyphen{}component in global frame
zzqdp   quadrupole tensor zz\sphinxhyphen{}component in global frame

MPLPOT  specifics of atomic multipole functions

m2scale factor by which 1\sphinxhyphen{}2 multipole interactions are scaled
m3scale factor by which 1\sphinxhyphen{}3 multipole interactions are scaled
m4scale factor by which 1\sphinxhyphen{}4 multipole interactions are scaled
m5scale factor by which 1\sphinxhyphen{}5 multipole interactions are scaled

MPOLE   multipole components for current structure

maxpole max components (monopole=1,dipole=4,quadrupole=13)
pole    multipole values for each site in the local frame
rpole   multipoles rotated to the global coordinate system
npole   total number of multipole sites in the system
ipole   number of the atom for each multipole site
polsiz  number of mutipole components at each multipole site
zaxis   number of the z\sphinxhyphen{}axis defining atom for each site
xaxis   number of the x\sphinxhyphen{}axis defining atom for each site
yaxis   number of the y\sphinxhyphen{}axis defining atom for each site
polaxe  local axis type for each multipole site

MUTANT  hybrid atoms for free energy perturbation

lambda  weighting of initial state in hybrid Hamiltonian
nhybrid number of atoms mutated from initial to final state
ihybrid atomic sites differing in initial and final state
type0   atom type of each atom in the initial state system
class0  atom class of each atom in the initial state system
type1   atom type of each atom in the final state system
class1  atom class of each atom in the final state system
alter   true if an atom is to be mutated, false otherwise

NUCLEO  parameters for nucleic acid structure

bkbone  phosphate backbone angles for each nucleotide
glyco   glycosidic torsional angle for each nucleotide
pucker  sugar pucker, either 2=2’\sphinxhyphen{}endo or 3=3’\sphinxhyphen{}endo
dblhlx  flag to mark system as nucleic acid double helix
deoxy   flag to mark deoxyribose or ribose sugar units
hlxform helix form (A, B or Z) of polynucleotide strands

OMEGA   dihedrals for torsional space computations

dihed   current value in radians of each dihedral angle
nomega  number of dihedral angles allowed to rotate
iomega  numbers of two atoms defining rotation axis
zline   line number in Z\sphinxhyphen{}matrix of each dihedral angle

OPBEND  out\sphinxhyphen{}of\sphinxhyphen{}plane bends in the current structure

kopb    force constant values for out\sphinxhyphen{}of\sphinxhyphen{}plane bending
nopbend total number of out\sphinxhyphen{}of\sphinxhyphen{}plane bends in the system
iopb    bond angle numbers used in out\sphinxhyphen{}of\sphinxhyphen{}plane bending

OPDIST  out\sphinxhyphen{}of\sphinxhyphen{}plane distances in current structure

kopd    force constant values for out\sphinxhyphen{}of\sphinxhyphen{}plane distance
nopdist total number of out\sphinxhyphen{}of\sphinxhyphen{}plane distances in the system
iopb    numbers of the atoms in each out\sphinxhyphen{}of\sphinxhyphen{}plane distance

ORBITS  orbital energies for conjugated pisystem

q       number of pi\sphinxhyphen{}electrons contributed by each atom
w       ionization potential of each pisystem atom
em      repulsion integral for each pisystem atom
nfill   number of filled pisystem molecular orbitals

OUTPUT  control of coordinate output file format

archive logical flag to save structures in an archive
noversion       logical flag governing use of filename versions
overwrite       logical flag to overwrite intermediate files inplace
cyclesave       logical flag to mark use of numbered cycle files
coordtype       selects Cartesian, internal, rigid body or none

PARAMS  contents of force field parameter file

nprm    number of nonblank lines in the parameter file
prmline contents of each individual parameter file line

PATHS   parameters for Elber reaction path method

p0      reactant Cartesian coordinates as variables
p1      product Cartesian coordinates as variables
pmid    midpoint between the reactant and product
pvect   vector connecting the reactant and product
pstep   step per cycle along reactant\sphinxhyphen{}product vector
pzet    current projection on reactant\sphinxhyphen{}product vector
pnorm   length of the reactant\sphinxhyphen{}product vector
acoeff  transformation matrix ‘A’ from Elber paper
gc      gradients of the path constraints

PDB     definition of a Protein Data Bank structure

xpdb    x\sphinxhyphen{}coordinate of each atom stored in PDB format
ypdb    y\sphinxhyphen{}coordinate of each atom stored in PDB format
zpdb    z\sphinxhyphen{}coordinate of each atom stored in PDB format
npdb    number of atoms stored in Protein Data Bank format
resnum  number of the residue to which each atom belongs
npdb12  number of atoms directly bonded to each CONECT atom
ipdb12  atom numbers of atoms connected to each CONECT atom
pdblist list of the Protein Data Bank atom number of each atom
pdbtyp  Protein Data Bank record type assigned to each atom
atmnam  Protein Data Bank atom name assigned to each atom
resnam  Protein Data Bank residue name assigned to each atom

PHIPSI  phi\sphinxhyphen{}psi\sphinxhyphen{}omega\sphinxhyphen{}chi angles for a protein

phi     value of the phi angle for each amino acid residue
psi     value of the psi angle for each amino acid residue
omega   value of the omega angle for each amino acid residue
chi     values of the chi angles for each amino acid residue
chiral  chirality of each amino acid residue (1=L, \sphinxhyphen{}1=D)
disulf  residue joined to each residue via a disulfide link

PIORBS  conjugated system in the current structure

norbit  total number of pisystem orbitals in the system
iorbit  numbers of the atoms containing pisystem orbitals
reorbit number of evaluations between orbital updates
piperp  atoms defining a normal plane to each orbital
nbpi    total number of bonds affected by the pisystem
bpi     bond and piatom numbers for each pisystem bond
ntpi    total number of torsions affected by the pisystem
tpi     torsion and pibond numbers for each pisystem torsion
listpi  atom list indicating whether each atom has an orbital

PISTUF  bonds and torsions in the current pisystem

bkpi    bond stretch force constants for pi\sphinxhyphen{}bond order of 1.0
blpi    ideal bond length values for a pi\sphinxhyphen{}bond order of 1.0
kslope  rate of force constant decrease with bond order decrease
lslope  rate of bond length increase with a bond order decrease
torsp2  2\sphinxhyphen{}fold torsional energy barrier for pi\sphinxhyphen{}bond order of 1.0

PITORS  pi\sphinxhyphen{}orbital torsions in the current structure

kpit    2\sphinxhyphen{}fold pi\sphinxhyphen{}orbital torsional force constants
npitors total number of pi\sphinxhyphen{}orbital torsional interactions
ipit    numbers of the atoms in each pi\sphinxhyphen{}orbital torsion

PME     parameters for particle mesh Ewald summation

maxfft  maximum number of points along each FFT direction
maxorder        maximum order of the B\sphinxhyphen{}spline approximation
maxtable        maximum size of the FFT table array
maxgrid maximum dimension of the PME charge grid array
bsmod1  B\sphinxhyphen{}spline moduli along the a\sphinxhyphen{}axis direction
bsmod2  B\sphinxhyphen{}spline moduli along the b\sphinxhyphen{}axis direction
bsmod3  B\sphinxhyphen{}spline moduli along the c\sphinxhyphen{}axis direction
table   intermediate array used by the FFT calculation
nfft1   number of grid points along the a\sphinxhyphen{}axis direction
nfft2   number of grid points along the b\sphinxhyphen{}axis direction
nfft3   number of grid points along the c\sphinxhyphen{}axis direction
bsorder order of the PME B\sphinxhyphen{}spline approximation

POLAR   polarizabilities and induced dipole moments

polarity        dipole polarizability for each multipole site (Ang**3)
pdamp   value of polarizability damping factor for each site
uind    induced dipole components at each multipole site
uinp    induced dipoles in field used for energy interactions
npolar  total number of polarizable sites in the system

POLGRP  polarizable site group connectivity lists

maxp11  maximum number of atoms in a polarization group
maxp12  maximum number of atoms in groups 1\sphinxhyphen{}2 to an atom
maxp13  maximum number of atoms in groups 1\sphinxhyphen{}3 to an atom
maxp14  maximum number of atoms in groups 1\sphinxhyphen{}4 to an atom
np11    number of atoms in polarization group of each atom
ip11    atom numbers of atoms in same group as each atom
np12    number of atoms in groups 1\sphinxhyphen{}2 to each atom
ip12    atom numbers of atoms in groups 1\sphinxhyphen{}2 to each atom
np13    number of atoms in groups 1\sphinxhyphen{}3 to each atom
ip13    atom numbers of atoms in groups 1\sphinxhyphen{}3 to each atom
np14    number of atoms in groups 1\sphinxhyphen{}4 to each atom
ip14    atom numbers of atoms in groups 1\sphinxhyphen{}4 to each atom

POLPOT  specifics of polarization functional form

poleps  induced dipole convergence criterion (rms Debyes/atom)
polsor  induced dipole SOR convergence acceleration factor
pgamma  prefactor in exponential polarization damping term
p2scale field 1\sphinxhyphen{}2 scale factor for energy evaluations
p3scale field 1\sphinxhyphen{}3 scale factor for energy evaluations
p4scale field 1\sphinxhyphen{}4 scale factor for energy evaluations
p5scale field 1\sphinxhyphen{}5 scale factor for energy evaluations
d1scale field intra\sphinxhyphen{}group scale factor for direct induced
d2scale field 1\sphinxhyphen{}2 group scale factor for direct induced
d3scale field 1\sphinxhyphen{}3 group scale factor for direct induced
d4scale field 1\sphinxhyphen{}4 group scale factor for direct induced
u1scale field intra\sphinxhyphen{}group scale factor for mutual induced
u2scale field 1\sphinxhyphen{}2 group scale factor for mutual induced
u3scale field 1\sphinxhyphen{}3 group scale factor for mutual induced
u4scale field 1\sphinxhyphen{}4 group scale factor for mutual induced
poltyp  type of polarization potential (direct or mutual)

POTENT  usage of each potential energy component

use\_bond        logical flag governing use of bond stretch potential
use\_angle       logical flag governing use of angle bend potential
use\_strbnd      logical flag governing use of stretch\sphinxhyphen{}bend potential
use\_urey        logical flag governing use of Urey\sphinxhyphen{}Bradley potential
use\_angang      logical flag governing use of angle\sphinxhyphen{}angle cross term
use\_opbend      logical flag governing use of out\sphinxhyphen{}of\sphinxhyphen{}plane bend term
use\_opdist      logical flag governing use of out\sphinxhyphen{}of\sphinxhyphen{}plane distance
use\_improp      logical flag governing use of improper dihedral term
use\_imptor      logical flag governing use of improper torsion term
use\_tors        logical flag governing use of torsional potential
use\_pitors      logical flag governing use of pi\sphinxhyphen{}orbital torsion term
use\_strtor      logical flag governing use of stretch\sphinxhyphen{}torsion term
use\_tortor      logical flag governing use of torsion\sphinxhyphen{}torsion term
use\_vdw logical flag governing use of vdw der Waals potential
use\_charge      logical flag governing use of charge\sphinxhyphen{}charge potential
use\_chgdpl      logical flag governing use of charge\sphinxhyphen{}dipole potential
use\_dipole      logical flag governing use of dipole\sphinxhyphen{}dipole potential
use\_mpole       logical flag governing use of multipole potential
use\_polar       logical flag governing use of polarization term
use\_rxnfld      logical flag governing use of reaction field term
use\_solv        logical flag governing use of surface area solvation
use\_gbsa        logical flag governing use of GB/SA solvation term
use\_metal       logical flag governing use of ligand field term
use\_geom        logical flag governing use of geometric restraints
use\_extra       logical flag governing use of extra potential term
use\_orbit       logical flag governing use of pisystem computation

PRECIS  values of machine precision tolerances

tiny    the smallest positive floating point value
small   the smallest relative floating point spacing
huge    the largest relative floating point spacing

REFER   storage of reference atomic coordinate set

xref    reference x\sphinxhyphen{}coordinate for each atom in the system
yref    reference y\sphinxhyphen{}coordinate for each atom in the system
zref    reference z\sphinxhyphen{}coordinate for each atom in the system
nref    total number of atoms in the reference system
reftyp  atom type for each atom in the reference system
n12ref  number of atoms bonded to each reference atom
i12ref  atom numbers of atoms 1\sphinxhyphen{}2 connected to each atom
refleng length in characters of the reference filename
refltitle       length in characters of the reference title string
refnam  atom name for each atom in the reference system
reffile base filename for the reference structure
reftitle        title used to describe the reference structure

RESDUE  standard biopolymer residue abbreviations

amino   three\sphinxhyphen{}letter abbreviations for amino acids types
nuclz   three\sphinxhyphen{}letter abbreviations for nucleic acids types
amino1  one\sphinxhyphen{}letter abbreviations for amino acids types
nuclz1  one\sphinxhyphen{}letter abbreviations for nucleic acids types

RGDDYN  velocities and momenta for rigid body MD

vcm     current translational velocity of each rigid body
wcm     current angular velocity of each rigid body
lm      current angular momentum of each rigid body
linear  logical flag to mark group as linear or nonlinear

RIGID   rigid body coordinates for atom groups

xrb     rigid body reference x\sphinxhyphen{}coordinate for each atom
yrb     rigid body reference y\sphinxhyphen{}coordinate for each atom
zrb     rigid body reference z\sphinxhyphen{}coordinate for each atom
rbc     current rigid body coordinates for each group
use\_rigid       flag to mark use of rigid body coordinate system

RING    number and location of small ring structures

nring3  total number of 3\sphinxhyphen{}membered rings in the system
iring3  numbers of the atoms involved in each 3\sphinxhyphen{}ring
nring4  total number of 4\sphinxhyphen{}membered rings in the system
iring4  numbers of the atoms involved in each 4\sphinxhyphen{}ring
nring5  total number of 5\sphinxhyphen{}membered rings in the system
iring5  numbers of the atoms involved in each 5\sphinxhyphen{}ring
nring6  total number of 6\sphinxhyphen{}membered rings in the system
iring6  numbers of the atoms involved in each 6\sphinxhyphen{}ring

ROTATE  molecule partitions for rotation of a bond

nrot    total number of atoms moving when bond rotates
rot     atom numbers of atoms moving when bond rotates
use\_short       logical flag governing use of shortest atom list

RXNFLD  reaction field matrix elements and indices

b1      first reaction field matrix element array
b2      second reaction field matrix element array
ijk     indices into the reaction field element arrays

RXNPOT  specifics of reaction field functional form

rfsize  radius of reaction field sphere centered at origin
rfbulkd bulk dielectric constant of reaction field continuum
rfterms number of terms to use in reaction field summation

SCALES  parameter scale factors for optimization

scale   multiplicative factor for each optimization parameter
set\_scale       logical flag to show if scale factors have been set

SEQUEN  sequence information for a biopolymer

nseq    total number of residues in biopolymer sequences
nchain  number of separate biopolymer sequence chains
ichain  first and last residue in each biopolymer chain
seqtyp  residue type for each residue in the sequence
seq     three\sphinxhyphen{}letter code for each residue in the sequence
chnnam  one\sphinxhyphen{}letter identifier for each sequence chain

SHAKE   definition of Shake/Rattle constraints

krat    ideal distance value for rattle constraint
nrat    number of rattle distance constraints to apply
nratx   number of atom group spatial constraints to apply
irat    atom numbers of atoms in a rattle constraint
iratx   group number of group in a spatial constraint
kratx   spatial constraint type (1=plane, 2=line, 3=point)
ratimage        flag to use minimum image for rattle constraint
use\_rattle      logical flag to set use of rattle contraints

SHUNT   polynomial switching function coefficients

off     distance at which the potential energy goes to zero
off2    square of distance at which the potential goes to zero
cut     distance at which switching of the potential begins
cut2    square of distance at which the switching begins
c0      zeroth order coefficient of multiplicative switch
c1      first order coefficient of multiplicative switch
c2      second order coefficient of multiplicative switch
c3      third order coefficient of multiplicative switch
c4      fourth order coefficient of multiplicative switch
c5      fifth order coefficient of multiplicative switch
f0      zeroth order coefficient of additive switch function
f1      first order coefficient of additive switch function
f2      second order coefficient of additive switch function
f3      third order coefficient of additive switch function
f4      fourth order coefficient of additive switch function
f5      fifth order coefficient of additive switch function
f6      sixth order coefficient of additive switch function
f7      seventh order coefficient of additive switch function

SIZES   parameter values to set array dimensions

“sizes.i” sets values for critical array dimensions used throughout the software; these parameters will fix the size of the largest systems that can be handled; values too large for   the computer’s memory and/or swap space to accomodate will result in poor performance or outright failure

parameter:      maximum allowed number of:

maxatm  atoms in the molecular system
maxval  atoms directly bonded to an atom
maxgrp  user\sphinxhyphen{}defined groups of atoms
maxtyp  force field atom type definitions
maxclass        force field atom class definitions
maxprm  lines in the parameter file
maxkey  lines in the keyword file
maxrot  bonds for torsional rotation
maxvar  optimization variables (vector storage)
maxopt  optimization variables (matrix storage)
maxhess off\sphinxhyphen{}diagonal Hessian elements
maxlight        sites for method of lights neighbors
maxvib  vibrational frequencies
maxgeo  distance geometry points
maxcell unit cells in replicated crystal
maxring 3\sphinxhyphen{}, 4\sphinxhyphen{}, or 5\sphinxhyphen{}membered rings
maxfix  geometric constraints and restraints
maxbio  biopolymer atom definitions
maxres  residues in the macromolecule
maxamino        amino acid residue types
maxnuc  nucleic acid residue types
maxbnd  covalent bonds in molecular system
maxang  bond angles in molecular system
maxtors torsional angles in molecular system
maxbitor        bitorsions in molecular system
maxpi   atoms in conjugated pisystem
maxpib  covalent bonds involving pisystem
maxpit  torsional angles involving pisystem

SOCKET  control parameters for socket communication

runtyp  calculation type for passing socket information
cstep   current optimization or dynamics step number
cdt     current dynamics cumulative simulation time
cenergy current potential energy from simulation
cdx     current gradient components along the x\sphinxhyphen{}axis
cdy     current gradient components along the y\sphinxhyphen{}axis
cdz     current gradient components along the z\sphinxhyphen{}axis
skt\_init        logical flag set to true after socket initialization
use\_socket      logical flag governing use of external sockets
use\_gui logical flag to show Tinker was invoked from GUI
closing logical flag to indicate JVM and server shutdown

SOLUTE  parameters for continuum solvation models

rsolv   atomic radius of each atom for continuum solvation
vsolv   atomic volume of each atom for continuum solvation
asolv   atomic solvation parameters (kcal/mole/Ang**2)
rborn   Born radius of each atom for GB/SA solvation
drb     solvation derivatives with respect to Born radii
doffset dielectric offset to continuum solvation atomic radii
p1      single\sphinxhyphen{}atom scale factor for analytical Still GB/SA
p2      1\sphinxhyphen{}2 interaction scale factor for analytical Still GB/SA
p3      1\sphinxhyphen{}3 interaction scale factor for analytical Still GB/SA
p4      nonbonded scale factor for analytical Still GB/SA
p5      soft cutoff parameter for analytical Still GB/SA
gpol    polarization self\sphinxhyphen{}energy values for each atom
shct    overlap scaling factors for Hawkins\sphinxhyphen{}Cramer\sphinxhyphen{}Truhlar GB/SA
wace    “omega” values for atom class pairs for use with ACE
s2ace   “sigma\textasciicircum{}2” values for atom class pairs for use with ACE
uace    “mu” values for atom class pairs for use with ACE
solvtyp solvation model (ASP, SASA, ONION, STILL, HCT, ACE)

STODYN  frictional coefficients for SD trajectory

friction        global frictional coefficient for exposed particle
gamma   atomic frictional coefficients for each atom
use\_sdarea      logical flag to use surface area friction scaling

STRBND  stretch\sphinxhyphen{}bends in the current structure

ksb     force constant for stretch\sphinxhyphen{}bend terms
nstrbnd total number of stretch\sphinxhyphen{}bend interactions
isb     angle and bond numbers used in stretch\sphinxhyphen{}bend

STRTOR  stretch\sphinxhyphen{}torsions in the current structure

kst     1\sphinxhyphen{}, 2\sphinxhyphen{} and 3\sphinxhyphen{}fold stretch\sphinxhyphen{}torsion force constants
nstrtor total number of stretch\sphinxhyphen{}torsion interactions
ist     torsion and bond numbers used in stretch\sphinxhyphen{}torsion

SYNTRN  definition of synchronous transit path

t       value of the path coordinate (0=reactant, 1=product)
pm      path coordinate for extra point in quadratic transit
xmin1   reactant coordinates as array of optimization variables
xmin2   product coordinates as array of optimization variables
xm      extra coordinate set for quadratic synchronous transit

TITLES  title for the current molecular system

ltitle  length in characters of the nonblank title string
title   title used to describe the current structure

TORPOT  specifics of torsional functional forms

idihunit        convert improper dihedral energy to kcal/mole
itorunit        convert improper torsion amplitudes to kcal/mole
torsunit        convert torsional parameter amplitudes to kcal/mole
ptorunit        convert pi\sphinxhyphen{}orbital torsion energy to kcal/mole
storunit        convert stretch\sphinxhyphen{}torsion energy to kcal/mole
ttorunit        convert stretch\sphinxhyphen{}torsion energy to kcal/mole

TORS    torsional angles within the current structure

tors1   1\sphinxhyphen{}fold amplitude and phase for each torsional angle
tors2   2\sphinxhyphen{}fold amplitude and phase for each torsional angle
tors3   3\sphinxhyphen{}fold amplitude and phase for each torsional angle
tors4   4\sphinxhyphen{}fold amplitude and phase for each torsional angle
tors5   5\sphinxhyphen{}fold amplitude and phase for each torsional angle
tors6   6\sphinxhyphen{}fold amplitude and phase for each torsional angle
ntors   total number of torsional angles in the system
itors   numbers of the atoms in each torsional angle

TORTOR  torsion\sphinxhyphen{}torsions in the current structure

ntortor total number of torsion\sphinxhyphen{}torsion interactions
itt     atoms and parameter indices for torsion\sphinxhyphen{}torsion

TREE    potential smoothing \& search tree levels

maxpss  maximum number of potential smoothing levels
etree   energy reference value at the top of the tree
ilevel  smoothing deformation value at each tree level
nlevel  number of levels of potential smoothing used

UNITS   physical constants and unit conversions

avogadro        Avogadro’s number (N) in particles/mole
boltzmann       Boltzmann constant (kB) in g*Ang**2/ps**2/K/mole
gasconst        ideal gas constant (R) in kcal/mole/K
lightspd        speed of light in vacuum (c) in cm/ps
bohr    conversion from Bohrs to Angstroms
joule   conversion from calories to joules
evolt   conversion from Hartree to electron\sphinxhyphen{}volts
hartree conversion from Hartree to kcal/mole
electric        conversion from electron**2/Ang to kcal/mole
debye   conversion from electron\sphinxhyphen{}Ang to Debyes
prescon conversion from kcal/mole/Ang**3 to Atm
convert conversion from kcal to g*Ang**2/ps**2

UREY    Urey\sphinxhyphen{}Bradley interactions in the structure

uk      Urey\sphinxhyphen{}Bradley force constants (kcal/mole/Ang**2)
ul      ideal 1\sphinxhyphen{}3 distance values in Angstroms
nurey   total number of Urey\sphinxhyphen{}Bradley terms in the system
iury    numbers of the atoms in each Urey\sphinxhyphen{}Bradley interaction

URYPOT  specifics of Urey\sphinxhyphen{}Bradley functional form

cury    cubic coefficient in Urey\sphinxhyphen{}Bradley potential
qury    quartic coefficient in Urey\sphinxhyphen{}Bradley potential
ureyunit        convert Urey\sphinxhyphen{}Bradley energy to kcal/mole

USAGE   atoms active during energy computation

nuse    number of active atoms used in energy calculation
use     true if an atom is active, false if inactive

VDW     van der Waals parameters for current structure

radmin  minimum energy distance for each atom class pair
epsilon well depth parameter for each atom class pair
radmin4 minimum energy distance for 1\sphinxhyphen{}4 interaction pairs
epsilon4        well depth parameter for 1\sphinxhyphen{}4 interaction pairs
radhbnd minimum energy distance for hydrogen bonding pairs
epshbnd well depth parameter for hydrogen bonding pairs
kred    value of reduction factor parameter for each atom
ired    attached atom from which reduction factor is applied
nvdw    total number van der Waals active sites in the system
ivdw    number of the atom for each van der Waals active site

VDWPOT  specifics of van der Waals functional form

abuck   value of “A” constant in Buckingham vdw potential
bbuck   value of “B” constant in Buckingham vdw potential
cbuck   value of “C” constant in Buckingham vdw potential
ghal    value of “gamma” in buffered 14\sphinxhyphen{}7 vdw potential
dhal    value of “delta” in buffered 14\sphinxhyphen{}7 vdw potential
v2scale factor by which 1\sphinxhyphen{}2 vdw interactions are scaled
v3scale factor by which 1\sphinxhyphen{}3 vdw interactions are scaled
v4scale factor by which 1\sphinxhyphen{}4 vdw interactions are scaled
v5scale factor by which 1\sphinxhyphen{}5 vdw interactions are scaled
igauss  coefficients of Gaussian fit to vdw potential
ngauss  number of Gaussians used in fit to vdw potential
vdwtyp  type of van der Waals potential energy function
radtyp  type of parameter (sigma or R\sphinxhyphen{}min) for atomic size
radsiz  atomic size provided as radius or diameter
radrule combining rule for atomic size parameters
epsrule combining rule for vdw well depth parameters
gausstyp        type of Gaussian fit to van der Waals potential

VIRIAL  components of internal virial tensor

vir     total internal virial Cartesian tensor components

WARP    parameters for potential surface smoothing

m2      second moment of the GDA gaussian for each atom
deform  value of the smoothing deformation parameter
difft   diffusion coefficient for torsional potential
diffv   diffusion coefficient for van der Waals potential
diffc   diffusion coefficient for charge\sphinxhyphen{}charge potential
use\_smooth      flag to use a potential energy smoothing method
use\_dem flag to use diffusion equation method potential
use\_gda flag to use gaussian density annealing potential
use\_tophat      flag to use analytical tophat smoothed potential
use\_stophat     flag to use shifted tophat smoothed potential

XTALS   crystal structures for parameter fitting

e0\_lattice      ideal lattice energy for the current crystal
moment\_0        ideal dipole moment for monomer from crystal
nxtal   number of crystal structures to be stored
nvary   number of potential parameters to optimize
ivary   index for the types of potential parameters
vary    atom numbers involved in potential parameters
iresid  crystal structure to which each residual refers
rsdtyp  experimental variable for each of the residuals
vartyp  type of potential parameter to be optimized

ZCLOSE  ring openings and closures for Z\sphinxhyphen{}matrix

nadd    number of added bonds between Z\sphinxhyphen{}matrix atoms
iadd    numbers of the atom pairs defining added bonds
ndel    number of bonds between Z\sphinxhyphen{}matrix bonds to delete
idel    numbers of the atom pairs defining deleted bonds

ZCOORD  Z\sphinxhyphen{}matrix internal coordinate definitions

zbond   bond length used to define each Z\sphinxhyphen{}matrix atom
zang    bond angle used to define each Z\sphinxhyphen{}matrix atom
ztors   angle or torsion used to define Z\sphinxhyphen{}matrix atom
iz      defining atom numbers for each Z\sphinxhyphen{}matrix atom


\chapter{Test Cases \& Examples}
\label{\detokenize{text/test-cases:test-cases-examples}}\label{\detokenize{text/test-cases::doc}}
This section contains brief descriptions of the sample calculations found in the EXAMPLE subdirectory of the Tinker distribution. These examples exercise several of the current Tinker programs and are intended to provide a flavor of the capabilities of the package.

\sphinxstylestrong{ANION Test}

Computes an estimation of the free energy of hydration of Cl\sphinxhyphen{} anion vs. Br\sphinxhyphen{} anion via a 2 picosecond simulation on a “hybrid” anion in a box of water followed by a free energy perturbation calculation.

\sphinxstylestrong{ARGON Test}

Performs an initial energy minimization on a periodic box containing 150 argon atoms followed by 6 picoseconds of a molecular dynamics using a modified Beeman integration algorithm and a Bersedsen thermostat.

\sphinxstylestrong{CLUSTER Test}

Performs a set of 10 Gaussian density annealing (GDA) trials on a cluster of 13 argon atoms in an attempt to locate the global minimum energy structure.

\sphinxstylestrong{CRAMBIN Test}

Generates a Tinker file from a PDB file, followed by a single point energy computation and determination of the molecular volume and surface area.

\sphinxstylestrong{CYCLOHEX Test}

First approximately locates the transition state between chair and boat cyclohexane, followed by subsequent refinement of the transition state and a final vibrational analysis to show that a single negative frequency is associated with the saddle point.

\sphinxstylestrong{DHFR Test}

Performs 10 steps of molecular dynamics on a pre\sphinxhyphen{}equilibrated system of DHFR protein in a box or water using the AMOEBA force field. Note this test case is the so\sphinxhyphen{}called Joint Amber\sphinxhyphen{}CHARMM “JAC” benchmark containing 23558 total atoms.

\sphinxstylestrong{DIALANINE Test}

Finds all the local minima of alanine dipeptide via a potential energy surface scan using torsional modes to jump between the minima.

\sphinxstylestrong{ENKEPHALIN Test}

Produces coordinates from the met\sphinxhyphen{}enkephalin amino acid sequence and phi/psi angles, followed by truncated Newton energy minimization and determination of the lowest frequency normal mode.

\sphinxstylestrong{ETHANOL Test}

Performs fitting of torsional parameter values for the ethanol C\sphinxhyphen{}C\sphinxhyphen{}O\sphinxhyphen{}H bond based on relative quantum mechanical (G09) energies for rotating the C\sphinxhyphen{}O bond.

\sphinxstylestrong{FORMAMIDE Test}

Generates a unit cell from fractional coordinates, followed by full crystal energy minimization and determination of optimal carbonyl oxygen energy parameters from a fit to lattice energy and structure.

\sphinxstylestrong{GPCR Test}

Finds the lowest\sphinxhyphen{}frequency normal mode of bacteriorhodopsin using vibrational analysis via a sliding block iterative matrix diagonalization. Alter the gpcr.run script to save the file gpcr.001 for later viewing of the mode.

\sphinxstylestrong{HELIX Test}

Performs a rigid\sphinxhyphen{}body optimization of the packing of two idealized polyalanine helices using only van der Waals interactions.

\sphinxstylestrong{ICE Test}

Performs a short MD simulation of the monoclinic ice V crystal form using the iAMOEBA water model, pairwise neighbor lists and PME electrostatics.

\sphinxstylestrong{IFABP Test}

Generates three distance geometry structures for intestinal fatty acid binding protein from a set of NOE distance restraints and torsional restraints.

\sphinxstylestrong{METHANOL Test}

Processes distributed multipole analysis (DMA) output to extract coordinates and permanent multipoles, set local frames and polarization groups, remove intramolecular polarization, detect and average equivalent atomic sites.

\sphinxstylestrong{NITROGEN Test}

Calculates the self\sphinxhyphen{}diffusion constant and the N\sphinxhyphen{}N radial distribution function for liquid nitrogen via analysis of a 50ps MD trajectory.

\sphinxstylestrong{SALT Test}

Converts a sodium chloride assymetric unit to the corresponding unit cell, then runs a crystal minimization starting from the initial diffraction structure using Ewald summation to model the long\sphinxhyphen{}range electrostatic interactions.

\sphinxstylestrong{TETRAALA Test}

Generates capped alanine tetrapeptide in an extended conformation, then use Monte Carlo Minimization with random torsional moves to find the global minimum energy structure.

\sphinxstylestrong{WATER Test}

Fits the electrostatic potential around an AMOEBA water molecule to the QM\sphinxhyphen{}derived potential (MP2/aug\sphinxhyphen{}cc\sphinxhyphen{}pVTZ) on a grid of points outside the molecular surface.


\chapter{Benchmark Results}
\label{\detokenize{text/benchmarks:benchmark-results}}\label{\detokenize{text/benchmarks::doc}}
The tables in this section provide CPU benchmarks for basic Tinker energy and derivative evaluations, vibrational analysis and molecular dynamics. All times are in seconds and were measured with Tinker executables dimensioned to maxatm of 10000 and maxhess of 1000000 in the source file sizes.i. All calculations were run twice in rapid succession on a quiet machine. The times reported for each benchmark are the results from the second run. If you have built Tinker on an alternative machine type and are able to run the benchmarks on the additional machine type, please send the results for inclusion in a future listing.

BENCHMARK \#1:  Calmodulin Energy Evaluation

The system is an isolated molecule of the 148\sphinxhyphen{}residue protein calmodulin with 2264 atoms using the Amber ff94 force field. All interactions are computed with no use of cutoffs. Times listed are for calculation setup followed by a single energy, energy/gradient and Hessian evaluation.

MACHINE\sphinxhyphen{}OS\sphinxhyphen{}COMPILER TYPE        MHz     SETUP   ENERGY  GRAD    HESS
\begin{quote}

Athlon XP 2400+ (RH 8.0, Intel)        2000    0.13    0.28    0.60    2.96
Athlon XP 2400+ (RH 8.0, PGI)  2000    0.16    0.31    0.70    3.60
Athlon XP 2400+ (RH 8.0, g77 3.2)      2000    0.17    0.28    0.66    3.67
Athlon Thunderbird (RH 8.0, Intel)     1400    0.22    0.41    0.86    5.15
Athlon Thunderbird (RH 8.0, PGI)       1400    0.21    0.44    1.00    5.92
Athlon Thunderbird (RH 8.0, g77 3.2)   1400    0.19    0.40    0.94    5.81
Athlon Classic (RH 8.0, Intel) 950     0.30    0.64    1.42    7.07
Athlon Classic (RH 8.0, PGI)   950     0.30    0.69    1.65    7.96
Athlon Classic (RH 8.0, g77 3.2)       950     0.31    0.63    1.57    7.94
Compaq Evo N610c P4 (RH 8.0, Intel)    2000    0.18    0.45    0.87    3.08
Compaq Evo N610c P4 (RH 8.0, PGI)      2000    0.22    0.44    1.06    4.27
Compaq Evo N610c P4 (RH 8.0, Absoft)   2000    0.17    0.52    1.06    3.95
Compaq Evo N610c P4 (RH 8.0, g77 3.2)  2000    0.19    0.41    1.07    4.41
Compaq Evo N610c P4 (WinXP, CVF 6.6)   2000    0.16    0.38    0.98    3.54
Compaq Evo N610c P4 (WinXP, g77 3.2)   2000    0.16    0.40    1.08    4.45
Apple Power Mac G4 (OSX 10.2, Absoft)  733     0.41    2.96    5.12    17.83
Apple Power Mac G4 (OSX 10.2, g77 3.3) 733     0.37    1.98    3.79    14.48
Compaq AlphaServer DS10 (Tru64 5.0)    466     0.35    1.33    1.93    8.40
SGI IndigoII R10K (Irix 6.5, MIPS)     195     1.17    3.49    6.35    23.03
\end{quote}

BENCHMARK \#2:  Crambin Crystal Energy Evaluation

The system is a unit cell of the 46\sphinxhyphen{}residue protein crambin containing 2 polypeptide chains, 2 ethanol and 178 water molecules for a total of 1360 atoms using the OPLS\sphinxhyphen{}UA force field. Periodic boundaries are used with particle mesh Ewald for electrostatics and a 9.0 Angstrom cutoff for vdW interactions. Times listed are for calculation setup followed by a single energy, energy/ gradient and Hessian evaluation.

MACHINE\sphinxhyphen{}OS\sphinxhyphen{}COMPILER TYPE        MHz     SETUP   ENERGY  GRAD    HESS
\begin{quote}

Athlon XP 2400+ (RH 8.0, Intel)        2000    0.12    0.12    0.21    0.66
Athlon XP 2400+ (RH 8.0, PGI)  2000    0.13    0.14    0.24    0.63
Athlon XP 2400+ (RH 8.0, g77 3.2)      2000    0.14    0.13    0.28    0.81
Athlon Thunderbird (RH 8.0, Intel)     1400    0.19    0.17    0.30    0.91
Athlon Thunderbird (RH 8.0, PGI)       1400    0.18    0.18    0.32    0.91
Athlon Thunderbird (RH 8.0, g77 3.2)   1400    0.17    0.17    0.38    1.11
Athlon Classic (RH 8.0, Intel) 950     0.26    0.25    0.47    1.46
Athlon Classic (RH 8.0, PGI)   950     0.29    0.27    0.50    1.42
Athlon Classic (RH 8.0, g77 3.2)       950     0.27    0.27    0.56    1.70
Compaq Evo N610c P4 (RH 8.0, Intel)    2000    0.15    0.14    0.27    0.64
Compaq Evo N610c P4 (RH 8.0, PGI)      2000    0.22    0.19    0.33    0.88
Compaq Evo N610c P4 (RH 8.0, Absoft)   2000    0.14    0.22    0.39    0.84
Compaq Evo N610c P4 (RH 8.0, g77 3.2)  2000    0.15    0.20    0.45    1.13
Compaq Evo N610c P4 (WinXP, CVF 6.6)   2000    0.14    0.17    0.33    0.83
Compaq Evo N610c P4 (WinXP, g77 3.2)   2000    0.12    0.22    0.52    1.16
Apple Power Mac G4 (OSX 10.2, Absoft)  733     0.32    0.58    1.09    3.11
Apple Power Mac G4 (OSX 10.2, g77 3.3) 733     0.31    0.42    0.79    2.37
Compaq AlphaServer DS10 (Tru64 5.0)    466     0.29    0.38    0.64    1.95
SGI IndigoII R10K (Irix 6.5, MIPS)     195     0.92    0.74    1.41    3.89
\end{quote}

BENCHMARK \#3:  Peptide Normal Mode Calculation

The system is a minimum energy conformation of a 20\sphinxhyphen{}residue peptide containing one of each of the standard amino acids for a total of 328 atoms using the OPLS\sphinxhyphen{}AA force field without cutoffs. The time reported is for computation of the Hessian and calculation of the normal modes of the Hessian matrix and the vibration frequencies requiring two separate matrix diagonalization steps.

MACHINE\sphinxhyphen{}OS\sphinxhyphen{}COMPILER TYPE        MHz     NORMAL MODES
\begin{quote}

Athlon XP 2400+ (RH 8.0, Intel)        2000    22
Athlon XP 2400+ (RH 8.0, PGI)  2000    26
Athlon XP 2400+ (RH 8.0, g77 3.2)      2000    24
Athlon Thunderbird (RH 8.0, Intel)     1400    31
Athlon Thunderbird (RH 8.0, PGI)       1400    34
Athlon Thunderbird (RH 8.0, g77 3.2)   1400    33
Athlon Classic (RH 8.0, Intel) 950     46
Athlon Classic (RH 8.0, PGI)   950     51
Athlon Classic (RH 8.0, g77 3.2)       950     48
Compaq Evo N610c P4 (RH 8.0, Intel)    2000    19
Compaq Evo N610c P4 (RH 8.0, PGI)      2000    19
Compaq Evo N610c P4 (RH 8.0, Absoft)   2000    20
Compaq Evo N610c P4 (RH 8.0, g77 3.2)  2000    19
Compaq Evo N610c P4 (WinXP, CVF 6.6)   2000    19
Compaq Evo N610c P4 (WinXP, g77 3.2)   2000    20
Apple Power Mac G4 (OSX 10.2, Absoft)  733     67
Apple Power Mac G4 (OSX 10.2, g77 3.3) 733     62
Compaq AlphaServer DS10 (Tru64 5.0)    466     39
SGI IndigoII R10K (Irix 6.5, MIPS)     195     144
\end{quote}

BENCHMARK \#4:  TIP3P Water Box Molecular Dynamics

The system consists of 216 rigid TIP3P water molecules in a 18.643 Angstrom periodic box, 9.0 Angstrom shifted energy switch cutoffs for nonbonded interactions. The time reported is for 1000 dynamics steps of 1.0 fs each using the modified Beeman integrator and Rattle constraints on all bond lengths.

MACHINE\sphinxhyphen{}OS\sphinxhyphen{}COMPILER TYPE        MHz     DYNAMICS
\begin{quote}

Athlon XP 2400+ (RH 8.0, Intel)        2000    37
Athlon XP 2400+ (RH 8.0, PGI)  2000    34
Athlon XP 2400+ (RH 8.0, g77 3.2)      2000    45
Athlon Thunderbird (RH 8.0, Intel)     1400    52
Athlon Thunderbird (RH 8.0, PGI)       1400    47
Athlon Thunderbird (RH 8.0, g77 3.2)   1400    63
Athlon Classic (RH 8.0, Intel) 950     77
Athlon Classic (RH 8.0, PGI)   950     71
Athlon Classic (RH 8.0, g77 3.2)       950     96
Compaq Evo N610c P4 (RH 8.0, Intel)    2000    53
Compaq Evo N610c P4 (RH 8.0, PGI)      2000    54
Compaq Evo N610c P4 (RH 8.0, Absoft)   2000    55
Compaq Evo N610c P4 (RH 8.0, g77 3.2)  2000    91
Compaq Evo N610c P4 (WinXP, CVF 6.6)   2000    63
Compaq Evo N610c P4 (WinXP, g77 3.2)   2000    94
Apple Power Mac G4 (OSX 10.2, Absoft)  733     209
Apple Power Mac G4 (OSX 10.2, g77 3.3) 733     170
Compaq AlphaServer DS10 (Tru64 5.0)    466     106
SGI IndigoII R10K (Irix 6.5, MIPS)     195     280
\end{quote}

BENCHMARK \#5:  Tinker Water Box Molecular Dynamics

The system consists of 216 AMOEBA flexible polarizable atomic multipole water molecules in a 18.643 Angstrom periodic box using regular Ewald summation for the electrostatics and a 12.0 Angstrom switched cutoff for vdW interactions. The time reported is for 100 dynamics steps of 1.0 fs each using the modified Beeman integrator and 0.01 Debye rms convergence for induced dipole moments.

MACHINE\sphinxhyphen{}OS\sphinxhyphen{}COMPILER TYPE        MHz     DYNAMICS
\begin{quote}

Athlon XP 2400+ (RH 8.0, Intel)        2000    108
Athlon XP 2400+ (RH 8.0, PGI)  2000    104
Athlon XP 2400+ (RH 8.0, g77 3.2)      2000    128
Athlon Thunderbird (RH 8.0, Intel)     1400    165
Athlon Thunderbird (RH 8.0, PGI)       1400    158
Athlon Thunderbird (RH 8.0, g77 3.2)   1400    183
Athlon Classic (RH 8.0, Intel) 950     282
Athlon Classic (RH 8.0, PGI)   950     261
Athlon Classic (RH 8.0, g77 3.2)       950     307
Compaq Evo N610c P4 (RH 8.0, Intel)    2000    156
Compaq Evo N610c P4 (RH 8.0, PGI)      2000    191
Compaq Evo N610c P4 (RH 8.0, Absoft)   2000    226
Compaq Evo N610c P4 (RH 8.0, g77 3.2)  2000    243
Compaq Evo N610c P4 (WinXP, CVF 6.6)   2000    176
Compaq Evo N610c P4 (WinXP, g77 3.2)   2000    263
Apple Power Mac G4 (OSX 10.2, Absoft)  733     680
Apple Power Mac G4 (OSX 10.2, g77 3.3) 733     479
Compaq AlphaServer DS10 (Tru64 5.0)    466     358
SGI IndigoII R10K (Irix 6.5, MIPS)     195     868
\end{quote}


\chapter{Acknowledgments}
\label{\detokenize{text/acknowledgements:acknowledgments}}\label{\detokenize{text/acknowledgements::doc}}
The TINKER package has developed over a period of many years, very slowly during the late\sphinxhyphen{}1980s, and more rapidly since the mid\sphinxhyphen{}1990s in Jay Ponder’s research group at the Washington University School of Medicine in Saint Louis. Many people have played significant roles in the development of the package into its current form. The major contributors are listed below:
\begin{description}
\item[{Stew Rubenstein coordinate interconversions; original optimization methods}] \leavevmode
and torsional angle manipulation

\end{description}

Craig Kundrot   molecular surface area \& volume and their derivatives
\begin{description}
\item[{Shawn Huston    original AMBER/OPLS implementation; free energy}] \leavevmode
calculations; time correlation functions

\end{description}

Mike Dudek      initial multipole models for peptides and proteins
\begin{description}
\item[{Yong “Mike” Kong        multipole electrostatics; dipole polarization; reaction field}] \leavevmode
treatment; TINKER water model

\item[{Reece Hart      potential smoothing methodology; Scheraga’s DEM,}] \leavevmode
Straub’s GDA and extensions

\item[{Mike Hodsdon    extension of the TINKER distgeom program and its}] \leavevmode
application to NMR NOE structure determination

\item[{Rohit Pappu     potential smoothing methodology and PSS algorithms;}] \leavevmode
rigid body optimization; GB/SA solvation derivatives

\item[{Wijnand Mooij   MM3 directional hydrogen bonding term; crystal lattice}] \leavevmode
minimization code

\end{description}

Gerald Loeffler stochastic/Langevin dynamics implementation

Marina Vorobieva        nucleic acid building module and parameter translation
Nina Sokolova

Peter Bagossi   AMOEBA force field parameters for alkanes and diatomics
\begin{description}
\item[{Pengyu Ren      Ewald summation for polarizable atomic multipoles;}] \leavevmode
AMOEBA force field for water, organics and peptides

\end{description}

Anders Carlsson original ligand field potential energy term for transition metals

Andrey Kutepov  integrator for rigid\sphinxhyphen{}body dynamics trajectories
\begin{description}
\item[{Tom Darden      Particle Mesh Ewald (PME) code, and development of PME}] \leavevmode
for the AMOEBA force field

\end{description}

Alan Grossfield Monte Carlo minimization; tophat potential smoothing
\begin{description}
\item[{Michael Schnieders      Force Field Explorer GUI for TINKER; neighbor lists for}] \leavevmode
nonbonded interactions

\item[{Chuanjie Wu     solvation free energy calculations; AMOEBA nucleic acid force}] \leavevmode
field; parameterization tools for TINKER

\item[{Justin Xiang    angular overlap and valence bond potential models for}] \leavevmode
transition metals

\item[{David Gohara    OpenMP parallelization of energy terms including PME,}] \leavevmode
and parallel neighbor lists

\end{description}

It is critically important that TINKER’s distributed force field parameter sets exactly reproduce the intent of the original force field authors. We would like to thank Julian Tirado\sphinxhyphen{}Rives (OPLS\sphinxhyphen{}AA), Alex MacKerell (CHARMM27), Wilfred van Gunsteren (GROMOS), and Adrian Roitberg and Carlos Simmerling (AMBER) for their help in testing TINKER’s results against those given by the authentic programs and parameter sets. Lou Allinger provided updated parameters for MM2 and MM3 on several occasions. His very successful methods provided the original inspiration for the development of TINKER.

Still other workers have devoted considerable time in developing code that will hopefully be incorporated into future TINKER versions; for example, Jim Kress (UFF implementation) and Michael Sheets (numerous code optimizations, thermodynamic integration). Finally, we wish to thank the many users of the TINKER package for their suggestions and comments, praise and criticism, which have resulted in a variety of improvements.


\chapter{References}
\label{\detokenize{text/references:references}}\label{\detokenize{text/references::doc}}
This section contains a list of the references to general theory, algorithms and implementation details which have been of use during the development of the TINKER package. Methods described in some of the references have been implemented in detail within the TINKER source code. Other references contain useful background information although the algorithms themselves are now obsolete. Still other papers contain ideas or extensions planned for future inclusion in TINKER. References for specific force field parameter sets are provided in an earlier section of this User’s Guide. This list is heavily skewed toward biomolecules in general and proteins in particular. This bias reflects our group’s major interests; however an attempt has been made to include methods which should be generally applicable.


\section{Partial List of Molecular Mechanics Software Packages}
\label{\detokenize{text/references:partial-list-of-molecular-mechanics-software-packages}}
AMBER   Peter Kollman, University of California, San Francisco
AMMP    Rob Harrison, Thomas Jefferson University, Philadelphia
ARGOS   Andy McCammon, University of California, San Diego
BOSS    William Jorgensen, Yale University
BRUGEL  Shoshona Wodak, Free University of Brussels
CFF     Shneior Lifson, Weizmann Institute
CHARMM  Martin Karplus, Harvard University
CHARMM/GEMM     Bernard Brooks, National Institutes of Health, Bethesda
DELPHI  Bastian van de Graaf, Delft University of Technology
DISCOVER        Molecular Simulations Inc., San Diego
DL\_POLY W. Smith \& T. Forester, CCP5, Daresbury Laboratory
ECEPP   Harold Scheraga, Cornell University
ENCAD   Michael Levitt, Stanford University
FANTOM  Werner Braun, University of Texas, Galveston
FEDER/2 Nobuhiro Go, Kyoto University
GROMACS Herman Berendsen, University of Groningen
GROMOS  Wilfred van Gunsteren, BIOMOS and ETH, Zurich
IMPACT  Ronald Levy, Rutgers University
MACROMODEL      Schodinger, Inc., Jersey City, New Jersey
MM2/MM3/MM4     N. Lou Allinger, University of Georgia
MMC     Cliff Dykstra, Indiana University\sphinxhyphen{}Purdue University at Indianapolis
MMFF    Tom Halgren, Merck Research Laboratories, Rahway
MMTK    Konrad Hinsen, Inst. of Structural Biology, Grenoble
MOIL    Ron Elber, Cornell University
MOLARIS Arieh Warshal, University of Southern California
MOLDY   Keith Refson, Oxford University
MOSCITO Dietmar Paschek \& Alfons Geiger, Universit‰t Dortmund
NAMD    Klaus Schulten, University of Illinois, Urbana
OOMPAA  Andy McCammon, University of California, San Diego
ORAL    Karel Zimmerman, INRA, Jouy\sphinxhyphen{}en\sphinxhyphen{}Josas, France
ORIENT  Anthony Stone, Cambridge University
PCMODEL Kevin Gilbert, Serena Software, Bloomington, Indiana
PEFF    Jan Dillen, University of Pretoria, South Africa
Q       Johan Aqvist, Uppsala University
SIBFA   Nohad Gresh, INSERM, CNRS, Paris
SIGMA   Jan Hermans, University of North Carolina
SPASIBA Gerard Vergoten, UniversitÈ de Lille
SPASMS  David Spellmeyer and the Kollman Group, UCSF
TINKER  Jay Ponder, Washington University, St. Louis
XPLOR/CNS       Axel Br¸nger, Stanford University
YAMMP   Stephen Harvey, University of Alabama, Birmingham
YASP    Florian Mueller\sphinxhyphen{}Plathe, ETH Zentrum, Zurich
YETI    Angelo Vedani, Biografik\sphinxhyphen{}Labor 3R, Basel

AMBER     D. A Pearlman, D. A. Case, J. W. Caldwell, W. S. Ross, T. E. Cheatham III, S. DeBolt, D. Ferguson, G. Seibel and P. Kollman, AMBER, a Package of Computer Programs for Applying Molecular Mechanics, Normal Mode Analysis, Molecular Dynamics and Free Energy Calculations to Simulate the Structural and Energetic Properties of Molecules, Comp. Phys. Commun., 91, 1\sphinxhyphen{}41 (1995)

ARGOS     T. P. Straatsma and J. A. McCammon, ARGOS, a Vectorized General Molecular Dynamics Program, J. Comput. Chem., 11, 943\sphinxhyphen{}951 (1990)

CHARMM     B. R. Brooks, R. E. Bruccoleri, B. D. Olafson, D. J. States, S. Swaminathan and M. Karplus, CHARMM: A Program for Macromolecular Energy, Minimization, and Dynamics Calculations, J. Comput. Chem., 4, 187\sphinxhyphen{}217 (1983)

ENCAD     M. Levitt, M. Hirshberg, R. Sharon and V. Daggett, Potential Energy Function and Parameters for Simulations for the Molecular Dynamics of Proteins and Nucleic Acids in Solution, Comp. Phys. Commun., 91, 215\sphinxhyphen{}231 (1995)

FANTOM     T. Schaumann, W. Braun and K. Wurtrich, The Program FANTOM for Energy Refinement of Polypeptides and Proteins Using a Newton\sphinxhyphen{}Raphson Minimizer in Torsion Angle Space, Biopolymers, 29, 679\sphinxhyphen{}694 (1990)

FEDER/2     H. Wako, S. Endo, K. Nagayama and N. Go, FEDER/2: Program for Static and Dynamic Conformational Energy Analysis of Macro\sphinxhyphen{}molecules in Dihedral Angle Space, Comp. Phys. Commun., 91, 233\sphinxhyphen{}251 (1995)

GROMACS     E. Lindahl, B. Hess and D. van der Spoel, GROMACS 3.0: A Package for Molecular Simulation and Trajectory Analysis, J. Mol. Mol., 7, 306\sphinxhyphen{}317 (2001)

GROMOS     W. R. P. Scott, P. H. Hunenberger , I. G. Tironi, A. E. Mark, S. R. Billeter, J. Fennen, A. E. Torda, T. Huber, P. Kruger, W. F. van Gunsteren, The GROMOS Biomolecular Simulation Program Package, J. Phys. Chem. A, 103, 3596\sphinxhyphen{}3607 (1999)

IMPACT     D. B. Kitchen, F. Hirata, J. D. Westbrook, R. Levy, D. Kofke and M. Yarmush, Conserving Energy during Molecular Dynamics Simulations of Water, Proteins, and Proteins in Water, J. Comput. Chem., 10, 1169\sphinxhyphen{}1180 (1990)

MACROMODEL     F. Mahamadi, N. G. J. Richards, W. C. Guida, R. Liskamp, M. Lipton, C. Caufield, G. Chang, T. Hendrickson and W. C. Still, MacroModel: An Integrated Software System for Modeling Organic and Bioorganic Molecules Using Molecular Mechanics, J. Comput. Chem., 11, 440\sphinxhyphen{}467 (1990)

MM2     N. L. Allinger, Conformational Analysis. 130. MM2. A Hydrocarbon Force Field Utilizing V1 and V2 Torsional Terms, J. Am. Chem. Soc., 99, 8127\sphinxhyphen{}8134 (1977)

MM3     N. L. Allinger, Y. H. Yuh and J.\sphinxhyphen{}H. Lii, Molecular Mechanics. The MM3 Force Field for Hydrocarbons, J. Am. Chem. Soc., 111, 8551\sphinxhyphen{}8566 (1989)

MM4     N. L. Allinger, K. Chen and J.\sphinxhyphen{}H. Lii, An Improved Force Field (MM4) for Saturated Hydrocarbons, J. Comput. Chem., 17, 642\sphinxhyphen{}668 (1996)

MMC     C. E. Dykstra, Molecular Mechanics for Weakly Interacting Assemblies of Rare Gas Atoms and Small Molecules, J. Am. Chem. Soc., 111, 6168\sphinxhyphen{}6174 (1989)

MMFF     T. A. Halgren, Merck Molecular Force Field. I. Basis, Form, Scope, Parameterization, and Performance of MMFF94, J. Comput. Chem., 17, 490\sphinxhyphen{}516 (1996)

MOIL     R. Elber, A. Roitberg, C. Simmerling, R. Goldstein, H. Li, G. Verkhiver, C. Keasar, J. Zhang and A. Ulitsky, MOIL: A Program for Simulations of Macromolecules, Comp. Phys. Commun., 91, 159\sphinxhyphen{}189 (1995)

MOSCITO     See the web site at \sphinxurl{http:/ganter.chemie.uni-dortmund.de/~pas/moscito.html}

NAMD     L. KalÈ, R. Skeel, M. Bhandarkar, R. Brunner, A. Gursoy, N. Krawetz, J. Phillips, A. Shinozaki, K. Varadarajan and K. Schulten, NAMD2: Greater Scalability for Parallel Molecular Dynamics, J. Comput. Phys., 151, 283\sphinxhyphen{}312 (1999)

OOMPAA     G. A. Huber and J. A. McCammon, OOMPAA: Object\sphinxhyphen{}oriented Model for Probing Assemblages of Atoms, J. Comput. Phys., 151, 264\sphinxhyphen{}282 (1999)

ORAL     K. Zimmermann, ORAL: All Purpose Molecular Mechanics Simulator and Energy Minimizer, J. Comput. Chem., 12, 310\sphinxhyphen{}319 (1991)

PCMODEL     See the web site at \sphinxurl{http:/www.serenasoft.com}

PEFF     J. L. M. Dillen, PEFF: A Program for the Development of Empirical Force Fields, J. Comput. Chem., 13, 257\sphinxhyphen{}267 (1992)

Q     See the web site at \sphinxurl{http://aqvist.bmc.uu.se/Q}

SIBFA     N. Gresh, Inter\sphinxhyphen{} and Intramolecular Interactions. Inception and Refinements of the SIBFA, Molecular Mechanics (SMM) Procedure, a Separable, Polarizable Methodology Grounded on ab Initio SCF/MP2 Computations. Examples of Applications to Molecular Recognition Problems, J. Chim. Phys. PCB, 94, 1365\sphinxhyphen{}1416 (1997)

SIGMA     See the web site at \sphinxurl{http://femto.med.unc.edu/SIGMA}

SPASIBA     P. Derreumaux and G. Vergoten, A New Spectroscopic Molecular Mechanics Force\sphinxhyphen{}Field \sphinxhyphen{} Parameters For Proteins, J. Chem. Phys., 102, 8586\sphinxhyphen{}8605 (1995)

TINKER     See the web site at \sphinxurl{http://dasher.wustl.edu/tinker}

YAMMP     R. K.\sphinxhyphen{}Z. Tan and S. C. Harvey, Yammp: Development of a Molecular Mechanics Program Using the Modular Programming Method, J. Comput. Chem., 14, 455\sphinxhyphen{}470 (1993)

YETI     A. Vedani, YETI: An Interactive Molecular Mechanics Program for Small\sphinxhyphen{}Molecule Protein Complexes, J. Comput. Chem., 9, 269\sphinxhyphen{}280 (1988)


\section{Molecular Mechanics}
\label{\detokenize{text/references:molecular-mechanics}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{20}
\item {} 
Burkert and N. L. Allinger, Molecular Mechanics, American Chemical Society, Washington, D.C., 1982

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Comba and T. W. Hambley, Molecular Modeling of Inorganic Compounds, 2nd Ed., Wiley\sphinxhyphen{}VCH, New York, 2001

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} 
Machida, Principles of Molecular Mechanics, Kodansha/John Wiley \& Sons, Tokyo/New York, 1999

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{10}
\item {} 
RappÈ and C. J. Casewit, Molecular Mechanics across Chemistry, University Science Books, Sausalito, CA, 1997

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} 
Rasmussen, Potential Energy Functions in Conformational Analysis (Lecture Notes in Chemistry, Vol. 27), Springer\sphinxhyphen{}Verlag, Berlin, 1985

\end{enumerate}


\section{Computer Simulation Methods}
\label{\detokenize{text/references:computer-simulation-methods}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{15}
\item {} 
Allen and D. J. Tildesley, Computer Simulation of Liquids, Oxford University Press, Oxford, 1987

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Cramer, Essentials of Computational Chemistry: Theories and Models, John Wiley and Sons, New York, 2002

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Field, A Practical Introduction to the Simulation of Molecular Systems, Cambridge Univ. Press, Cambridge, 1999

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} 
Frankel and B. Smit, Understanding Molecular Simulation: From Algorithms to Applications, 2nd Ed., Academic Press, San Diego, CA, 2001

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{12}
\item {} 
Haile, Molecular Dynamics Simulation: Elementary Methods, John Wiley and Sons, New York, 1992

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{5}
\item {} 
Jensen, Introduction to Computational Chemistry, John Wiley and Sons, New York, 1998

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{17}
\item {} 
Leach, Molecular Modelling: Principles and Applications, 2nd Ed., Addison Wesley Longman, Essex, England, 2001

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Rapaport, The Art of Molecular Dynamics Simulation, 2nd Ed., Cambridge University Press, Cambridge, 2004

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} 
Schlick, Molecular Modeling and Simulation, Springer\sphinxhyphen{}Verlag, New York, 2002

\end{enumerate}


\section{Modeling of Biological Macromolecules}
\label{\detokenize{text/references:modeling-of-biological-macromolecules}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{14}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{12}
\item {} 
Becker, A. D. MacKerell, Jr., B. Roux and M. Watanabe, Eds., Computational Biochemistry and Biophysics, Marcel Dekker, New York, 2001

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Brooks III, M. Karplus and B. M. Pettitt, Proteins: A Theoretical Perspective of Dynamics, Structure, and Thermodynamics, John Wiley and Sons, New York, 1988

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{21}
\item {} 
Daggett, Ed., Protein Simulations (Advances in Protein Chemistry, Vol. 66), Academic Press/Elsevier, New York, 2003

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
McCammon and S. Harvey, Dynamics of Proteins and Nucleic Acids, Cambridge University Press, Cambridge, 1987

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{5}
\item {} 
van Gunsteren, P. K. Weiner and A. J. Wilkinson, Computer Simulation of Biomolecular Systems, Vol. 1\sphinxhyphen{}3, Kluwer Academic Publishers, Dordrecht, 1989\sphinxhyphen{}1997

\end{enumerate}

\end{enumerate}


\section{Conjugate Gradient and Quasi\sphinxhyphen{}Newton Optimization}
\label{\detokenize{text/references:conjugate-gradient-and-quasi-newton-optimization}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Nocedal and S. J. Wright, Numerical Optimization, Springer\sphinxhyphen{}Verlag, New York, 1999

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{6}
\item {} 
Nash and A. Sofer, Linear and Nonlinear Programming, McGraw\sphinxhyphen{}Hill, New York, 1996

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} 
Fletcher, Practical Methods of Optimization, John Wiley \& Sons Ltd., Chichester, 1987

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{6}
\item {} 
Luenberger, Linear and Nonlinear Programming, 2nd Ed., Addison\sphinxhyphen{}Wesley, Reading, MA, 1984

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Gill, W. Murray and M. H. Wright, Practical Optimization, Academic Press, New York, 1981

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Nocedal, Updating Quasi\sphinxhyphen{}Newton Matrices with Limited Storage, Math. Comp., 773\sphinxhyphen{}782 (1980)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Watowich, E. S. Meyer, R. Hagstrom and R. Josephs, A Stable, Rapidly Converging Conjugate Gradient Method for Energy Minimization, J. Comput. Chem., 9, 650\sphinxhyphen{}661 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Davidon, Optimally Conditioned Optimization Algorithms without Line Searches, Math. Prog., 9, 1\sphinxhyphen{}30 (1975)

\end{enumerate}

\end{enumerate}


\section{Truncated Newton Optimization}
\label{\detokenize{text/references:truncated-newton-optimization}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{22}
\item {} 
Ponder and F. M. Richards, An Efficient Newton\sphinxhyphen{}like Method for Molecular Mechanics Energy Minimization of Large Molecules, J. Comput. Chem., 8, 1016\sphinxhyphen{}1024 (1987)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Dembo and T. Steihaug, Truncated\sphinxhyphen{}Newton Algorithms for Large\sphinxhyphen{}Scale Unconstrained Optimization, Math. Prog., 26, 190\sphinxhyphen{}212 (1983)

\end{enumerate}

\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Eisenstat and H. F. Walker, Choosing the Forcing Terms in an Inexact Newton Method, SIAM J. Sci. Comput., 17, 16\sphinxhyphen{}32 (1996)

\end{enumerate}

\item {} 
Schlick and M. Overton, A Powerful Truncated Newton Method for Potential Energy Minimization, J. Comput. Chem., 8, 1025\sphinxhyphen{}1039 (1987)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Kershaw, The Incomplete Cholesky\sphinxhyphen{}Conjugate Gradient Method for the Iterative Solution of Systems of Linear Equations, J. Comput. Phys., 26, 43\sphinxhyphen{}65 (1978)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
Manteuffel, An Incomplete Factorization Technique for Positive Definite Linear Systems, Math. Comp., 34, 473\sphinxhyphen{}497 (1980)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Derreumaux, G. Zhang and T. Schlick and B. R. Brooks, A Truncated Newton Minimizer Adapted for CHARMM and Biomolecular Applications, J. Comput. Chem., 15, 532\sphinxhyphen{}552 (1994)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Roman}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Duff, A. M. Erisman and J. K. Reid, Direct Methods for Sparse Matrices, Oxford University Press, Oxford, 1986

\end{enumerate}

\end{enumerate}


\section{Potential Energy Smoothing}
\label{\detokenize{text/references:potential-energy-smoothing}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{21}
\item {} 
Pappu, R. K. Hart and J. W. Ponder, Analysis and Application of Potential Energy Smoothing Methods for Global Optimization, J. Phys. Chem. B, 102, 9725\sphinxhyphen{}9742 (1998)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} 
Piela, J. Kostrowicki and H. A. Scheraga, The Multiple\sphinxhyphen{}Minima Problem in the Conformational Analysis of Molecules. Deformation of the Potential Energy Hypersurface by the Diffusion Equation Method, J. Phys. Chem., 93, 3339\sphinxhyphen{}3346 (1989)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Ma and J. E. Straub, Simulated Annealing Using the Classical Density Distribution, J. Chem. Phys., 101, 533\sphinxhyphen{}541 (1994)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} 
Tsoo and C. L. Brooks, Cluster Structure Determination Using Gaussian Density Distribution Global Minimization Methods, J. Chem. Phys., 101, 6405\sphinxhyphen{}6411 (1994)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Nakamura, H. Hirose, M. Ikeguchi and J. Doi, Conformational Energy Minimization Using a Two\sphinxhyphen{}Stage Method, J. Phys. Chem., 99, 8374\sphinxhyphen{}8378 (1995)

\item {} 
Huber, A. E. Torda and W. F. van Gunsteren, Structure Optimization Combining Soft\sphinxhyphen{}Core Interaction Functions, the Diffusion Equation Method, and Molecular Dynamics, J. Phys. Chem. A, 101, 5926\sphinxhyphen{}5930 (1997)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Schelstraete and H. Verschelde, Finding Minimum\sphinxhyphen{}Energy Configurations of Lennard\sphinxhyphen{}Jones Clusters Using an Effective Potential, J. Phys. Chem. A, 101, 310\sphinxhyphen{}315 (1998)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Roman}{enumi}{enumii}{}{.}%
\item {} 
Andricioaei and J. E. Straub, Global Optimization Using Bad Derivatives: Derivative\sphinxhyphen{}Free Method for Molecular Energy Minimization, J. Comput. Chem., 19, 1445\sphinxhyphen{}1455 (1998)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} 
Piela, Search for the Most Stable Structures on Potential Energy Surfaces, Coll. Czech. Chem. Commun., 63, 1368\sphinxhyphen{}1380 (1998)

\end{enumerate}


\section{“Sniffer” Global Optimization}
\label{\detokenize{text/references:sniffer-global-optimization}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{14}
\item {} 
Griewank, Generalized Descent for Global Optimization, J. Opt. Theor. Appl., 34, 11\sphinxhyphen{}39 (1981)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumiii}{enumiv}{}{.}%
\setcounter{enumiii}{17}
\item {} 
Butler and E. E. Slaminka, An Evaluation of the Sniffer Global Optimization Algorithm Using Standard Test Functions, J. Comput. Phys., 99, 28\sphinxhyphen{}32 (1993)

\end{enumerate}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{22}
\item {} 
Rogers and R. A. Donnelly, Potential Transformation Methods for Large\sphinxhyphen{}Scale Global Optimization, SIAM J. Optim., 5, 871\sphinxhyphen{}891 (1995)

\end{enumerate}

\end{enumerate}


\section{Integration Methods for Molecular Dynamics}
\label{\detokenize{text/references:integration-methods-for-molecular-dynamics}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} 
Beeman, Some Multistep Methods for Use in Molecular Dynamics Calculations, J. Comput. Phys., 20, 130\sphinxhyphen{}139 (1976)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Levitt and H. Meirovitch, Integrating the Equations of Motion, J. Mol. Biol., 168, 617\sphinxhyphen{}620 (1983)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Aqvist, W. F. van Gunsteren, M. Leijonmarck and O. Tapia, A Molecular Dynamics Study of the C\sphinxhyphen{}Terminal Fragment of the L7/L12 Ribosomal Protein, J. Mol. Biol., 183, 461\sphinxhyphen{}477 (1985)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Swope, H. C. Andersen, P. H. Berens and K. R. Wilson, A Computer Simulation Method for the Calculation of Equilibrium Constants for the Formation of Physical Clusters of Molecules: Application to Small Water Clusters, J. Chem. Phys., 76, 637\sphinxhyphen{}649 (1982)

\end{enumerate}

\end{enumerate}


\section{Constraint Dynamics}
\label{\detokenize{text/references:constraint-dynamics}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{5}
\item {} 
van Gunsteren and H. J. C. Berendsen, Algorithms for Macromolecular Dynamics and Constraint Dynamics, Mol. Phys., 34, 1311\sphinxhyphen{}1327 (1977)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} 
Ciccotti, M. Ferrario and J.\sphinxhyphen{}P. Ryckaert, Molecular Dynamics of Rigid Systems in Cartesian Coordinates: A General Formulation, Mol. Phys., 47, 1253\sphinxhyphen{}1264 (1982)

\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Andersen, Rattle: A “Velocity” Version of the Shake Algorithm for Molecular Dynamics Calculations, J. Comput. Phys., 52, 24\sphinxhyphen{}34 (1983)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} 
Kutteh, RATTLE Recipe for General Holonomic Constraints: Angle and Torsion Constraints, CCP5 Newsletter, 46, 9\sphinxhyphen{}17 (1998) {[}available from the web site at \sphinxurl{http://www.dl.ac.uk/CCP/CCP5/newsletter\_index.html}{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{1}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Palmer, Direct Application of SHAKE to the Velocity Verlet Algorithm, J. Comput. Phys., 104, 470\sphinxhyphen{}472 (1993)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Miyamoto and P. A. Kollman, SETTLE: An Analytical Version of the SHAKE and RATTLE Algorithm for Rigid Water Models, J. Comput. Chem., 13, 952\sphinxhyphen{}962 (1992)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{1}
\item {} 
Hess, H. Bekker, H. J. C. Berendsen and J. G. E. M. Fraaije, LINCS: A Linear Constraint Solver for Molecular Simulations, J. Comput. Chem., 18, 1463\sphinxhyphen{}1472 (1997)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Slusher and P. T. Cummings, Non\sphinxhyphen{}Iterative Constraint Dynamics using Velocity\sphinxhyphen{}Explicit Verlet Methods, Mol. Simul., 18, 213\sphinxhyphen{}224 (1996)

\end{enumerate}

\end{enumerate}


\section{Langevin, Brownian and Stochastic Dynamics}
\label{\detokenize{text/references:langevin-brownian-and-stochastic-dynamics}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{15}
\item {} 
Allen, Brownian Dynamics Simulation of a Chemical Reaction in Solution, Mol. Phys., 40, 1073\sphinxhyphen{}1087 (1980)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{5}
\item {} 
van Gunsteren and H. J. C. Berendsen, Algorithms for Brownian Dynamics, Mol. Phys., 45, 637\sphinxhyphen{}647 (1982)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{5}
\item {} 
Guarnieri and W. C. Still, A Rapidly Convergent Simulation Method: Mixed Monte Carlo/Stochastic Dynamics, J. Comput. Chem., 15, 1302\sphinxhyphen{}1310 (1994)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{6}
\item {} 
Paterlini and D. M. Ferguson, Constant Temperature Simulations using the Langevin Equation with Velocity Verlet Integration, Chem. Phys., 236, 243\sphinxhyphen{}252 (1998)

\end{enumerate}

\end{enumerate}


\section{Constant Temperature and Pressure Dynamics}
\label{\detokenize{text/references:constant-temperature-and-pressure-dynamics}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{7}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumiii}{enumiv}{}{.}%
\setcounter{enumiii}{2}
\item {} 
Berendsen, J. P. M. Postma, W. F. van Gunsteren, A. DiNola and J. R. Haak, Molecular Dynamics with Coupling to an External Bath, J. Chem. Phys., 81, 3684\sphinxhyphen{}3690 (1984)

\end{enumerate}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{6}
\item {} 
Hoover, Canonical Dynamics: Equilibrium Phase\sphinxhyphen{}space Distributions, Phys. Rev. A, 31, 1695\sphinxhyphen{}1697 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Morales, S. Toxvaerd and L. F. Rull, Computer Simulation of a Phase Transition at Constant Temperature and Pressure, Phys. Rev. A, 34, 1495\sphinxhyphen{}1498 (1986)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{1}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{17}
\item {} 
Brooks, Algorithms for Molecular Dynamics at Constant Temperature and Pressure, Internal Report of Division of Computer Research and Technology, National Institutes of Health, 1988.

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Levitt, Molecular Dynamics of Native Protein: Computer Simulation of Trajectories, J. Mol. Biol., 168, 595\sphinxhyphen{}620 (1983)

\end{enumerate}


\section{Out\sphinxhyphen{}of\sphinxhyphen{}Plane Deformation Terms}
\label{\detokenize{text/references:out-of-plane-deformation-terms}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{17}
\item {} 
Maple, U. Dinar and A. T. Hagler, Derivation of Force Fields for Molecular Mechanics and Dynamics from ab initio Energy Surfaces, Proc. Natl. Acad. Sci. USA, 85, 5350\sphinxhyphen{}5354 (1988)

\end{enumerate}

\end{enumerate}

S.\sphinxhyphen{}H. Lee, K. Palmo and S. Krimm, New Out\sphinxhyphen{}of\sphinxhyphen{}Plane Angle and Bond Angle Internal Coordinates and Related Potential Energy Functions for Molecular Mechanics and Dynamics Simulations, J. Comput. Chem., 20, 1067\sphinxhyphen{}1084 (1999)


\section{Analytical Derivatives of Potential Functions}
\label{\detokenize{text/references:analytical-derivatives-of-potential-functions}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Miller, R. J. Hinde and J. Anderson, First and Second Derivative Matrix Elements for the Stretching, Bending, and Torsional Energy, J. Comput. Chem., 10, 63\sphinxhyphen{}76 (1989)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{7}
\item {} 
Faber and C. Altona, UTAH5: A Versatile Programme Package for the Calculation of Molecular Properties by Force Field Methods, Computers \& Chemistry, 1, 203\sphinxhyphen{}213 (1977)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Swope and D. M. Ferguson, Alternative Expressions for Energies and Forces Due to Angle Bending and Torsional Energy, Report G320\sphinxhyphen{}3561, J. Comput. Chem., 13, 585\sphinxhyphen{}594 (1992)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} 
Blondel and M. Karplus, New Formulation for Derivatives of Torsion Angles and Improper Torsion Angles in Molecular Mechanics: Elimination of Singularities, J. Comput. Chem., 17, 1132\sphinxhyphen{}1141 (1996)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Tuzun, D. W. Noid and B. G. Sumpter, Efficient Treatment of Out\sphinxhyphen{}of\sphinxhyphen{}Plane Bend and Improper Torsion Interactions in MM2, MM3, and MM4 Molecular Mechanics Calculations, J. Comput. Chem., 18, 1804\sphinxhyphen{}1811 (1997)

\end{enumerate}

\end{enumerate}


\section{Torsional Space Derivatives and Normal Modes}
\label{\detokenize{text/references:torsional-space-derivatives-and-normal-modes}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Levitt, C. Sander and P. S. Stern, Protein Normal\sphinxhyphen{}mode Dynamics:  Trypsin Inhibitor, Crambin, Ribonuclease and Lysozyme, J. Mol. Biol., 181, 423\sphinxhyphen{}447 (1985)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Levitt, Protein Folding by Restrained Energy Minimization and Molecular Dynamics, J. Mol. Biol., 170, 723\sphinxhyphen{}764 (1983)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{7}
\item {} 
Wako and N. Go, Algorithm for Rapid Calculation of Hessian of Conformational Energy Function of Proteins by Supercomputer, J. Comput. Chem., 8, 625\sphinxhyphen{}635 (1987)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{7}
\item {} 
Abe, W. Braun, T. Noguti and N. Go, Rapid Calculation of First and Second Derivatives of Conformational Energy with Respect to Dihedral Angles for Proteins: General Recurrent Equations, Computers \& Chemistry, 8, 239\sphinxhyphen{}247 (1984)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} 
Noguti and N. Go, A Method of Rapid Calculation of a Second Derivative Matrix of Conformational Energy for Large Molecules, J. Phys. Soc. Japan, 52, 3685\sphinxhyphen{}3690 (1983)

\end{enumerate}


\section{Analytical Surface Area and Volume}
\label{\detokenize{text/references:analytical-surface-area-and-volume}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Connolly, Analytical Molecular Surface Calculation, J. Appl. Cryst., 16, 548\sphinxhyphen{}558 (1983)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Connolly, Computation of Molecular Volume, J. Am. Chem. Soc., 107, 1118\sphinxhyphen{}1124 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Connolly, Molecular Surfaces: A Review, available from the web site at \sphinxurl{http://www.netsci.org/Science/Compchem/feature14.html}

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Kundrot, J. W. Ponder and F. M. Richards, Algorithms for Calculating Excluded Volume and Its Derivatives as a Function of Molecular Conformation and Their Use in Energy Minimization, J. Comput. Chem., 12, 402\sphinxhyphen{}409 (1991)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Richmond, Solvent Accessible Surface Area and Excluded Volume in Proteins, J. Mol. Biol., 178, 63\sphinxhyphen{}89 (1984)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} 
Wesson and D. Eisenberg, Atomic Solvation Parameters Applied to Molecular Dynamics of Proteins in Solution, Protein Science, 1, 227\sphinxhyphen{}235 (1992)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{21}
\item {} 
Gononea and E. Osawa, Implementation of Solvent Effect in Molecular Mechanics, Part 3. The First\sphinxhyphen{} and Second\sphinxhyphen{}order Analytical Derivatives of Excluded Volume, J. Mol. Struct. (Theochem), 311 305\sphinxhyphen{}324 (1994)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Gibson and H. A. Scheraga, Exact Calculation of the Volume and Surface Area of Fused Hard\sphinxhyphen{}sphere Molecules with Unequal Atomic Radii, Mol. Phys., 62, 1247\sphinxhyphen{}1265 (1987)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Gibson and H. A. Scheraga, Surface Area of the Intersection of Three Spheres with Unequal Radii: A Simplified Analytical Formula, Mol. Phys., 64, 641\sphinxhyphen{}644 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Sridharan, A. Nichols and K. A. Sharp, A Rapid Method for Calculating Derivatives of Solvent Accessible Surface Areas of Molecules, J. Comput, Chem., 16, 1038\sphinxhyphen{}1044 (1995)

\end{enumerate}


\section{Approximate Surface Area and Volume}
\label{\detokenize{text/references:approximate-surface-area-and-volume}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Wodak and J. Janin, Analytical Approximation to the Accessible Surface Area of Proteins, Proc. Natl. Acad. Sci. USA, 77, 1736\sphinxhyphen{}1740 (1980)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} 
Hasel, T. F. Hendrickson and W. C. Still, A Rapid Approximation to the Solvent Accessible Surface Areas of Atoms, Tetrahedron Comput. Method., 1, 103\sphinxhyphen{}116 (1988)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Weiser, P. S. Shenkin and W. C. Still, Approximate Solvent\sphinxhyphen{}Accessible Surface Areas from Tetrahedrally Directed Neighber Densities, Biopolymers, 50, 373\sphinxhyphen{}380 (1999)

\end{enumerate}


\section{Boundary Conditions and Neighbor Methods}
\label{\detokenize{text/references:boundary-conditions-and-neighbor-methods}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{5}
\item {} 
van Gunsteren, H. J. C. Berendsen, F. Colonna, D. Perahia, J. P. Hollenberg and D. Lellouch, On Searching Neighbors in Computer Simulations of Macromolecular Systems, J. Comput. Chem., 5, 272\sphinxhyphen{}279  (1984)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{5}
\item {} 
Sullivan, R. D. Mountain and J. O’Connell, Molecular Dynamics on Vector Computers, J. Comput. Phys., 61, 138\sphinxhyphen{}153 (1985)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Boris, A Vectorized “Near Neighbors” Algorithm of Order N Using a Monotonic Logical Grid, J. Comput. Phys., 66, 1\sphinxhyphen{}20 (1986)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{6}
\item {} 
Lambrakos and J. P. Boris, Geometric Properties of the Monotonic Lagrangian Grid Algorithm for Near Neighbors Calculations, J. Comput. Phys., 73, 183\sphinxhyphen{}202 (1987)

\end{enumerate}

\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
Andrea, W. C. Swope and H. C. Andersen, The Role of Long Ranged Forces in Determining the Structure and Properties of Liquid Water, J. Chem. Phys., 79, 4576\sphinxhyphen{}4584 (1983)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{13}
\item {} 
Theodorou and U. W. Suter, Geometrical Considerations in Model Systems with Periodic Boundary Conditions, J. Chem. Phys., 82, 955\sphinxhyphen{}966 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Barnes and P. Hut, A Hierarchical O(NlogN) Force\sphinxhyphen{}calculation Algorithm, Nature, 234, 446\sphinxhyphen{}449 (1986)

\end{enumerate}


\section{Cutoff and Truncation Methods}
\label{\detokenize{text/references:cutoff-and-truncation-methods}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Steinbach and B. R. Brooks, New Spherical\sphinxhyphen{}Cutoff Methods for Long\sphinxhyphen{}Range Forces in Macromolecular Simulation, J. Comput. Chem., 15, 667\sphinxhyphen{}683 (1993)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Loncharich and B. R. Brooks, The Effects of Truncating Long\sphinxhyphen{}Range Forces on Protein Dynamics, Proteins, 6, 32\sphinxhyphen{}45 (1989)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Brooks III, B. M. Pettitt and M. Karplus, Structural and Energetic Effects of Truncating Long Ranged Interactions in Ionic and Polar Fluids, J. Chem. Phys., 83, 5897\sphinxhyphen{}5908 (1985)

\end{enumerate}

\end{enumerate}


\section{Ewald Summation Techniques}
\label{\detokenize{text/references:ewald-summation-techniques}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{24}
\item {} 
Toukmaji and J. A. Board, Jr., Ewald Summation Techniques in Perspective: A Survey, Comp. Phys. Commun., 95, 73\sphinxhyphen{}92 (1996)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} 
Darden, L. Perera, L. Li and L. Pedersen, New Tricks for Modelers from the Crystallography Toolkit: The Particle Mesh Ewald Algorithm and its Use in Nucleic Acid Simulations, Structure, 7, R550\sphinxhyphen{}R60 (1999)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} 
Darden, D. York and L. G. Pedersen, Particle Mesh Ewald: An Nlog(N) Method for Ewald Sums in Large Systems, J. Chem. Phys., 98, 10089\sphinxhyphen{}10092 (1993)

\item {} 
Essmann, L. Perera, M. L. Berkowitz, T. Darden, H. Lee and L. G. Pedersen, A Smooth Particle Mesh Ewald Method, J. Chem. Phys., 103, 8577\sphinxhyphen{}8593 (1995)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} 
Smith, Point Multipoles in the Ewald Summation (Revisited), CCP5 Newsletter, 46, 18\sphinxhyphen{}30 (1998)  {[}available from \sphinxurl{http://www.dl.ac.uk/CCP/CCP5/newsletter\_index.html}{]}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Feller, R. W. Pastor, A. Rojnuckarin, S. Bogusz and B. R. Brooks, Effect of Electrostatic Force Truncation on Interfacial and Transport Properties of Water, J. Phys. Chem., 100, 17011\sphinxhyphen{}17020 (1996)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} 
Weber, P. H. H¸nenberger and J. A. McCammon, Molecular Dynamics Simulations of a Polyalanine Octapeptide under Ewald Boundary Conditions: Influence of Artificial Periodicity on Peptide Conformation, J. Phys. Chem. B, 104, 3668\sphinxhyphen{}3675 (2000)

\end{enumerate}


\section{Conjugated and Aromatic Systems}
\label{\detokenize{text/references:conjugated-and-aromatic-systems}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, F. Li, L. Yan and J. C. Tai, Molecular Mechanics (MM3) Calculations on Conjugated Hydrocarbons, J. Comput. Chem., 11, 868\sphinxhyphen{}895 (1990)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Sprague, J. C. Tai, Y. Yuh and N. L. Allinger, The MMP2 Calculational Method, J. Comput. Chem., 8, 581\sphinxhyphen{}603 (1987)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Kao, A Molecular Orbital Based Molecular Mechanics Approach to Study Conjugated Hydrocarbons, J. Am. Chem. Soc., 109, 3818\sphinxhyphen{}3829 (1987)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Kao and N. L. Allinger, Conformational Analysis: Heats of Formation of Conjugated Hydrocarbons by the Force Field Method, J. Am. Chem. Soc., 99, 975\sphinxhyphen{}986 (1977)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{7}
\item {} 
Lo and M. A. Whitehead, Accurate Heats of Atomization and Accurate Bond Lengths: Benzenoid Hydrocarbons, Can. J. Chem., 46, 2027\sphinxhyphen{}2040 (1968)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Zeiss and M. A. Whitehead, Hetero\sphinxhyphen{}atomic Molecules: Semi\sphinxhyphen{}empirical Molecular Orbital Calculations and Prediction of Physical Properties, J. Chem. Soc. A, 1727\sphinxhyphen{}1738 (1971)

\end{enumerate}

\end{enumerate}


\section{Free Energy Simulation Methods}
\label{\detokenize{text/references:free-energy-simulation-methods}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} 
Kollman, Free Energy Calculations: Applications to Chemical and Biochemical Phenomena, Chem. Rev., 93, 2395\sphinxhyphen{}2417 (1993)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{1}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Tembe and J. A. McCammon, Ligand\sphinxhyphen{}Receptor Interactions, Computers \& Chemistry, 8, 281\sphinxhyphen{}283 (1984)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen and C. Ravimohan, Monte Carlo Simulation of Differences in Free Energy of Hydration, J. Chem. Phys., 83, 3050\sphinxhyphen{}3054 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen, J. K. Buckner, S. Boudon and J. Tirado\sphinxhyphen{}Rives, Efficient Computation of Absolute Free Energies of Binding by Computer Simulations:  Application to the Methane Dimer in Water, J. Chem. Phys., 89, 3742\sphinxhyphen{}3746 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{7}
\item {} 
Fleischman and C. L. Brooks III, Thermodynamics of Aqueous Solvation:  Solution Properties of Alcohols and Alkanes, J. Chem. Phys., 87, 3029\sphinxhyphen{}3037 (1987)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{20}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Singh, F. K. Brown, P. A. Bash and P. A. Kollman, An Approach to the Application of Free Energy Perturbation Methods Using Molecular Dynamics, J. Am. Chem. Soc., 109, 1607\sphinxhyphen{}1614 (1987)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
Pearlman and P. A. Kollman, A New Method for Carrying out Free Energy Perturbation Calculations: Dynamically Modified Windows, J. Chem. Phys., 90, 2460\sphinxhyphen{}2470 (1989)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{15}
\item {} 
Straatsma, H. J. C. Berendsen and J. P. M. Postma, Free Energy of Hydrophobic Hydration:  A Molecular Dynamics Study of Noble Gases in Water, J. Chem. Phys., 85, 6720\sphinxhyphen{}6727 (1986)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{15}
\item {} 
Straatsma and H. J. C. Berendsen, Free Energy of Ionic Hydration:  Analysis of a Thermodynamic Integration Technique to Evaluate Free Energy Differences by Molecular Dynamics Simulations, J. Chem. Phys., 89, 5876\sphinxhyphen{}5886 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Mezei, The Finite Difference Thermodynamic Integration, Tested on Calculating the Hydration Free Energy Difference between Acetone and Dimethylamine in Water, J. Chem. Phys., 86, 7084\sphinxhyphen{}7088 (1987)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Mark and W. F. van Gunsteren, Decomposition of the Free Energy of a System in Terms of Specific Interactions, J. Mol. Biol., 240, 167\sphinxhyphen{}176 (1994)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Boresch and M. Karplus, The Meaning of Copmponent Analysis: Decomposition of the Free Energy in Terms of Specific Interactions, J. Mol. Biol., 254, 801\sphinxhyphen{}807 (1995)

\end{enumerate}


\section{Methods for Parameter Determination}
\label{\detokenize{text/references:methods-for-parameter-determination}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{13}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Allinger, X. Zhou and J. Bergsma, Molecular Mechanics Parameters, J. Mol. Struct. (THEOCHEM), 312, 69\sphinxhyphen{}83 (1994)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Pertsin and A. I. Kitaigorodsky, The Atom\sphinxhyphen{}Atom Potential Method: Application to Organic Molecular Solids, Springer\sphinxhyphen{}Verlag, Berlin, 1987

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Williams, Transferable Empirical Nonbonded Potential Functions, in Crystal Cohesion and Conformational Energies, Ed. by R. M. Metzger, Springer\sphinxhyphen{}Verlag, Berlin, 1981

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Hagler and S. Lifson, A Procedure for Obtaining Energy Parameters from Crystal Packing, Acta Cryst., B30, 1336\sphinxhyphen{}1341 (1974)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Hagler, S. Lifson and P. Dauber, Consistent Force Field Studies of Intermolecular Forces in Hydrogen\sphinxhyphen{}Bonded Crystals:  A Benchmark for the Objective Comparison of Alternative Force Fields, J. Am. Chem. Soc., 101, 5122\sphinxhyphen{}5130 (1979)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen, J. D. Madura and C. J. Swenson, Optimized Intermolecular Potential Functions for Liquid Hydrocarbons, J. Am. Chem. Soc., 106, 6638\sphinxhyphen{}6646 (1984)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Jorgensen and C. J. Swenson, Optimized Intermolecular Potential Functions for Amides and Peptides: Structure and Properties of Liquid Amides, J. Am. Chem. Soc., 107, 569\sphinxhyphen{}578 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{17}
\item {} 
Maple, U. Dinur and A. T. Hagler, Derivation of Force Fields for Molecular Mechanics and Dynamics from ab Initio Surfaces, Proc. Nat. Acad. Sci. USA, 85, 5350\sphinxhyphen{}5354 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{20}
\item {} 
Dinur and A. T. Hagler, Direct Evaluation of Nonbonding Interactions from ab Initio Calculations, J. Am. Chem. Soc., 111, 5149\sphinxhyphen{}5151 (1989)

\end{enumerate}


\section{Electrostatic Interactions}
\label{\detokenize{text/references:electrostatic-interactions}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{11}
\item {} 
Price, Towards More Accurate Model Intermolecular Potentials for Organic Molecules, Rev. Comput. Chem., 14, 225\sphinxhyphen{}289 (2000)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{7}
\item {} 
Faerman and S. L. Price, A Transferable Distributed Multipole Model for the Electrostatic Interactions of Peptides and Amides, J. Am. Chem. Soc., 112, 4915\sphinxhyphen{}4926 (1990)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Dykstra, Electrostatic Interaction Potentials in Molecular Force Fields, Chem. Rev., 93, 2339\sphinxhyphen{}2353 (1993)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Dudek and J. W. Ponder, Accurate Modeling of the Intramolecular Electrostatic Energy of Proteins, J. Comput. Chem., 16, 791\sphinxhyphen{}816 (1995)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{20}
\item {} 
Koch and E. Egert, An Improved Description of the Molecular Charge Density in Force Fields with Atomic Multipole Moments, J. Comput. Chem., 16, 937\sphinxhyphen{}944 (1995)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Williams, Representation of the Molecular Electrostatic Potential by Atomic Multipole and Bond Dipole Models, J. Comput. Chem., 9, 745\sphinxhyphen{}763 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{5}
\item {} 
Colonna, E. Evleth and J. G. Angyan, Critical Analysis of Electric Field Modeling: Formamide, J. Comput. Chem., 13, 1234\sphinxhyphen{}1245 (1992)

\end{enumerate}


\section{Polarization Effects}
\label{\detokenize{text/references:polarization-effects}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Kuwajima and A. Warshel, Incorporating Electric Polarizabilities in Water\sphinxhyphen{}Water Interaction Potentials, J. Phys. Chem., 94, 460\sphinxhyphen{}466 (1990)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{22}
\item {} 
Caldwell and P. A. Kollman, Structure and Properties of Neat Liquids Using Nonadditive Molecular Dynamics: Water, Methanol, and N\sphinxhyphen{}Methylacetamide, J. Phys. Chem., 99, 6208\sphinxhyphen{}6219 (1995)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{13}
\item {} 
Bernardo, Y. Ding, K. Kroegh\sphinxhyphen{}Jespersen and R. M. Levy, An Anisotropic Polarizable Water Model: Incorporation of All\sphinxhyphen{}Atom Polarizabilities into Molecular Mechanics Force Fields, J. Phys. Chem., 98, 4180\sphinxhyphen{}4187 (1994)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{15}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
van Duijnen and M. Swart, Molecular and Atomic Polarizabilities: Thole’s Model Revisited, J. Phys. Chem. A, 102, 2399\sphinxhyphen{}2407 (1998)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Miller, Calculation of the Molecular Polarizability Tensor, J. Am. Chem. Soc., 112, 8543\sphinxhyphen{}8551 (1990)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Applequist, J. R. Carl and K.\sphinxhyphen{}K. Fung, An Atom Dipole Interaction Model for Molecular Polarizability. Application to Polyatomic Molecules and Determination of Atom Polarizabilities, J. Am. Chem. Soc., 94, 2952\sphinxhyphen{}2960 (1972)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} 
Applequist, Atom Charge Transfer in Molecular Polarizabilities. Application of the Olson\sphinxhyphen{}Sundberg Model to Aliphatic and Aromatic Hydrocarbons, J. Phys. Chem., 97, 6016\sphinxhyphen{}6023 (1993)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Stone, Distributed Polarizabilities, Mol. Phys., 56, 1065\sphinxhyphen{}1082 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{12}
\item {} 
Stout and C. E. Dykstra, A Distributed Model of the Electrical Response of Organic Molecules, J. Phys. Chem. A, 102, 1576\sphinxhyphen{}1582 (1998)

\end{enumerate}

\end{enumerate}


\section{Macroscopic Treatment of Solvent}
\label{\detokenize{text/references:macroscopic-treatment-of-solvent}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{2}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Cramer and D. G. Truhlar, Continuum Solvation Models: Classical and Quantum Mechanical Implementations, Rev. Comput. Chem., 6, 1\sphinxhyphen{}72 (1995)

\end{enumerate}

\end{enumerate}

B.Roux and T. Simonson, Implicit Solvation Models, Biophys. Chem., 78, 1\sphinxhyphen{}20 (1999)
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{10}
\item {} 
Gilson, Introduction to Continuum Electrostatics with Molecular Applications, available from \sphinxurl{http://gilsonlab.umbi.umd.edu}

\end{enumerate}

\end{enumerate}


\section{Surface Area\sphinxhyphen{}Based Solvation Models}
\label{\detokenize{text/references:surface-area-based-solvation-models}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} 
Eisenberg and A. D. McLachlan, Solvation Energy in Protein Folding and Binding, Nature, 319, 199\sphinxhyphen{}203 (1986)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{11}
\item {} 
Wesson and D. Eisenberg, Atomic Solvation Parameters Applied to Molecular Dynamics of Proteins in Solution, Prot. Sci., 1, 227\sphinxhyphen{}235 (1992)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} 
Ooi, M. Oobatake, G. Nemethy and H. A. Scheraga, Accessible Surface Areas as a Measure of the Thermodynamic Parameters of Hydration of Peptides, Proc. Natl. Acad. Sci. USA, 84, 3086\sphinxhyphen{}3090 (1987)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Augspurger and H. A. Scheraga, An Efficient, Differentiable Hydration Potential for Peptides and Proteins, J. Comput. Chem., 17, 1549\sphinxhyphen{}1558 (1996)

\end{enumerate}

\end{enumerate}


\section{Generalized Born Solvation Models}
\label{\detokenize{text/references:generalized-born-solvation-models}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{22}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Still, A. Tempczyk, R. C. Hawley and T. Hendrickson, A Semiempirical Treatment of Solvation for Molecular Mechanics and Dynamics, J. Am. Chem. Soc., 112, 6127\sphinxhyphen{}6129 (1990)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} 
Qiu, P. S. Shenkin, F. P. Hollinger and W. C. Still, The GB/SA Continuum Model for Solvation. A Fast Analytical Method for the Calculation of Approximate Born Radii, J. Phys. Chem. A, 101, 3005\sphinxhyphen{}3014 (1997)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Hawkins, C. J. Cramer and D. G. Truhlar, Pairwise Solute Descreening of Solute Charges from a Dielectric Medium, Chem. Phys. Lett., 246, 122\sphinxhyphen{}129 (1995)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
Hawkins, C. J. Cramer and D. G. Truhlar, Parametrized Models of Aqueous Free Energies of Solvation Based on Pairwise Descreening of Solute Atomic Charges from a Dielectric Medium, J. Phys. Chem., 100, 19824\sphinxhyphen{}19839 (1996)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} 
Onufriev, D. Bashford and D. A. Case, Modification of the Generalized Born Model Suitable for Macromolecules, J. Phys. Chem. B, 104, 3712\sphinxhyphen{}3720 (2000)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Schaefer and M. Karplus, A Comprehensive Analytical Treatment of Continuum Electrostatics, J. Phys. Chem., 100, 1578\sphinxhyphen{}1599 (1996)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{12}
\item {} 
Schaefer, C. Bartels and M. Karplus, Solution Conformations and Thermodynamics of Structured Peptides: Molecular Dynamics Simulation with an Implicit Solvation Model, J. Mol. Biol., 284, 835\sphinxhyphen{}848 (1998)

\end{enumerate}


\section{Superposition of Coordinate Sets}
\label{\detokenize{text/references:superposition-of-coordinate-sets}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{9}
\item {} 
Kearsley, An Algorithm for the Simultaneous Superposition of a Structural Series, J. Comput. Chem., 11, 1187\sphinxhyphen{}1192 (1990)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} 
Diamond, A Note on the Rotational Superposition Problem, Acta Cryst., A44, 211\sphinxhyphen{}216 (1988)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{3}
\item {} 
McLachlan, Rapid Comparison of Protein Structures, Acta Cryst., A38, 871\sphinxhyphen{}873 (1982)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{2}
\item {} 
Nyburg, Some Uses of a Best Molecular Fit Routine, Acta Cryst., B30, 251\sphinxhyphen{}253 (1974)

\end{enumerate}

\end{enumerate}


\section{Location of Transition States}
\label{\detokenize{text/references:location-of-transition-states}}\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} 
Czerminski and R. Elber, Reaction Path Study of Conformational Transitions and Helix Formation in a Tetrapeptide, Proc. Nat. Acad. Sci. USA, 86, 6963 (1989)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{18}
\item {} 
Berry, H. L. Davis and T. L. Beck, Finding Saddles on Multidimensional Potential Surfaces, Chem. Phys. Lett., 147, 13 (1988)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{10}
\item {} 
Muller, Reaction Paths on Multidimensional Energy Hypersurfaces, Ang. Chem. Int. Ed. Engl., 19, 1\sphinxhyphen{}13 (1980)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Bell and J. S. Crighton, Locating Transition States, J. Chem. Phys., 80, 2464\sphinxhyphen{}2475 (1984)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{18}
\item {} 
Fischer and M. Karplus, Conjugate Peak Refinement: An Algorithm for Finding Reaction Paths and Accurate Transition States in Systems with Many Degrees of Freedom, Chem. Phys. Lett., 194, 252\sphinxhyphen{}261 (1992)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{9}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{4}
\item {} 
Sinclair and R. Fletcher, A New Method of Saddle\sphinxhyphen{}Point Location for the Calculation of Defect Migration Energies, J. Phys. C, 7, 864\sphinxhyphen{}870 (1974)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{17}
\item {} 
Elber and M. Karplus, A Method for Determining Reaction Paths in Large Molecules:  Application to Myoglobin, Chem. Phys. Lett., 139, 375\sphinxhyphen{}380 (1987)

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{3}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Nguyen and D. A. Case, On Finding Stationary States on Large\sphinxhyphen{}Molecule Potential Energy Surfaces, J. Phys. Chem., 89, 4020\sphinxhyphen{}4026 (1985)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{19}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\item {} 
Halgren and W. N. Lipscomb, The Synchronous\sphinxhyphen{}Transit Method for Determining Reaction Pathways and Locating Molecular Transition States, Chem. Phys. Lett., 49, 225\sphinxhyphen{}232 (1977)

\end{enumerate}

\end{enumerate}
\begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumi}{enumii}{}{.}%
\setcounter{enumi}{6}
\item {} \begin{enumerate}
\sphinxsetlistlabels{\Alph}{enumii}{enumiii}{}{.}%
\setcounter{enumii}{19}
\item {} 
Barkema and N. Mousseau, Event\sphinxhyphen{}Based Relaxation of Continuous Disordered Systems, Phys. Rev. Lett., 77, 4358\sphinxhyphen{}4361 (1996)

\end{enumerate}

\end{enumerate}



\renewcommand{\indexname}{Index}
\printindex
\end{document}